-- challenge_platform data

INSERT INTO challenge_platform (id, slug, name, avatar_key, website_url)
VALUES (
    1,
    'synapse',
    'Synapse',
    'logo/synapse.png',
    'https://synapse.org/'
  ),
  (
    2,
    'cagi',
    'CAGI',
    'logo/cagi.png',
    'https://genomeinterpretation.org/challenges.html'
  ),
  (
    3,
    'cami',
    'CAMI',
    'logo/cami.png',
    'https://data.cami-challenge.org/'
  ),
  (
    4,
    'casp',
    'CASP',
    'logo/casp.png',
    'https://predictioncenter.org/'
  ),
  (
    5,
    'grand-challenge',
    'Grand Challenge',
    'logo/grand-challenge.png',
    'https://grand-challenge.org/'
  ),
  (
    6,
    'precision-fda',
    'precisionFDA',
    'logo/precisionfda.png',
    'https://precision.fda.gov/challenges'
  ),
  (
    7,
    'easychair',
    'EasyChair',
    'logo/easy-chair.jpg',
    'https://easychair.org/'
  ),
  (
    8,
    'kaggle',
    'Kaggle',
    'logo/kaggle.png',
    'https://www.kaggle.com/'
  ),
  (
    9,
    'codalab',
    'CodaLab',
    'logo/codalab.jpg',
    'https://codalab.lisn.upsaclay.fr/'
  ),
  (
    10,
    'codabench',
    'CodaBench',
    'logo/codalab.jpg',
    'https://www.codabench.org/'
  ),
  (
    11,
    'openml',
    'OpenML',
    'logo/openml.jpg',
    'https://www.openml.org/'
  ),
  (
    12,
    'papers-with-code',
    'PapersWithCode',
    'logo/papers-with-code.jpg',
    'https://paperswithcode.com/'
  ),
  (
    13,
    'eterna',
    'Eterna',
    'logo/eterna.svg',
    'https://eternagame.org/'
  ),
  (
    14,
    'other',
    'Other',
    '',
    ''
  ),
  (
    15,
    'nightingale-os',
    'Nightingale OS',
    'logo/nightingale-os.png',
    'https://app.nightingalescience.org/'
  );

-- challenge_input_data_type

INSERT INTO challenge_input_data_type (id, slug, name)
VALUES (1, 'genomic', 'genomic'),
  (2, 'proteomic', 'proteomic'),
  (3, 'gene-expression', 'gene expression'),
  (4, 'metabolomic', 'metabolomic');

-- challenge data

INSERT INTO challenge (
    id,
    slug,
    name,
    headline,
    description,
    website_url,
    status,
    difficulty,
    platform_id,
    doi,
    start_date,
    end_date,
    created_at,
    updated_at
  )
VALUES (
    1,
    'network-topology-and-parameter-inference',
    'Network Topology and Parameter Inference',
    '',
    'Participants are asked to develop and/or apply optimization methods, including the selection of the most informative experiments, to accurately estimate parameters and
 predict outcomes of perturbations in Systems Biology models.',
    'https://www.synapse.org/#!Synapse:syn2821735',
    'completed',
    'intermediate',
    '1',
    '',
    '2012-06-01',
    '2012-10-01',
    '2023-06-23 00:00:00',
    '2023-08-09 08:42:17'
  ),
  (
    2,
    'breast-cancer-prognosis',
    'Breast Cancer Prognosis',
    '',
    'The goal of the breast cancer prognosis Challenge is to assess the accuracy of computational models designed to predict breast cancer survival, based on clinical information about the patient''s tumor as well as genome-wide molecular profiling data including gene expression and copy number profiles.',
    'https://www.synapse.org/#!Synapse:syn2813426',
    'completed',
    'intermediate',
    '1',
    '',
    '2012-07-12',
    '2012-10-15',
    '2023-06-23 00:00:00',
    '2023-07-26 19:47:13'
  ),
  (
    3,
    'phil-bowen-als-prediction-prize4life',
    'Phil Bowen ALS Prediction Prize4Life',
    '',
    'Amyotrophic Lateral Sclerosis (ALS)–also known as Lou Gehrig''s disease (in the US) or Motor Neurone disease (outside the US)–is a fatal neurological disease causing death of the nerve cells in the brain and spinal cord which control voluntary muscle movements. This leaves patients struggling with a progressive loss of motor function while leaving cognitive functions intact. Symptoms usually do not manifest until the age of 50 but can start earlier. At any given time, approximately five out of every 100,000 people worldwide suffer from ALS, though there would be a higher prevalence if the disease did not progress so rapidly, leading to the death of the patient.  There are no known risk factors for developing ALS other than having a family member who has a hereditary form of the disease, which accounts for about 5-10% of ALS patients. There is also no known cure for ALS. The only FDA-approved drug for the disease is Riluzole, which has been shown to prolong the life span of someone...',
    'https://www.synapse.org/#!Synapse:syn2826267',
    'completed',
    'intermediate',
    '1',
    '',
    '2012-06-01',
    '2012-10-01',
    '2023-06-23 00:00:00',
    '2023-07-26 19:47:15'
  ),
  (
    4,
    'drug-sensitivity-and-drug-synergy-prediction',
    'Drug Sensitivity and Drug Synergy Prediction',
    '',
    'Development of new cancer therapeutics currently requires a long and protracted process of experimentation and testing. Human cancer cell lines represent a good model to help identify associations between molecular subtypes, pathways, and drug response. In recent years there have been several efforts to generate genomic profiles of collections of cell lines and to determine their response to panels of candidate therapeutic compounds. These data provide the basis for the development of in silico models of sensitivity based either on the unperturbed genetic potential of a cancer cell, or by using perturbation data to incorporate knowledge of actual cell response. Making predictions from either of these data profiles will be beneficial in identifying single and combinatorial chemotherapeutic response in patients. To that end, the present challenge seeks computational methods, derived from the molecular profiling of cell lines both in a static state and in response to perturbation of ...',
    'https://www.synapse.org/#!Synapse:syn2785778',
    'completed',
    'intermediate',
    '1',
    '',
    '2012-06-01',
    '2012-10-01',
    '2023-06-23 00:00:00',
    '2023-08-09 08:42:24'
  ),
  (
    5,
    'niehs-ncats-unc-toxicogenetics',
    'NIEHS-NCATS-UNC Toxicogenetics',
    '',
    'This challenge is designed to build predictive models of cytotoxicity as mediated by exposure to environmental toxicants and drugs. To approach this question, we will provide a dataset containing cytotoxicity estimates as measured in lymphoblastoid cell lines derived from 884 individuals following in vitro exposure to 156 chemical compounds. In subchallenge 1, participants will be asked to model interindividual variability in cytotoxicity based on genomic profiles in order to predict cytotoxicity in unknown individuals. In subchallenge 2, participants will be asked to predict population-level parameters of cytotoxicity across chemicals based on structural attributes of compounds in order to predict median cytotoxicity and mean variance in toxicity for unknown compounds.',
    'https://www.synapse.org/#!Synapse:syn1761567',
    'completed',
    'intermediate',
    '1',
    '',
    '2013-06-10',
    '2013-09-15',
    '2023-06-23 00:00:00',
    '2023-08-09 08:42:24'
  ),
  (
    6,
    'whole-cell-parameter-estimation',
    'Whole-Cell Parameter Estimation',
    '',
    'The goal of this challenge is to explore and compare innovative approaches to parameter estimation of large, heterogeneous computational models. Participants are encouraged to develop and/or apply optimization methods, including the selection of the most informative experiments. The organizers encourage participants to form teams to collaboratively solve the challenge.',
    'https://www.synapse.org/#!Synapse:syn1876068',
    'completed',
    'intermediate',
    '1',
    '',
    '2013-06-10',
    '2013-09-23',
    '2023-06-23 00:00:00',
    '2023-07-26 19:47:21'
  ),
  (
    7,
    'hpn-dream-breast-cancer-network-inference',
    'HPN-DREAM Breast Cancer Network Inference',
    '',
    'The overall goal of the Heritage-DREAM breast cancer network inference challenge is to quickly and effectively advance our ability to infer causal signaling networks and predict protein phosphorylation dynamics in cancer. We provide extensive training data from experiments on four breast cancer cell lines stimulated with various ligands. The data comprise protein abundance time-courses under inhibitor perturbations.',
    'https://www.synapse.org/#!Synapse:syn1720047',
    'completed',
    'intermediate',
    '1',
    '',
    '2013-06-10',
    '2013-09-16',
    '2023-06-23 00:00:00',
    '2023-07-26 19:47:22'
  ),
  (
    8,
    'rheumatoid-arthritis-responder',
    'Rheumatoid Arthritis Responder',
    '',
    'The goal of this project is to use a crowd-based competition framework to develop a validated molecular predictor of anti-TNF response in RA. There is an increasing need for predictors of response to therapy in inflammatory disease driven by the observation that most clinically defined diseases show variable response and the growing availability of alternative therapies. Anti-TNF drugs in Rheumatoid Arthritis represent a prototypical example of this opportunity. A number of studies have tried, over the past decade, to develop a robust predictor of response. We believe the time is right to try a different approach to developing such a biomarker with a crowd-sourced collaborative competition. This is based on DREAM and Sage Bionetworks'' experience with running competitions and the availability of new unpublished large-scale data relating to RA treatment response.THIS CHALLENGE RAN FROM FEBRUARY TO OCTOBER 2014 AND IS NOW CLOSED.',
    'https://www.synapse.org/#!Synapse:syn1734172',
    'completed',
    'intermediate',
    '1',
    '',
    '2014-02-10',
    '2014-06-04',
    '2023-06-23 00:00:00',
    '2023-07-26 19:47:24'
  ),
  (
    9,
    'icgc-tcga-dream-mutation-calling',
    'ICGC-TCGA DREAM Mutation Calling',
    '',
    'The ICGC-TCGA DREAM Genomic Mutation Calling Challenge (herein, The Challenge) is an international effort to improve standard methods for identifying cancer-associated mutations and rearrangements in whole-genome sequencing (WGS) data. Leaders of the International Cancer Genome Consortium (ICGC) and The Cancer Genome Atlas (TCGA) cancer genomics projects are joining with Sage Bionetworks and IBM-DREAM to initiate this innovative open crowd-sourced Challenge [1-3].',
    'https://www.synapse.org/#!Synapse:syn312572',
    'completed',
    'intermediate',
    '1',
    '',
    '2013-12-14',
    '2016-04-22',
    '2023-06-23 00:00:00',
    '2023-09-13 21:11:42'
  ),
  (
    10,
    'acute-myeloid-leukemia-outcome-prediction',
    'Acute Myeloid Leukemia Outcome Prediction',
    '',
    'The AML Outcome Prediction Challenge provides a unique opportunity to access and interpret a rich dataset for AML patients that includes clinical covariates, select gene mutation status and proteomic data. Capitalizing on a unique AML reverse phase protein array (RPPA) dataset obtained at M.D. Anderson Cancer Center that captures 271 measurements for each patient, participants of the DREAM 9 Challenge will help uncover what drives AML. Outcomes of this Challenge have the potential to be used immediately to tailor therapies for newly diagnosed leukemia patients and to accelerate the development of new drugs for leukemia.',
    'https://www.synapse.org/#!Synapse:syn2455683',
    'completed',
    'intermediate',
    '1',
    '',
    '2014-06-02',
    '2014-09-15',
    '2023-06-23 00:00:00',
    '2023-07-26 19:47:28'
  ),
  (
    11,
    'broad-dream-gene-essentiality-prediction',
    'Broad-DREAM Gene Essentiality Prediction',
    '',
    'The goal of this project is to use a crowd-based competition to develop predictive models that can infer gene dependency scores in cancer cells (genes that are essential to cancer cell viability when suppressed) using features of those cell lines. An additional goal is to find a small set of biomarkers (gene expression, copy number, and mutation features) that can best predict a single gene or set of genes.',
    'https://www.synapse.org/#!Synapse:syn2384331',
    'completed',
    'intermediate',
    '1',
    '',
    '2014-06-02',
    '2014-09-29',
    '2023-06-23 00:00:00',
    '2023-07-26 19:47:33'
  ),
  (
    12,
    'alzheimers-disease-big-data',
    'Alzheimer''s Disease Big Data',
    '',
    'The goal of the Alzheimer''s Disease Big Data DREAM Challenge #1 (AD#1) was to apply an open science approach to rapidly identify accurate predictive AD biomarkers that can be used by the scientific, industrial and regulatory communities to improve AD diagnosis and treatment. AD#1 will be the first in a series of AD Data Challenges to leverage genetics and brain imaging in combination with cognitive assessments, biomarkers and demographic information from cohorts ranging from cognitively normal to mild cognitively impaired to individuals with AD.',
    'https://www.synapse.org/#!Synapse:syn2290704',
    'completed',
    'intermediate',
    '1',
    '',
    '2014-06-02',
    '2014-10-17',
    '2023-06-23 00:00:00',
    '2023-07-26 19:47:34'
  ),
  (
    13,
    'olfaction-prediction',
    'Olfaction Prediction',
    '',
    'The goal of the DREAM Olfaction Prediction Challenge is to find models that can predict how a molecule smells from its physical and chemical features. A model that allows us to predict a smell from a molecule will provide fundamental insights into how odor chemicals are transformed into a smell percept in the brain. Further, being able to predict how a chemical smells will greatly accelerate the design of new molecules to be used as fragrances. Currently, fragrance chemists synthesize many molecules to obtain a new ingredient, but most of these will not have the desired qualities.',
    'https://www.synapse.org/#!Synapse:syn2811262',
    'completed',
    'intermediate',
    '1',
    '',
    '2015-01-15',
    '2015-05-01',
    '2023-06-23 00:00:00',
    '2023-07-26 19:47:36'
  ),
  (
    14,
    'prostate-cancer',
    'Prostate Cancer',
    '',
    'This challenge will attempt to improve the prediction of survival and toxicity of docetaxel treatment in patients with metastatic castration-resistant prostate cancer (mCRPC). The primary benefit of this Challenge will be to establish new quantitative benchmarks for prognostic modeling in mCRPC, with a potential impact for clinical decision making and ultimately understanding the mechanism of disease progression. Participating teams will be asked to submit predictive models based on clinical variables from the comparator arms of four phase III clinical trials with over 2,000 mCRPC patients treated with first-line docetaxel. The comparator arm of a clinical trial represents the patients that receive a treatment that is considered to be effective. This arm of the clinical trial is used to evaluate the effectiveness of the new therapy being tested.',
    'https://www.synapse.org/#!Synapse:syn2813558',
    'completed',
    'intermediate',
    '1',
    '',
    '2015-03-16',
    '2015-07-27',
    '2023-06-23 00:00:00',
    '2023-07-26 19:47:38'
  ),
  (
    15,
    'als-stratification-prize4life',
    'ALS Stratification Prize4Life',
    '',
    'As illustrated by the overview figure below, (a) Challenge Data includes data from ALS clinical trials and ALS registries. ALS clinical trials consist of patients from clinical trials available open access on the PRO-ACT database and patients from 6 clinical trials not yet added into the database. Data from ALS registries was collected from patients in national ALS registries. (b) Data is divided into three subsets: training data provided to solvers in full, leaderboard, and validation data that is available only to the organizers and is reserved for the scoring of the challenge. (c) The goal of this challenge is then to predict the Clinical Targets, i.e. the disease progression as ALSFRS slope as well as survival. (d) For Building the Models, participants create two algorithms - one that selects features and one that predicts outcomes. To perform predictions, data from a given patient (1) is fed into the selector (2). The selector selects 6 features and a cluster/model ID (3), e....',
    'https://www.synapse.org/#!Synapse:syn2873386',
    'completed',
    'intermediate',
    '1',
    '',
    '2015-06-22',
    '2015-10-04',
    '2023-06-23 00:00:00',
    '2023-07-26 19:47:40'
  ),
  (
    16,
    'astrazeneca-sanger-drug-combination-prediction',
    'AstraZeneca-Sanger Drug Combination Prediction',
    '',
    'To accelerate the understanding of drug synergy, AstraZeneca has partnered with the European Bioinformatic Institute, the Sanger Institute, Sage Bionetworks, and the distributed DREAM community to launch the AstraZeneca-Sanger Drug Combination Prediction DREAM Challenge. This Challenge is designed to explore fundamental traits that underlie effective combination treatments and synergistic drug behavior using baseline genomic data, i.e. data collected pretreatment. As the basis of the Challenge, AstraZeneca is releasing ~11.5k experimentally tested drug combinations measuring cell viability over 118 drugs and 85 cancer cell lines (primarily colon, lung, and breast), and monotherapy drug response data for each drug and cell line. Moreover, in coordination with the Genomics of Drug Sensitivity in Cancer and COSMIC teams at the Sanger Institute, genomic data including gene expression, mutations (whole exome), copy-number alterations, and methylation data will be released into the publ...',
    'https://www.synapse.org/#!Synapse:syn4231880',
    'completed',
    'intermediate',
    '1',
    '',
    '2015-09-03',
    '2016-03-14',
    '2023-06-23 00:00:00',
    '2023-07-26 19:47:41'
  ),
  (
    17,
    'smc-dna-meta',
    'SMC-DNA Meta',
    '',
    'The goal of this Challenge is to identify the most accurate meta-pipeline for somatic mutation detection, and establish the state-of-the-art. The algorithms in this Challenge must use as input mutations predicted by one or more variant callers and output mutation calls associated with cancer. An additional goal is to highlight the complementarity of the calling algorithms and help understand their individual advantages/deficiencies.',
    'https://www.synapse.org/#!Synapse:syn4588939',
    'completed',
    'intermediate',
    '1',
    '',
    '2015-08-17',
    '2016-04-10',
    '2023-06-23 00:00:00',
    '2023-07-26 19:47:43'
  ),
  (
    18,
    'smc-het',
    'SMC-Het',
    '',
    'The ICGC-TCGA DREAM Somatic Mutation Calling - Tumour Heterogeneity Challenge (SMC-Het) is an international effort to improve standard methods for subclonal reconstruction: to quantify and genotype each individual cell population present within a tumor. Leaders of the International Cancer Genome Consortium (ICGC) and The Cancer Genome Atlas (TCGA) cancer genomics projects are joining with Sage Bionetworks and IBM-DREAM to initiate this innovative open crowd-sourced Challenge [1-3].',
    'https://www.synapse.org/#!Synapse:syn2813581',
    'completed',
    'intermediate',
    '1',
    '',
    '2015-11-16',
    '2016-06-30',
    '2023-06-23 00:00:00',
    '2023-07-26 20:40:13'
  ),
  (
    19,
    'respiratory-viral',
    'Respiratory Viral',
    '',
    'Respiratory viruses are highly infectious and cause acute illness in millions of people every year. However, there is wide variation in the physiologic response to exposure at the individual level. Some people that are exposed to virus are able to completely avoid infection. Others contract virus but are able to fight it off without exhibiting any symptoms of illness such as coughing, sneezing, sore throat or fever. It is not well understood what characteristics may protect individuals from respiratory viral infection. These individual responses are likely influenced by multiple processes including both the basal state of the human host upon exposure and the dynamics of host immune response in the early hours immediately following exposure. Many of these processes play out in the peripheral blood through activation and recruitment of circulating immune cells. Global gene expression patterns measured in peripheral blood at the time of symptom onset - several days after viral exposu...',
    'https://www.synapse.org/#!Synapse:syn5647810',
    'completed',
    'intermediate',
    '1',
    '',
    '2016-05-16',
    '2016-09-28',
    '2023-06-23 00:00:00',
    '2023-07-26 19:47:46'
  ),
  (
    20,
    'disease-module-identification',
    'Disease Module Identification',
    '',
    'The Disease Module Identification DREAM Challenge is an open community effort to systematically assess module identification methods on a panel of state-of-the-art genomic networks and leverage the “wisdom of crowds” to discover novel modules and pathways underlying complex diseases.',
    'https://www.synapse.org/#!Synapse:syn6156761',
    'completed',
    'intermediate',
    '1',
    '',
    '2016-06-24',
    '2016-10-01',
    '2023-06-23 00:00:00',
    '2023-07-26 19:48:43'
  ),
  (
    21,
    'encode',
    'ENCODE',
    '',
    'Transcription factors (TFs) are regulatory proteins that bind specific DNA sequence patterns (motifs) in the genome and affect transcription rates of target genes. Binding sites of TFs differ across cell types and experimental conditions. Chromatin immunoprecipitation followed by sequencing (ChIP-seq) is an experimental method that is commonly used to obtain the genome-wide binding profile of a TF of interest in a specific cell type/condition. However, profiling the binding landscape of every TF in every cell type/condition is infeasible due to constraints on cost, material and effort. Hence, accurate computational prediction of in vivo TF binding sites is critical to complement experimental results.',
    'https://www.synapse.org/#!Synapse:syn6131484',
    'completed',
    'intermediate',
    '1',
    '',
    '2016-07-07',
    '2017-01-11',
    '2023-06-23 00:00:00',
    '2023-07-26 19:48:44'
  ),
  (
    22,
    'idea',
    'Idea',
    '',
    'The DREAM Idea Challenge is designed to collaboratively shape and enable the solution of a question fundamental to improving human health. In the process, all proposals and their evaluation will be made publicly available for the explicit purpose of connecting modelers and experimentalists who want to address the same question. This Wall of Models will enable new collaborations, and help turn every good modeling idea into a success story. It will further serve as a basis for new DREAM challenges.',
    'https://www.synapse.org/#!Synapse:syn5659209',
    'completed',
    'advanced',
    '1',
    '',
    '2016-06-15',
    '2017-04-30',
    '2023-06-23 00:00:00',
    '2023-07-26 19:48:45'
  ),
  (
    23,
    'smc-rna',
    'SMC-RNA',
    '',
    'The ICGC-TCGA DREAM Somatic Mutation Calling - RNA Challenge (SMC-RNA) is an international effort to improve standard methods for identifying cancer-associated rearrangements in RNA sequencing (RNA-seq) data. Leaders of the International Cancer Genome Consortium (ICGC) and The Cancer Genome Atlas (TCGA) cancer genomics projects are joining with Sage Bionetworks and IBM-DREAM to initiate this innovative open crowd-sourced Challenge [1-3].',
    'https://www.synapse.org/#!Synapse:syn2813589',
    'completed',
    'intermediate',
    '1',
    '',
    '2016-06-29',
    '2017-05-02',
    '2023-06-23 00:00:00',
    '2023-07-26 19:48:46'
  ),
  (
    24,
    'digital-mammography',
    'Digital Mammography',
    '',
    'The Digital Mammography DREAM Challenge will attempt to improve the predictive accuracy of digital mammography for the early detection of breast cancer. The primary benefit of this Challenge will be to establish new quantitative tools - machine learning, deep learning or other - that can help decrease the recall rate of screening mammography, with a potential impact on shifting the balance of routine breast cancer screening towards more benefit and less harm. Participating teams will be asked to submit predictive models based on over 640,000 de-identified digital mammography images from over 86000 subjects, with corresponding clinical variables.',
    'https://www.synapse.org/#!Synapse:syn4224222',
    'completed',
    'advanced',
    '1',
    '10.1001/jamanetworkopen.2020.0265',
    '2016-11-18',
    '2017-05-16',
    '2023-06-23 00:00:00',
    '2023-07-26 19:48:48'
  ),
  (
    25,
    'multiple-myeloma',
    'Multiple Myeloma',
    '',
    'Multiple myeloma (MM) is a cancer of the plasma cells in the bone marrow, with about 25,000 newly diagnosed patients per year in the United States alone. The disease''s clinical course depends on a complex interplay of clinical traits and molecular characteristics of the plasma cells.1 Since risk-adapted therapy is becoming standard of care, there is an urgent need for a precise risk stratification model to assist in therapeutic decision-making and research. While progress has been made, there remains a significant opportunity to improve patient stratification to optimize treatment and to develop new therapies for high-risk patients. A DREAM Challenge represents a chance not only to integrate available data and analytical approaches to tackle this important problem, but also provides the ability to benchmark potential methods to identify those with the greatest potential to yield patient care benefits in the future.',
    'https://www.synapse.org/#!Synapse:syn6187098',
    'completed',
    'intermediate',
    '1',
    '',
    '2017-06-30',
    '2017-11-08',
    '2023-06-23 00:00:00',
    '2023-07-26 19:48:48'
  ),
  (
    26,
    'ga4gh-dream-workflow-execution',
    'GA4GH-DREAM Workflow Execution',
    '',
    'The highly distributed and disparate nature of genomic and clinical data generated around the world presents an enormous challenge for those scientists who wish to integrate and analyze these data. The sheer volume of data often exceeds the capacity for storage at any one site and prohibits the efficient transfer between sites. To address this challenge, researchers must bring their computation to the data. Numerous groups are now developing technologies and best practice methodologies for running portable and reproducible genomic analysis pipelines as well as tools and APIs for discovering genomic analysis resources. Software development, deployment, and sharing efforts in these groups commonly rely on the use of modular workflow pipelines and virtualization based on Docker containers and related tools.',
    'https://www.synapse.org/#!Synapse:syn8507133',
    'completed',
    'intermediate',
    '1',
    '',
    '2017-07-21',
    '2017-12-31',
    '2023-06-23 00:00:00',
    '2023-07-26 19:48:51'
  ),
  (
    27,
    'parkinsons-disease-digital-biomarker',
    'Parkinson''s Disease Digital Biomarker',
    '',
    'The Parkinson''s Disease Digital Biomarker DREAM Challenge is a first of it''s kind challenge, designed to benchmark methods for the processing of sensor data for development of digital signatures reflective of Parkinson''s Disease. Participants will be provided with raw sensor (accelerometer, gyroscope, and magnetometer) time series data recorded during the performance of pre-specified motor tasks, and will be asked to extract data features which are predictive of PD pathology. In contrast to traditional DREAM challenges, this one will focus on feature extraction rather than predictive modeling, and submissions will be evaluated based on their ability to predict disease phenotype using an array of standard machine learning algorithms.',
    'https://www.synapse.org/#!Synapse:syn8717496',
    'completed',
    'intermediate',
    '1',
    '',
    '2017-07-06',
    '2017-11-10',
    '2023-06-23 00:00:00',
    '2023-07-26 19:48:52'
  ),
  (
    28,
    'nci-cptac-proteogenomics',
    'NCI-CPTAC Proteogenomics',
    '',
    'Cancer is driven by aberrations in the genome [1,2], and these alterations manifest themselves largely in the changes in the structure and abundance of proteins, the main functional gene products. Hence, characterization and analyses of alterations in the proteome has the promise to shed light into cancer development and may improve development of both biomarkers and therapeutics. Measuring the proteome is very challenging, but recent rapid technology developments in mass spectrometry are enabling deep proteomics analysis [3]. Multiple initiatives have been launched to take advantage of this development to characterize the proteome of tumours, such as the Clinical Proteomic Tumor Analysis Consortium (CPTAC). These efforts hold the promise to revolutionize cancer research, but this will only be possible if the community develops computational tools powerful enough to extract the most information from the proteome, and to understand the association between genome, transcriptome and ...',
    'https://www.synapse.org/#!Synapse:syn8228304',
    'completed',
    'intermediate',
    '1',
    '',
    '2017-06-26',
    '2017-11-20',
    '2023-06-23 00:00:00',
    '2023-07-26 19:48:52'
  ),
  (
    29,
    'multi-targeting-drug',
    'Multi-Targeting Drug',
    '',
    'The objective of this challenge is to incentivize development of methods for predicting compounds that bind to multiple targets. In particular, methods that are generalizable to multiple prediction problems are sought. To achieve this, participants will be asked to predict 2 separate compounds, each having specific targets to which they should bind, and a list of anti-targets to avoid. Participants should use the same methods to produce answers for questions 1 and 2.',
    'https://www.synapse.org/#!Synapse:syn8404040',
    'completed',
    'intermediate',
    '1',
    '',
    '2017-10-05',
    '2018-02-26',
    '2023-06-23 00:00:00',
    '2023-07-26 19:48:53'
  ),
  (
    30,
    'single-cell-transcriptomics',
    'Single Cell Transcriptomics',
    '',
    'In this Challenge on Single-Cell Transcriptomics, participants will reconstruct the location of single cells in the Drosophila embryo using single-cell transcriptomic data. Data will be made available in late August and participating challenge teams can work on the data and submit their results previous to the DREAM Conference. The best performers will be announced at the DREAM conference on Dec 8.',
    'https://www.synapse.org/#!Synapse:syn15665609',
    'completed',
    'intermediate',
    '1',
    '',
    '2018-09-04',
    '2018-11-21',
    '2023-06-23 00:00:00',
    '2023-07-26 19:48:54'
  ),
  (
    31,
    'idg-drug-kinase-binding',
    'IDG Drug-Kinase Binding',
    '',
    'This IDG-DREAM Drug-Kinase Binding Prediction Challenge seeks to evaluate the power of statistical and machine learning models as a systematic and cost-effective means for catalyzing compound-target interaction mapping efforts by prioritizing most potent interactions for further experimental evaluation. The Challenge will focus on kinase inhibitors, due to their clinical importance [2], and will be implemented in a screening-based, pre-competitive drug discovery project in collaboration with theIlluminating the Druggable Genome (IDG) Kinase-focused Data and Resource Generation Center, consortium, with the aim to establish kinome-wide target profiles of small-molecule agents, with the goal of extending the druggability of the human kinome space.',
    'https://www.synapse.org/#!Synapse:syn15667962',
    'completed',
    'intermediate',
    '1',
    '',
    '2018-10-01',
    '2019-04-18',
    '2023-06-23 00:00:00',
    '2023-07-26 19:48:57'
  ),
  (
    32,
    'malaria',
    'Malaria',
    '',
    'The Malaria DREAM Challenge is open to anyone interested in contributing to the development of computational models that address important problems in advancing the fight against malaria. The overall goal of the first Malaria DREAM Challenge is to predict Artemisinin (Art) drug resistance level of a test set of malaria parasites using their in vitro transcription data and a training set consisting of published in vivo and unpublished in vitrotranscriptomes. The in vivodataset consists of ~1000 transcription samples from various geographic locations covering a wide range of life cycles and resistance levels, with other accompanying data such as patient age, geographic location, Art combination therapy used, etc [Mok et al (2015) Science]. The in vitro transcription dataset consists of 55 isolates, with transcription collected at two timepoints (6 and 24 hours post-invasion), in the absence or presence of an Art perturbation, for two biological replicates using a custom microarray a...',
    'https://www.synapse.org/#!Synapse:syn16924919',
    'completed',
    'intermediate',
    '1',
    '',
    '2019-04-30',
    '2019-08-15',
    '2023-06-23 00:00:00',
    '2023-07-26 19:48:58'
  ),
  (
    33,
    'preterm-birth-prediction-transcriptomics',
    'Preterm Birth Prediction - Transcriptomics',
    '',
    'A basic need in pregnancy care is to establish gestational age, and inaccurate estimates may lead to unnecessary interventions and sub-optimal patient management. Current approaches to establish gestational age rely on patient''s recollection of her last menstrual period and/or ultrasound, with the latter being not only costly but also less accurate if not performed during the first trimester of pregnancy. Therefore development of an inexpensive and accurate molecular clock of pregnancy would be of benefit to patients and health care systems. Participants in sub-challenge 1 (Prediction of gestational age) will be given whole blood gene expression data collected from pregnant women to develop prediction models for the gestational age at blood draw.  Another challenge in obstetrics, in both low and high-income countries, is identification and treatment of women at risk of developing the ‘great obstetrical syndromes‘. Of these, preterm birth (PTB), defined as giving birth prior to co...',
    'https://www.synapse.org/#!Synapse:syn18380862',
    'completed',
    'good_for_beginners',
    '1',
    '',
    '2019-05-04',
    '2019-12-05',
    '2023-06-23 00:00:00',
    '2023-07-26 19:48:58'
  ),
  (
    34,
    'single-cell-signaling-in-breast-cancer',
    'Single-Cell Signaling in Breast Cancer',
    '',
    'Signaling underlines nearly every cellular event. Individual cells, even if genetically identical, respond to perturbation in different ways. This underscores the relevance of cellular heterogeneity, in particular in how cells respond to drugs. This is of high relevance since the fact that a subset of cells do not respond (or only weakly) to drugs can render this drug an ineffective treatment. In spite of its relevance to many diseases, comprehensive studies on the heterogeneous signaling in single cells are still lacking.  We have generated the, to our knowledge, currently largest single cell signaling dataset on a panel of 67 well-characterized breast cancer cell lines by mass cytometry (3''015 conditions, ~80 mio single cells, 38 markers; Bandura et al. 2009; Bendall et al., 2011; Bodenmiller et al., 2012; Lun et al., 2017; Lun et al., 2019). These cell lines are, among others, also characterized at the genomic, transcriptomic, and proteomic level (Marcotte et al., 2016).  We a...',
    'https://www.synapse.org/#!Synapse:syn20366914',
    'completed',
    'intermediate',
    '1',
    '',
    '2018-08-20',
    '2019-11-15',
    '2023-06-23 00:00:00',
    '2023-07-26 19:48:59'
  ),
  (
    35,
    'ehr-dream-challenge-patient-mortality-prediction',
    'EHR DREAM Challenge: Patient Mortality Prediction',
    '',
    'The recent advent of new CRISPR-based molecular tools allows the reconstruction of cell lineages based on the phylogenetical analysis of DNA mutations induced by CRISPR during development and promises to solve the lineage of complex model organisms at single-cell resolution (see image from McKenna et al Science 2016). To date, however, no lineage reconstruction algorithms have been rigorously examined for their performance/robustness across diverse molecular tools, datasets, and number of cells/size of lineage trees. It also remains unclear whether new Machine-Learning algorithms that go beyond the classical ones developed for reconstructing phylogenetic trees, could consistently reconstruct cell lineages to a high degree of accuracy. The challenge - a partnership between The Allen Institute and DREAM - will comprise 3 subchallenges that consist of reconstructing cell lineage trees of different sizes and nature. In subchallenge 1, participants will be given experimental molecular ...',
    'https://www.synapse.org/#!Synapse:syn18405991',
    'completed',
    'intermediate',
    '1',
    'https://doi.org/10.1093/jamia/ocad159',
    '2019-09-09',
    '2020-01-23',
    '2023-06-23 00:00:00',
    '2023-07-26 19:49:00'
  ),
  (
    36,
    'allen-institute-cell-lineage-reconstruction',
    'Allen Institute Cell Lineage Reconstruction',
    '',
    'The recent advent of new CRISPR-based molecular tools allows the reconstruction of cell lineages based on the phylogenetical analysis of DNA mutations induced by CRISPR during development and promises to solve the lineage of complex model organisms at single-cell resolution.  To date, however, no lineage reconstruction algorithms have been rigorously examined for their performance/robustness across diverse molecular tools, datasets, and number of cells/size of lineage trees. It also remains unclear whether new Machine-Learning algorithms that go beyond the classical ones developed for reconstructing phylogenetic trees, could consistently reconstruct cell lineages to a high degree of accuracy. The challenge - a partnership between The Allen Institute and DREAM - will comprise 3 subchallenges that consist of reconstructing cell lineage trees of different sizes and nature. In subchallenge 1, participants will be given experimental molecular data to reconstruct in vitro cell lineages ...',
    'https://www.synapse.org/#!Synapse:syn20692755',
    'completed',
    'intermediate',
    '1',
    '',
    '2019-10-15',
    '2020-02-06',
    '2023-06-23 00:00:00',
    '2023-08-09 08:39:22'
  ),
  (
    37,
    'tumor-deconvolution',
    'Tumor Deconvolution',
    '',
    'The extent of stromal and immune cell infiltration within solid tumors has prognostic and predictive significance. Unfortunately, expression profiling of tumors has, until very recently, largely been undertaken using bulk techniques (e.g., microarray and RNA-seq). Unlike single-cell methods (e.g., single-cell RNA-seq, FACS, mass cytometry, or immunohistochemistry), bulk approaches average expression across all cells (cancer, stromal, and immune) within the sample and, hence, do not directly quantitate tumor infiltration. This information can be recovered by computational tumor deconvolution methods, which would thus allow interrogation of immune subpopulations across the large collection of public bulk expression datasets.  The goal of this Challenge is to evaluate the ability of computational methods to deconvolve bulk expression data, reflecting a mixture of cell types, into individual immune components. Methods will be assessed based on in vitro and in silico admixtures specifi...',
    'https://www.synapse.org/#!Synapse:syn15589870',
    'completed',
    'intermediate',
    '1',
    '',
    '2019-06-26',
    '2020-04-30',
    '2023-06-23 00:00:00',
    '2023-07-26 19:49:01'
  ),
  (
    38,
    'ctd2-pancancer-drug-activity',
    'CTD2 Pancancer Drug Activity',
    '',
    'NCI Genomic Data Commons,Harvard Chan School, Blue Collar Bioinformatics,ENCODE DCC, Stanford,CCHMC, Barski Lab,KnowEnG, UIUC,UCSC,OICR, ICGC,Broad Institute, GATK',
    'https://www.synapse.org/#!Synapse:syn20968331',
    'completed',
    'good_for_beginners',
    '1',
    '',
    '2019-12-02',
    '2020-02-13',
    '2023-06-23 00:00:00',
    '2023-07-26 19:49:04'
  ),
  (
    39,
    'ctd2-beataml',
    'CTD2 BeatAML',
    '',
    'In the era of precision medicine, AML patients have few therapeutic options, with “7 + 3” induction chemotherapy having been the standard for decades (Bertoli et al. 2017). While several agents targeting the myeloid marker CD33 or alterations in FLT3 or IDH2 have demonstrated efficacy in patients (Wei and Tiong 2017), responses are uncertain in some populations (Castaigne et al. 2012) and relapse remains prevalent (Stone et al. 2017). These drugs highlight both the promise of targeted therapies in AML and the urgent need for additional treatment options that are tailored to more refined patient subpopulations in order to achieve durable responses.  The BeatAML initiative was launched as a comprehensive study of the relationship between molecular alterations and ex-vivo drug sensitivity in patients with AML. One of the primary goals of this multi-center study was to develop a discovery cohort that could yield new drug target hypotheses and predictive biomarkers of therapeutic respo...',
    'https://www.synapse.org/#!Synapse:syn20940518',
    'completed',
    'good_for_beginners',
    '1',
    '',
    '2019-12-19',
    '2020-04-28',
    '2023-06-23 00:00:00',
    '2023-07-26 19:49:04'
  ),
  (
    40,
    'metadata-automation',
    'Metadata Automation',
    '',
    'The Cancer Research Data Commons (CRDC) will collate data across diverse groups of cancer researchers, each collecting biomedical data in different formats. This means the data must be retrospectively harmonized and transformed to enable this data to be submitted. In addition, to be findable by the broader scientific community, coherent information (metadata) is necessary about the data fields and values. Coherent metadata annotation of the data fields and their values can enable computational data transformation, query, and analysis. Creation of this type of descriptive metadata can require biomedical expertise to determine the best annotations and thus is a time-consuming and manual task which is both an obstacle and a bottleneck in data sharing and submissions.  Goal: Using structured biomedical data files, challenge participants will develop tools to semi-automate annotation of metadata fields and values, using available research data annotations (e.g. caDSR CDEs) as well as e...',
    'https://www.synapse.org/#!Synapse:syn18065891',
    'completed',
    'intermediate',
    '1',
    '',
    '2020-01-14',
    '2020-06-02',
    '2023-06-23 00:00:00',
    '2023-07-26 19:49:05'
  ),
  (
    41,
    'automated-scoring-of-radiographic-joint-damage',
    'Automated Scoring of Radiographic Joint Damage',
    '',
    'The purpose of the RA2-DREAM Challenge is to develop an automated method to quickly and accurately quantify the degree of joint damage associated with rheumatoid arthritis (RA). Based on radiographs of the hands and feet, a novel, automated scoring method could be applied broadly for patient care and research. We challenge participants to develop algorithms to automatically assess joint space narrowing and erosions using a large set of existing radiographs with damage scores generated by visual assessment of images by trained readers using standard protocols. The end result will be a generalizable, publicly available, automated method to generate accurate, reproducible and unbiased RA damage scores to replace the current tedious, expensive, and non-scalable method of scoring by human visual inspection.',
    'https://www.synapse.org/#!Synapse:syn20545111',
    'completed',
    'intermediate',
    '1',
    '',
    '2019-11-04',
    '2020-05-21',
    '2023-06-23 00:00:00',
    '2023-07-26 19:49:08'
  ),
  (
    42,
    'beat-pd',
    'BEAT-PD',
    '',
    'Recent advances in mobile health have demonstrated great potential to leverage sensor-based technologies for quantitative, remote monitoring of health and disease - particularly for diseases affecting motor function such as Parkinson''s disease. Such approaches have been rolled out using research-grade wearable sensors and, increasingly, through the use of smartphones and consumer wearables, such as smart watches and fitness trackers. These devices not only provide the ability to measure much more detailed disease phenotypes but also provide the ability to follow patients longitudinally with much higher frequency than is possible through clinical exams. However, the conversion of sensor-based data streams into digital biomarkers is complex and no methodological standards have yet evolved to guide this process.  Parkinson''s disease (PD) is a neurodegenerative disease that primarily affects the motor system but also exhibits other symptoms. Typical motor symptoms of the disease inc...',
    'https://www.synapse.org/#!Synapse:syn20825169',
    'completed',
    'intermediate',
    '1',
    '',
    '2020-01-13',
    '2020-05-13',
    '2023-06-23 00:00:00',
    '2023-07-26 19:49:09'
  ),
  (
    43,
    'ctd2-pancancer-chemosensitivity',
    'CTD2 Pancancer Chemosensitivity',
    '',
    'Over the last two years, the Columbia CTD2 Center developed PANACEA (Pancancer Analysis of Chemical Entity Activity), a comprehensive repertoire of dose response curves and molecular profiles representative of cellular responses to drug perturbations. PANACEA covers a broad spectrum of cellular contexts representative of poor outcome malignancies, including rare ones such as GIST sarcoma and gastroenteropancreatic neuroendocrine tumors (GEP-NETs). PANACEA is uniquely suited to support DREAM Challenges related to the elucidation of drug mechanism of action (MOA), drug sensitivity, and drug synergy.  The goal of this Challenge is to foster development and benchmarking of algorithms to predict the sensitivity, as measured by the area under the dose-response curve, of a cell line to a compound based on the baseline transcriptional profiles of the cell line.  The drug perturbational RNAseq profiles of 11 cell lines for 30 chosen compounds will be provided to challenge participants, wit...',
    'https://www.synapse.org/#!Synapse:syn21763589',
    'completed',
    'good_for_beginners',
    '1',
    '',
    '2020-04-28',
    '2020-07-27',
    '2023-06-23 00:00:00',
    '2023-07-26 19:49:10'
  ),
  (
    44,
    'ehr-dream-challenge-covid-19',
    'EHR DREAM Challenge: COVID-19',
    '',
    'The rapid rise of COVID-19 has challenged healthcare globally. The underlying risks and outcomes of infection are still incompletely characterized even as the world surpasses 4 million infections. Due to the importance and emergent need for better understanding of the condition and the development of patient specific clinical risk scores and early warning tools, we have developed a platform to support testing analytic and machine learning hypotheses on clinical data without data sharing as a platform to rapidly discover and implement approaches for care. We have previously applied this approach in the successful EHR DREAM Challenge focusing on Patient Mortality Prediction with UW Medicine.  We have the goal of incorporating machine learning and predictive algorithms into clinical care and COVID-19 is an important and highly urgent challenge. In our first iteration, we will facilitate understanding risk factors that lead to a positive test utilizing electronic health recorded data ...',
    'https://www.synapse.org/#!Synapse:syn21849255',
    'completed',
    'intermediate',
    '1',
    'https://doi.org/10.1001/jamanetworkopen.2021.24946',
    '2020-04-30',
    '2021-07-01',
    '2023-06-23 00:00:00',
    '2023-07-27 19:18:50'
  ),
  (
    45,
    'anti-pd1-response-prediction',
    'Anti-PD1 Response Prediction',
    '',
    'While durable responses and prolonged survival have been demonstrated in some lung cancer patients treated with immuno-oncology (I-O) anti-PD-1 therapy, there remains a need to improve the ability to predict which patients are more likely to receive benefit from treatment with I-O. The goal of this challenge is to leverage clinical and biomarker data to develop predictive models of response to I-O therapy in lung cancer and ultimately gain insights that may facilitate potential novel monotherapies or combinations with I-O.',
    'https://www.synapse.org/#!Synapse:syn18404605',
    'completed',
    'intermediate',
    '1',
    '',
    '2020-11-17',
    '2021-02-25',
    '2023-06-23 00:00:00',
    '2023-07-26 19:49:12'
  ),
  (
    46,
    'brats-2021-challenge',
    'BraTS 2021 Challenge',
    '',
    'Glioblastoma, and diffuse astrocytic glioma with molecular features of glioblastoma (WHO Grade 4 astrocytoma), are the most common and aggressive malignant primary tumor of the central nervous system in adults, with extreme intrinsic heterogeneity in appearance, shape, and histology. Glioblastoma patients have very poor prognosis, and the current standard of care treatment comprises surgery, followed by radiotherapy and chemotherapy. The International Brain Tumor Segmentation (BraTS) Challenges —which have been running since 2012— assess state-of-the-art machine learning (ML) methods used for brain tumor image analysis in mpMRI scans.',
    'https://www.synapse.org/#!Synapse:syn25829067',
    'completed',
    'advanced',
    '1',
    '',
    '2021-07-07',
    '2021-10-15',
    '2023-06-23 00:00:00',
    '2023-07-26 19:49:17'
  ),
  (
    47,
    'cancer-data-registry-nlp',
    'Cancer Data Registry NLP',
    '',
    'A critical bottleneck in translational and clinical research is access to large volumes of high-quality clinical data. While structured data exist in medical EHR systems, a large portion of patient information including patient status, treatments, and outcomes is contained in unstructured text fields. Research in Natural Language Processing (NLP) aims to unlock this hidden and often inaccessible information. However, numerous challenges exist in developing and evaluating NLP methods, much of it centered on having “gold-standard” metrics for evaluation, and access to data that may contain personal health information (PHI).  This DREAM Challenge will focus on the development and evaluation of of NLP algorithms that can improve clinical trial matching and recruitment.',
    'https://www.synapse.org/#!Synapse:syn18361217',
    'upcoming',
    'intermediate',
    '1',
    '',
    NULL,
    NULL,
    '2023-06-23 00:00:00',
    '2023-07-26 19:49:19'
  ),
  (
    48,
    'barda-community-challenge-pediatric-covid-19-data-challenge',
    'BARDA Community Challenge - Pediatric COVID-19 Data Challenge',
    '',
    'While most children with COVID-19 are asymptomatic or have mild symptoms, healthcare providers have difficulty determining which among their pediatric patients will progress to moderate or severe COVID-19 early in the progression. Some of these patients develop multisystem inflammatory syndrome in children (MIS-C), a life-threatening inflammation of organs and tissues. Methods to distinguish children at risk for severe COVID-19 complications, including conditions such as MIS-C, are needed for earlier interventions to improve pediatric patient outcomes. Multiple HHS divisions are coming together for a data challenge competition that will leverage de-identified electronic health record data to develop, train and validate computational models that can predict severe COVID-19 complications in children, equipping healthcare providers with the information and tools they need to identify pediatric patients at risk.',
    'https://www.synapse.org/#!Synapse:syn25875374/wiki/611225',
    'completed',
    'intermediate',
    '1',
    '',
    '2021-08-19',
    '2021-12-17',
    '2023-06-23 00:00:00',
    '2023-07-26 19:49:20'
  ),
  (
    49,
    'brats-continuous-evaluation',
    'BraTS Continuous Evaluation',
    '',
    'Brain tumors are among the deadliest types of cancer. Specifically, glioblastoma, and diffuse astrocytic glioma with molecular features of glioblastoma (WHO Grade 4 astrocytoma), are the most common and aggressive malignant primary tumor of the central nervous system in adults, with extreme intrinsic heterogeneity in appearance, shape, and histology, with a median survival of approximately 15 months. Brain tumors in general are challenging to diagnose, hard to treat and inherently resistant to conventional therapy because of the challenges in delivering drugs to the brain, as well as the inherent high heterogeneity of these tumors in their radiographic, morphologic, and molecular landscapes. Years of extensive research to improve diagnosis, characterization, and treatment have decreased mortality rates in the U.S by 7% over the past 30 years. Although modest, these research innovations have not translated to improvements in survival for adults and children in low- and middle-incom...',
    'https://www.synapse.org/brats_ce',
    'completed',
    'advanced',
    '1',
    '',
    '2022-01-01',
    NULL,
    '2023-06-23 00:00:00',
    '2023-08-09 08:42:59'
  ),
  (
    50,
    'fets-2022',
    'FeTS 2022',
    '',
    'FeTS 2022 focuses on benchmarking methods for federated learning (FL), and particularly i) weight aggregation methods for federated training, and ii) algorithmic generalizability on out-of-sample data based on federated evaluation. In line with its last instance (FeTS 2021 - the 1st FL challenge ever organized), FeTS 2022 targets the task of brain tumor segmentation and builds upon i) the centralized dataset of >8,000 clinically-acquired multi-institutional MRI scans (from the RSNA-ASNR-MICCAI BraTS 2021 challenge) with their real-world partitioning, and ii) the collaborative network of remote independent institutions included in a real-world federation. Participants are welcome to compete in either of the two challenge tasks:  Task 1 (“Federated Training”) seeks effective weight aggregation methods for the creation of a consensus model given a pre-defined segmentation algorithm for training, while also (optionally) accounting for network outages.  Task 2 (“Federated Evaluation”) ...',
    'https://www.synapse.org/#!Synapse:syn28546456/wiki/617093',
    'completed',
    'advanced',
    '1',
    '',
    '2022-04-08',
    '2022-08-15',
    '2023-06-23 00:00:00',
    '2023-08-09 08:40:15'
  ),
  (
    51,
    'random-promotor',
    'Random Promotor',
    '',
    'Decoding how gene expression is regulated is critical to understanding disease. Regulatory DNA is decoded by the cell in a process termed “cis-regulatory logic”, where proteins called Transcription Factors (TFs) bind to specific DNA sequences within the genome and work together to produce as output a level of gene expression for downstream adjacent genes. This process is exceedingly complex to model as a large number of parameters is needed to fully describe the process (see Rationale, de Boer et al. 2020; Zeitingler J. 2020). Understanding the cis-regulatory logic of the human genome is an important goal and would provide insight into the origins of many diseases. However, learning models from human data is challenging due to limitations in the diversity of sequences present within the human genome (e.g. extensive repetitive DNA), the vast number of cell types that differ in how they interpret regulatory DNA, limited reporter assay data, and substantial technical biases present i...',
    'https://www.synapse.org/#!Synapse:syn28469146/wiki/617075',
    'completed',
    'intermediate',
    '1',
    '',
    '2022-05-02',
    '2022-08-07',
    '2023-06-23 00:00:00',
    '2023-07-26 19:49:22'
  ),
  (
    52,
    'preterm-birth-prediction-microbiome',
    'Preterm Birth Prediction - Microbiome',
    '',
    'Globally, about 11% of infants every year are born preterm, defined as birth prior to 37 weeks of gestation, totaling nearly 15 million births.(5) In addition to the emotional and financial toll on families, preterm births have higher rates of neonatal death, nearly 1 million deaths each year, and long-term health consequences for some children. Infants born preterm are at risk for a variety of adverse outcomes, such as respiratory illnesses, cerebral palsy, infections, and blindness, with infants born very preterm (i.e., before 32 weeks) at increased risk of these conditions.(6) The ability to accurately predict which women are at a higher risk for preterm birth would help healthcare providers to treat in a timely manner those at higher risk of delivering preterm. Currently available treatments for pregnant women at risk of preterm delivery include corticosteroids for fetal maturation and magnesium sulfate provided prior to 32 weeks to prevent cerebral palsy.(7) There are several...',
    'https://www.synapse.org/#!Synapse:syn26133770/wiki/612541',
    'completed',
    'advanced',
    '1',
    '',
    '2022-07-19',
    '2022-09-16',
    '2023-06-23 00:00:00',
    '2023-07-26 19:49:23'
  ),
  (
    53,
    'finrisk',
    'FINRISK - Heart Failure and Microbiome',
    '',
    'Cardiovascular diseases are the leading cause of death both in men and women worldwide. Heart failure (HF) is the most common form of heart disease, characterised by the heart''s inability to pump a sufficient supply of blood to meet the needs of the body. The lifetime risk of developing HF is roughly 20%, yet, it remains difficult to diagnose due to its and a lack of agreement of diagnostic criteria. As the diagnosis of HF is dependent on ascertainment of clinical histories and appropriate screening of symptomatic individuals, identifying those at risk of HF is essential. This DREAM challenge focuses on the prediction of HF using a combination of gut microbiome and clinical variables. This challenge is designed to predict incident risk for heart failure in a large human population study of Finnish adults, FINRISK 2002 (Borodulin et al., 2018). The FINRISK study has been conducted in Finland to investigate the risk factors for cardiovascular disease every 5 years since 1972. A ran...',
    'https://www.synapse.org/#!Synapse:syn27130803/wiki/616705',
    'completed',
    'advanced',
    '1',
    '',
    '2022-09-20',
    '2023-01-30',
    '2023-06-23 00:00:00',
    '2023-07-26 19:51:19'
  ),
  (
    54,
    'scrna-seq-and-scatac-seq-data-analysis',
    'scRNA-seq and scATAC-seq Data Analysis',
    '',
    'Understanding transcriptional regulation at individual cell resolution is fundamental to understanding complex biological systems such as tissues and organs. Emerging high-throughput sequencing technologies now allow for transcript quantification and chromatin accessibility at the single cell level. These technologies present unique challenges due to inherent data sparsity. Proper signal correction is key to accurate gene expression quantification via scRNA-seq, which propagates into downstream analyses such as differential gene expression analysis and cell-type identification. In the even more sparse scATAC-seq data, the correct identification of informative features is key to assessing cell heterogeneity at the chromatin level. The aims of this challenge will be two-fold:  1) To evaluate computational methods for signal correction and peak identification in scRNA-seq and scATAC-seq, respectively;  2) To assess the impact of these methods on downstream analysis',
    'https://www.synapse.org/#!Synapse:syn26720920/wiki/615338',
    'completed',
    'advanced',
    '1',
    '',
    '2022-11-29',
    '2023-02-08',
    '2023-06-23 00:00:00',
    '2023-08-09 08:40:41'
  ),
  (
    55,
    'cough-diagnostic-algorithm-for-tuberculosis',
    'COugh Diagnostic Algorithm for Tuberculosis',
    '',
    'Tuberculosis (TB), a communicable disease caused by Mycobacterium tuberculosis, is a major cause of ill health and one of the leading causes of death worldwide. Until the COVID-19 pandemic, TB was the leading cause of death from a single infectious agent, ranking even above HIV/AIDS. In 2020, an estimated 9.9 million people fell ill with TB and 1.3 million died of TB worldwide. However, approximately 40% of people with TB were not diagnosed or reported to public health authorities because of challenges in accessing health facilities or failure to be tested or treated when they do. The development of low-cost, non-invasive digital screening tools may improve some of the gaps in diagnosis. As cough is a common symptom of TB, it has the potential to be used as a biomarker for diagnosis of disease. Several previous studies have demonstrated the potential for cough sounds to be used to screen for TB[1-3], though these were typically done in small samples or limited settings. Further de...',
    'https://www.synapse.org/#!Synapse:syn31472953/wiki/617828',
    'completed',
    'advanced',
    '1',
    '',
    '2022-10-16',
    '2023-02-13',
    '2023-06-23 00:00:00',
    '2023-07-26 19:49:25'
  ),
  (
    56,
    'nih-long-covid-computational-challenge',
    'NIH Long COVID Computational Challenge',
    '',
    'The overall prevalence of post-acute sequelae of SARS-CoV-2 (PASC) is currently unknown, but there is growing evidence that more than half of COVID-19 survivors experience at least one symptom of PASC/Long COVID at six months after recovery of the acute illness. Reports also reflect an underlying heterogeneity of symptoms, multi-organ involvement, and persistence of PASC/Long COVID in some patients. Research is ongoing to understand prevalence, duration, and clinical outcomes of PASC/Long COVID. Symptoms of fatigue, cognitive impairment, shortness of breath, and cardiac damage, among others, have been observed in patients who had only mild initial disease. The breadth and complexity of data created in today''s health care encounters require advanced analytics to extract meaning from longitudinal data on symptoms, laboratory results, images, functional tests, genomics, mobile health/wearable devices, written notes, electronic health records (EHR), and other relevant data types. Adv...',
    'https://www.synapse.org/#!Synapse:syn33576900/wiki/618451',
    'completed',
    'intermediate',
    '1',
    '',
    '2022-08-25',
    '2022-12-15',
    '2023-06-23 00:00:00',
    '2023-07-26 19:49:26'
  ),
  (
    57,
    'bridge2ai',
    'Bridge2AI',
    'What makes a good color palette?',
    'What makes a good color palette?',
    '',
    'upcoming',
    'good_for_beginners',
    '1',
    '',
    NULL,
    NULL,
    '2023-06-23 00:00:00',
    '2023-07-26 19:49:28'
  ),
  (
    58,
    'rare-x-open-data-science',
    'RARE-X Open Data Science',
    '',
    'The Xcelerate RARE: A Rare Disease Open Science Data Challenge is bringing together researchers and data scientists in a collaborative and competitive environment to make the best use of patient-provided data to solve big unknowns in healthcare. The Challenge will launch to researchers in late May 2023, focused on rare pediatric neurodevelopmental diseases. ',
    'https://www.synapse.org/#!Synapse:syn51198355/wiki/621435',
    'completed',
    'intermediate',
    '1',
    '',
    '2023-05-17',
    '2023-08-16',
    '2023-06-23 00:00:00',
    '2023-08-07 20:19:38'
  ),
  (
    59,
    'cagi5-regulation-saturation',
    'CAGI5: Regulation saturation',
    '',
    '17,500 single nucleotide variants (SNVs) in 5 human disease associated enhancers (including IRF4, IRF6, MYC, SORT1) and 9 promoters (including TERT, LDLR, F9, HBG1) were assessed in a saturation mutagenesis massively parallel reporter assay. Promoters were cloned into a plasmid upstream of a tagged reporter construct, and reporter expression was measured relative to the plasmid DNA to determine the impact of promoter variants. Enhancers were placed upstream of a minimal promoter and assayed similarly. The challenge is to predict the functional effects of these variants in the regulatory regions as measured from the reporter expression.',
    'https://genomeinterpretation.org/cagi5-regulation-saturation.html',
    'completed',
    'intermediate',
    '2',
    '',
    '2018-01-04',
    '2018-05-03',
    '2023-06-23 00:00:00',
    '2023-07-26 19:49:31'
  ),
  (
    60,
    'cagi5-calm1',
    'CAGI5: CALM1',
    '',
    'Calmodulin is a calcium-sensing protein that modulates the activity of a large number of proteins in the cell. It is involved in many cellular processes, and is especially important for neuron and muscle cell function. Variants that affect calmodulin function have been found to be causally associated with cardiac arrhythmias. A large library of calmodulin missense variants was assessed with respect to their effects on protein function using a high-throughput yeast complementation assay. The challenge is to predict the functional effects of these calmodulin variants on competitive growth in a high-throughput yeast complementation assay.',
    'https://genomeinterpretation.org/cagi5-calm1.html',
    'completed',
    'intermediate',
    '2',
    '',
    '2017-10-21',
    '2017-12-20',
    '2023-06-23 00:00:00',
    '2023-07-26 19:49:33'
  ),
  (
    61,
    'cagi5-pcm1',
    'CAGI5: PCM1',
    '',
    'The PCM1 (Pericentriolar Material 1) gene is a component of centriolar satellites occurring around centrosomes in vertebrate cells. Several studies have implicated PCM1 variants as a risk factor for schizophrenia. Ventricular enlargement is one of the most consistent abnormal structural brain findings in schizophrenia Therefore 38 transgenic human PCM1 missense mutations implicated in schizophrenia were assayed in a zebrafish model to determine their impact on the posterior ventricle area. The challenge is to predict whether variants implicated in schizophrenia impact zebrafish ventricular area.',
    'https://genomeinterpretation.org/cagi5-pcm1.html',
    'completed',
    'intermediate',
    '2',
    '',
    '2017-11-09',
    '2018-04-19',
    '2023-06-23 00:00:00',
    '2023-07-26 19:49:36'
  ),
  (
    62,
    'cagi5-frataxin',
    'CAGI5: Frataxin',
    '',
    'Fraxatin is a highly-conserved protein found in prokaryotes and eukaryotes that is required for efficient regulation of cellular iron homeostasis. Humans with a frataxin deficiency have the cardio- and neurodegenerative disorder Friedreich''s ataxia. A library of eight missense variants was assessed by near and far-UV circular dichroism and intrinsic fluorescence spectra to determine thermodynamic stability at different concentration of denaturant. These were used to calculate a ΔΔGH20 value, the difference in unfolding free energy (ΔGH20) between the mutant and wild-type proteins for each variant. The challenge is to predict ΔΔGH20 for each frataxin variant.',
    'https://genomeinterpretation.org/cagi5-frataxin.html',
    'completed',
    'intermediate',
    '2',
    '',
    '2017-11-30',
    '2018-04-18',
    '2023-06-23 00:00:00',
    '2023-07-26 19:49:36'
  ),
  (
    63,
    'cagi5-tpmt-and-p10',
    'CAGI5: TPMT and p10',
    '',
    'The gene p10 encodes for PTEN (Phosphatase and TEnsin Homolog), an important secondary messenger molecule promoting cell growth and survival through signaling cascades including those controlled by AKT and mTOR. Thiopurine S-methyl transferase (TPMT) is a key enzyme involved in the metabolism of thiopurine drugs and functions by catalyzing the S-methylation of aromatic and heterocyclic sulfhydryl groups. A library of thousands of PTEN and TPMT mutations was assessed to measure the stability of the variant protein using a multiplexed variant stability profiling (VSP) assay, which detects the presence of EGFP fused to the mutated PTEN and TPMT protein respectively. The stability of the variant protein dictates the abundance of the fusion protein and thus the EGFP level of the cell. The challenge is to predict the effect of each variant on TPMT and/or PTEN protein stability.',
    'https://genomeinterpretation.org/cagi5-tpmt.html',
    'completed',
    'intermediate',
    '2',
    '',
    '2017-11-30',
    '2017-12-01',
    '2023-06-23 00:00:00',
    '2023-07-26 19:49:37'
  ),
  (
    64,
    'cagi5-annotate-all-nonsynonymous-variants',
    'CAGI5: Annotate all nonsynonymous variants',
    '',
    'dbNSFP describes 810,848,49 possible protein-altering variants in the human genome. The challenge is to predict the functional effect of every such variant. For the vast majority of these missense variants, the functional impact is not currently known, but experimental and clinical evidence are accruing rapidly. Rather than drawing upon a single discrete dataset as typical with CAGI, predictions will be assessed by comparing with experimental or clinical annotations made available after the prediction submission date, on an ongoing basis. if predictors assent, predictions will also incorporated into dbNSFP.',
    'https://genomeinterpretation.org/cagi5-annotate-all-missense.html',
    'completed',
    'intermediate',
    '2',
    '',
    '2017-11-30',
    '2018-05-09',
    '2023-06-23 00:00:00',
    '2023-07-26 19:49:38'
  ),
  (
    65,
    'cagi5-gaa',
    'CAGI5: GAA',
    '',
    'Acid alpha-glucosidase (GAA) is a lysosomal alpha-glucosidase. Some mutations in GAA cause a rare disorder, Pompe disease, (Glycogen Storage Disease II). Rare GAA missense variants found in a human population sample have been assayed for enzymatic activity in transfected cell lysates. The assessment of this challenge will include evaluations that recognize novelty of approach. The challenge is to predict the fractional enzyme activity of each mutant protein compared to the wild-type enzyme.',
    'https://genomeinterpretation.org/cagi5-gaa.html',
    'completed',
    'intermediate',
    '2',
    '',
    '2017-11-09',
    '2018-04-25',
    '2023-06-23 00:00:00',
    '2023-07-26 19:49:39'
  ),
  (
    66,
    'cagi5-chek2',
    'CAGI5: CHEK2',
    '',
    'Variants in the CHEK2 gene are associated with breast cancer. This challenge includes CHEK2 gene variants from approximately 1200 Latino breast cancer cases and 1200 ethnically matched controls. This challenge is to estimate the probability of each gene variant occurring in an individual from the cancer affected cohort.',
    'https://genomeinterpretation.org/cagi5-chek2.html',
    'completed',
    'intermediate',
    '2',
    '',
    '2017-12-20',
    '2018-04-24',
    '2023-06-23 00:00:00',
    '2023-07-26 19:49:40'
  ),
  (
    67,
    'cagi5-enigma',
    'CAGI5: ENIGMA',
    '',
    'Breast cancer is the most prevalent cancer among women worldwide. The association between germline mutations in the BRCA1 and BRCA2 genes and the development of cancer has been well established. The most common high-risk mutations associated with breast cancer are those in the autosomal dominant breast cancer genes 1 and 2 (BRCA1 and BRCA2). Mutations in these genes are found in 1-3% of breast cancer cases. The challenge is to predict which variants are associated with increased risk for breast cancer.',
    'https://genomeinterpretation.org/cagi5-enigma.html',
    'completed',
    'intermediate',
    '2',
    '',
    '2017-12-20',
    '2018-05-01',
    '2023-06-23 00:00:00',
    '2023-07-26 19:49:42'
  ),
  (
    68,
    'cagi5-mapsy',
    'CAGI5: MaPSy',
    '',
    'The Massively Parallel Splicing Assay (MaPSy) approach was used to screen 797 reported exonic disease mutations using a mini-gene system, assaying both in vivo via transfection in tissue culture, and in vitro via incubation in cell nuclear extract. The challenge is to predict the degree to which a given variant causes changes in splicing.',
    'https://genomeinterpretation.org/cagi5-mapsy.html',
    'completed',
    'intermediate',
    '2',
    '',
    '2017-11-29',
    '2018-05-07',
    '2023-06-23 00:00:00',
    '2023-07-26 19:49:42'
  ),
  (
    69,
    'cagi5-vex-seq',
    'CAGI5: Vex-seq',
    '',
    'A barcoding approach called Variant exon sequencing (Vex-seq) was applied to assess effect of 2,059 natural single nucleotide variants and short indels on splicing of a globin mini-gene construct transfected into HepG2 cells. This is reported as ΔΨ (delta PSI, or Percent Spliced In), between the variant Ψand the reference Ψ. The challenge is to predict ΔΨ for each variant.',
    'https://genomeinterpretation.org/cagi5-vex-seq.html',
    'completed',
    'intermediate',
    '2',
    '',
    '2017-12-14',
    '2018-05-02',
    '2023-06-23 00:00:00',
    '2023-07-26 19:49:43'
  ),
  (
    70,
    'cagi5-sickkids-clinical-genomes',
    'CAGI5: SickKids clinical genomes',
    '',
    'This challenge involves 30 children with suspected genetic disorders who were referred for clinical genome sequencing. Predictors are given the 30 genome sequences, and are also provided with the phenotypic descriptions as shared with the diagnostic laboratory. The challenge is to predict what class of disease is associated with each genome, and which genome corresponds to which clinical description. Predictors may additionally identify the diagnostic variant(s) underlying the predictions, and identify predictive secondary variants conferring high risk of other diseases whose phenotypes are not reported in the clinical descriptions.',
    'https://genomeinterpretation.org/cagi5-sickkids5.html',
    'completed',
    'intermediate',
    '2',
    '',
    '2017-12-22',
    '2018-04-26',
    '2023-06-23 00:00:00',
    '2023-07-26 19:49:43'
  ),
  (
    71,
    'cagi5-id-panel',
    'CAGI5: ID Panel',
    '',
    'The challenge presented here is to use computational methods to predict a patient''s clinical phenotype and the causal variant(s) based on analysis of their gene panel sequence data. Sequence data for 74 genes associated with intellectual disability (ID) and/or Autism spectrum disorders (ASD) from a cohort of 150 patients with a range of neurodevelopmental presentations (ID, autism, epilepsy, etc..) have been made available for this challenge. For each patient, predictors must report the causative variants and which of seven phenotypes are present.',
    'https://genomeinterpretation.org/cagi5-intellectual-disability.html',
    'completed',
    'intermediate',
    '2',
    '',
    '2017-12-22',
    '2018-04-30',
    '2023-06-23 00:00:00',
    '2023-07-26 19:49:44'
  ),
  (
    72,
    'cagi5-clotting-disease-exomes',
    'CAGI5: Clotting disease exomes',
    '',
    'African Americans have a higher incidence of developing venous thromboembolisms (VTE), which includes deep vein thrombosis (DVT) and pulmonary embolism (PE), than people of European ancestry. Participants are provided with exome data and clinical covariates for a cohort of African Americans who have been prescribed Warfarin either because they had experienced a VTE event or had been diagnosed with atrial fibrillation (which predisposes to clotting). The challenge is to distinguish between these conditions. At present, in contrast to European ancestry, there are no genetic methods for anticipating which African Americans are most at risk of a venous thromboembolism, and the results of this challenge may contribute to the development of such tools.',
    'https://genomeinterpretation.org/cagi5-clotting-disease.html',
    'completed',
    'intermediate',
    '2',
    '',
    '2017-11-23',
    '2018-04-28',
    '2023-06-23 00:00:00',
    '2023-08-09 08:43:32'
  ),
  (
    73,
    'cagi6-rare-genomes-project',
    'CAGI6: Rare Genomes Project',
    '',
    'The Rare Genomes Project (RGP) is a direct-to-participant research study on the utility of genome sequencing for rare disease diagnosis and gene discovery. The study is led by genomics experts and clinicians at the Broad Institute of MIT and Harvard. Research subjects are consented for genomic sequencing and the sharing of their sequence and phenotype information with researchers working to understand the molecular causes of rare disease. When a candidate disease variant believed to be related to the phenotype is identified, the variant is confirmed with Sanger sequencing in a clinical setting and returned to the participant via his or her local physician. In this challenge, whole genome sequence data and phenotype data from a subset of the solved and unsolved RGP families will be provided. Participants in the challenge will try to identify the causative variant(s) in each case. For the unsolved cases, prioritized variants from the participating teams will be examined to see if ad...',
    'https://genomeinterpretation.org/cagi6-rgp.html',
    'completed',
    'intermediate',
    '1',
    '',
    '2021-06-08',
    '2021-10-11',
    '2023-06-23 00:00:00',
    '2023-08-09 08:43:32'
  ),
  (
    74,
    'cagi6-intellectual-disability-panel',
    'CAGI6: Intellectual Disability Panel',
    '',
    'The objective in this challenge is to predict a patient''s clinical phenotype and the causal variant(s) based on their gene panel sequences. Sequence data for 74 genes from a cohort of 500 patients with a range of neurodevelopmental presentations (intellectual disability, autistic spectrum disorder, epilepsy, microcephaly, macrocephaly, hypotonia, ataxia) has been made available for this challenge. Additional data from 150 patients from the same clinical study is available for training and validation.',
    'https://genomeinterpretation.org/cagi6-id-panel.html',
    'completed',
    'intermediate',
    '1',
    '',
    '2021-06-08',
    '2021-10-11',
    '2023-06-23 00:00:00',
    '2023-08-09 08:43:34'
  ),
  (
    75,
    'cagi6-sickkids-clinical-genomes-and-transcriptomes',
    'CAGI6: SickKids clinical genomes and transcriptomes',
    '',
    'This challenge involves data from 79 children who were referred to The Hospital for Sick Children''s (SickKids) Genome Clinic for genome sequencing because of suspected but undiagnosed genetic disorders. Research subjects are consented for sharing of their sequence data and phenotype information with researchers working to understand the molecular causes of rare disease. When a candidate disease variant believed to be related to the phenotype is identified, the variant is adjudicated and confirmed in a clinical setting. In this challenge, transcriptomic and phenotype data from a subset of the “solved” (diagnosed) and “unsolved” SickKids patients will be provided, along with corresponding genomic sequence data. The challenge is to use a transcriptome-driven approach to identify the gene(s) and molecular mechanisms underlying the phenotypic descriptions in each case. For the unsolved cases, prioritized variants from the participating teams will be examined to see if additional diagn...',
    'https://genomeinterpretation.org/cagi6-sickkids.html',
    'completed',
    'intermediate',
    '1',
    '',
    '2021-08-04',
    '2021-12-31',
    '2023-06-23 00:00:00',
    '2023-08-09 08:43:37'
  ),
  (
    76,
    'cagi6-polygenic-risk-scores',
    'CAGI6: Polygenic Risk Scores',
    '',
    'Polygenic risk scores (PRS) have potential clinical utility for risk surveillance, prevention and personalized medicine. Participants will be provided with datasets of four real phenotypes (Type 2 Diabetes, Breast Cancer, Inflammatory Bowel Disease and Coronary Artery Disease) and of thirty simulated phenotypes representing a range of genetic architectures of common polygenic diseases. The challenge is to predict the disease outcomes of individuals in held-out validation cohorts.',
    'https://genomeinterpretation.org/cagi6-prs.html',
    'completed',
    'intermediate',
    '1',
    '',
    '2021-06-08',
    '2021-10-11',
    '2023-06-23 00:00:00',
    '2023-08-09 08:43:39'
  ),
  (
    77,
    'cagi6-hmbs',
    'CAGI6: HMBS',
    '',
    'Hydroxymethylbilane synthase (HMBS), also known as porphobilinogen deaminase (PBGD) or uroporphyrinogen I synthase, is an enzyme involved in heme production. In humans, variants that affect HMBS function result in acute intermittent porphyria (AIP), an autosomal dominant genetic disorder caused by a build-up of porphobilinogen in the cytoplasm. A large library of HMBS missense variants was assessed with respect to their effects on protein function using a high-throughput yeast complementation assay. The challenge is to predict the functional effects of these variants.',
    'https://genomeinterpretation.org/cagi6-hmbs.html',
    'completed',
    'intermediate',
    '1',
    '',
    '2021-06-08',
    '2021-10-11',
    '2023-06-23 00:00:00',
    '2023-08-09 08:43:40'
  ),
  (
    78,
    'cagi6-cam',
    'CAGI6: CaM',
    '',
    'Calmodulin (CaM) is a ubiquitous calcium (Ca2+) sensor protein interacting with more than 200 molecular partners, thereby regulating a variety of biological processes. Missense point mutations in the genes encoding CaM have been associated with ventricular tachycardia and sudden cardiac death. A library encompassing up to 17 point mutations was assessed by far-UV circular dichroism (CD) by measuring melting temperature (Tm) and percentage of unfolding (%unfold) upon thermal denaturation at pH and salt concentration that mimic the physiological conditions. The challenge is to predict: (1) the Tm and %unfold values for isolated CaM variants under Ca2+-saturating conditions (Ca2+-CaM) and in the Ca2+-free (apo) state; (2) whether the point mutation stabilizes or destabilizes the protein (based on Tm and %unfold).',
    'https://genomeinterpretation.org/cagi6-cam.html',
    'completed',
    'intermediate',
    '1',
    '',
    '2021-06-08',
    '2021-10-11',
    '2023-06-23 00:00:00',
    '2023-08-09 08:43:41'
  ),
  (
    79,
    'cagi6-annotate-all-missense',
    'CAGI6: Annotate All Missense',
    '',
    'dbNSFP currently describes 81,782,923 possible protein-altering variants in the human genome. The challenge is to predict the functional effect of every such variant. For the vast majority of these missense and nonsense variants, the functional impact is not currently known, but experimental and clinical evidence is accruing rapidly. Rather than drawing upon a single discrete dataset as typical with CAGI, predictions will be assessed by comparing with experimental or clinical annotations made available after the prediction submission date, on an ongoing basis. If predictors assent, predictions will also be incorporated into dbNSFP.',
    'https://genomeinterpretation.org/cagi6-annotate-all-missense.html',
    'completed',
    'intermediate',
    '1',
    '',
    '2021-06-01',
    '2021-10-11',
    '2023-06-23 00:00:00',
    '2023-08-09 08:43:42'
  ),
  (
    80,
    'cagi6-stk11',
    'CAGI6: STK11',
    '',
    'Serine/Threonine Kinase 11 (STK11) is considered a master kinase that functions as a tumor suppressor and nutrient sensor within a heterotrimeric complex with pseudo-kinase STRAD-alpha and structural protein MO25. Germline variants resulting in loss of STK11 define Peutz-Jaghers Syndrome, an autosomal dominant cancer predisposition syndrome marked by gastrointestinal hamartomas and freckling of the oral mucosa. Somatic loss of function variants, both nonsense and missense, occur in 15-30% of non-small cell lung adenocarcinomas, where they correlate clinically with insensitivity to anti-PD1 monoclonal antibody therapy. The challenge is to predict the impact on STK11 function for each missense variant in relation to wildtype STK11.',
    'https://genomeinterpretation.org/cagi6-stk11.html',
    'completed',
    'intermediate',
    '1',
    '',
    '2021-06-08',
    '2021-09-01',
    '2023-06-23 00:00:00',
    '2023-08-09 08:43:43'
  ),
  (
    81,
    'cagi6-mapk1',
    'CAGI6: MAPK1',
    '',
    'MAPK1 (ERK2) is active as serine/threonine kinase in the Ras-Raf-MEK-ERK signal transduction cascade that regulates cell proliferation, transcription, differentiation, and cell cycle progression. MAPK1 is activated by phosphorylation which occurs with strict specificity by MEK1/2 on Thr185 and Tyr187, and may also act as a transcriptional repressor independent of its kinase activity. A library of eleven missense variants selected from the COSMIC database was assessed by near and far-UV circular dichroism and intrinsic fluorescence spectra to determine thermodynamic stability at different concentrations of denaturant. These data were used to calculate a ΔΔGH20 value; i.e., the difference in unfolding free energy ΔGH20 between each variant and the wildtype protein, both in phosphorylated and unphosphorylated forms. The challenge is to predict these two ΔΔGH20 values and the catalytic efficiency (kcat/km)mut/(kcat/km)wt, as determined by a fluorescence assay, of the phosphorylated fo...',
    'https://genomeinterpretation.org/cagi6-mapk1.html',
    'completed',
    'intermediate',
    '1',
    '',
    '2021-07-08',
    '2021-10-11',
    '2023-06-23 00:00:00',
    '2023-08-09 08:43:44'
  ),
  (
    82,
    'cagi6-mapk3',
    'CAGI6: MAPK3',
    '',
    'MAPK3 (ERK1) is active as serine/threonine kinase in the Ras-Raf-MEK-ERK signal transduction cascade that regulates cell proliferation, transcription, differentiation, and cell cycle progression. MAPK3 is activated by phosphorylation which occurs with strict specificity by MEK1/2 on Thr202 and Tyr204, and may also act as a transcriptional repressor independent of its kinase activity. A library of twelve missense variants selected from the COSMIC database was assessed by near and far-UV circular dichroism and intrinsic fluorescence spectra to determine thermodynamic stability at different concentrations of denaturant. These data were used to calculate a ΔΔGH20 value; i.e., the difference in unfolding free energy ΔGH20 between each variant and the wildtype protein, both in phosphorylated and unphosphorylated forms. The challenge is to predict these two ΔΔGH20 values and the catalytic efficiency (kcat/km)mut/(kcat/km)wt, as determined by a fluorescence assay, of the phosphorylated fo...',
    'https://genomeinterpretation.org/cagi6-mapk3.html',
    'completed',
    'intermediate',
    '1',
    '',
    '2021-08-04',
    '2021-10-11',
    '2023-06-23 00:00:00',
    '2023-08-09 08:43:45'
  ),
  (
    83,
    'cagi6-mthfr',
    'CAGI6: MTHFR',
    '',
    'Methylenetetrahydrofolate reductase (MTHFR) catalyzes the production of 5-methyltetrahydrofolate, which is needed for conversion of homocysteine to methionine. Humans with variants affecting MTHFR function present with a wide range of phenotypes, including homocystinuria, homocysteinemia, developmental delay, severe mental retardation, psychiatric disturbances, and late-onset neurodegenerative disorders. A further complication to interpretation of variants in this gene is a common variant, Ala222Val, carried by a large fraction of the human population. A large library of MTHFR missense variants was assessed with respect to their effects on protein function using a high-throughput yeast complementation assay. The challenge is to predict the functional effects of these variants in two different settings: (1) for the wildtype protein, and (2) for the protein with the common Ala222Val variant.',
    'https://genomeinterpretation.org/cagi6-mthfr.html',
    'completed',
    'intermediate',
    '1',
    '',
    '2021-05-03',
    '2021-06-30',
    '2023-06-23 00:00:00',
    '2023-08-09 08:43:45'
  ),
  (
    84,
    'cagi6-splicing-vus',
    'CAGI6: Splicing VUS',
    '',
    'Variants causing aberrant splicing have been implicated in a range of common and rare disorders, including retinitis pigmentosa, autism spectrum disorder, amyotrophic lateral sclerosis, and a variety of cancers. However, such variants are frequently overlooked by diagnostic sequencing pipelines, leading to missed diagnoses for patients. Clinically ascertained variants of unknown significance underwent whole-blood based RT-PCR to test for impact on splicing. The challenge is to predict which of the tested variants disrupt splicing.',
    'https://genomeinterpretation.org/cagi6-splicing-vus.html',
    'completed',
    'intermediate',
    '1',
    '',
    '2021-06-08',
    '2021-10-11',
    '2023-06-23 00:00:00',
    '2023-08-09 08:43:47'
  ),
  (
    85,
    'cagi6-sherloc-clinical-classification',
    'CAGI6: Sherloc clinical classification',
    '',
    'Invitae is a genetic testing company that publishes their variant interpretations to ClinVar. In this challenge, over 122,000 previously uncharacterized variants are provided, spanning the range of effects seen in the clinic. Following the close of this challenge, Invitae will submit their interpretations for these variants to ClinVar. Predictors are asked to interpret the pathogenicity of these variants, and the clinical utility of predictions will be assessed across multiple categories by Invitae.',
    'https://genomeinterpretation.org/cagi6-invitae.html',
    'completed',
    'intermediate',
    '1',
    '',
    '2021-07-08',
    '2021-12-01',
    '2023-06-23 00:00:00',
    '2023-08-09 08:43:48'
  ),
  (
    86,
    'cami-ii',
    'CAMI II',
    '',
    'CAMI II offers several challenges: an assembly, a genome binning, a taxonomic binning and a taxonomic profiling challenge, on several multi-sample data sets from different environments, including long and short read data. This includes a marine data set and a high-strain diversity data set, with a third data set to follow later. A pathogen detection challenge on a clinical sample is also provided.',
    'https://www.microbiome-cosi.org/cami/cami/cami2',
    'completed',
    'intermediate',
    '3',
    '',
    '2019-01-14',
    '2021-01-31',
    '2023-06-23 00:00:00',
    '2023-07-26 19:49:45'
  ),
  (
    87,
    'camda18-metasub-forensics',
    'CAMDA18: MetaSUB Forensics',
    '',
    'The MetaSUB International Consortium is building a longitudinal metagenomic map of mass-transit systems and other public spaces across the globe. The consortium maintains a strategic partnership with CAMDA and this year provides data from global City Sampling Days for the first-ever multi-city forensic analyses.',
    'http://camda2018.bioinf.jku.at/doku.php/contest_dataset#metasub_forensics_challenge',
    'completed',
    'intermediate',
    '7',
    '',
    NULL,
    NULL,
    '2023-06-23 00:00:00',
    '2023-07-26 19:49:46'
  ),
  (
    88,
    'camda18-cmap-drug-safety',
    'CAMDA18: CMap Drug Safety',
    '',
    'Attrition in drug discovery and development due to safety / toxicity issues remains a significant concern, and there are strong efforts to identify and mitigate risk as early as possible. Drug-induced liver injury (DILI) is one of the primary problems in drug development and regulatory clearance due to the poor performance of existing preclinical models. There is a pressing need to evaluate alternative methods for predicting DILI, with great hopes being placed in modern approaches from statistics and machine learning applied to genome scale profiling data. A critical question thus is if we can better integrate, understand, and exploit information from cell-based screens like the Broad Institute Connectivity Map (CMap, Science 313, Nature Reviews Cancer 7).
 This CAMDA challenge focuses on understanding or predicting drug induced liver injury in humans from cell-based screens, specifically the CMap gene expression responses of two different cancer cell lines (MCF7 and PC3) to 276 d...',
    'http://camda2018.bioinf.jku.at/doku.php/contest_dataset#cmap_drug_safety_challenge',
    'completed',
    'intermediate',
    '7',
    '',
    NULL,
    NULL,
    '2023-06-23 00:00:00',
    '2023-07-26 19:49:46'
  ),
  (
    89,
    'camda18-cancer-data-integration',
    'CAMDA18: Cancer Data Integration',
    '',
    'Examine the power of data integration in a real-world clinical settings. Many approaches work well on some data-sets yet not on others. We here challenge you to demonstrate a unified single approach to data-integration that matches or outperforms the current state of the art on two different diseases, breast cancer and neuroblastoma. Breast cancer affects about 3 million women every year (McGuire et al, Cancers 7), and this number is growing fast, especially in developed countries. Can you improve on the large Metabric study (Curtis et al., Nature 486, and Dream Challenge, Margolin et al, Sci Transl Med 5)? The cohort is biologically heterogeneous with all five distinct PAM50 breast cancer subtypes represented. Matched profiles for microarray and copy number data as well as clinical information (survival times, multiple prognostic markers, therapy data) are available for about 2,000 patients. Neuroblastoma is the most common extracranial solid tumor in children. The base study com...',
    'http://camda2018.bioinf.jku.at/doku.php/contest_dataset#cancer_data_integration_challenge',
    'completed',
    'intermediate',
    '7',
    '',
    NULL,
    NULL,
    '2023-06-23 00:00:00',
    '2023-07-26 19:49:47'
  ),
  (
    90,
    'cafa-4',
    'CAFA 4',
    '',
    'The goal of the Critical Assessment of Functional Annotation(CAFA) challenge is to evaluate automated protein function prediction algorithms in the task of predicting Gene Ontology and Human Phenotype Ontology terms for a given set of protein sequences. For the GO-based predictions, the evaluation will be carried out for the Molecular Function Ontology, Biological Process Ontology and Cellular Component Ontology. Participants develop protein function prediction algorithms using training protein sequence data and submit their predictions on target protein sequence data.',
    'https://www.biofunctionprediction.org/cafa/',
    'completed',
    'intermediate',
    '1',
    '',
    '2019-10-21',
    '2020-02-12',
    '2023-06-23 00:00:00',
    '2023-08-05 00:32:49'
  ),
  (
    91,
    'casp13',
    'CASP13',
    '',
    'CASP (Critical Assessment of Structure Prediction) is a community wide experiment to determine and advance the state of the art in modeling protein structure from amino acid sequence. Every two years, participants are invited to submit models for a set of proteins for which the experimental structures are not yet public. Independent assessors then compare the models with experiment. Assessments and results are published in a special issue of the journal PROTEINS. In the most recent CASP round, CASP12, nearly 100 groups from around the world submitted more than 50,000 models on 82 modeling targets',
    'https://predictioncenter.org/casp13/index.cgi',
    'completed',
    'intermediate',
    '4',
    '',
    '2018-04-18',
    '2018-08-20',
    '2023-06-23 00:00:00',
    '2023-07-26 19:49:48'
  ),
  (
    92,
    'casp14',
    'CASP14',
    '',
    'CASP (Critical Assessment of Structure Prediction) is a community wide experiment to determine and advance the state of the art in modeling protein structure from amino acid sequence. Every two years, participants are invited to submit models for a set of proteins for which the experimental structures are not yet public. Independent assessors then compare the models with experiment. Assessments and results are published in a special issue of the journal PROTEINS. In the most recent CASP round, CASP14, nearly 100 groups from around the world submitted more than 67,000 models on 90 modeling targets.',
    'https://predictioncenter.org/casp14/index.cgi',
    'completed',
    'intermediate',
    '4',
    '',
    '2020-05-04',
    '2020-09-07',
    '2023-06-23 00:00:00',
    '2023-07-26 19:49:51'
  ),
  (
    93,
    'cfsan-pathogen-detection',
    'CFSAN Pathogen Detection',
    '',
    'In the U.S. alone, one in six individuals, an estimated 48 million people, fall prey to foodborne illness, resulting in 128,000 hospitalizations and 3,000 deaths per year. Economic burdens are estimated cumulatively at $152 billion dollars annually, including $39 billion due to contamination of fresh and processed produce. One longstanding problem is the ability to rapidly identify the food-source associated with the outbreak being investigated. The faster an outbreak is identified and the increased certainty that a given source (e.g., papayas from Mexico) and patients are linked, the faster the outbreak can be stopped, limiting morbidity and mortality.  In the last few years, the application of next-generation sequencing (NGS) technology for whole genome sequencing (WGS) of foodborne pathogens has revolutionized food pathogen outbreak surveillance. WGS of foodborne pathogens enables high-resolution identification of pathogens isolated from food or environmental samples. These pat...',
    'https://precision.fda.gov/challenges/2',
    'completed',
    'intermediate',
    '6',
    '',
    '2018-02-15',
    '2018-04-26',
    '2023-06-23 00:00:00',
    '2023-07-26 19:49:51'
  ),
  (
    94,
    'cdrh-biothreat',
    'CDRH Biothreat',
    '',
    'Many infectious diseases have similar signs and symptoms, making it challenging for healthcare providers to identify the disease-causing agent. Clinical samples are often tested by multiple test methods to help reveal the microbe that is causing the infectious disease. The results of these test methods can help healthcare professionals determine the best treatment for patients. Today, High-Throughput Sequencing (HTS) or Next Generation Sequencing (NGS) technology has the capability, as a single test, to accomplish what might have required several different tests in the past.  NGS technology may allow the diagnosis of infections without prior knowledge of disease(s) cause. NGS technology can potentially reveal the presence of all microorganisms in a patient sample. Using infectious disease NGS (ID-NGS) technology, each microbial pathogen may be identified by its unique genomic fingerprint. The vision of ID-NGS technology is to further improve patient care by delivering diagnostics ...',
    'https://precision.fda.gov/challenges/3',
    'completed',
    'intermediate',
    '6',
    '',
    '2018-08-03',
    '2018-10-18',
    '2023-06-23 00:00:00',
    '2023-07-26 19:49:52'
  ),
  (
    95,
    'multi-omics-enabled-sample-mislabeling-correction',
    'Multi-omics Enabled Sample Mislabeling Correction',
    '',
    'In biomedical research, sample mislabeling (accidental swapping of patient samples) or data mislabeling (accidental swapping of patient omics data) has been a long-standing problem that contributes to irreproducible results and invalid conclusions. These problems are particularly prevalent in large scale multi-omics studies, in which multiple different omics experiments are carried out at different time periods and/or in different labs. Human errors could arise during sample transferring, sample tracking, large-scale data generation, and data sharing/management. Thus, there is a pressing need to identify and correct sample and data mislabeling events to ensure the right data for the right patient. Simultaneous use of multiple types of omics platforms to characterize a large set of biological samples, as utilized in The Cancer Genome Atlas (TCGA) and the Clinical Proteomic Tumor Analysis Consortium (CPTAC) projects, has been demonstrated as a powerful approach to understanding the ...',
    'https://precision.fda.gov/challenges/4',
    'completed',
    'intermediate',
    '6',
    '',
    '2018-09-24',
    '2018-12-19',
    '2023-06-23 00:00:00',
    '2023-07-26 19:49:52'
  ),
  (
    96,
    'biocompute-object-app-a-thon',
    'BioCompute Object App-a-thon',
    '',
    'Like scientific laboratory experiments, bioinformatics analysis results and interpretation are faced with reproducibility challenges due to the variability in multiple computational parameters, including input format, prerequisites, platform dependencies, and more. Even small changes in these computational parameters may have a large impact on the results and carry big implications for their scientific validity. Because there are currently no standardized schemas for reporting computational scientific workflows and parameters together with their results, the ways in which these workflows are communicated is highly variable, incomplete, and difficult or impossible to reproduce.  The US Food and Drug Administration (FDA) High Performance Virtual Environment (HIVE) group and George Washington University (GW) have partnered to establish a framework for community-based standards development and harmonization of high-throughput sequencing (HTS) computations and data formats based around...',
    'https://precision.fda.gov/challenges/7/',
    'completed',
    'intermediate',
    '6',
    '',
    '2019-05-14',
    '2019-10-18',
    '2023-06-23 00:00:00',
    '2023-07-26 19:49:53'
  ),
  (
    97,
    'brain-cancer-predictive-modeling-and-biomarker-discovery',
    'Brain Cancer Predictive Modeling and Biomarker Discovery',
    '',
    'An estimated 86,970 new cases of primary brain and other central nervous system tumors are expected to be diagnosed in the US in 2019. Brain tumors comprise a particularly deadly subset of all cancers due to limited treatment options and the high cost of care. Only a few prognostic and predictive markers have been successfully implemented in the clinic so far for gliomas, the most common malignant brain tumor type. These markers include MGMT promoter methylation in high-grade astrocytomas, co-deletion of 1p/19q in oligodendrogliomas, and mutations in IDH1 or IDH2 genes (Staedtke et al. 2016). There remains significant potential for identifying new clinical biomarkers in gliomas.  Clinical investigators at Georgetown University are seeking to advance precision medicine techniques for the prognosis and treatment of brain tumors through the identification of novel multi-omics biomarkers. In support of this goal, precisionFDA and the Georgetown Lombardi Comprehensive Cancer Center and...',
    'https://precision.fda.gov/challenges/8/',
    'completed',
    'advanced',
    '6',
    '',
    '2019-11-01',
    '2020-02-14',
    '2023-06-23 00:00:00',
    '2023-07-26 19:49:53'
  ),
  (
    98,
    'gaining-new-insights-by-detecting-adverse-event-anomalies',
    'Gaining New Insights by Detecting Adverse Event Anomalies',
    '',
    'The Food and Drug Administration (FDA) calls on the public to develop computational algorithms for automatic detection of adverse event anomalies using publicly available data.',
    'https://precision.fda.gov/challenges/9/',
    'completed',
    'intermediate',
    '6',
    '',
    '2020-01-17',
    '2020-05-18',
    '2023-06-23 00:00:00',
    '2023-07-26 19:49:54'
  ),
  (
    99,
    'calling-variants-in-difficult-to-map-regions',
    'Calling Variants in Difficult-to-Map Regions',
    '',
    'This challenge calls on the public to assess variant calling pipeline performance on a common frame of reference, with a focus on benchmarking in difficult-to-map regions, segmental duplications, and the Major Histocompatibility Complex (MHC).',
    'https://precision.fda.gov/challenges/10/',
    'completed',
    'intermediate',
    '6',
    '',
    '2020-05-01',
    '2020-06-15',
    '2023-06-23 00:00:00',
    '2023-07-26 19:49:55'
  ),
  (
    100,
    'vha-innovation-ecosystem-and-covid-19-risk-factor-modeling',
    'VHA Innovation Ecosystem and COVID-19 Risk Factor Modeling',
    '',
    'The novel coronavirus disease 2019 (COVID-19) is a respiratory disease caused by a new type of coronavirus, known as “severe acute respiratory syndrome coronavirus 2,” or SARS-CoV-2. On March 11, 2020, the World Health Organization (WHO) declared the outbreak a global pandemic. As of Monday, June 1, the Johns Hopkins University COVID-19 dashboard reports over 6.21 million total confirmed cases worldwide, including over 1.79 million cases in the United States. Although most people have mild to moderate symptoms, the disease can cause severe medical complications leading to death in some people.  The Centers for Disease Control and Prevention (CDC) have identified several groups at elevated risk for severe illness, including people 65 years and older, individuals living in nursing homes or long term care facilities, and those with serious underlying medical conditions, such as severe obesity, diabetes, chronic lung disease or moderate to severe asthma, chronic kidney or liver diseas...',
    'https://precision.fda.gov/challenges/11/',
    'completed',
    'intermediate',
    '6',
    '',
    '2020-06-02',
    '2020-07-03',
    '2023-06-23 00:00:00',
    '2023-07-26 19:49:56'
  ),
  (
    101,
    'covid-19-precision-immunology-app-a-thon',
    'COVID-19 Precision Immunology App-a-thon',
    '',
    'The novel coronavirus disease 2019 (COVID-19), a respiratory disease caused by a new type of coronavirus, known as “severe acute respiratory syndrome coronavirus 2” or SARS-CoV-2, was declared a global pandemic by the World Health Organization on March 11, 2020. To date, the Johns Hopkins University COVID-19 dashboard reports over 62 million confirmed cases worldwide, with a wide range of disease severity from asymptomatic to deaths (over 1.46 million). To effectively combat the widespread transmission of COVID-19 infection and save lives especially of those vulnerable individuals, it is imperative to better understand its pathophysiology to enable effective diagnosis, prognosis and treatment strategies using rapidly shared data.',
    'https://precision.fda.gov/challenges/12/',
    'completed',
    'intermediate',
    '6',
    '',
    '2020-11-30',
    '2021-01-29',
    '2023-06-23 00:00:00',
    '2023-07-26 19:49:57'
  ),
  (
    102,
    'smarter-food-safety-low-cost-tech-enabled-traceability',
    'Smarter Food Safety Low Cost Tech-Enabled Traceability',
    '',
    'The motivation is tapping into new technologies and integrating data streams will help to advance the widespread, consistent implementation of traceability systems across the food industry. However, the affordability of such technologies, particularly for smaller companies, can be a barrier to implementing tech-enabled traceability systems.  FDA''s New Era of Smarter Food Safety initiative strives to work with stakeholders to explore low-cost or no-cost options so that our approaches are inclusive of and viable for human and animal food operations of all sizes. Democratizing the benefits of digitizing data will allow the entire food system to move more rapidly towards digital traceability systems. The primary goal is to encourage stakeholders, including technology providers, public health advocates, entrepreneurs, and innovators from all disciplines and around the world, to develop traceability hardware, software, or data analytics platforms that are low-cost or no-cost to the end...',
    'https://precision.fda.gov/challenges/13',
    'completed',
    'intermediate',
    '6',
    '',
    '2021-06-01',
    '2021-07-30',
    '2023-06-23 00:00:00',
    '2023-08-09 08:46:13'
  ),
  (
    103,
    'tumor-mutational-burden-tmb-challenge-phase-1',
    'Tumor Mutational Burden (TMB) Challenge Phase 1',
    '',
    'Tumor mutational burden (TMB) is generally defined as the number of mutations detected in a patient''s tumor sample per megabase of DNA sequenced. However different algorithms use different methods for calculating TMB. Mutations in genes in tumor cells may lead to the creation of neoantigens, which have the potential to activate an immune system response against the tumor, and the likelihood of an immune system response may increase with the number of mutations. Thus, TMB is a biomarker for some immunotherapy drugs, called immune checkpoint inhibitors, such as those that target the PD-1 and PD-L1 pathways (Chan et al., 2019).  An outstanding problem is the lack of standardization for TMB calculation and reporting between different assays. To address this problem, the Friends of Cancer Research convened a working group of industry and regulatory stakeholders to develop guidance and tools for TMB harmonization. Results from the first phase of this effort were presented at AACR 2020 ...',
    'https://precision.fda.gov/challenges/17',
    'completed',
    'advanced',
    '6',
    '',
    '2021-06-21',
    '2021-09-13',
    '2023-06-23 00:00:00',
    '2023-07-26 19:49:57'
  ),
  (
    104,
    'kidney-and-kidney-tumor-segmentation',
    'Kidney and Kidney Tumor Segmentation',
    '',
    'The 2021 Kidney and Kidney Tumor Segmentation challenge (abbreviated KiTS21) is a competition in which teams compete to develop the best system for automatic semantic segmentation of renal tumors and surrounding anatomy. Kidney cancer is one of the most common malignancies in adults around the world, and its incidence is thought to be increasing [1]. Fortunately, most kidney tumors are discovered early while they''re still localized and operable. However, there are important questions concerning management of localized kidney tumors that remain unanswered [2], and metastatic renal cancer remains almost uniformly fatal [3]. Kidney tumors are notorious for their conspicuous appearance in computed tomography (CT) imaging, and this has enabled important work by radiologists and surgeons to study the relationship between tumor size, shape, and appearance and its prospects for treatment [4,5,6]. It''s laborious work, however, and it relies on assessments that are often subjective and im...',
    'https://kits21.kits-challenge.org/',
    'completed',
    'advanced',
    '5',
    '',
    '2021-08-23',
    '2021-09-17',
    '2023-06-23 00:00:00',
    '2023-07-26 19:49:58'
  ),
  (
    105,
    'cross-modality-da-for-medical-image-segmentation',
    'Cross-Modality DA for Medical Image Segmentation',
    '',
    'Domain Adaptation (DA) has recently raised strong interests in the medical imaging community. By encouraging algorithms to be robust to unseen situations or different input data domains, Domain Adaptation improves the applicability of machine learning approaches to various clinical settings. While a large variety of DA techniques has been proposed for image segmentation, most of these techniques have been validated either on private datasets or on small publicly available datasets. Moreover, these datasets mostly address single-class problems. To tackle these limitations, the crossMoDA challenge introduces the first large and multi-class dataset for unsupervised cross-modality Domain Adaptation.',
    'https://crossmoda-challenge.ml/',
    'completed',
    'advanced',
    '5',
    '',
    '2021-04-05',
    '2021-09-27',
    '2023-06-23 00:00:00',
    '2023-07-26 19:49:59'
  ),
  (
    106,
    'real-noise-mri',
    'Real Noise MRI',
    '',
    'In recent years, there is a growing focus on the application of fast magnetic resonance imaging (MRI) based on prior knowledge. In the 1980s and 2000s the community used either purely mathematical models such as the partial Fourier transform or solutions derived through advanced engineering such as parallel imaging to speed up MRI acquisition. Since the mid-2000''s, compressed sensing and artificial intelligence have been employed to speed up MRI acquisition. These newer methods rely on under sampling the data acquired in Fourier (aka k-) space and then interpolating or augmenting k-space data based on training data content. One of the underlying problems for the development of fast imaging techniques, that just as in e.g. [1], it is common to use a fully sampled image as ground truth and then under sample it in k-space in order to simulate under sampled data. The problem with this approach is that in cases were the under sampled data is corrupted, through e.g. motion, this under ...',
    'https://realnoisemri.grand-challenge.org/Description/',
    'completed',
    'intermediate',
    '5',
    '',
    '2021-09-21',
    '2021-12-06',
    '2023-06-23 00:00:00',
    '2023-07-26 19:49:59'
  ),
  (
    107,
    'deep-generative-model-challenge-for-da-in-surgery',
    'Deep Generative Model Challenge for DA in Surgery',
    '',
    'Mitral regurgitation (MR) is the second most frequent indication for valve surgery in Europe and may occur for organic or functional causes [1]. Mitral valve repair, although considerably more difficult, is prefered over mitral valve replacement, since the native tissue of the valve is preserved. It is a complex on-pump heart surgery, often conducted only by a handful of surgeons in high-volume centers. Minimally invasive procedures, which are performed with endoscopic video recordings, became more and more popular in recent years. However, data availability and data privacy concerns are still an issue for the development of automatic scene analysis algorithms. The AdaptOR challenge aims to address these issues by formulating a domain adaptation problem from simulation to surgery. We provide a smaller number of datasets from real surgeries, and a larger number of annotated recordings of training and planning sessions from a physical mitral valve simulator. The goal is to reduce th...',
    'https://adaptor2021.github.io/',
    'completed',
    'intermediate',
    '1',
    '',
    '2021-04-01',
    '2021-07-16',
    '2023-06-23 00:00:00',
    '2023-07-26 19:50:00'
  ),
  (
    108,
    'aimdatathon',
    'AIM Datathon 2020',
    'Join the AI in Medicine ( AIM ) Datathon 2020',
    'Join the AI in Medicine ( AIM ) Datathon 2020',
    'https://www.kaggle.com/competitions/aimdatathon',
    'completed',
    'intermediate',
    '8',
    '',
    '2020-11-09',
    '2020-11-22',
    '2023-06-23 00:00:00',
    '2023-07-26 19:50:02'
  ),
  (
    109,
    'opc-recurrence',
    'Oropharynx Cancer (OPC) Radiomics Challenge :: Local Recurrence Prediction',
    'Determine from CT data whether a tumor will be controlled by definitive radi...',
    'Determine from CT data whether a tumor will be controlled by definitive radiation therapy.',
    'https://www.kaggle.com/competitions/opc-recurrence',
    'completed',
    'intermediate',
    '8',
    '',
    '2016-07-26',
    '2016-09-12',
    '2023-06-23 00:00:00',
    '2023-07-26 19:50:03'
  ),
  (
    110,
    'oropharynx-radiomics-hpv',
    'Oropharynx Cancer (OPC) Radiomics Challenge :: Human Papilloma Virus (HPV) Status Prediction',
    'Predict from CT data the HPV phenotype of oropharynx tumors; compare to grou...',
    'Predict from CT data the HPV phenotype of oropharynx tumors; compare to ground-truth results previously obtained by p16 or HPV testing.',
    'https://www.kaggle.com/competitions/oropharynx-radiomics-hpv',
    'completed',
    'intermediate',
    '8',
    '',
    '2016-07-26',
    '2016-09-12',
    '2023-06-23 00:00:00',
    '2023-07-26 19:50:03'
  ),
  (
    111,
    'data-science-bowl-2017',
    'Data Science Bowl 2017',
    'Can you improve lung cancer detection?',
    'Can you improve lung cancer detection?',
    'https://www.kaggle.com/competitions/data-science-bowl-2017',
    'completed',
    'intermediate',
    '8',
    '',
    '2017-01-12',
    '2017-04-12',
    '2023-06-23 00:00:00',
    '2023-07-26 19:50:04'
  ),
  (
    112,
    'predict-impact-of-air-quality-on-death-rates',
    'Predict impact of air quality on mortality rates',
    'Predict CVD and cancer caused mortality rates in England using air quality d...',
    'Predict CVD and cancer caused mortality rates in England using air quality data available from Copernicus Atmosphere Monitoring Service',
    'https://www.kaggle.com/competitions/predict-impact-of-air-quality-on-death-rates',
    'completed',
    'intermediate',
    '8',
    '',
    '2017-02-13',
    '2017-05-05',
    '2023-06-23 00:00:00',
    '2023-07-26 19:50:04'
  ),
  (
    113,
    'intel-mobileodt-cervical-cancer-screening',
    'Intel & MobileODT Cervical Cancer Screening',
    'Which cancer treatment will be most effective?',
    'Which cancer treatment will be most effective?',
    'https://www.kaggle.com/competitions/intel-mobileodt-cervical-cancer-screening',
    'completed',
    'intermediate',
    '8',
    '',
    '2017-03-15',
    '2017-06-21',
    '2023-06-23 00:00:00',
    '2023-07-26 19:50:05'
  ),
  (
    114,
    'msk-redefining-cancer-treatment',
    'Personalized Medicine: Redefining Cancer Treatment',
    'Predict the effect of Genetic Variants to enable Personalized Medicine',
    'Predict the effect of Genetic Variants to enable Personalized Medicine',
    'https://www.kaggle.com/competitions/msk-redefining-cancer-treatment',
    'completed',
    'intermediate',
    '8',
    '',
    '2017-06-26',
    '2017-10-02',
    '2023-06-23 00:00:00',
    '2023-07-26 19:50:05'
  ),
  (
    115,
    'mubravo',
    'Predicting Cancer Diagnosis',
    'Bravo''s machine learning competition!',
    'Bravo''s machine learning competition!',
    'https://www.kaggle.com/competitions/mubravo',
    'completed',
    'intermediate',
    '8',
    '',
    '2018-07-31',
    '2018-08-13',
    '2023-06-23 00:00:00',
    '2023-07-26 19:50:08'
  ),
  (
    116,
    'histopathologic-cancer-detection',
    'Histopathologic Cancer Detection',
    'Identify metastatic tissue in histopathologic scans of lymph node sections',
    'Identify metastatic tissue in histopathologic scans of lymph node sections',
    'https://www.kaggle.com/competitions/histopathologic-cancer-detection',
    'completed',
    'intermediate',
    '8',
    '',
    '2018-11-16',
    '2019-03-30',
    '2023-06-23 00:00:00',
    '2023-07-26 19:50:10'
  ),
  (
    117,
    'tjml1920-decision-trees',
    'TJML 2019-20 Breast Cancer Detection Competition',
    'Use a decision tree to identify malignant breast cancer tumors',
    'Use a decision tree to identify malignant breast cancer tumors',
    'https://www.kaggle.com/competitions/tjml1920-decision-trees',
    'completed',
    'intermediate',
    '8',
    '',
    '2019-09-22',
    '2019-10-16',
    '2023-06-23 00:00:00',
    '2023-07-26 19:50:09'
  ),
  (
    118,
    'prostate-cancer-grade-assessment',
    'Prostate cANcer graDe Assessment (PANDA) Challenge',
    'Prostate cancer diagnosis using the Gleason grading system',
    'Prostate cancer diagnosis using the Gleason grading system',
    'https://www.kaggle.com/competitions/prostate-cancer-grade-assessment',
    'completed',
    'intermediate',
    '8',
    '',
    '2020-04-21',
    '2020-07-22',
    '2023-06-23 00:00:00',
    '2023-07-26 19:50:10'
  ),
  (
    119,
    'breast-cancer',
    'Breast Cancer',
    'Use cell nuclei categories to predict breast cancer tumor.',
    'Use cell nuclei categories to predict breast cancer tumor.',
    'https://www.kaggle.com/competitions/breast-cancer',
    'completed',
    'intermediate',
    '8',
    '',
    '2020-08-12',
    '2020-08-13',
    '2023-06-23 00:00:00',
    '2023-07-26 19:50:10'
  ),
  (
    120,
    'breast-cancer-detection',
    'Breast Cancer Detection',
    'breast cancer detection',
    'breast cancer detection',
    'https://www.kaggle.com/competitions/breast-cancer-detection',
    'completed',
    'intermediate',
    '8',
    '',
    '2020-09-25',
    '2020-12-31',
    '2023-06-23 00:00:00',
    '2023-07-26 19:50:11'
  ),
  (
    121,
    'hrpred',
    'Prediction of High Risk Patients',
    'Classification of high and low risk cancer patients',
    'Classification of high and low risk cancer patients',
    'https://www.kaggle.com/competitions/hrpred',
    'completed',
    'intermediate',
    '8',
    '',
    '2020-11-25',
    '2020-12-05',
    '2023-06-23 00:00:00',
    '2023-07-26 19:50:11'
  ),
  (
    122,
    'ml4moleng-cancer',
    'MIT ML4MolEng: Predicting Cancer Progression',
    'MIT 3.100, 10.402, 20.301 In class ML competition (Spring 2021)',
    'MIT 3.100, 10.402, 20.301 In class ML competition (Spring 2021)',
    'https://www.kaggle.com/competitions/ml4moleng-cancer',
    'completed',
    'intermediate',
    '8',
    '',
    '2021-05-06',
    '2021-05-21',
    '2023-06-23 00:00:00',
    '2023-07-26 19:50:12'
  ),
  (
    123,
    'uw-madison-gi-tract-image-segmentation',
    'UW-Madison GI Tract Image Segmentation',
    'Track healthy organs in medical scans to improve cancer treatment',
    'Track healthy organs in medical scans to improve cancer treatment',
    'https://www.kaggle.com/competitions/uw-madison-gi-tract-image-segmentation',
    'completed',
    'intermediate',
    '8',
    '',
    '2022-04-14',
    '2022-07-14',
    '2023-06-23 00:00:00',
    '2023-07-26 19:50:13'
  ),
  (
    124,
    'rsna-miccai-brain-tumor-radiogenomic-classification',
    'RSNA-MICCAI Brain Tumor Radiogenomic Classification',
    'Predict the status of a genetic biomarker important for brain cancer treatment',
    'The Brain Tumor Segmentation (BraTS) challenge celebrates its 10th anniversary, and this year is jointly organized by the Radiological Society of North America (RSNA), the American Society of Neuroradiology (ASNR), and the Medical Image Computing and Computer Assisted Interventions (MICCAI) society. The RSNA-ASNR-MICCAI BraTS 2021 challenge utilizes multi-institutional pre-operative baseline multi-parametric magnetic resonance imaging (mpMRI) scans, and focuses on the evaluation of state-of-the-art methods for (Task 1) the segmentation of intrinsically heterogeneous brain glioblastoma sub-regions in mpMRI scans. Furthemore, this BraTS 2021 challenge also focuses on the evaluation of (Task 2) classification methods to predict the MGMT promoter methylation status. Participants are free to choose whether they want to focus only on one or both tasks.',
    'https://www.kaggle.com/competitions/rsna-miccai-brain-tumor-radiogenomic-classification',
    'completed',
    'advanced',
    '8',
    '',
    '2021-07-13',
    '2021-10-15',
    '2023-06-23 00:00:00',
    '2023-07-26 19:50:13'
  ),
  (
    125,
    'breastcancer',
    'Breast Cancer - Beginners ML',
    'Beginners hands-on experience with ML basics',
    'Beginners hands-on experience with ML basics',
    'https://www.kaggle.com/competitions/breastcancer',
    'completed',
    'intermediate',
    '8',
    '',
    '2021-12-21',
    '2022-02-12',
    '2023-06-23 00:00:00',
    '2023-07-26 19:50:13'
  ),
  (
    126,
    'ml-olympiad-health-and-education',
    'ML Olympiad - Let''s Fight lung cancer',
    'Use your ML expertise to help us step another step toward defeating cancer [...',
    'Use your ML expertise to help us step another step toward defeating cancer [ Starts on the 14th February ]',
    'https://www.kaggle.com/competitions/ml-olympiad-health-and-education',
    'completed',
    'intermediate',
    '8',
    '',
    '2022-01-31',
    '2022-03-19',
    '2023-06-23 00:00:00',
    '2023-07-26 19:50:14'
  ),
  (
    127,
    'cs98-22-dl-task1',
    'CS98X-22-DL-Task1',
    'This competition is related to Task 1 in coursework - breast cancer classifi...',
    'This competition is related to Task 1 in coursework - breast cancer classification',
    'https://www.kaggle.com/competitions/CS98-22-DL-Task1',
    'completed',
    'intermediate',
    '8',
    '',
    '2022-02-28',
    '2022-04-11',
    '2023-06-23 00:00:00',
    '2023-07-26 19:50:16'
  ),
  (
    128,
    'parasitedetection-iiitb2019',
    'Parasite detection',
    'detect if cell image has parasite or is uninfected',
    'detect if cell image has parasite or is uninfected',
    'https://www.kaggle.com/competitions/parasitedetection-iiitb2019',
    'completed',
    'intermediate',
    '8',
    '',
    '2019-10-13',
    '2019-11-25',
    '2023-06-23 00:00:00',
    '2023-07-26 19:50:16'
  ),
  (
    129,
    'hpa-single-cell-image-classification',
    'Human Protein Atlas - Single Cell Classification',
    'Find individual human cell differences in microscope images',
    'Find individual human cell differences in microscope images',
    'https://www.kaggle.com/competitions/hpa-single-cell-image-classification',
    'completed',
    'intermediate',
    '8',
    '',
    '2021-01-26',
    '2021-05-11',
    '2023-06-23 00:00:00',
    '2023-07-26 19:50:17'
  ),
  (
    130,
    'stem-cell-predcition',
    'Stem Cell Predcition',
    'Classify stem and non-stem cells using RNA-seq data',
    'Classify stem and non-stem cells using RNA-seq data',
    'https://www.kaggle.com/competitions/stem-cell-predcition',
    'completed',
    'intermediate',
    '8',
    '',
    '2021-04-01',
    '2021-07-01',
    '2023-06-23 00:00:00',
    '2023-07-26 19:50:18'
  ),
  (
    131,
    'sartorius-cell-instance-segmentation',
    'Sartorius - Cell Instance Segmentation',
    'Detect single neuronal cells in microscopy images',
    'In this competition, you’ll detect and delineate distinct objects of interest in biological images depicting neuronal cell types commonly used in the study of neurological disorders. More specifically, you''ll use phase contrast microscopy images to train and test your model for instance segmentation of neuronal cells. Successful models will do this with a high level of accuracy. If successful, you''ll help further research in neurobiology thanks to the collection of robust quantitative data. Researchers may be able to use this to more easily measure the effects of disease and treatment conditions on neuronal cells. As a result, new drugs could be discovered to treat the millions of people with these leading causes of death and disability.',
    'https://www.kaggle.com/competitions/sartorius-cell-instance-segmentation',
    'completed',
    'intermediate',
    '8',
    '',
    '2021-10-14',
    '2021-12-30',
    '2023-06-23 00:00:00',
    '2023-08-08 17:53:06'
  ),
  (
    132,
    'pvelad',
    'Photovoltaic cell anomaly detection',
    'Hosted by Hebei University of Technology (AIHebut research group) and Beihan...',
    'Hosted by Hebei University of Technology (AIHebut research group) and Beihang University (NAVE research group)',
    'https://www.kaggle.com/competitions/pvelad',
    'completed',
    'intermediate',
    '8',
    '',
    '2022-03-15',
    '2022-07-30',
    '2023-06-23 00:00:00',
    '2023-07-26 19:50:19'
  ),
  (
    133,
    'blood-mnist',
    'Blood-MNIST',
    'Classifying blood cell types using Weights and Biases',
    'Classifying blood cell types using Weights and Biases',
    'https://www.kaggle.com/competitions/blood-mnist',
    'completed',
    'intermediate',
    '8',
    '',
    '2022-03-19',
    '2022-03-19',
    '2023-06-23 00:00:00',
    '2023-07-26 19:50:19'
  ),
  (
    134,
    'insilicomolhack',
    'MolHack',
    'Apply deep learning to speedup drug validation',
    'Apply deep learning to speedup drug validation',
    'https://www.kaggle.com/competitions/insilicomolhack',
    'completed',
    'intermediate',
    '8',
    '',
    '2018-04-02',
    '2018-05-25',
    '2023-06-23 00:00:00',
    '2023-07-26 19:50:20'
  ),
  (
    135,
    'codata2019challenge',
    'Cell Response Classification',
    'From recorded timeseries of many cells in a well, predict which drug treatme...',
    'From recorded timeseries of many cells in a well, predict which drug treatment has been applied',
    'https://www.kaggle.com/competitions/codata2019challenge',
    'completed',
    'intermediate',
    '8',
    '',
    '2019-04-08',
    '2019-05-07',
    '2023-06-23 00:00:00',
    '2023-07-26 19:50:21'
  ),
  (
    136,
    'drug-solubility-challenge',
    'Drug solubility challenge',
    'Solubility is vital to achieve desired concentration of drug for anticipated...',
    'Solubility is vital to achieve desired concentration of drug for anticipated pharmacological response.',
    'https://www.kaggle.com/competitions/drug-solubility-challenge',
    'completed',
    'intermediate',
    '8',
    '',
    '2019-05-18',
    '2019-10-18',
    '2023-06-23 00:00:00',
    '2023-07-26 19:50:22'
  ),
  (
    137,
    'kinase-inhibition-challenge',
    'Kinase inhibition challenge',
    'Protein kinases have become a major class of drug targets, accumulating a hu...',
    'Protein kinases have become a major class of drug targets, accumulating a huge amount of data',
    'https://www.kaggle.com/competitions/kinase-inhibition-challenge',
    'completed',
    'intermediate',
    '8',
    '',
    '2019-05-20',
    '2019-12-28',
    '2023-06-23 00:00:00',
    '2023-07-26 19:50:22'
  ),
  (
    138,
    'ai-drug-discovery',
    'AI Drug Discovery Workshop and Coding Challenge',
    'Developing Fundamental AI Programming Skills for Drug Discovery',
    'Developing Fundamental AI Programming Skills for Drug Discovery',
    'https://www.kaggle.com/competitions/ai-drug-discovery',
    'completed',
    'intermediate',
    '8',
    '',
    '2021-11-12',
    '2021-12-31',
    '2023-06-23 00:00:00',
    '2023-07-26 19:50:23'
  ),
  (
    139,
    'protein-compound-affinity',
    'Structure-free protein-ligand affinity prediction - Task 1 Fitting',
    'Developing new AI models for drug discovery, main portal (Task-1 fitting)',
    'Developing new AI models for drug discovery, main portal (Task-1 fitting)',
    'https://www.kaggle.com/competitions/protein-compound-affinity',
    'completed',
    'intermediate',
    '8',
    '',
    '2021-12-06',
    '2021-12-31',
    '2023-06-23 00:00:00',
    '2023-07-26 19:50:24'
  ),
  (
    140,
    'cisc873-dm-f21-a5',
    'CISC873-DM-F21-A5',
    'Anti-Cancer Drug Activity Prediction',
    'Anti-Cancer Drug Activity Prediction',
    'https://www.kaggle.com/competitions/cisc873-dm-f21-a5',
    'completed',
    'intermediate',
    '8',
    '',
    '2021-11-26',
    '2021-12-10',
    '2023-06-23 00:00:00',
    '2023-07-26 19:50:24'
  ),
  (
    141,
    'pro-lig-aff-task2-mse',
    'Structure-free protein-ligand affinity prediction - Task 2 Fitting',
    'Developing new AI models for drug discovery (Task-2 fitting)',
    'Developing new AI models for drug discovery (Task-2 fitting)',
    'https://www.kaggle.com/competitions/pro-lig-aff-task2-mse',
    'completed',
    'intermediate',
    '8',
    '',
    '2021-12-08',
    '2021-12-31',
    '2023-06-23 00:00:00',
    '2023-07-26 19:50:25'
  ),
  (
    142,
    'pro-lig-aff-task1-pearsonr',
    'Structure-free protein-ligand affinity prediction - Task 1 Ranking',
    'Developing new AI models for drug discovery (Task-1 ranking)',
    'Developing new AI models for drug discovery (Task-1 ranking)',
    'https://www.kaggle.com/competitions/pro-lig-aff-task1-pearsonr',
    'completed',
    'intermediate',
    '8',
    '',
    '2021-12-08',
    '2021-12-31',
    '2023-06-23 00:00:00',
    '2023-07-26 19:50:25'
  ),
  (
    143,
    'pro-lig-aff-task2-pearsonr',
    'Structure-free protein-ligand affinity prediction - Task 2 Ranking',
    'Developing new AI models for drug discovery (Task-2 ranking)',
    'Developing new AI models for drug discovery (Task-2 ranking)',
    'https://www.kaggle.com/competitions/pro-lig-aff-task2-pearsonr',
    'completed',
    'intermediate',
    '8',
    '',
    '2021-12-08',
    '2021-12-31',
    '2023-06-23 00:00:00',
    '2023-07-26 19:50:38'
  ),
  (
    144,
    'pro-lig-aff-task3-spearmanr',
    'Structure-free protein-ligand affinity prediction - Task 3 Ranking',
    'Developing new AI models for drug discovery (Task-3 ranking)',
    'Developing new AI models for drug discovery (Task-3 ranking)',
    'https://www.kaggle.com/competitions/pro-lig-aff-task3-spearmanr',
    'completed',
    'intermediate',
    '8',
    '',
    '2021-12-08',
    '2021-12-31',
    '2023-06-23 00:00:00',
    '2023-07-26 19:50:39'
  ),
  (
    145,
    'hhp',
    'Heritage Health Prize',
    'Identify patients who will be admitted to a hospital within the next year us...',
    'Identify patients who will be admitted to a hospital within the next year using historical claims data. (Enter by 06:59:59 UTC Oct 4 2012)',
    'https://www.kaggle.com/competitions/hhp',
    'completed',
    'intermediate',
    '8',
    '',
    '2011-04-04',
    '2013-04-04',
    '2023-06-23 00:00:00',
    '2023-07-26 19:50:39'
  ),
  (
    146,
    'pf2012',
    'Practice Fusion Analyze This! 2012 - Prediction Challenge',
    'Start digging into electronic health records and submit your ideas for the m...',
    'Start digging into electronic health records and submit your ideas for the most promising, impactful or interesting predictive modeling competitions',
    'https://www.kaggle.com/competitions/pf2012',
    'completed',
    'intermediate',
    '8',
    '',
    '2012-06-07',
    '2012-06-30',
    '2023-06-23 00:00:00',
    '2023-07-26 19:50:40'
  ),
  (
    147,
    'pf2012-at',
    'Practice Fusion Analyze This! 2012 - Open Challenge',
    'Start digging into electronic health records and submit your creative, insig...',
    'Start digging into electronic health records and submit your creative, insightful, and visually striking analyses.',
    'https://www.kaggle.com/competitions/pf2012-at',
    'completed',
    'intermediate',
    '8',
    '',
    '2012-06-07',
    '2012-09-10',
    '2023-06-23 00:00:00',
    '2023-07-26 19:50:41'
  ),
  (
    148,
    'seizure-detection',
    'UPenn and Mayo Clinic''s Seizure Detection Challenge',
    'Detect seizures in intracranial EEG recordings',
    'Detect seizures in intracranial EEG recordings',
    'https://www.kaggle.com/competitions/seizure-detection',
    'completed',
    'intermediate',
    '8',
    '',
    '2014-05-19',
    '2014-08-19',
    '2023-06-23 00:00:00',
    '2023-07-26 19:50:41'
  ),
  (
    149,
    'seizure-prediction',
    'American Epilepsy Society Seizure Prediction Challenge',
    'Predict seizures in intracranial EEG recordings',
    'Predict seizures in intracranial EEG recordings',
    'https://www.kaggle.com/competitions/seizure-prediction',
    'completed',
    'intermediate',
    '8',
    '',
    '2014-08-25',
    '2014-11-17',
    '2023-06-23 00:00:00',
    '2023-07-26 19:50:42'
  ),
  (
    150,
    'deephealth-1',
    'Deep Health - alcohol',
    'Find Correlations and patterns with Medical data',
    'Find Correlations and patterns with Medical data',
    'https://www.kaggle.com/competitions/deephealth-1',
    'completed',
    'intermediate',
    '8',
    '',
    '2017-02-13',
    '2017-02-19',
    '2023-06-23 00:00:00',
    '2023-07-26 19:50:44'
  ),
  (
    151,
    'deep-health-3',
    'Deep Health - Diabetes 2',
    'This competition is for those attending the Deep Health Hackathon.  Predict ...',
    'This competition is for those attending the Deep Health Hackathon.  Predict the next occurrence of diabetes',
    'https://www.kaggle.com/competitions/deep-health-3',
    'completed',
    'intermediate',
    '8',
    '',
    '2017-02-15',
    '2017-02-19',
    '2023-06-23 00:00:00',
    '2023-07-26 19:50:44'
  ),
  (
    152,
    'd012554-2021',
    'D012554 - 2021',
    'Classify the health of a fetus using CTG data',
    'Classify the health of a fetus using CTG data',
    'https://www.kaggle.com/competitions/d012554-2021',
    'completed',
    'intermediate',
    '8',
    '',
    '2021-04-11',
    '2021-05-09',
    '2023-06-23 00:00:00',
    '2023-07-26 19:50:45'
  ),
  (
    153,
    'idao-2022-bootcamp-insomnia',
    'IDAO 2022. ML Bootcamp - Insomnia',
    'Predict sleep disorder on given human health data',
    'Predict sleep disorder on given human health data',
    'https://www.kaggle.com/competitions/idao-2022-bootcamp-insomnia',
    'completed',
    'intermediate',
    '8',
    '',
    '2021-12-04',
    '2021-12-05',
    '2023-06-23 00:00:00',
    '2023-07-26 19:50:46'
  ),
  (
    154,
    'tweet-mental-health-classification',
    'Tweet Mental Health Classification',
    'Build Models to classify tweets to determine mental health',
    'Build Models to classify tweets to determine mental health',
    'https://www.kaggle.com/competitions/tweet-mental-health-classification',
    'completed',
    'intermediate',
    '8',
    '',
    '2021-12-27',
    '2022-01-31',
    '2023-06-23 00:00:00',
    '2023-07-26 19:50:46'
  ),
  (
    155,
    'ml-olympiad-good-health-and-well-being',
    'ML Olympiad - GOOD HEALTH AND WELL BEING',
    'Use your ML expertise to classify if a patient has heart disease or not',
    'Use your ML expertise to classify if a patient has heart disease or not',
    'https://www.kaggle.com/competitions/ml-olympiad-good-health-and-well-being',
    'completed',
    'intermediate',
    '8',
    '',
    '2022-02-03',
    '2022-03-01',
    '2023-06-23 00:00:00',
    '2023-07-26 19:50:47'
  ),
  (
    156,
    'rsna-breast-cancer-detection',
    'RSNA Screening Mammography Breast Cancer Detection',
    'Find breast cancers in screening mammograms',
    'Find breast cancers in screening mammograms',
    'https://www.kaggle.com/competitions/rsna-breast-cancer-detection',
    'completed',
    'intermediate',
    '8',
    '',
    '2022-11-28',
    '2023-02-27',
    '2023-06-23 00:00:00',
    '2023-08-09 08:47:37'
  ),
  (
    157,
    'biocreative-vii-text-mining-drug-and-chemical-protein-interactions-drugprot',
    'BioCreative VII: Text mining drug and chemical-protein interactions (DrugProt)',
    '',
    'With the rapid accumulation of biomedical literature, it is getting increasingly challenging to exploit efficiently drug-related information described in the scientific literature. One of the most relevant aspects of drugs and chemical compounds are their relationships with certain biomedical entities, in particular genes and proteins.
 The aim of the DrugProt track (similar to the previous CHEMPROT task of BioCreative VI) is to promote the development and evaluation of systems that are able to automatically detect in relations between chemical compounds/drug and genes/proteins.
 There are a range of different types of drug-gene/protein interactions, and their systematic extraction and characterization is essential to analyze, predict and explore key biomedical properties underlying high impact biomedical applications. These application scenarios include use cases related to drug discovery, drug repurposing, drug design, metabolic engineering, modeling drug response, pharmacogenet...',
    'https://biocreative.bioinformatics.udel.edu/tasks/biocreative-vii/track-1/',
    'completed',
    'intermediate',
    '7',
    '',
    '2021-06-15',
    '2021-09-22',
    '2023-06-23 00:00:00',
    '2023-07-26 19:50:48'
  ),
  (
    158,
    'extended-literature-ai-for-drug-induced-liver-injury',
    'Extended Literature AI for Drug Induced Liver Injury',
    '',
    'Unexpected Drug-Induced Liver Injury (DILI) still is one of the main killers of promising novel drug candidates. It is a clinically significant disease that can lead to severe outcomes such as acute liver failure and even death. It remains one of the primary liabilities in drug development and regulatory clearance due to the limited performance of mandated preclinical models even today. The free text of scientific publications is still the main medium carrying DILI results from clinical practice or experimental studies. The textual data still has to be analysed manually. This process, however, is tedious and prone to human mistakes or omissions, as results are very rarely available in a standardized form or organized form. There is thus great hope that modern techniques from machine learning or natural language processing could provide powerful tools to better process and derive the underlying knowledge within free form texts. The pressing need to faster process potential drug can...',
    'http://camda2022.bioinf.jku.at/contest_dataset#extended_literature_ai_for_drug_induced_liver_injury',
    'completed',
    'intermediate',
    '7',
    '',
    NULL,
    '2022-05-20',
    '2023-06-23 00:00:00',
    '2023-07-26 19:50:49'
  ),
  (
    159,
    'anti-microbial-resistance-forensics',
    'Anti-Microbial Resistance Forensics',
    '',
    'Bacteriophages, being the re-occuring mystery in the history of science are believed to be they key for understanding of microbial evolution and the transfer of AMR genes. Recent studies show that there is a significant correlation between occurence of Phages and AMR genes, indicating that they are indeed taking part in the spread of them. While taking part in AMR dissemination the phages are also considered as the potential alternative to antibiotics. In such contradictory world there is a huge potential as well as urgent need for precise classification, description and analysis of capabilities. Due to pandemic of SARS-CoV-2, advance in phylogenetic algorithms and k-mer based methods have been extremely rapid and those improvements are witing to be adapted to different branches of life sciences.',
    'http://camda2022.bioinf.jku.at/contest_dataset#anti-microbial_resistance_forensics',
    'completed',
    'intermediate',
    '7',
    '',
    NULL,
    '2022-05-20',
    '2023-06-23 00:00:00',
    '2023-07-26 19:50:49'
  ),
  (
    160,
    'disease-maps-to-modelling-covid-19',
    'Disease Maps to Modelling COVID-19',
    'Use the COVID-19 disease map to suggest drugs candidate for repurposing, tha...',
    'The Disease Maps to modeling COVID-19 Challenge provides highly detailed expert-curated molecular mechanistic maps for COVID-19. Combine them with available omic data to expand the current biological knowledge on COVID-19 mechanism of infection and downstream consequences. The main topic for this year''s challenge is drug repurposing with the possibility of Real World Data based validation of the most promising candidates suggested.',
    'http://camda2022.bioinf.jku.at/contest_dataset#disease_maps_to_modelling_covid-19',
    'completed',
    'intermediate',
    '7',
    '',
    NULL,
    '2022-05-20',
    '2023-06-23 00:00:00',
    '2023-07-26 19:50:50'
  ),
  (
    161,
    'crowdsourced-evaluation-of-inchi-based-tautomer-identification',
    'Crowdsourced Evaluation of InChI-based Tautomer Identification',
    'Calling on scientists from industry, government, and academia dealing with c...',
    'This challenge focuses on the International Chemical Identifier (InChI), which was developed and is maintained under the auspices of the International Union of Pure and Applied Chemistry (IUPAC) and the InChI Trust. The InChI Trust, the IUPAC Working Group on Tautomers, and the U.S. Food and Drug Administration (FDA) call on the scientific community dealing with chemical repositories/data sets and analytics of compounds to test the recently modified InChI algorithm, which was designed for advanced recognition of tautomers. Participants will evaluate this algorithm against real chemical samples in this Crowdsourced Evaluation of InChI-based Tautomer Identification.',
    'https://precision.fda.gov/challenges/29',
    'completed',
    'intermediate',
    '6',
    '',
    '2022-11-01',
    '2023-03-01',
    '2023-06-23 00:00:00',
    '2023-07-26 19:50:51'
  ),
  (
    162,
    'nctr-indel-calling-from-oncopanel-sequencing-challenge-phase-2',
    'NCTR Indel Calling from Oncopanel Sequencing Challenge Phase 2',
    'In Phase 2, participants who completed in Phase 1 of the challenge have the ...',
    'The high value of clinically actionable information obtained by oncopanel sequencing makes it a crucial tool for precision oncology[1,2]. With the surge in availability of oncopanels, it is critical to ensure that they have been thoroughly tested and are properly used. FDA has initiated the Sequencing Quality Control phase II (SEQC2) project[3] to develop standard analysis protocols and quality control metrics for fit-for-purpose use of Next Generation Sequencing (NGS) data including oncopanel sequencing to inform regulatory science research and precision medicine. The Oncopanel Sequencing Working Group of FDA-led SEQC2 has developed a reference sample[4] suitable for benchmarking oncopanels and comprehensively assessed the analytical performance of several oncopanels[1,2]. The genomic deoxyribonucleic acid (gDNA) reference sample was derived from ten Universal Human Reference RNA (UHRR, Agilent Technologies, Inc) cell-lines and made publicly available by Agilent. Substantial gene...',
    'https://precision.fda.gov/challenges/22',
    'completed',
    'intermediate',
    '6',
    '',
    '2022-07-11',
    '2022-07-26',
    '2023-06-23 00:00:00',
    '2023-07-26 19:50:51'
  ),
  (
    163,
    'nctr-indel-calling-from-oncopanel-sequencing-data-challenge-phase-1',
    'NCTR Indel Calling from Oncopanel Sequencing Data Challenge Phase 1',
    'Genetic variation involving indels (insertions and deletions) in the cancer ...',
    'The high value of clinically actionable information obtained by oncopanel sequencing makes it a crucial tool for precision oncology[1,2]. With the surge in availability of oncopanels, it is critical to ensure that they have been thoroughly tested and are properly used. FDA has initiated the Sequencing Quality Control phase II (SEQC2) project[3] to develop standard analysis protocols and quality control metrics for fit-for-purpose use of Next Generation Sequencing (NGS) data including oncopanel sequencing to inform regulatory science research and precision medicine. The Oncopanel Sequencing Working Group of FDA-led SEQC2 has developed a reference sample[4] suitable for benchmarking oncopanels and comprehensively assessed the analytical performance of several oncopanels[1,2]. The genomic deoxyribonucleic acid (gDNA) reference sample was derived from ten Universal Human Reference RNA (UHRR, Agilent Technologies, Inc) cell-lines and made publicly available by Agilent. Substantial gene...',
    'https://precision.fda.gov/challenges/21',
    'completed',
    'intermediate',
    '6',
    '',
    '2022-05-02',
    '2022-07-08',
    '2023-06-23 00:00:00',
    '2023-07-26 19:50:54'
  ),
  (
    164,
    'vha-innovation-ecosystem-and-precisionfda-covid-19-risk-factor-modeling-challenge-phase-2',
    'VHA Innovation Ecosystem and precisionFDA COVID-19 Risk Factor Modeling Challenge Phase 2',
    'The focus of Phase 2 was to validate the top performing models on two additi...',
    'The novel coronavirus disease 2019 (COVID-19) is a respiratory disease caused by a new type of coronavirus, known as “severe acute respiratory syndrome coronavirus 2,” or SARS-CoV-2. On March 11, 2020, the World Health Organization (WHO) declared the outbreak a global pandemic. As of January 22nd, 2022, the Johns Hopkins University COVID-19 dashboard reports over 338 million total confirmed cases worldwide. Although most people have mild to moderate symptoms, the disease can cause severe medical complications leading to death in some people. The Centers for Disease Control and Prevention (CDC) have identified several risk factors for severe COVID-19 illness, including people 65 years and older, individuals living in nursing homes or long-term care facilities, and those with serious underlying medical conditions. The Veteran population has a higher prevalence of several of the known risk factors for severe COVID-19 illness, such as advanced age, heart disease, and diabetes. Identif...',
    'https://precision.fda.gov/challenges/20',
    'completed',
    'intermediate',
    '6',
    '',
    '2021-04-14',
    '2022-01-28',
    '2023-06-23 00:00:00',
    '2023-07-26 19:50:54'
  ),
  (
    165,
    'tumor-mutational-burden-tmb-challenge-phase-2',
    'Tumor Mutational Burden (TMB) Challenge Phase 2',
    'The goal of the Friends of Cancer Research and precisionFDA Tumor Mutational...',
    'Tumor mutational burden (TMB) is generally defined as the number of mutations detected in a patient''s tumor sample per megabase of DNA sequenced. However different algorithms use different methods for calculating TMB. Mutations in genes in tumor cells may lead to the creation of neoantigens, which have the potential to activate an immune system response against the tumor, and the likelihood of an immune system response may increase with the number of mutations. Thus, TMB is a biomarker for some immunotherapy drugs, called immune checkpoint inhibitors, such as those that target the PD-1 and PD-L1 pathways (Chan et al., 2019). An outstanding problem is the lack of standardization for TMB calculation and reporting between different assays. To address this problem, the Friends of Cancer Research convened a working group of industry and regulatory stakeholders to develop guidance and tools for TMB harmonization. Results from the first phase of this effort were presented at AACR 2020 (...',
    'https://precision.fda.gov/challenges/18',
    'completed',
    'intermediate',
    '6',
    '',
    '2021-07-19',
    '2021-09-12',
    '2023-06-23 00:00:00',
    '2023-07-26 19:50:55'
  ),
  (
    166,
    'predicting-gene-expression-using-millions-of-random-promoter-sequences',
    'Predicting Gene Expression Using Millions of Random Promoter Sequences',
    '',
    'Decoding how gene expression is regulated is critical to understanding disease. Regulatory DNA is decoded by the cell in a process termed “cis-regulatory logic”, where proteins called Transcription Factors (TFs) bind to specific DNA sequences within the genome and work together to produce as output a level of gene expression for downstream adjacent genes. This process is exceedingly complex to model as a large number of parameters is needed to fully describe the process (see Rationale, de Boer et al. 2020; Zeitingler J. 2020). Understanding the cis-regulatory logic of the human genome is an important goal and would provide insight into the origins of many diseases. However, learning models from human data is challenging due to limitations in the diversity of sequences present within the human genome (e.g. extensive repetitive DNA), the vast number of cell types that differ in how they interpret regulatory DNA, limited reporter assay data, and substantial technical biases present i...',
    'https://www.synapse.org/#!Synapse:syn28469146/wiki/617075',
    'completed',
    'intermediate',
    '1',
    '',
    '2022-06-15',
    '2022-08-07',
    '2023-06-23 00:00:00',
    '2023-08-07 20:20:45'
  ),
  (
    167,
    'brats-2023',
    'BraTS 2023',
    '',
    'The International Brain Tumor Segmentation (BraTS) challenge. BraTS, since 2012, has focused on the generation of a benchmarking environment and dataset for the delineation of adult brain gliomas. The focus of this year’s challenge remains the generation of a common benchmarking environment, but its dataset is substantially expanded to ~4,500 cases towards addressing additional i) populations (e.g., sub-Saharan Africa patients), ii) tumors (e.g., meningioma), iii) clinical concerns (e.g., missing data), and iv) technical considerations (e.g., augmentations). Specifically, the focus of BraTS 2023 is to identify the current state-of-the-art algorithms for addressing (Task 1) the same adult glioma population as in the RSNA-ANSR-MICCAI BraTS challenge, as well as (Task 2) the underserved sub-Saharan African brain glioma patient population, (Task 3) intracranial meningioma, (Task 4) brain metastasis, (Task 5) pediatric brain tumor patients, (Task 6) global & local missing data, (Task 7...',
    'https://www.synapse.org/brats',
    'active',
    'advanced',
    '1',
    '',
    '2023-06-01',
    '2023-08-25',
    '2023-06-23 00:00:00',
    '2023-08-04 21:51:07'
  ),
  (
    168,
    'cagi7',
    'CAGI7',
    'The seventh round of CAGI',
    'There have been six editions of CAGI experiments, held between 2010 and 2022. The seventh round of CAGI is planned to take place over the Summer of 2024.',
    'https://genomeinterpretation.org/challenges.html',
    'upcoming',
    'intermediate',
    '2',
    '',
    NULL,
    NULL,
    '2023-08-04 21:47:38',
    '2023-08-08 18:52:54'
  ),
  (
    169,
    'casp15',
    'CASP15',
    'Establish the state-of-art in modeling proteins and protein complexes',
    'CASP14 (2020) saw an enormous jump in the accuracy of single protein and domain models such that many are competitive with experiment. That advance is largely the result of the successful application of deep learning methods, particularly by the AlphaFold and, since that CASP, RosettaFold. As a consequence, computed protein structures are becoming much more widely used in a broadening range of applications. CASP has responded to this new landscape with a revised set of modeling categories. Some old categories have been dropped (refinement, contact prediction, and aspects of model accuracy estimation) and new ones have been added (RNA structures, protein ligand complexes, protein ensembles, and accuracy estimation for protein complexes). We are also strengthening our interactions with our partners CAPRI and CAMEO. We hope that these changes will maximize the insight that CASP15 provides, particularly in new applications of deep learning. ',
    'https://predictioncenter.org/casp15/index.cgi',
    'completed',
    'intermediate',
    '4',
    '',
    '2022-04-18',
    NULL,
    '2023-08-04 21:52:12',
    '2023-08-08 18:52:57'
  ),
  (
    170,
    'synthrad2023',
    'SynthRAD2023',
    'Synthesizing computed tomography for radiotherapy',
    'This challenge aims to provide the first platform offering public data evaluation metrics to compare the latest developments in sCT generation methods. The accepted challenge design approved by MICCAI can be found at https://doi.org/10.5281/zenodo.7746019. A type 2 challenge will be run, where the participant needs to submit their algorithm packaged in a docker both for validation and test.',
    'https://synthrad2023.grand-challenge.org/',
    'active',
    'intermediate',
    '5',
    '',
    '2023-04-01',
    '2023-08-22',
    '2023-08-04 21:54:31',
    '2023-08-07 20:20:05'
  ),
  (
    171,
    'syn-iss',
    'Synthetic Data for Instrument Segmentation in Surgery (Syn-ISS)',
    '',
    'A common limitation noted by the surgical data science community is the size of datasets and the resources needed to generate training data at scale for building reliable and high-performing machine learning models. Beyond unsupervised and self-supervised approaches another solution within the broader machine learning community has been a growing volume of literature in the use of synthetic data (simulation) for training algorithms than can be applied to real world data. Synthetic data has multiple benefits like free groundtruth at large scale, possibility to collect larger sample of rare events, include anatomical variations, etc. A first step towards proving the validity of using synthetic data for real world applications is to demonstrate the feasibility within the simulation world itself. Our proposed challenge is to train machine learning methods for instrument segmentation using synthetic datasets and test their performance on synthetic datasets. That is, the challenge parti...',
    'https://www.synapse.org/#!Synapse:syn50908388/wiki/620516',
    'active',
    'intermediate',
    '1',
    '',
    '2023-07-19',
    '2023-09-07',
    '2023-08-04 23:49:44',
    '2023-08-07 20:20:07'
  ),
  (
    172,
    'pitvis',
    'PitVis',
    'Surgical workflow and instrument recognition in endonasal surgery',
    'The pituitary gland, found just off the base of the brain, is commonly known as “the master gland”, performing essential functions required for sustaining human life. Clinically relevant tumours that have grown on the pituitary gland have an estimated prevalence of 1 in 1000 of the population, and if left untreated can be life-limiting. The “gold standard” treatment is endoscopic pituitary surgery, where the tumour is directly removed by entering through a nostril. This surgery is particularly challenging due to the small working space which limits both vision and instrument manoeuvrability and thus can lead to poor surgical technique causing adverse outcomes for the patient. Computer-assisted intervention can help overcome these challenges by providing guidance for senior surgeons and operative staff during surgery, and for junior surgeons during training.',
    'https://www.synapse.org/#!Synapse:syn51232283/wiki/',
    'active',
    'intermediate',
    '1',
    '',
    '2023-06-29',
    '2023-09-10',
    '2023-08-04 23:58:01',
    '2023-08-07 20:20:08'
  ),
  (
    173,
    'mvseg2023',
    'MVSEG2023',
    'Automatically segment mitral valve leaflets from single frame 3D trans-esoph...',
    'Mitral valve (MV) disease is a common pathologic problem occurring in approximately 2 % of the general population but climbing to 10 % in those over the age of 75. The preferred intervention for mitral regurgitation is valve repair, due to superior patient outcomes compared to those following valve replacement. Mitral valve interventions are technically challenging due to the functional and anatomical complexity of mitral pathologies. Repair must be tailored to the patient-specific anatomy and pathology, which requires considerable expert training and experience. Automatic segmentation of the mitral valve leaflets from 3D transesophageal echocardiography (TEE) may play an important role in treatment planning, as well as physical and computational modelling of patient-specific valve pathologies and potential repair approaches. This may have important implications in the drive towards personalized care and has the potential to impact clinical outcomes for those undergoing mitral val...',
    'https://www.synapse.org/#!Synapse:syn51186045/wiki/621356',
    'completed',
    'intermediate',
    '1',
    '',
    '2023-05-29',
    '2023-08-07',
    '2023-08-05 0:04:36',
    '2023-08-05 0:06:32'
  ),
  (
    174,
    'crossmoda23',
    'CrossMoDA23',
    'This challenge proposes is the third edition of the first medical imaging be...',
    'Domain Adaptation (DA) has recently raised strong interest in the medical imaging community. By encouraging algorithms to be robust to unseen situations or different input data domains, Domain Adaptation improves the applicability of machine learning approaches to various clinical settings. While a large variety of DA techniques has been proposed, most of these techniques have been validated either on private datasets or on small publicly available datasets. Moreover, these datasets mostly address single-class problems. To tackle these limitations, the crossMoDA challenge introduced the first large and multi-class dataset for unsupervised cross-modality Domain Adaptation. From an application perspective, crossMoDA focuses on MRI segmentation for Vestibular Schwannoma. Compared to the previous crossMoDA instance, which made use of multi-institutional data acquired in controlled conditions for radiosurgery planning and focused on a 2 class segmentation task (tumour and cochlea), the...',
    'https://www.synapse.org/#!Synapse:syn51236108/wiki/621615',
    'completed',
    'intermediate',
    '1',
    '',
    '2023-04-15',
    '2023-07-10',
    '2023-08-05 0:13:23',
    '2023-08-05 0:22:59'
  ),
  (
    175,
    'icr-identify-age-related-conditions',
    'ICR - Identifying Age-Related Conditions',
    'Use Machine Learning to detect conditions with measurements of anonymous cha...',
    'The goal of this competition is to predict if a person has any of three medical conditions. You are being asked to predict if the person has one or more of any of the three medical conditions (Class 1), or none of the three medical conditions (Class 0). You will create a model trained on measurements of health characteristics. To determine if someone has these medical conditions requires a long and intrusive process to collect information from patients. With predictive models, we can shorten this process and keep patient details private by collecting key characteristics relative to the conditions, then encoding these characteristics.',
    'https://www.kaggle.com/competitions/icr-identify-age-related-conditions',
    'completed',
    'intermediate',
    '8',
    '',
    '2023-05-11',
    '2023-08-10',
    '2023-08-05 0:32:01',
    '2023-08-08 21:59:15'
  ),
  (
    176,
    'cafa-5-protein-function-prediction',
    'CAFA 5 Protein Function Prediction',
    'Predict the biological function of a protein',
    'The goal of this competition is to predict the function of a set of proteins. You will develop a model trained on the amino-acid sequences of the proteins and on other data. Your work will help ​​researchers better understand the function of proteins, which is important for discovering how cells, tissues, and organs work. This may also aid in the development of new drugs and therapies for various diseases.',
    'https://www.kaggle.com/competitions/cafa-5-protein-function-prediction',
    'completed',
    'intermediate',
    '8',
    '',
    '2023-04-18',
    '2023-08-21',
    '2023-08-05 5:18:40',
    '2023-08-07 20:20:11'
  ),
  (
    177,
    'rsna-2023-abdominal-trauma-detection',
    'RSNA 2023 Abdominal Trauma Detection',
    'Detect and classify traumatic abdominal injuries',
    'Traumatic injury is the most common cause of death in the first four decades of life and a major public health problem around the world. There are estimated to be more than 5 million annual deaths worldwide from traumatic injury. Prompt and accurate diagnosis of traumatic injuries is crucial for initiating appropriate and timely interventions, which can significantly improve patient outcomes and survival rates. Computed tomography (CT) has become an indispensable tool in evaluating patients with suspected abdominal injuries due to its ability to provide detailed cross-sectional images of the abdomen. Interpreting CT scans for abdominal trauma, however, can be a complex and time-consuming task, especially when multiple injuries or areas of subtle active bleeding are present. This challenge seeks to harness the power of artificial intelligence and machine learning to assist medical professionals in rapidly and precisely detecting injuries and grading their severity. The development ...',
    'https://www.kaggle.com/competitions/rsna-2023-abdominal-trauma-detection',
    'active',
    'intermediate',
    '8',
    '',
    '2023-07-26',
    '2023-10-13',
    '2023-08-05 5:24:09',
    '2023-08-07 20:20:12'
  ),
  (
    178,
    'hubmap-hacking-the-human-vasculature',
    'HuBMAP - Hacking the Human Vasculature',
    'Segment instances of microvascular structures from healthy human kidney tiss...',
    'The goal of this competition is to segment instances of microvascular structures, including capillaries, arterioles, and venules. You''ll create a model trained on 2D PAS-stained histology images from healthy human kidney tissue slides. Your help in automating the segmentation of microvasculature structures will improve researchers'' understanding of how the blood vessels are arranged in human tissues.',
    'https://www.kaggle.com/competitions/hubmap-hacking-the-human-vasculature',
    'completed',
    'intermediate',
    '8',
    '',
    '2023-05-22',
    '2023-07-31',
    '2023-08-05 5:31:12',
    '2023-08-05 5:36:41'
  ),
  (
    179,
    'amp-parkinsons-disease-progression-prediction',
    'AMP(R)-Parkinson''s Disease Progression Prediction',
    'Use protein and peptide data measurements from Parkinson''s Disease patients...',
    'The goal of this competition is to predict MDS-UPDR scores, which measure progression in patients with Parkinson''s disease. The Movement Disorder Society-Sponsored Revision of the Unified Parkinson''s Disease Rating Scale (MDS-UPDRS) is a comprehensive assessment of both motor and non-motor symptoms associated with Parkinson''s. You will develop a model trained on data of protein and peptide levels over time in subjects with Parkinson’s disease versus normal age-matched control subjects. Your work could help provide important breakthrough information about which molecules change as Parkinson’s disease progresses.',
    'https://www.kaggle.com/competitions/amp-parkinsons-disease-progression-prediction',
    'completed',
    'intermediate',
    '8',
    '',
    '2023-02-16',
    '2023-05-18',
    '2023-08-05 5:37:12',
    '2023-08-05 5:39:03'
  ),
  (
    180,
    'open-problems-multimodal',
    'Open Problems - Multimodal Single-Cell Integration',
    'Predict how DNA, RNA & protein measurements co-vary in single cells',
    'The goal of this competition is to predict how DNA, RNA, and protein measurements co-vary in single cells as bone marrow stem cells develop into more mature blood cells. You will develop a model trained on a subset of 300,000-cell time course dataset of CD34+ hematopoietic stem and progenitor cells (HSPC) from four human donors at five time points generated for this competition by Cellarity, a cell-centric drug creation company. In the test set, taken from an unseen later time point in the dataset, competitors will be provided with one modality and be tasked with predicting a paired modality measured in the same cell. The added challenge of this competition is that the test data will be from a later time point than any time point in the training data. Your work will help accelerate innovation in methods of mapping genetic information across layers of cellular state. If we can predict one modality from another, we may expand our understanding of the rules governing these complex re...',
    'https://www.kaggle.com/competitions/open-problems-multimodal',
    'completed',
    'intermediate',
    '8',
    '',
    '2022-08-15',
    '2022-11-15',
    '2023-08-05 5:43:25',
    '2023-08-05 5:44:44'
  ),
  (
    181,
    'multi-atlas-labeling-beyond-the-cranial-vault',
    'Multi-Atlas Labeling Beyond the Cranial Vault',
    '',
    'Multi-atlas labeling has proven to be an effective paradigm for creating segmentation algorithms from training data. These approaches have been extraordinarily successful for brain and cranial structures (e.g., our prior MICCAI workshops: MLSF’11, MAL’12, SATA’13). After the original challenges closed, the data continue to drive scientific innovation; 144 groups have registered for the 2012 challenge (brain only) and 115 groups for the 2013 challenge (brain/heart/canine leg). However, innovation in application outside of the head and to soft tissues has been more limited. This workshop will provide a snapshot of the current progress in the field through extended discussions and provide researchers an opportunity to characterize their methods on a newly created and released standardized dataset of abdominal anatomy on clinically acquired CT. The datasets will be freely available both during and after the challenge. We have two separate new challenges: abdomen and cervix on routinel...',
    'https://www.synapse.org/#!Synapse:syn3193805/wiki/89480',
    'active',
    'intermediate',
    '1',
    '',
    '2015-04-15',
    NULL,
    '2023-08-07 20:21:22',
    '2023-08-10 5:38:08'
  ),
  (
    182,
    'hubmap-organ-segmentation',
    'HuBMAP + HPA - Hacking the Human Body',
    'Segment multi-organ functional tissue units',
    'In this competition, you’ll identify and segment functional tissue units (FTUs) across five human organs. You''ll build your model using a dataset of tissue section images, with the best submissions segmenting FTUs as accurately as possible. If successful, you''ll help accelerate the world’s understanding of the relationships between cell and tissue organization. With a better idea of the relationship of cells, researchers will have more insight into the function of cells that impact human health. Further, the Human Reference Atlas constructed by HuBMAP will be freely available for use by researchers and pharmaceutical companies alike, potentially improving and prolonging human life.',
    'https://www.kaggle.com/competitions/hubmap-organ-segmentation',
    'completed',
    'intermediate',
    '8',
    '',
    '2022-06-22',
    '2022-09-22',
    '2023-08-08 16:30:22',
    '2023-08-08 21:47:25'
  ),
  (
    183,
    'hubmap-kidney-segmentation',
    'HuBMAP - Hacking the Kidney',
    'Identify glomeruli in human kidney tissue images',
    'This competition, “Hacking the Kidney," starts by mapping the human kidney at single cell resolution. Your challenge is to detect functional tissue units (FTUs) across different tissue preparation pipelines. An FTU is defined as a “three-dimensional block of cells centered around a capillary, such that each cell in this block is within diffusion distance from any other cell in the same block” ([de Bono, 2013](https://www.ncbi.nlm.nih.gov/pubmed/24103658)). The goal of this competition is the implementation of a successful and robust glomeruli FTU detector. You will also have the opportunity to present your findings to a panel of judges for additional consideration. Successful submissions will construct the tools, resources, and cell atlases needed to determine how the relationships between cells can affect the health of an individual. Advancements in HuBMAP will accelerate the world’s understanding of the relationships between cell and tissue organization and function and human health.',
    'https://www.kaggle.com/competitions/hubmap-kidney-segmentation',
    'completed',
    'intermediate',
    '8',
    '',
    '2020-11-16',
    '2021-05-10',
    '2023-08-08 17:31:46',
    '2023-08-08 21:47:21'
  ),
  (
    184,
    'ventilator-pressure-prediction',
    'Google Brain - Ventilator Pressure Prediction',
    'Simulate a ventilator connected to a sedated patient''s lung',
    'In this competition, you’ll simulate a ventilator connected to a sedated patient''s lung. The best submissions will take lung attributes compliance and resistance into account. If successful, you''ll help overcome the cost barrier of developing new methods for controlling mechanical ventilators. This will pave the way for algorithms that adapt to patients and reduce the burden on clinicians during these novel times and beyond. As a result, ventilator treatments may become more widely available to help patients breathe.',
    'https://www.kaggle.com/competitions/ventilator-pressure-prediction',
    'completed',
    'intermediate',
    '8',
    '',
    '2021-09-22',
    '2021-11-03',
    '2023-08-08 17:53:33',
    '2023-08-08 17:54:50'
  ),
  (
    185,
    'stanford-covid-vaccine',
    'OpenVaccine: COVID-19 mRNA Vaccine Degradation Prediction',
    'Urgent need to bring the COVID-19 vaccine to mass production',
    'In this competition, we are looking to leverage the data science expertise of the Kaggle community to develop models and design rules for RNA degradation. Your model will predict likely degradation rates at each base of an RNA molecule, trained on a subset of an Eterna dataset comprising over 3000 RNA molecules (which span a panoply of sequences and structures) and their degradation rates at each position. We will then score your models on a second generation of RNA sequences that have just been devised by Eterna players for COVID-19 mRNA vaccines. These final test sequences are currently being synthesized and experimentally characterized at Stanford University in parallel to your modeling efforts -- Nature will score your models!',
    'https://www.kaggle.com/competitions/stanford-covid-vaccine',
    'completed',
    'intermediate',
    '8',
    '',
    '2020-09-10',
    '2020-10-06',
    '2023-08-08 18:06:17',
    '2023-08-08 18:07:30'
  ),
  (
    186,
    'openvaccine',
    'OpenVaccine',
    'To develop mRNA vaccines stable enough to be deployed to everyone in the wor...',
    'mRNA vaccines are a relatively new technology that have come into the limelight with the onset of COVID-19. They were the first COVID-19 vaccines to start clinical trials (initially formulated in a matter of days) and the first to be approved and distributed. mRNA vaccines have the potential to transform immunization, being significantly faster to formulate and produce, cheaper, and more effective - including against mutant strains. However, there is one key bottleneck to their widespread viability and our ability to immunize the entire world: poor refrigerator stability in prefilled syringes. The OpenVaccine challenge aims to allow a worldwide community of game players to create an enhanced vaccine to be injected into millions of people. The challenge: design an mRNA that codes for the same amino acid sequence of the spike protein, but is 2x-10x+ more stable. Through a number of academic partnerships and the launch of a Kaggle machine learning challenge to create best-in-class al...',
    'https://eternagame.org/challenges/10845741',
    'completed',
    'intermediate',
    '13',
    'https://doi.org/10.1038/s41467-022-28776-w',
    NULL,
    '2021-12-12',
    '2023-08-08 18:22:49',
    '2023-08-08 19:08:57'
  ),
  (
    187,
    'opentb',
    'OpenTB',
    'What if we could use RNA to detect a gene sequence found to be present only ...',
    'OpenTB used a recently reported gene signature for active tuberculosis based on three RNAs in the blood. This signature could form the basis for a fast, color-based test for TB, similar to an over-the-counter pregnancy test. What was needed was a sensor that could detect the concentrations of three RNAs, carry out the needed calculation, and report the result by binding another molecule. Over four rounds, players designed RNA sensors that can do the math on these 3 genes. Through experimental feedback, they honed their skills and techniques, which resulted in the creation of multiple designs that have been shown to be successful. These findings are being prepared to be published, and future work will be done to develop diagnostic devices integrating these designs',
    'https://eternagame.org/challenges/10845742',
    'completed',
    'intermediate',
    '13',
    '',
    '2016-05-04',
    '2018-04-15',
    '2023-08-08 18:43:09',
    '2023-08-08 19:04:00'
  ),
  (
    188,
    'opencrispr',
    'OpenCRISPR',
    'A project to discover design patterns for guide RNAs to make gene editing mo...',
    'CRISPR gene editing is a RNA-based method that can target essentially any gene in a living organism for genetic changes. Since its first demonstration, CRISPR has been revolutionizing biology and promises to change how we tackle numerous human diseases from malaria to cancer. Stanford''s Center for Personal Dynamic Regulomes and UC Berkeley''s Innovative Genomics Institute have challenged Eterna players to solve a remaining hurdle in making this technology safe for use. Scientists want the power to turn on and off CRISPR on demand with small molecules. This is almost a perfect match to the small-molecule switches that the Eterna community has worked on. In fact, the MS2 RNA hairpin often used in Eterna is routinely used to recruit new functionality to CRISPR complexes through other molecules tethered to the MS2 protein. The puzzles began with OpenCRISPR Controls, looking for solutions to lock in or lock out the MS2 RNA hairpin within a special loop in the CRISPR RNA. We hope the r...',
    'https://eternagame.org/challenges/10845743',
    'completed',
    'intermediate',
    '13',
    'https://doi.org/10.1021/acssynbio.9b00142',
    '2017-08-26',
    NULL,
    '2023-08-08 18:43:14',
    '2023-08-10 5:38:26'
  ),
  (
    189,
    'openknot',
    'OpenKnot',
    'Many important biological processes depend on RNAs that form pseudoknots, an...',
    'RNA pseudoknots have significant biological importance in various processes. They participate in gene regulation by influencing translation initiation or termination in mRNA molecules. Pseudoknots also play a role in programmed ribosomal frameshifting, leading to the production of different protein products from a single mRNA. RNA viruses, including SARS-CoV-2 and Dengue virus, utilize pseudoknots to regulate their replication and control the synthesis of viral proteins. Additionally, certain RNA molecules with pseudoknot structures exhibit enzymatic activity, acting as ribozymes and catalyzing biochemical reactions. These functions highlight the crucial role of RNA pseudoknots in gene expression, proteomic diversity, viral replication, and enzymatic processes. Several unanswered scientific questions surround RNA pseudoknots. One key area of inquiry is understanding the folding pathways of pseudoknots and how they form from linear RNA sequences. Elucidating the structural dynamics...',
    'https://eternagame.org/challenges/11843006',
    'active',
    'intermediate',
    '13',
    '',
    '2022-06-17',
    NULL,
    '2023-08-08 18:43:22',
    '2023-08-10 5:38:35'
  ),
  (
    190,
    'openaso',
    'OpenASO',
    'A research initiative aimed at developing innovative design principles for R...',
    'The DNA genome is the blueprint for building and operating cells, but this information must be decoded into RNA molecules to be useful. Transcription is the process of decoding DNA genomic information into RNA, resulting in RNA transcripts. Genes are specific sequences of DNA that contain information to produce a specific RNA transcript. The fate of most mRNA molecules in the cell is to be translated by ribosomes into protein molecules. However, mRNA splicing is a crucial step that occurs between the formation of an RNA transcript and protein translation. This step is essential because genes contain non-protein coding introns and protein-coding exons. Splicing removes introns and joins exons to produce a mature mRNA molecule that can be decoded into the correct protein molecule. When the splicing process is corrupted due to genetic mutations, the resulting RNA can become toxic, leading to the synthesis of non-functional proteins or no protein at all, causing various human diseases...',
    'https://eternagame.org/challenges/11546273',
    'active',
    'intermediate',
    '13',
    '',
    '2023-02-20',
    NULL,
    '2023-08-08 18:43:25',
    '2023-08-10 5:38:36'
  ),
  (
    191,
    'openribosome',
    'OpenRibosome',
    'We aim to 1) gain fundamental insights into the ribosome''s RNA sequence-fol...',
    'Our modern world has many challenges - challenges like climate change, increasing waste production, and human health. Imagine: we could replace petrochemistry with biology, single-use plastics with selectively degradable polymers, broad chemotherapeutics with targeted medicines for fighting specific cancer cells, and complex health equipment with point-of-care diagnostics. These innovations and many more can empower us to confront the challenges affecting humanity, our world, and beyond. But how do we actually create these smart materials and medicines? Is it possible to do so by repurposing one of Nature''s molecular machines? We think we can. The answer? Customized ribosomes. In Nature, ribosomes are the catalysts for protein assembly. And proteins are more or less similar, chemically, to the smart materials and medicines we want to synthesize. If we could modify ribosomes to build polymers with diverse components - beyond the canonical amino acids us',
    'https://eternagame.org/challenges/11043833',
    'active',
    'intermediate',
    '13',
    'https://doi.org/10.1038/s41467-023-35827-3',
    '2019-01-31',
    NULL,
    '2023-08-08 18:43:27',
    '2023-08-10 5:42:45'
  ),
  (
    192,
    'lish-moa',
    'Mechanisms of Action (MoA) Prediction',
    'Can you improve the algorithm that classifies drugs based on their biologica...',
    'Can you improve the algorithm that classifies drugs based on their biological activity?',
    'https://www.kaggle.com/competitions/lish-moa',
    'completed',
    'intermediate',
    '8',
    '',
    '2020-09-03',
    '2020-11-30',
    '2023-08-08 19:09:31',
    '2023-08-08 19:10:54'
  ),
  (
    193,
    'recursion-cellular-image-classification',
    'Recursion Cellular Image Classification',
    'CellSignal: Disentangling biological signal from experimental noise in cellu...',
    'This competition will have you disentangling experimental noise from real biological signals. Your entry will classify images of cells under one of 1,108 different genetic perturbations. You can help eliminate the noise introduced by technical execution and environmental variation between experiments. If successful, you could dramatically improve the industry’s ability to model cellular images according to their relevant biology. In turn, applying AI could greatly decrease the cost of treatments, and ensure these treatments get to patients faster.',
    'https://www.kaggle.com/competitions/recursion-cellular-image-classification',
    'completed',
    'intermediate',
    '8',
    '',
    '2019-06-27',
    '2019-09-26',
    '2023-08-08 19:38:42',
    '2023-08-08 19:40:06'
  ),
  (
    194,
    'tlvmc-parkinsons-freezing-gait-prediction',
    'Parkinson''s Freezing of Gait Prediction',
    'Event detection from wearable sensor data',
    'The goal of this competition is to detect freezing of gait (FOG), a debilitating symptom that afflicts many people with Parkinson’s disease. You will develop a machine learning model trained on data collected from a wearable 3D lower back sensor. Your work will help researchers better understand when and why FOG episodes occur. This will improve the ability of medical professionals to optimally evaluate, monitor, and ultimately, prevent FOG events.',
    'https://www.kaggle.com/competitions/tlvmc-parkinsons-freezing-gait-prediction',
    'completed',
    'intermediate',
    '8',
    '',
    '2023-03-09',
    '2023-06-08',
    '2023-08-08 19:47:54',
    '2023-08-08 21:47:57'
  ),
  (
    195,
    'chaimeleon',
    'CHAIMELEON Open Challenges',
    '',
    'The CHAIMELEON Open Challenges is a competition designed to train and refine AI models to answer clinical questions about five types of cancer: prostate, lung, breast, colon, and rectal. Participants are challenged to collaborate and develop innovative AI-powered solutions that can significantly impact cancer diagnosis, management, and treatment. They will be evaluated considering a balance between the performance of their AI algorithms to predict different clinical endpoints such as disease staging, treatment response or progression free survival and their trustworthiness.  The challenges are open to the whole scientific and tech community interested in AI. They are a unique opportunity to showcase how AI can be used to advance medical research and improve patient outcomes within the CHAIMELEON project.',
    'https://chaimeleon.grand-challenge.org/',
    'upcoming',
    'intermediate',
    '5',
    '',
    NULL,
    '2023-12-31',
    '2023-08-09 17:13:09',
    '2023-08-09 22:40:12'
  ),
  (
    196,
    'topcow23',
    'Topology-Aware Anatomical Segmentation of the Circle of Willis for CTA and MRA',
    '',
    'The aim of the challenge is to extract the CoW angio-architecture from 3D angiographic imaging by segmentation of the vessel components. There are two sub-tasks: binary segmentation of CoW vessels, and multi-class CoW anatomical segmentation. We release a new dataset of joint-modalities, CTA and MRA of the same patient cohort, both with annotations of the anatomy of CoW. Our challenge has two tracks for the same segmentation task, namely CTA track and MRA track. We made use of the clinical information from both modalities during our annotation. And participants can pick whichever modality they want, both CTA and MRA, and choose to tackle the task for either modality.',
    'https://topcow23.grand-challenge.org/',
    'active',
    'intermediate',
    '5',
    '',
    '2023-08-20',
    '2023-09-25',
    '2023-08-09 17:16:22',
    '2023-08-09 22:41:03'
  ),
  (
    197,
    'crown2023',
    'Circle of Willis Intracranial Artery Classification and Quantification Challenge 2023',
    '',
    'The purpose of this challenge is to compare automatic methods for classification of the circle of Willis (CoW) configuration and quantification of the CoW major artery diameters and bifurcation angles.',
    'https://crown.isi.uu.nl/',
    'completed',
    'intermediate',
    '14',
    '',
    '2023-05-01',
    '2023-08-15',
    '2023-08-09 22:13:24',
    '2023-08-10 5:44:38'
  ),
  (
    198,
    'making-sense-of-electronic-health-record-ehr-race-and-ethnicity-data',
    'Making Sense of Electronic Health Record (EHR) Race and Ethnicity Data',
    'The US Food and Drug Administration (FDA) calls on stakeholders, including t...',
    'The urgency of the coronavirus disease 2019 (COVID-19) pandemic has heightened interest in the use of real-world data (RWD) to obtain timely information about patients and populations and has focused attention on EHRs. The pandemic has also heightened awareness of long-standing racial and ethnic health disparities along a continuum from underlying social determinants of health, exposure to risk, access to insurance and care, quality of care, and responses to treatments. This highlighted the potential that EHRs can be used to describe and contribute to our understanding of racial and ethnic health disparities and their solutions. The OMB Revisions to the Standards for the Classification of Federal Data on Race and Ethnicity provides minimum standards for maintaining, collecting, and presenting data on race and ethnicity for all Federal reporting purposes, and defines the two separate constructs of race and ethnicity.',
    'https://precision.fda.gov/challenges/30',
    'completed',
    'intermediate',
    '6',
    '',
    '2023-05-31',
    '2023-06-23',
    '2023-08-10 18:28:06',
    '2023-08-10 21:18:15'
  ),
  (
    199,
    'v-champs',
    'The Veterans Cardiac Health and AI Model Predictions (V-CHAMPS)',
    'The Veterans Health Administration Innovation Ecosystem, the Digital Health ...',
    'To better understand the risk and protective factors in the Veteran population, the VHA IE and its collaborating partners are calling upon the public to develop AI/ML models to predict cardiovascular health outcomes, including readmission and mortality, using synthetically generated Veteran health records.  The Challenge consists of two Phases: Phase 1 is focused on synthetic data. In this Phase of the Challenge, AI/ML models will be developed by Challenge participants and trained and tested on the synthetic data sets provided to them, with a view towards predicting outcome variables for Veterans who have been diagnosed with chronic heart failure (please note that in Phase 1, the data is synthetic Veteran health records). Phase 2 will focus on validating and further exploring the limits of the AI/ML models. During this Phase, high-performing AI/ML models from Phase 1 will be brought into the VA system and validated on the real-world Veterans health data within the VHA. These model...',
    'https://precision.fda.gov/challenges/31',
    'completed',
    'intermediate',
    '6',
    '',
    '2023-05-25',
    '2023-08-02',
    '2023-08-10 21:41:10',
    '2023-08-10 22:08:01'
  ),
  (
    200,
    'predicting-high-risk-breast-cancer-phase-1',
    'Predicting High Risk Breast Cancer - Phase 1',
    'Predicting High Risk Breast Cancer: a Nightingale OS & AHLI data challenge',
    ' Every year, 40 million women get a mammogram; some go on to have an invasive biopsy to better examine a concerning area. Underneath these routine tests lies a deep—and disturbing—mystery. Since the 1990s, we have found far more ‘cancers’, which has in turn prompted vastly more surgical procedures and chemotherapy. But death rates from metastatic breast cancer have hardly changed. When a pathologist looks at a biopsy slide, she is looking for known signs of cancer: tubules, cells with atypical looking nuclei, evidence of rapid cell division. These features, first identified in 1928, still underlie critical decisions today: which women must receive urgent treatment with surgery and chemotherapy? And which can be prescribed “watchful waiting”, sparing them invasive procedures for cancers that would not harm them? There is already evidence that algorithms can predict which cancers will metastasize and harm patients on the basis of the biopsy image. Fascinatingly, these algorithms als...',
    'https://app.nightingalescience.org/contests/3jmp2y128nxd',
    'completed',
    'intermediate',
    '15',
    '',
    '2022-06-01',
    '2023-01-12',
    '2023-08-22 17:07:00',
    '2023-09-12 23:52:21'
  ),
  (
    201,
    'predicting-high-risk-breast-cancer-phase-2',
    'Predicting High Risk Breast Cancer - Phase 2',
    'Predicting High Risk Breast Cancer: a Nightingale OS & AHLI data challenge',
    ' Every year, 40 million women get a mammogram; some go on to have an invasive biopsy to better examine a concerning area. Underneath these routine tests lies a deep—and disturbing—mystery. Since the 1990s, we have found far more ‘cancers’, which has in turn prompted vastly more surgical procedures and chemotherapy. But death rates from metastatic breast cancer have hardly changed. When a pathologist looks at a biopsy slide, she is looking for known signs of cancer: tubules, cells with atypical looking nuclei, evidence of rapid cell division. These features, first identified in 1928, still underlie critical decisions today: which women must receive urgent treatment with surgery and chemotherapy? And which can be prescribed “watchful waiting”, sparing them invasive procedures for cancers that would not harm them? There is already evidence that algorithms can predict which cancers will metastasize and harm patients on the basis of the biopsy image. Fascinatingly, these algorithms als...',
    'https://app.nightingalescience.org/contests/vd8g98zv9w0p',
    'completed',
    'intermediate',
    '15',
    '',
    '2023-02-03',
    '2023-05-13',
    '2023-08-22 17:07:01',
    '2023-09-12 23:52:25'
  ),
  (
    202,
    'dream-2-in-silico-network-inference',
    'DREAM 2 - In Silico Network Inference',
    'Predicting the connectivity and properties of in-silico networks.',
    'Three in-silico networks were created and endowed with a dynamics that simulate biological interactions. The challenge consists of predicting the connectivity and some of the properties of one or more of these three networks.',
    'https://www.synapse.org/#!Synapse:syn2825394/wiki/71150',
    'completed',
    'intermediate',
    '1',
    '',
    '2007-03-25',
    NULL,
    '2023-08-24 18:54:05',
    '2023-09-12 23:52:36'
  ),
  (
    203,
    'dream-3-in-silico-network-challenge',
    'DREAM 3 - In Silico Network Challenge',
    'The goal of the in silico challenges is the reverse engineering of gene netw...',
    'The goal of the in silico challenges is the reverse engineering of gene networks from steady state and time series data. Participants are challenged to predict the directed unsigned network topology from the given in silico generated gene expression datasets.',
    'https://www.synapse.org/#!Synapse:syn2853594/wiki/71567',
    'completed',
    'intermediate',
    '1',
    'https://doi.org/10.1089/cmb.2008.09TT',
    '2008-06-09',
    NULL,
    '2023-08-25 16:43:41',
    '2023-09-12 23:52:32'
  ),
  (
    204,
    'dream-4-in-silico-network-challenge',
    'DREAM 4 - In Silico Network Challenge',
    'The goal of the in silico network challenge is to reverse engineer gene regu...',
    'The goal of the in silico network challenge is to reverse engineer gene regulation networks from simulated steady-state and time-series data. Participants are challenged to infer the network structure from the given in silico gene expression datasets. Optionally, participants may also predict the response of the networks to a set of novel perturbations that were not included in the provided datasets.',
    'https://www.synapse.org/#!Synapse:syn3049712/wiki/74628',
    'completed',
    'intermediate',
    '1',
    'https://doi.org/10.1073/pnas.0913357107',
    '2009-06-09',
    NULL,
    '2023-08-25 16:43:42',
    '2023-09-12 23:52:35'
  ),
  (
    205,
    'dream-5-network-inference-challenge',
    'DREAM 5 - Network Inference Challenge',
    'The goal of this Network Inference Challenge is to reverse engineer gene reg...',
    'The goal of this Network Inference Challenge is to reverse engineer gene regulatory networks from gene expression datasets. Participants are given four microarray compendia and are challenged to infer the structure of the underlying transcriptional regulatory networks. Three of the four compendia were obtained from microorganisms, some of which are pathogens of clinical relevance. The fourth compendium is based on an in-silico (i.e., simulated) network. Each compendium consists of hundreds of microarray experiments, which include a wide range of genetic, drug, and environmental perturbations (or in the in-silico network case, simulations thereof). Network predictions will be evaluated on a subset of known interactions for each organism, or on the known network for the in-silico case.',
    'https://www.synapse.org/#!Synapse:syn2787209/wiki/70349',
    'completed',
    'intermediate',
    '1',
    'https://doi.org/10.1038/nmeth.2016',
    '2010-06-09',
    '2010-10-31',
    '2023-08-25 16:43:43',
    '2023-09-12 23:52:37'
  ),
  (
    206,
    'nlp-sandbox-date-annotation',
    'NLP Sandbox Date Annotation',
    'Identify dates in clinical notes.',
    'An NLP Sandbox Date Annotator takes as input a clinical note and outputs a list of predicted date annotations found in the clinical note.',
    'https://www.synapse.org/#!Synapse:syn22277123/wiki/609134',
    'completed',
    'intermediate',
    '1',
    'https://doi.org/10.7303/syn22277123',
    '2021-06-04',
    '2023-09-01',
    '2023-08-25 16:45:22',
    '2023-09-08 16:44:46'
  ),
  (
    207,
    'nlp-sandbox-person-name-annotation',
    'NLP Sandbox Person Name Annotation',
    'Identify person names in clinical notes.',
    'An NLP Sandbox Person Name Annotator takes as input a clinical note and outputs a list of predicted person name annotations found in the clinical note.',
    'https://www.synapse.org/#!Synapse:syn22277123/wiki/609134',
    'completed',
    'intermediate',
    '1',
    'https://doi.org/10.7303/syn22277123',
    '2021-06-04',
    '2023-09-01',
    '2023-09-08 16:44:20',
    '2023-09-08 16:44:20'
  ),
  (
    208,
    'nlp-sandbox-location-annotation',
    'NLP Sandbox Location Annotation',
    'Identify location information in clinical notes.',
    'An NLP Sandbox Location Annotator takes as input a clinical note and outputs a list of predicted location annotations found in the clinical note.',
    'https://www.synapse.org/#!Synapse:syn22277123/wiki/609134',
    'completed',
    'intermediate',
    '1',
    'https://doi.org/10.7303/syn22277123',
    '2021-06-04',
    '2023-09-01',
    '2023-09-08 16:44:21',
    '2023-09-08 16:44:21'
  ),
  (
    209,
    'nlp-sandbox-contact-annotation',
    'NLP Sandbox Contact Annotation',
    'Identify contact information in clinical notes.',
    'An NLP Sandbox contact annotator takes as input a clinical note and outputs a list of predicted contact annotations found in the clinical note.',
    'https://www.synapse.org/#!Synapse:syn22277123/wiki/609134',
    'completed',
    'intermediate',
    '1',
    'https://doi.org/10.7303/syn22277123',
    '2021-06-04',
    '2023-09-01',
    '2023-09-08 16:44:22',
    '2023-09-08 16:44:22'
  ),
  (
    210,
    'nlp-sandbox-id-annotation',
    'NLP Sandbox ID Annotation',
    'Identify identifiers in clinical notes.',
    'An NLP Sandbox ID annotator takes as input a clinical note and outputs a list of predicted ID annotations found in the clinical note.',
    'https://www.synapse.org/#!Synapse:syn22277123/wiki/609134',
    'completed',
    'intermediate',
    '1',
    'https://doi.org/10.7303/syn22277123',
    '2021-06-04',
    '2023-09-01',
    '2023-09-08 16:44:22',
    '2023-09-09 2:50:13'
  ),
  (
    211,
    'dream-2-bcl6-transcriptomic-target-prediction',
    'DREAM 2 – BCL6 Transcriptomic Target Prediction',
    '',
    'A number of potential transcriptional targets of BCL6, a gene that encodes for a transcription factor active in B cells, have been identified with ChIP-on-chip data and functionally validated by perturbing the BCL6 pathway with CD40 and anti-IgM, and by over-expressing exogenous BCL6 in Ramos cell. We subselected a number of targets found in this way (the "gold standard positive" set), and added a number decoys (genes that have no evidence of being BCL6 targets, named the "gold standard negative" set), compiling a list of 200 genes in total. Given this list of 200 genes, the challenge consists of identifying which ones are the true targets and which ones are the decoys, using an independent panel of gene expression data.',
    'https://www.synapse.org/#!Synapse:syn3034857/wiki/',
    'completed',
    'intermediate',
    '1',
    'https://doi.org/10.1073/pnas.0437996100',
    '2007-04-19',
    NULL,
    '2023-09-12 21:26:22',
    '2023-09-12 23:53:56'
  ),
  (
    212,
    'dream-2-protein-protein-interaction-network-inference',
    'DREAM 2 – Protein-Protein Interaction Network Inference',
    'Predict a PPI network of 47 proteins',
    'For many pairs of bait and prey genes, yeast protein-protein interactions were tested in an unbiased fashion using a high saturation, high-stringency variant of the yeast two-hybrid (Y2H) method. A high confidence subset of gene pairs that were found to interact in at least three repetitions of the experiment but that hadn’t been reported in the literature was extracted. There were 47 yeast genes involved in these pairs. Including self interactions, there are a total of 47*48/2 possible pairs of genes that can be formed with these 47 genes. As mentioned above some of these gene pairs were seen to consistently interact in at least three repetitions of the Y2H experiments: these gene pairs form the "gold standard positive" set. A second set among these gene pairs were seen never to interact in repeated experiments and were not reported as interacting in the literature; we call this the "gold standard negative" set. Finally in a third set of gene pairs, which we shall call the "undec...',
    'https://www.synapse.org/#!Synapse:syn2825374/wiki/',
    'completed',
    'intermediate',
    '1',
    'https://doi.org/10.1126/science.1158684',
    '2007-05-24',
    NULL,
    '2023-09-12 21:26:28',
    '2023-09-12 23:56:04'
  ),
  (
    213,
    'dream-2-genome-scale-network-inference',
    'DREAM 2 – Genome-Scale Network Inference',
    '',
    'A panel of single-channel microarrays was collected for a particular microorganism, including some already published and some in-print data. The data was appropriately normalized (to the logarithmic scale). The challenge consists of reconstructing a genome-scale transcriptional network for this organism. The accuracy of network inference will be judged using chromatin precipitation and otherwise experimentally verified Transcription Factor (TF)-target interactions.',
    'https://www.synapse.org/#!Synapse:syn3034894/wiki/74418',
    'completed',
    'intermediate',
    '1',
    'https://doi.org/10.1371/journal.pbio.0050008',
    '2007-06-05',
    '2007-10-31',
    '2023-09-12 21:26:34',
    '2023-09-13 0:02:45'
  ),
  (
    214,
    'dream-2-synthetic-five-gene-network-inference',
    'DREAM 2 – Synthetic Five-Gene Network Inference',
    '',
    'A synthetic-biology network consisting of 5 interacting genes was created and transfected to an in-vivo model organism. The challenge consists of predicting the connectivity of the five-gene network from in-vivo measurements.',
    'https://www.synapse.org/#!Synapse:syn3034869/wiki/74411',
    'completed',
    'intermediate',
    '1',
    'https://doi.org/10.1016/j.cell.2009.01.055',
    '2007-06-20',
    '2007-10-31',
    '2023-09-12 21:26:56',
    '2023-09-13 0:02:34'
  ),
  (
    215,
    'dream-3-signaling-cascade-identification',
    'DREAM 3 – Signaling Cascade Identification',
    '',
    'The concentration of four intracellular proteins or phospho-proteins (X1, X2, X3 and X4) participating in a signaling cascade were measured in about 104 cells by antibody staining and flow cytometry. The idea of this challenge is to explore what key aspects of the dynamics and topology of interactions of a signaling cascade can be inferred from incomplete flow cytometry data.',
    'https://www.synapse.org/#!Synapse:syn3033068/wiki/74362',
    'completed',
    'intermediate',
    '1',
    '',
    '2008-06-01',
    '2008-10-31',
    '2023-09-12 21:27:04',
    '2023-09-13 0:03:50'
  ),
  (
    216,
    'dream-3-gene-expression-prediction',
    'DREAM 3 – Gene Expression Prediction',
    '',
    'Gene expression time course data is provided for four different strains of yeast (S. Cerevisiae), after perturbation of the cells. The challenge is to predict the rank order of induction/repression of a small subset of genes (the "prediction targets") in one of the four strains, given complete data for three of the strains, and data for all genes except the prediction targets in the other strain. You are also allowed to use any information that is in the public domain and are expected to be forthcoming about what information was used.',
    'https://www.synapse.org/#!Synapse:syn3033083/wiki/74369',
    'completed',
    'intermediate',
    '1',
    '',
    '2008-06-01',
    '2008-10-31',
    '2023-09-12 21:27:12',
    '2023-09-13 0:05:21'
  ),
  (
    217,
    'dream-4-predictive-signaling-network-modelling',
    'DREAM 4 – Predictive Signaling Network Modelling',
    'Cell-type specific high-throughput experimental data',
    'This challenge explores the extent to which our current knowledge of signaling pathways, collected from a variety of cell types, agrees with cell-type specific high-throughput experimental data. Specifically, we ask the challenge participants to create a cell-type specific model of signal transduction using the measured activity levels of signaling proteins in HepG2 cell lines. The model, which can leverage prior information encoded in a generic signaling pathway provided in the challenge, should be biologically interpretable as a network, and capable of predicting the outcome of new experiments.',
    'https://www.synapse.org/#!Synapse:syn2825304/wiki/71129',
    'completed',
    'intermediate',
    '1',
    '',
    '2009-03-09',
    NULL,
    '2023-09-12 21:27:14',
    '2023-09-13 0:07:43'
  ),
  (
    218,
    'dream-3-signaling-response-prediction',
    'DREAM 3 – Signaling Response Prediction',
    'Predict missing protein concentrations from a large corpus of measurements',
    'Approximately 10,000 intracellular measurements (fluorescence signals proportional to the concentrations of phosphorylated proteins) and extracellular measurements (concentrations of cytokines released in response to cell stimulation) were acquired in human normal hepatocytes and the hepatocellular carcinoma cell line HepG2 cells. The datasets consist of measurements of 17 phospho-proteins (at 0 min, 30 min, and 3 hrs) and 20 cytokines (at 0 min, 3 hrs, and 24 hrs) in two cell types (normal and cancer) after perturbations to the pathway induced by the combinatorial treatment of 7 stimuli and 7 selective inhibitors.',
    'https://www.synapse.org/#!Synapse:syn2825325/wiki/',
    'completed',
    'intermediate',
    '1',
    'https://doi.org/10.1126%2Fscisignal.2002212',
    '2009-03-09',
    NULL,
    '2023-09-12 21:27:20',
    '2023-09-13 0:12:14'
  ),
  (
    219,
    'dream-4-peptide-recognition-domain-prd-specificity-prediction',
    'DREAM 4 – Peptide Recognition Domain (PRD) Specificity Prediction',
    '',
    'Many important protein-protein interactions are mediated by peptide recognition domains (PRD), which bind short linear sequence motifs in other proteins. For example, SH3 domains typically recognize proline-rich motifs, PDZ domains recognize hydrophobic C-terminal tails, and kinases recognize short sequence regions around a phosphorylatable residue (Pawson, 2003).  Given the sequence of the domains, the challenge consists of predicting a position weight matrix (PWM) that describes the specificity profile of each of the given domains to their target peptides. Any publicly accessible peptide specificity information available for the domain may be used.',
    'https://www.synapse.org/#!Synapse:syn2925957/wiki/72976',
    'completed',
    'intermediate',
    '1',
    '',
    '2009-06-01',
    '2009-10-31',
    '2023-09-12 21:27:35',
    '2023-09-13 0:15:42'
  ),
  (
    220,
    'dream-5-transcription-factor-dna-motif-recognition-challenge',
    'DREAM 5 – Transcription-Factor, DNA-Motif Recognition Challenge',
    '',
    'Transcription factors (TFs) control the expression of genes through sequence-specific interactions with genomic DNA. Different TFs bind preferentially to different sequences, with the majority recognizing short (6-12 base), degenerate ‘motifs’. Modeling the sequence specificities of TFs is a central problem in understanding the function and evolution of the genome, because many types of genomic analyses involve scanning for potential TF binding sites. Models of TF binding specificity are also important for understanding the function and evolution of the TFs themselves. The challenge consists of predicting the signal intensities for the remaining TFs.',
    'https://www.synapse.org/#!Synapse:syn2887863/wiki/72185',
    'completed',
    'intermediate',
    '1',
    'https://doi.org/10.1038/nbt.2486',
    '2011-06-01',
    '2011-09-30',
    '2023-09-12 21:27:41',
    '2023-09-13 0:20:22'
  ),
  (
    221,
    'dream-5-epitope-antibody-recognition-ear-challenge',
    'DREAM 5 – Epitope-Antibody Recognition (EAR) Challenge',
    'Predict the binding specificity of peptide-antibody interactions.',
    'Humoral immune responses are mediated through antibodies. About 1010 to 1012 different antigen binding sites called paratopes are generated by genomic recombination. These antibodies are capable to bind to a variety of structures ranging from small molecules to protein complexes, including any posttranslational modification thereof. When studying protein-antibody interactions, two types of epitopes (the region paratopes interact with) are to be distinguished from each other: i) conformational and ii) linear epitopes. All potential linear epitopes of a protein can be represented by short peptides derived from the primary amino acid sequence. These peptides can be synthesized and arrayed on solid supports, e.g. glass slides (see Lorenz et al., 2009 [1]). By incubating these peptide arrays with antibody mixtures such as human serum or plasma, peptides can be determined that interact with antibodies in a specific fashion.',
    'https://www.synapse.org/#!Synapse:syn2820433/wiki/71017',
    'completed',
    'intermediate',
    '1',
    '',
    '2010-06-09',
    NULL,
    '2023-09-12 21:27:44',
    '2023-09-13 0:25:03'
  ),
  (
    222,
    'dream-gene-expression-prediction-challenge',
    'DREAM Gene Expression Prediction Challenge',
    'Predict gene expression levels from promoter sequences in eukaryotes',
    'The level by which genes are transcribed is determined in large part by the DNA sequence upstream to the gene, known as the promoter region. Although widely studied, we are still far from a quantitative and predictive understanding of how transcriptional regulation is encoded in gene promoters. One obstacle in the field is obtaining accurate measurements of transcription derived by different promoters. To address this, an experimental system was designed to measure the transcription derived by different promoters, all of which are inserted into the same genomic location upstream to a reporter gene – a yellow florescence protein gene (YFP). The challenge consists of the prediction of the promoter activity given a promoter sequence and a specific experimental condition. To study a set of promoters that share many elements of the regulatory program, and thus are suitable for computational learning, the data pertains to promoters of most of the ribosomal protein genes (RP) of yeast (S...',
    'https://www.synapse.org/#!Synapse:syn2820426/wiki/71010',
    'completed',
    'intermediate',
    '1',
    '',
    '2010-07-09',
    NULL,
    '2023-09-12 21:28:00',
    '2023-09-13 0:31:58'
  ),
  (
    223,
    'dream-5-systems-genetics-challenge',
    'DREAM 5 – Systems Genetics Challenge',
    'Predict disease phenotypes and infer Gene Networks from Systems Genetics data',
    'The central goal of systems biology is to gain a predictive, system-level understanding of biological networks. This can be done, for example, by inferring causal networks from observations on a perturbed biological system. An ideal experimental design for causal inference is randomized, multifactorial perturbation. The recognition that the genetic variation in a segregating population represents randomized, multifactorial perturbations (Jansen and Nap (2001), Jansen (2003)) gave rise to Systems Genetics (SG), where a segregating or genetically randomized population is genotyped for many DNA variants, and profiled for phenotypes of interest (e.g. disease phenotypes), gene expression, and potentially other ‘omics’ variables (protein expression, metabolomics, DNA methylation, etc.; Figure 1. Figure 1 was taken from Jansen and Nap (2001)). In this challenge we explore the use of Systems Genetics data for elucidating causal network models among genes, i.e. Gene Networks (DREAM5 SYSGEN...',
    'https://www.synapse.org/#!Synapse:syn2820440/wiki/',
    'completed',
    'intermediate',
    '1',
    '',
    '2010-07-09',
    NULL,
    '2023-09-12 21:28:10',
    '2023-09-13 0:32:10'
  ),
  (
    224,
    'dream-6-estimation-of-model-parameters-challenge',
    'DREAM 6 – Estimation of Model Parameters Challenge',
    '',
    'Given the complete model structures (including expressions for the kinetic rate laws) for three gene regulatory networks, participants are asked to develop and/or apply optimization methods, including the selection of the most informative experiments, to accurately estimate parameters and predict outcomes of perturbations in Systems Biology models.',
    'https://www.synapse.org/#!Synapse:syn2841366/wiki/71372',
    'completed',
    'intermediate',
    '1',
    '',
    '2011-06-01',
    '2011-10-31',
    '2023-09-12 21:28:12',
    '2023-09-13 0:32:54'
  ),
  (
    225,
    'dream-6-flowcap2-molecular-classification-of-acute-myeloid-leukemia-challenge',
    'DREAM 6 – FlowCAP2 Molecular Classification of Acute Myeloid Leukemia Challenge',
    'The goal of this challenge is to diagnose Acute Myeloid Leukaemia from patie...',
    'Flow cytometry (FCM) has been widely used by immunologists and cancer biologists for more than 30 years as a biomedical research tool to distinguish different cell types in mixed populations, based on the expression of cellular markers. It has also become a widely used diagnostic tool for clinicians to identify abnormal cell populations associated with disease. In the last decade, advances in instrumentation and reagent technologies have enabled simultaneous single-cell measurement of tens of surface and intracellular markers, as well as tens of signaling molecules, positioning FCM to play an even bigger role in medicine and systems biology [1,2]. However, the rapid expansion of FCM applications has outpaced the functionality of traditional analysis tools used to interpret FCM data such that scientists are faced with the daunting prospect of manually identifying interesting cell populations in 20 dimensional data from a collection of millions of cells. For these reasons a reliable...',
    'https://www.synapse.org/#!Synapse:syn2887788/wiki/72178',
    'completed',
    'intermediate',
    '1',
    'https://doi.org/10.1038/nmeth.2365',
    '2011-06-01',
    '2011-09-30',
    '2023-09-12 21:28:19',
    '2023-09-13 0:37:41'
  ),
  (
    226,
    'dream-6-alternative-splicing-challenge',
    'DREAM 6 – Alternative Splicing Challenge',
    '',
    'The goal of the mRNA-seq alternative splicing challenge is to assess the accuracy of the reconstruction of alternatively spliced mRNA transcripts from Illumina short-read mRNA-seq. Reconstructed transcripts will be scored against Pacific Biosciences long-read mRNA-seq. The ensuing analysis of the transcriptomes from mandrill and rhinoceros fibroblasts and their derived induced pluripotent stem cells (iPSC), as well as the transcriptome for human Embrionic Stem Cells (hESC) is an opportunity to discover novel biology as well as investigate species-bias of different methods.',
    'https://www.synapse.org/#!Synapse:syn2817724/wiki/',
    'completed',
    'intermediate',
    '1',
    '',
    '2011-08-09',
    NULL,
    '2023-09-12 21:28:25',
    '2023-09-13 0:40:05'
  ),
  (
    227,
    'causalbench-challenge',
    'CausalBench Challenge',
    'A machine learning contest for gene network inference from single-cell pertu...',
    'Mapping gene–gene interactions in cellular systems is a fundamental step in early-stage drug discovery that helps generate hypotheses on what molecular mechanisms may effectively be targeted by potential future medicines. In the CausalBench Challenge, we invite the machine-learning community to advance the state-of-the-art in deriving gene–gene networks from large-scale real-world perturbational single-cell datasets to improve our ability to glean causal insights into disease-relevant biology.',
    'https://www.gsk.ai/causalbench-challenge/',
    'completed',
    'intermediate',
    '14',
    'https://doi.org/10.48550/arXiv.2308.15395',
    '2023-03-01',
    '2023-04-21',
    '2023-09-13 0:41:58',
    '2023-09-13 17:26:24'
  ),
  (
    228,
    'iclr-computational-geometry-and-topology-challenge-2022',
    'ICLR Computational Geometry & Topology Challenge 2022',
    '',
    'The purpose of this challenge is to foster reproducible research in geometric (deep) learning, by crowdsourcing the open-source implementation of learning algorithms on manifolds. Participants are asked to contribute code for a published/unpublished algorithm, following Scikit-Learn/Geomstats'' or pytorch''s APIs and computational primitives, benchmark it, and demonstrate its use in real-world scenarios.',
    'https://github.com/geomstats/challenge-iclr-2022',
    'completed',
    'intermediate',
    '14',
    '',
    NULL,
    '2022-04-04',
    '2023-09-13 16:54:06',
    '2023-09-13 17:26:29'
  ),
  (
    229,
    'iclr-computational-geometry-and-topology-challenge-2021',
    'ICLR Computational Geometry & Topology Challenge 2021',
    '',
    'The purpose of this challenge is to push forward the fields of computational differential geometry and topology, by creating the best data analysis, computational method, or numerical experiment relying on state-of-the-art geometric and topological Python packages.',
    'https://github.com/geomstats/challenge-iclr-2021',
    'completed',
    'intermediate',
    '14',
    'https://doi.org/10.48550/arXiv.2108.09810',
    NULL,
    '2021-05-02',
    '2023-09-13 17:02:12',
    '2023-09-13 17:26:31'
  ),
  (
    230,
    'genedisco-challenge',
    'GeneDisco Challenge',
    '',
    'The GeneDisco challenge is a machine learning community challenge for evaluating batch active learning algorithms for exploring the vast experimental design space in genetic perturbation experiments. Genetic perturbation experiments, using for example CRISPR technologies to perturb the genome, are a vital component of early-stage drug discovery, including target discovery and target validation. The GeneDisco challenge is organized in conjunction with the Machine Learning for Drug Discovery workshop at ICLR-22.',
    'https://www.gsk.ai/genedisco-challenge/',
    'completed',
    'intermediate',
    '14',
    'https://doi.org/10.48550/arXiv.2110.11875',
    '2022-01-31',
    '2022-03-31',
    '2023-09-13 17:20:30',
    '2023-09-13 17:26:20'
  );

  -- challenge_organization_role data

INSERT INTO challenge_contribution (id, challenge_id, organization_id, role)
VALUES (1, 1, 75, 'sponsor'),
  (2, 2, 28, 'data_contributor'),
  (3, 2, 45, 'data_contributor'),
  (4, 2, 151, 'data_contributor'),
  (5, 2, 52, 'sponsor'),
  (6, 3, 154, 'data_contributor'),
  (7, 3, 118, 'data_contributor'),
  (8, 3, 17, 'data_contributor'),
  (9, 3, 142, 'data_contributor'),
  (10, 4, 150, 'data_contributor'),
  (11, 4, 52, 'data_contributor'),
  (12, 4, 131, 'sponsor'),
  (13, 5, 134, 'challenge_organizer'),
  (14, 5, 132, 'challenge_organizer'),
  (15, 5, 211, 'challenge_organizer'),
  (16, 5, 101, 'data_contributor'),
  (17, 6, 54, 'challenge_organizer'),
  (18, 6, 146, 'challenge_organizer'),
  (19, 6, 171, 'challenge_organizer'),
  (20, 6, 227, 'challenge_organizer'),
  (21, 6, 27, 'challenge_organizer'),
  (22, 6, 128, 'challenge_organizer'),
  (23, 6, 93, 'challenge_organizer'),
  (24, 6, 179, 'challenge_organizer'),
  (25, 6, 89, 'challenge_organizer'),
  (26, 6, 73, 'challenge_organizer'),
  (27, 6, 170, 'challenge_organizer'),
  (28, 6, 119, 'challenge_organizer'),
  (29, 6, 171, 'sponsor'),
  (30, 6, 93, 'sponsor'),
  (31, 6, 57, 'sponsor'),
  (32, 6, 146, 'sponsor'),
  (33, 6, 120, 'sponsor'),
  (34, 7, 171, 'challenge_organizer'),
  (35, 7, 128, 'challenge_organizer'),
  (36, 7, 93, 'challenge_organizer'),
  (37, 7, 71, 'challenge_organizer'),
  (38, 7, 89, 'challenge_organizer'),
  (39, 7, 73, 'challenge_organizer'),
  (40, 7, 170, 'challenge_organizer'),
  (41, 7, 150, 'challenge_organizer'),
  (42, 7, 130, 'challenge_organizer'),
  (43, 8, 171, 'challenge_organizer'),
  (44, 8, 206, 'challenge_organizer'),
  (45, 8, 128, 'challenge_organizer'),
  (46, 8, 93, 'challenge_organizer'),
  (47, 8, 56, 'challenge_organizer'),
  (48, 8, 37, 'challenge_organizer'),
  (49, 8, 25, 'challenge_organizer'),
  (50, 8, 78, 'challenge_organizer'),
  (51, 8, 125, 'challenge_organizer'),
  (52, 8, 23, 'challenge_organizer'),
  (53, 8, 55, 'data_contributor'),
  (54, 8, 24, 'sponsor'),
  (55, 8, 125, 'sponsor'),
  (56, 8, 145, 'sponsor'),
  (57, 8, 181, 'sponsor'),
  (58, 8, 93, 'sponsor'),
  (59, 9, 171, 'challenge_organizer'),
  (60, 9, 150, 'challenge_organizer'),
  (61, 9, 128, 'challenge_organizer'),
  (62, 9, 93, 'challenge_organizer'),
  (63, 9, 197, 'challenge_organizer'),
  (64, 9, 149, 'challenge_organizer'),
  (65, 9, 23, 'challenge_organizer'),
  (66, 9, 99, 'data_contributor'),
  (67, 10, 165, 'challenge_organizer'),
  (68, 10, 171, 'challenge_organizer'),
  (69, 10, 128, 'challenge_organizer'),
  (70, 10, 93, 'challenge_organizer'),
  (71, 10, 23, 'challenge_organizer'),
  (72, 10, 122, 'challenge_organizer'),
  (73, 10, 122, 'data_contributor'),
  (74, 10, 122, 'sponsor'),
  (75, 10, 93, 'sponsor'),
  (76, 10, 171, 'sponsor'),
  (77, 10, 165, 'sponsor'),
  (78, 11, 171, 'challenge_organizer'),
  (79, 11, 150, 'challenge_organizer'),
  (80, 11, 128, 'challenge_organizer'),
  (81, 11, 41, 'challenge_organizer'),
  (82, 11, 58, 'challenge_organizer'),
  (83, 11, 205, 'challenge_organizer'),
  (84, 11, 93, 'challenge_organizer'),
  (85, 11, 23, 'challenge_organizer'),
  (86, 11, 41, 'data_contributor'),
  (87, 11, 58, 'data_contributor'),
  (88, 11, 46, 'sponsor'),
  (89, 11, 109, 'sponsor'),
  (90, 12, 107, 'challenge_organizer'),
  (91, 12, 171, 'challenge_organizer'),
  (92, 12, 204, 'challenge_organizer'),
  (93, 12, 128, 'challenge_organizer'),
  (94, 12, 93, 'challenge_organizer'),
  (95, 12, 38, 'challenge_organizer'),
  (96, 12, 223, 'challenge_organizer'),
  (97, 12, 119, 'challenge_organizer'),
  (98, 12, 23, 'challenge_organizer'),
  (99, 12, 82, 'challenge_organizer'),
  (100, 12, 169, 'data_contributor'),
  (101, 12, 18, 'data_contributor'),
  (102, 12, 94, 'data_contributor'),
  (103, 12, 19, 'sponsor'),
  (104, 12, 39, 'sponsor'),
  (105, 12, 74, 'sponsor'),
  (106, 12, 153, 'sponsor'),
  (107, 12, 164, 'sponsor'),
  (108, 12, 168, 'sponsor'),
  (109, 12, 172, 'sponsor'),
  (110, 12, 181, 'sponsor'),
  (111, 13, 206, 'challenge_organizer'),
  (112, 13, 167, 'challenge_organizer'),
  (113, 13, 171, 'challenge_organizer'),
  (114, 13, 93, 'challenge_organizer'),
  (115, 13, 128, 'challenge_organizer'),
  (116, 13, 23, 'challenge_organizer'),
  (117, 13, 167, 'data_contributor'),
  (118, 13, 100, 'sponsor'),
  (119, 13, 93, 'sponsor'),
  (120, 13, 171, 'sponsor'),
  (121, 13, 1, 'sponsor'),
  (122, 14, 58, 'challenge_organizer'),
  (123, 14, 93, 'challenge_organizer'),
  (124, 14, 124, 'challenge_organizer'),
  (125, 14, 155, 'challenge_organizer'),
  (126, 14, 157, 'challenge_organizer'),
  (127, 14, 171, 'challenge_organizer'),
  (128, 14, 188, 'challenge_organizer'),
  (129, 14, 196, 'challenge_organizer'),
  (130, 14, 199, 'challenge_organizer'),
  (131, 14, 219, 'challenge_organizer'),
  (132, 14, 26, 'data_contributor'),
  (133, 14, 47, 'data_contributor'),
  (134, 14, 172, 'data_contributor'),
  (135, 14, 124, 'data_contributor'),
  (136, 14, 157, 'sponsor'),
  (137, 14, 131, 'sponsor'),
  (138, 14, 21, 'sponsor'),
  (139, 15, 128, 'challenge_organizer'),
  (140, 15, 93, 'challenge_organizer'),
  (141, 15, 171, 'challenge_organizer'),
  (142, 15, 118, 'challenge_organizer'),
  (143, 15, 154, 'challenge_organizer'),
  (144, 15, 187, 'challenge_organizer'),
  (145, 15, 115, 'challenge_organizer'),
  (146, 15, 23, 'challenge_organizer'),
  (147, 15, 154, 'data_contributor'),
  (148, 15, 140, 'data_contributor'),
  (149, 15, 142, 'data_contributor'),
  (150, 15, 154, 'sponsor'),
  (151, 15, 93, 'sponsor'),
  (152, 15, 171, 'sponsor'),
  (153, 15, 1, 'sponsor'),
  (154, 15, 32, 'sponsor'),
  (155, 15, 67, 'sponsor'),
  (156, 16, 128, 'challenge_organizer'),
  (157, 16, 93, 'challenge_organizer'),
  (158, 16, 26, 'challenge_organizer'),
  (159, 16, 89, 'challenge_organizer'),
  (160, 16, 73, 'challenge_organizer'),
  (161, 16, 170, 'challenge_organizer'),
  (162, 16, 171, 'challenge_organizer'),
  (163, 16, 23, 'challenge_organizer'),
  (164, 16, 26, 'data_contributor'),
  (165, 16, 235, 'data_contributor'),
  (166, 16, 29, 'data_contributor'),
  (167, 16, 26, 'sponsor'),
  (168, 17, 128, 'challenge_organizer'),
  (169, 17, 93, 'challenge_organizer'),
  (170, 17, 197, 'challenge_organizer'),
  (171, 17, 149, 'challenge_organizer'),
  (172, 17, 171, 'challenge_organizer'),
  (173, 17, 23, 'challenge_organizer'),
  (174, 17, 99, 'data_contributor'),
  (175, 17, 149, 'sponsor'),
  (176, 17, 81, 'sponsor'),
  (177, 17, 138, 'sponsor'),
  (178, 17, 171, 'sponsor'),
  (179, 17, 197, 'sponsor'),
  (180, 17, 150, 'sponsor'),
  (181, 18, 235, 'challenge_organizer'),
  (182, 18, 128, 'challenge_organizer'),
  (183, 18, 93, 'challenge_organizer'),
  (184, 18, 197, 'challenge_organizer'),
  (185, 18, 150, 'challenge_organizer'),
  (186, 18, 149, 'challenge_organizer'),
  (187, 18, 79, 'challenge_organizer'),
  (188, 18, 220, 'challenge_organizer'),
  (189, 18, 171, 'challenge_organizer'),
  (190, 18, 23, 'challenge_organizer'),
  (191, 18, 149, 'data_contributor'),
  (192, 18, 99, 'data_contributor'),
  (193, 18, 149, 'data_contributor'),
  (194, 18, 156, 'sponsor'),
  (195, 18, 81, 'sponsor'),
  (196, 18, 138, 'sponsor'),
  (197, 19, 171, 'challenge_organizer'),
  (198, 19, 64, 'challenge_organizer'),
  (199, 19, 229, 'challenge_organizer'),
  (200, 19, 59, 'challenge_organizer'),
  (201, 19, 59, 'sponsor'),
  (202, 20, 171, 'challenge_organizer'),
  (203, 20, 205, 'challenge_organizer'),
  (204, 20, 128, 'challenge_organizer'),
  (205, 20, 93, 'challenge_organizer'),
  (206, 20, 89, 'challenge_organizer'),
  (207, 20, 73, 'challenge_organizer'),
  (208, 20, 170, 'challenge_organizer'),
  (209, 20, 118, 'challenge_organizer'),
  (210, 20, 23, 'challenge_organizer'),
  (211, 20, 111, 'challenge_organizer'),
  (212, 20, 205, 'data_contributor'),
  (213, 20, 180, 'sponsor'),
  (214, 21, 179, 'challenge_organizer'),
  (215, 21, 171, 'challenge_organizer'),
  (216, 21, 128, 'challenge_organizer'),
  (217, 21, 93, 'challenge_organizer'),
  (218, 21, 199, 'challenge_organizer'),
  (219, 21, 150, 'challenge_organizer'),
  (220, 21, 115, 'challenge_organizer'),
  (221, 21, 23, 'challenge_organizer'),
  (222, 21, 69, 'data_contributor'),
  (223, 22, 220, 'challenge_organizer'),
  (224, 22, 91, 'challenge_organizer'),
  (225, 22, 127, 'challenge_organizer'),
  (226, 22, 95, 'challenge_organizer'),
  (227, 22, 128, 'challenge_organizer'),
  (228, 22, 93, 'challenge_organizer'),
  (229, 22, 171, 'challenge_organizer'),
  (230, 22, 23, 'challenge_organizer'),
  (231, 22, 64, 'data_contributor'),
  (232, 22, 229, 'data_contributor'),
  (233, 23, 232, 'challenge_organizer'),
  (234, 23, 128, 'challenge_organizer'),
  (235, 23, 93, 'challenge_organizer'),
  (236, 23, 197, 'challenge_organizer'),
  (237, 23, 150, 'challenge_organizer'),
  (238, 23, 149, 'challenge_organizer'),
  (239, 23, 171, 'challenge_organizer'),
  (240, 23, 23, 'challenge_organizer'),
  (241, 23, 99, 'data_contributor'),
  (242, 23, 131, 'data_contributor'),
  (243, 23, 149, 'data_contributor'),
  (244, 23, 138, 'sponsor'),
  (245, 23, 81, 'sponsor'),
  (246, 23, 156, 'sponsor'),
  (247, 23, 131, 'sponsor'),
  (248, 24, 171, 'challenge_organizer'),
  (249, 24, 93, 'challenge_organizer'),
  (250, 24, 106, 'challenge_organizer'),
  (251, 24, 128, 'challenge_organizer'),
  (252, 24, 175, 'challenge_organizer'),
  (253, 24, 189, 'challenge_organizer'),
  (254, 24, 36, 'data_contributor'),
  (255, 24, 128, 'data_contributor'),
  (256, 24, 110, 'sponsor'),
  (257, 24, 80, 'sponsor'),
  (258, 24, 236, 'sponsor'),
  (259, 24, 93, 'sponsor'),
  (260, 24, 20, 'sponsor'),
  (261, 25, 47, 'challenge_organizer'),
  (262, 25, 129, 'challenge_organizer'),
  (263, 25, 171, 'challenge_organizer'),
  (264, 25, 193, 'data_contributor'),
  (265, 25, 58, 'data_contributor'),
  (266, 25, 86, 'data_contributor'),
  (267, 25, 89, 'data_contributor'),
  (268, 25, 129, 'data_contributor'),
  (269, 25, 47, 'sponsor'),
  (270, 26, 85, 'challenge_organizer'),
  (271, 26, 197, 'challenge_organizer'),
  (272, 26, 63, 'challenge_organizer'),
  (273, 26, 149, 'challenge_organizer'),
  (274, 26, 171, 'challenge_organizer'),
  (275, 26, 131, 'data_contributor'),
  (276, 26, 88, 'data_contributor'),
  (277, 26, 179, 'data_contributor'),
  (278, 26, 49, 'data_contributor'),
  (279, 26, 108, 'data_contributor'),
  (280, 26, 202, 'data_contributor'),
  (281, 26, 197, 'data_contributor'),
  (282, 26, 149, 'data_contributor'),
  (283, 26, 99, 'data_contributor'),
  (284, 26, 41, 'data_contributor'),
  (285, 27, 126, 'challenge_organizer'),
  (286, 27, 65, 'challenge_organizer'),
  (287, 27, 230, 'challenge_organizer'),
  (288, 27, 88, 'challenge_organizer'),
  (289, 27, 171, 'challenge_organizer'),
  (290, 27, 224, 'challenge_organizer'),
  (291, 27, 208, 'challenge_organizer'),
  (292, 27, 216, 'challenge_organizer'),
  (293, 27, 126, 'data_contributor'),
  (294, 27, 171, 'data_contributor'),
  (295, 27, 126, 'sponsor'),
  (296, 27, 166, 'sponsor'),
  (297, 28, 141, 'challenge_organizer'),
  (298, 28, 128, 'challenge_organizer'),
  (299, 28, 93, 'challenge_organizer'),
  (300, 28, 131, 'challenge_organizer'),
  (301, 28, 89, 'challenge_organizer'),
  (302, 28, 73, 'challenge_organizer'),
  (303, 28, 170, 'challenge_organizer'),
  (304, 28, 149, 'challenge_organizer'),
  (305, 28, 59, 'challenge_organizer'),
  (306, 28, 152, 'challenge_organizer'),
  (307, 28, 171, 'challenge_organizer'),
  (308, 28, 51, 'data_contributor'),
  (309, 28, 51, 'sponsor'),
  (310, 28, 59, 'sponsor'),
  (311, 28, 147, 'sponsor'),
  (312, 29, 211, 'challenge_organizer'),
  (313, 29, 128, 'challenge_organizer'),
  (314, 29, 210, 'challenge_organizer'),
  (315, 29, 171, 'challenge_organizer'),
  (316, 29, 195, 'challenge_organizer'),
  (317, 29, 128, 'data_contributor'),
  (318, 29, 128, 'sponsor'),
  (319, 30, 89, 'challenge_organizer'),
  (320, 30, 128, 'challenge_organizer'),
  (321, 30, 93, 'challenge_organizer'),
  (322, 30, 73, 'challenge_organizer'),
  (323, 30, 170, 'challenge_organizer'),
  (324, 30, 121, 'challenge_organizer'),
  (325, 30, 171, 'challenge_organizer'),
  (326, 30, 121, 'data_contributor'),
  (327, 31, 171, 'challenge_organizer'),
  (328, 31, 96, 'challenge_organizer'),
  (329, 31, 128, 'challenge_organizer'),
  (330, 31, 93, 'challenge_organizer'),
  (331, 31, 190, 'data_contributor'),
  (332, 31, 15, 'sponsor'),
  (333, 32, 87, 'challenge_organizer'),
  (334, 32, 48, 'challenge_organizer'),
  (335, 32, 66, 'challenge_organizer'),
  (336, 32, 128, 'challenge_organizer'),
  (337, 32, 93, 'challenge_organizer'),
  (338, 32, 171, 'challenge_organizer'),
  (339, 32, 182, 'data_contributor'),
  (340, 32, 116, 'data_contributor'),
  (341, 32, 15, 'data_contributor'),
  (342, 32, 31, 'sponsor'),
  (343, 33, 233, 'challenge_organizer'),
  (344, 33, 196, 'challenge_organizer'),
  (345, 33, 128, 'challenge_organizer'),
  (346, 33, 93, 'challenge_organizer'),
  (347, 33, 199, 'challenge_organizer'),
  (348, 33, 179, 'challenge_organizer'),
  (349, 33, 72, 'challenge_organizer'),
  (350, 33, 72, 'data_contributor'),
  (351, 33, 233, 'data_contributor'),
  (352, 33, 117, 'data_contributor'),
  (353, 34, 89, 'challenge_organizer'),
  (354, 34, 226, 'challenge_organizer'),
  (355, 34, 171, 'challenge_organizer'),
  (356, 34, 73, 'challenge_organizer'),
  (357, 34, 170, 'challenge_organizer'),
  (358, 34, 71, 'challenge_organizer'),
  (359, 34, 226, 'data_contributor'),
  (360, 34, 71, 'data_contributor'),
  (361, 35, 171, 'challenge_organizer'),
  (362, 35, 224, 'challenge_organizer'),
  (363, 35, 224, 'data_contributor'),
  (364, 35, 133, 'sponsor'),
  (365, 35, 97, 'sponsor'),
  (366, 35, 224, 'sponsor'),
  (367, 36, 43, 'challenge_organizer'),
  (368, 36, 171, 'challenge_organizer'),
  (369, 36, 234, 'challenge_organizer'),
  (370, 36, 114, 'challenge_organizer'),
  (371, 36, 224, 'challenge_organizer'),
  (372, 36, 16, 'challenge_organizer'),
  (373, 36, 93, 'challenge_organizer'),
  (374, 36, 43, 'data_contributor'),
  (375, 36, 114, 'data_contributor'),
  (376, 36, 234, 'data_contributor'),
  (377, 37, 95, 'challenge_organizer'),
  (378, 37, 113, 'challenge_organizer'),
  (379, 37, 150, 'challenge_organizer'),
  (380, 37, 171, 'challenge_organizer'),
  (381, 37, 179, 'challenge_organizer'),
  (382, 37, 16, 'data_contributor'),
  (383, 37, 234, 'data_contributor'),
  (384, 37, 114, 'data_contributor'),
  (385, 37, 43, 'data_contributor'),
  (386, 37, 171, 'data_contributor'),
  (387, 37, 131, 'sponsor'),
  (388, 38, 171, 'challenge_organizer'),
  (389, 38, 52, 'challenge_organizer'),
  (390, 38, 176, 'challenge_organizer'),
  (391, 38, 15, 'challenge_organizer'),
  (392, 38, 131, 'challenge_organizer'),
  (393, 38, 46, 'challenge_organizer'),
  (394, 38, 128, 'challenge_organizer'),
  (395, 38, 93, 'challenge_organizer'),
  (396, 38, 89, 'challenge_organizer'),
  (397, 38, 73, 'challenge_organizer'),
  (398, 38, 170, 'challenge_organizer'),
  (399, 38, 52, 'data_contributor'),
  (400, 39, 171, 'challenge_organizer'),
  (401, 39, 150, 'challenge_organizer'),
  (402, 39, 128, 'challenge_organizer'),
  (403, 39, 15, 'challenge_organizer'),
  (404, 39, 131, 'challenge_organizer'),
  (405, 39, 46, 'challenge_organizer'),
  (406, 39, 112, 'data_contributor'),
  (407, 39, 150, 'data_contributor'),
  (408, 40, 131, 'challenge_organizer'),
  (409, 40, 171, 'challenge_organizer'),
  (410, 40, 150, 'challenge_organizer'),
  (411, 40, 22, 'data_contributor'),
  (412, 40, 44, 'data_contributor'),
  (413, 40, 131, 'sponsor'),
  (414, 41, 192, 'challenge_organizer'),
  (415, 41, 128, 'challenge_organizer'),
  (416, 41, 93, 'challenge_organizer'),
  (417, 41, 199, 'challenge_organizer'),
  (418, 41, 171, 'challenge_organizer'),
  (419, 41, 192, 'data_contributor'),
  (420, 41, 133, 'sponsor'),
  (421, 41, 40, 'sponsor'),
  (422, 42, 171, 'challenge_organizer'),
  (423, 42, 144, 'challenge_organizer'),
  (424, 42, 126, 'challenge_organizer'),
  (425, 42, 76, 'challenge_organizer'),
  (426, 42, 126, 'data_contributor'),
  (427, 42, 144, 'data_contributor'),
  (428, 42, 216, 'data_contributor'),
  (429, 42, 191, 'data_contributor'),
  (430, 42, 198, 'data_contributor'),
  (431, 42, 161, 'data_contributor'),
  (432, 43, 171, 'challenge_organizer'),
  (433, 43, 52, 'challenge_organizer'),
  (434, 43, 176, 'challenge_organizer'),
  (435, 43, 15, 'challenge_organizer'),
  (436, 43, 46, 'challenge_organizer'),
  (437, 43, 128, 'challenge_organizer'),
  (438, 43, 89, 'challenge_organizer'),
  (439, 43, 52, 'data_contributor'),
  (440, 43, 46, 'sponsor'),
  (441, 44, 171, 'challenge_organizer'),
  (442, 44, 224, 'challenge_organizer'),
  (443, 44, 224, 'data_contributor'),
  (444, 45, 211, 'challenge_organizer'),
  (445, 45, 40, 'challenge_organizer'),
  (446, 45, 148, 'challenge_organizer'),
  (447, 45, 171, 'challenge_organizer'),
  (448, 45, 40, 'data_contributor'),
  (449, 46, 215, 'challenge_organizer'),
  (450, 46, 131, 'challenge_organizer'),
  (451, 46, 196, 'data_contributor'),
  (452, 46, 220, 'data_contributor'),
  (453, 46, 171, 'challenge_organizer'),
  (454, 46, 185, 'data_contributor'),
  (455, 46, 61, 'data_contributor'),
  (456, 46, 226, 'challenge_organizer'),
  (457, 46, 148, 'data_contributor'),
  (458, 47, 47, 'challenge_organizer'),
  (459, 47, 171, 'challenge_organizer'),
  (460, 47, 158, 'challenge_organizer'),
  (461, 47, 124, 'challenge_organizer'),
  (462, 47, 58, 'challenge_organizer'),
  (463, 47, 160, 'challenge_organizer'),
  (464, 47, 47, 'sponsor'),
  (465, 59, 30, 'data_contributor'),
  (466, 59, 224, 'data_contributor'),
  (467, 59, 196, 'data_contributor'),
  (468, 61, 64, 'data_contributor'),
  (469, 62, 173, 'data_contributor'),
  (470, 63, 224, 'data_contributor'),
  (471, 64, 184, 'data_contributor'),
  (472, 65, 33, 'data_contributor'),
  (473, 66, 196, 'data_contributor'),
  (474, 67, 159, 'data_contributor'),
  (475, 67, 70, 'data_contributor'),
  (476, 68, 42, 'data_contributor'),
  (477, 69, 200, 'data_contributor'),
  (478, 70, 91, 'data_contributor'),
  (479, 71, 214, 'data_contributor'),
  (480, 72, 179, 'data_contributor'),
  (481, 73, 41, 'data_contributor'),
  (482, 74, 213, 'data_contributor'),
  (483, 75, 91, 'data_contributor'),
  (484, 76, 88, 'data_contributor'),
  (485, 77, 220, 'data_contributor'),
  (486, 78, 222, 'data_contributor'),
  (487, 79, 217, 'data_contributor'),
  (488, 80, 221, 'data_contributor'),
  (489, 81, 173, 'data_contributor'),
  (490, 82, 173, 'data_contributor'),
  (491, 83, 220, 'data_contributor'),
  (492, 84, 218, 'data_contributor'),
  (493, 85, 41, 'data_contributor'),
  (494, 90, 104, 'challenge_organizer'),
  (495, 90, 3, 'challenge_organizer'),
  (496, 90, 102, 'challenge_organizer'),
  (497, 90, 215, 'challenge_organizer'),
  (498, 90, 143, 'challenge_organizer'),
  (499, 90, 224, 'challenge_organizer'),
  (500, 90, 203, 'challenge_organizer'),
  (501, 90, 219, 'challenge_organizer'),
  (502, 90, 137, 'sponsor'),
  (503, 90, 15, 'sponsor'),
  (504, 90, 60, 'sponsor'),
  (505, 90, 68, 'sponsor'),
  (506, 91, 209, 'challenge_organizer'),
  (507, 91, 183, 'challenge_organizer'),
  (508, 91, 194, 'challenge_organizer'),
  (509, 91, 207, 'challenge_organizer'),
  (510, 91, 135, 'sponsor'),
  (511, 92, 209, 'challenge_organizer'),
  (512, 92, 183, 'challenge_organizer'),
  (513, 92, 194, 'challenge_organizer'),
  (514, 92, 207, 'challenge_organizer'),
  (515, 92, 135, 'sponsor'),
  (516, 93, 189, 'sponsor'),
  (517, 94, 13, 'challenge_organizer'),
  (518, 94, 189, 'challenge_organizer'),
  (519, 94, 34, 'challenge_organizer'),
  (520, 94, 62, 'challenge_organizer'),
  (521, 94, 228, 'challenge_organizer'),
  (522, 94, 136, 'challenge_organizer'),
  (523, 94, 189, 'sponsor'),
  (524, 95, 13, 'challenge_organizer'),
  (525, 95, 189, 'challenge_organizer'),
  (526, 95, 152, 'challenge_organizer'),
  (527, 95, 201, 'challenge_organizer'),
  (528, 95, 128, 'challenge_organizer'),
  (529, 95, 93, 'challenge_organizer'),
  (530, 95, 62, 'challenge_organizer'),
  (531, 95, 34, 'challenge_organizer'),
  (532, 95, 189, 'sponsor'),
  (533, 96, 13, 'challenge_organizer'),
  (534, 96, 189, 'challenge_organizer'),
  (535, 96, 82, 'challenge_organizer'),
  (536, 96, 62, 'challenge_organizer'),
  (537, 96, 34, 'challenge_organizer'),
  (538, 96, 189, 'sponsor'),
  (539, 97, 13, 'challenge_organizer'),
  (540, 97, 189, 'challenge_organizer'),
  (541, 97, 83, 'challenge_organizer'),
  (542, 97, 34, 'challenge_organizer'),
  (543, 97, 189, 'sponsor'),
  (544, 98, 13, 'challenge_organizer'),
  (545, 98, 189, 'challenge_organizer'),
  (546, 98, 62, 'challenge_organizer'),
  (547, 98, 53, 'challenge_organizer'),
  (548, 98, 34, 'challenge_organizer'),
  (549, 98, 189, 'sponsor'),
  (550, 99, 136, 'challenge_organizer'),
  (551, 99, 13, 'challenge_organizer'),
  (552, 99, 189, 'challenge_organizer'),
  (553, 99, 62, 'challenge_organizer'),
  (554, 99, 34, 'challenge_organizer'),
  (555, 99, 189, 'sponsor'),
  (556, 100, 13, 'challenge_organizer'),
  (557, 100, 189, 'challenge_organizer'),
  (558, 100, 231, 'challenge_organizer'),
  (559, 100, 34, 'challenge_organizer'),
  (560, 100, 62, 'challenge_organizer'),
  (561, 100, 189, 'sponsor'),
  (562, 101, 13, 'challenge_organizer'),
  (563, 101, 189, 'challenge_organizer'),
  (564, 101, 34, 'challenge_organizer'),
  (565, 101, 62, 'challenge_organizer'),
  (566, 101, 189, 'sponsor'),
  (567, 102, 13, 'challenge_organizer'),
  (568, 102, 189, 'challenge_organizer'),
  (569, 102, 34, 'challenge_organizer'),
  (570, 102, 62, 'challenge_organizer'),
  (571, 102, 189, 'sponsor'),
  (572, 103, 13, 'challenge_organizer'),
  (573, 103, 189, 'challenge_organizer'),
  (574, 103, 34, 'challenge_organizer'),
  (575, 103, 62, 'challenge_organizer'),
  (576, 103, 131, 'challenge_organizer'),
  (577, 103, 177, 'challenge_organizer'),
  (578, 103, 189, 'sponsor'),
  (579, 104, 12, 'challenge_organizer'),
  (580, 104, 215, 'challenge_organizer'),
  (581, 104, 84, 'challenge_organizer'),
  (582, 104, 7, 'challenge_organizer'),
  (583, 104, 15, 'sponsor'),
  (584, 104, 90, 'sponsor'),
  (585, 104, 103, 'sponsor'),
  (586, 104, 50, 'sponsor'),
  (587, 105, 12, 'challenge_organizer'),
  (588, 105, 147, 'sponsor'),
  (589, 105, 35, 'sponsor'),
  (590, 106, 12, 'challenge_organizer'),
  (591, 106, 186, 'sponsor'),
  (592, 106, 178, 'sponsor'),
  (593, 107, 12, 'challenge_organizer'),
  (594, 107, 77, 'sponsor'),
  (595, 124, 12, 'challenge_organizer'),
  (596, 124, 215, 'challenge_organizer'),
  (597, 124, 84, 'challenge_organizer'),
  (598, 124, 7, 'challenge_organizer'),
  (599, 124, 98, 'sponsor'),
  (600, 124, 139, 'sponsor'),
  (601, 124, 162, 'sponsor'),
  (602, 161, 13, 'challenge_organizer'),
  (603, 161, 237, 'challenge_organizer'),
  (604, 162, 13, 'challenge_organizer'),
  (605, 162, 238, 'challenge_organizer'),
  (606, 163, 13, 'challenge_organizer'),
  (607, 163, 238, 'challenge_organizer'),
  (608, 164, 13, 'challenge_organizer'),
  (609, 164, 231, 'challenge_organizer'),
  (610, 165, 13, 'challenge_organizer'),
  (611, 55, 1, 'sponsor'),
  (612, 53, 1, 'sponsor'),
  (613, 52, 1, 'sponsor'),
  (614, 54, 1, 'sponsor'),
  (615, 166, 1, 'sponsor'),
  (616, 45, 1, 'sponsor'),
  (617, 44, 1, 'sponsor'),
  (618, 43, 1, 'sponsor'),
  (619, 40, 1, 'sponsor'),
  (620, 42, 1, 'sponsor'),
  (621, 39, 1, 'sponsor'),
  (622, 41, 1, 'sponsor'),
  (623, 35, 1, 'sponsor'),
  (624, 36, 1, 'sponsor'),
  (625, 34, 1, 'sponsor'),
  (626, 37, 1, 'sponsor'),
  (627, 33, 1, 'sponsor'),
  (628, 32, 1, 'sponsor'),
  (629, 31, 1, 'sponsor'),
  (630, 30, 1, 'sponsor'),
  (631, 29, 1, 'sponsor'),
  (632, 21, 1, 'sponsor'),
  (633, 27, 1, 'sponsor'),
  (634, 26, 1, 'sponsor'),
  (635, 28, 1, 'sponsor'),
  (636, 24, 1, 'sponsor'),
  (637, 25, 1, 'sponsor'),
  (638, 23, 1, 'sponsor'),
  (639, 20, 1, 'sponsor'),
  (640, 19, 1, 'sponsor'),
  (641, 18, 1, 'sponsor'),
  (642, 17, 1, 'sponsor'),
  (643, 16, 1, 'sponsor'),
  (644, 14, 1, 'sponsor'),
  (645, 8, 1, 'sponsor'),
  (646, 7, 1, 'sponsor'),
  (647, 11, 1, 'sponsor'),
  (648, 12, 1, 'sponsor'),
  (649, 10, 1, 'sponsor'),
  (650, 9, 1, 'sponsor'),
  (651, 6, 1, 'sponsor'),
  (652, 5, 1, 'sponsor'),
  (653, 4, 1, 'sponsor'),
  (654, 1, 1, 'sponsor'),
  (655, 3, 1, 'sponsor'),
  (656, 167, 215, 'challenge_organizer'),
  (657, 167, 171, 'challenge_organizer'),
  (658, 167, 239, 'challenge_organizer'),
  (659, 167, 240, 'challenge_organizer'),
  (660, 167, 195, 'challenge_organizer'),
  (661, 167, 226, 'challenge_organizer'),
  (662, 167, 241, 'challenge_organizer'),
  (663, 167, 242, 'challenge_organizer'),
  (664, 167, 185, 'challenge_organizer'),
  (665, 167, 196, 'challenge_organizer'),
  (666, 167, 243, 'challenge_organizer'),
  (667, 167, 244, 'challenge_organizer'),
  (668, 167, 245, 'challenge_organizer'),
  (669, 167, 246, 'challenge_organizer'),
  (670, 167, 247, 'challenge_organizer'),
  (671, 167, 248, 'challenge_organizer'),
  (672, 167, 13, 'challenge_organizer'),
  (673, 167, 249, 'challenge_organizer'),
  (674, 167, 34, 'challenge_organizer'),
  (675, 167, 62, 'challenge_organizer'),
  (676, 167, 215, 'data_contributor'),
  (677, 167, 240, 'data_contributor'),
  (678, 167, 250, 'data_contributor'),
  (679, 167, 251, 'data_contributor'),
  (680, 167, 241, 'data_contributor'),
  (681, 167, 185, 'data_contributor'),
  (682, 167, 243, 'data_contributor'),
  (683, 167, 245, 'data_contributor'),
  (684, 167, 49, 'data_contributor'),
  (685, 167, 252, 'data_contributor'),
  (686, 167, 253, 'data_contributor'),
  (687, 167, 98, 'sponsor'),
  (688, 167, 139, 'sponsor'),
  (689, 167, 254, 'sponsor'),
  (690, 167, 255, 'sponsor'),
  (691, 169, 209, 'challenge_organizer'),
  (692, 169, 195, 'challenge_organizer'),
  (693, 169, 194, 'challenge_organizer'),
  (694, 169, 257, 'challenge_organizer'),
  (695, 169, 135, 'sponsor'),
  (696, 170, 258, 'challenge_organizer'),
  (697, 170, 258, 'data_contributor'),
  (698, 170, 161, 'challenge_organizer'),
  (699, 170, 161, 'data_contributor'),
  (700, 170, 259, 'challenge_organizer'),
  (701, 170, 260, 'challenge_organizer'),
  (702, 170, 261, 'challenge_organizer'),
  (703, 170, 261, 'data_contributor'),
  (704, 170, 262, 'challenge_organizer'),
  (705, 170, 263, 'challenge_organizer'),
  (706, 170, 264, 'challenge_organizer'),
  (707, 170, 265, 'sponsor'),
  (708, 170, 260, 'sponsor'),
  (709, 170, 266, 'sponsor'),
  (710, 170, 261, 'sponsor'),
  (711, 171, 267, 'challenge_organizer'),
  (712, 171, 267, 'sponsor'),
  (713, 171, 267, 'data_contributor'),
  (714, 172, 269, 'sponsor'),
  (715, 172, 268, 'challenge_organizer'),
  (716, 173, 272, 'challenge_organizer'),
  (717, 173, 271, 'challenge_organizer'),
  (718, 173, 270, 'challenge_organizer'),
  (719, 174, 273, 'sponsor'),
  (720, 174, 107, 'challenge_organizer'),
  (721, 174, 256, 'challenge_organizer'),
  (722, 174, 270, 'challenge_organizer'),
  (723, 174, 147, 'challenge_organizer'),
  (724, 174, 215, 'challenge_organizer'),
  (725, 174, 270, 'data_contributor'),
  (726, 176, 104, 'challenge_organizer'),
  (727, 176, 143, 'challenge_organizer'),
  (728, 176, 213, 'challenge_organizer'),
  (729, 176, 275, 'challenge_organizer'),
  (730, 176, 102, 'challenge_organizer'),
  (731, 174, 274, 'data_contributor'),
  (732, 174, 276, 'data_contributor'),
  (733, 174, 277, 'data_contributor'),
  (734, 177, 162, 'challenge_organizer'),
  (735, 176, 3, 'challenge_organizer'),
  (736, 175, 278, 'challenge_organizer'),
  (737, 179, 279, 'challenge_organizer'),
  (738, 174, 12, 'sponsor'),
  (739, 172, 12, 'sponsor'),
  (740, 173, 12, 'sponsor'),
  (741, 171, 12, 'sponsor'),
  (742, 167, 12, 'sponsor'),
  (743, 181, 12, 'sponsor'),
  (744, 181, 280, 'challenge_organizer'),
  (745, 181, 281, 'challenge_organizer'),
  (746, 181, 211, 'challenge_organizer'),
  (747, 181, 282, 'challenge_organizer'),
  (748, 181, 171, 'challenge_organizer'),
  (749, 182, 283, 'challenge_organizer'),
  (750, 182, 284, 'challenge_organizer'),
  (751, 182, 285, 'challenge_organizer'),
  (752, 182, 67, 'challenge_organizer'),
  (753, 182, 286, 'challenge_organizer'),
  (754, 182, 12, 'sponsor'),
  (755, 182, 15, 'sponsor'),
  (756, 183, 283, 'challenge_organizer'),
  (757, 183, 67, 'challenge_organizer'),
  (758, 183, 287, 'challenge_organizer'),
  (759, 183, 286, 'challenge_organizer'),
  (760, 183, 288, 'challenge_organizer'),
  (761, 183, 289, 'challenge_organizer'),
  (762, 183, 12, 'sponsor'),
  (763, 183, 15, 'sponsor'),
  (764, 184, 290, 'challenge_organizer'),
  (765, 184, 291, 'challenge_organizer'),
  (766, 185, 179, 'challenge_organizer'),
  (767, 186, 179, 'challenge_organizer'),
  (768, 186, 292, 'challenge_organizer'),
  (769, 187, 179, 'challenge_organizer'),
  (770, 187, 119, 'challenge_organizer'),
  (771, 188, 179, 'challenge_organizer'),
  (772, 188, 293, 'challenge_organizer'),
  (773, 189, 179, 'challenge_organizer'),
  (774, 190, 197, 'challenge_organizer'),
  (775, 191, 179, 'challenge_organizer'),
  (776, 191, 294, 'challenge_organizer'),
  (777, 191, 144, 'challenge_organizer'),
  (778, 191, 295, 'challenge_organizer'),
  (779, 192, 41, 'challenge_organizer'),
  (780, 192, 296, 'challenge_organizer'),
  (781, 192, 297, 'challenge_organizer'),
  (782, 193, 298, 'challenge_organizer'),
  (783, 193, 299, 'sponsor'),
  (784, 193, 300, 'sponsor'),
  (785, 193, 147, 'sponsor'),
  (786, 193, 301, 'sponsor'),
  (787, 194, 302, 'challenge_organizer'),
  (788, 194, 302, 'data_contributor'),
  (789, 194, 303, 'data_contributor'),
  (790, 194, 304, 'data_contributor'),
  (791, 194, 126, 'sponsor'),
  (792, 195, 75, 'sponsor'),
  (793, 195, 312, 'challenge_organizer'),
  (794, 196, 12, 'sponsor'),
  (795, 196, 226, 'challenge_organizer'),
  (796, 196, 306, 'challenge_organizer'),
  (797, 196, 307, 'challenge_organizer'),
  (798, 196, 308, 'challenge_organizer'),
  (799, 196, 248, 'challenge_organizer'),
  (800, 196, 305, 'challenge_organizer'),
  (801, 196, 309, 'challenge_organizer'),
  (802, 196, 310, 'challenge_organizer'),
  (803, 196, 220, 'challenge_organizer'),
  (804, 196, 261, 'data_contributor'),
  (805, 196, 311, 'data_contributor'),
  (806, 197, 261, 'challenge_organizer'),
  (807, 197, 261, 'data_contributor'),
  (808, 197, 12, 'sponsor'),
  (809, 196, 305, 'data_contributor'),
  (810, 198, 13, 'challenge_organizer'),
  (811, 198, 313, 'challenge_organizer'),
  (812, 198, 314, 'challenge_organizer'),
  (813, 198, 315, 'challenge_organizer'),
  (814, 199, 231, 'challenge_organizer'),
  (815, 199, 249, 'challenge_organizer'),
  (816, 199, 313, 'challenge_organizer'),
  (817, 199, 315, 'challenge_organizer'),
  (818, 199, 136, 'challenge_organizer'),
  (819, 199, 316, 'challenge_organizer'),
  (820, 199, 34, 'challenge_organizer'),
  (821, 199, 317, 'challenge_organizer'),
  (822, 199, 318, 'challenge_organizer'),
  (823, 206, 171, 'challenge_organizer'),
  (824, 207, 171, 'challenge_organizer'),
  (825, 208, 171, 'challenge_organizer'),
  (826, 209, 171, 'challenge_organizer'),
  (827, 210, 171, 'challenge_organizer'),
  (828, 206, 319, 'data_contributor'),
  (829, 207, 319, 'data_contributor'),
  (830, 208, 319, 'data_contributor'),
  (831, 209, 319, 'data_contributor'),
  (832, 210, 319, 'data_contributor'),
  (833, 206, 247, 'data_contributor'),
  (834, 207, 247, 'data_contributor'),
  (835, 208, 247, 'data_contributor'),
  (836, 209, 247, 'data_contributor'),
  (837, 210, 247, 'data_contributor'),
  (838, 211, 1, 'sponsor'),
  (839, 212, 1, 'sponsor'),
  (840, 213, 1, 'sponsor'),
  (841, 214, 1, 'sponsor'),
  (842, 215, 1, 'sponsor'),
  (843, 216, 1, 'sponsor'),
  (844, 217, 1, 'sponsor'),
  (845, 218, 1, 'sponsor'),
  (846, 219, 1, 'sponsor'),
  (847, 220, 1, 'sponsor'),
  (848, 221, 1, 'sponsor'),
  (849, 222, 1, 'sponsor'),
  (850, 223, 1, 'sponsor'),
  (851, 224, 1, 'sponsor'),
  (852, 225, 1, 'sponsor'),
  (853, 226, 1, 'sponsor'),
  (854, 202, 1, 'sponsor'),
  (855, 203, 1, 'sponsor'),
  (856, 204, 1, 'sponsor'),
  (857, 205, 1, 'sponsor'),
  (858, 211, 52, 'data_contributor'),
  (859, 212, 58, 'data_contributor'),
  (860, 213, 320, 'data_contributor'),
  (861, 214, 321, 'data_contributor'),
  (862, 215, 124, 'data_contributor'),
  (863, 216, 322, 'data_contributor'),
  (864, 217, 256, 'data_contributor'),
  (865, 217, 119, 'data_contributor'),
  (866, 217, 93, 'challenge_organizer'),
  (867, 218, 256, 'data_contributor'),
  (868, 218, 119, 'data_contributor'),
  (869, 218, 93, 'challenge_organizer'),
  (870, 218, 256, 'challenge_organizer'),
  (871, 218, 119, 'challenge_organizer'),
  (872, 219, 220, 'data_contributor'),
  (873, 219, 242, 'data_contributor'),
  (874, 219, 93, 'challenge_organizer'),
  (875, 220, 220, 'data_contributor'),
  (876, 220, 93, 'challenge_organizer'),
  (877, 220, 256, 'challenge_organizer'),
  (878, 220, 119, 'challenge_organizer'),
  (879, 221, 93, 'challenge_organizer'),
  (880, 221, 256, 'challenge_organizer'),
  (881, 221, 119, 'challenge_organizer'),
  (882, 221, 323, 'challenge_organizer'),
  (883, 221, 323, 'data_contributor'),
  (884, 222, 234, 'data_contributor'),
  (885, 222, 93, 'challenge_organizer'),
  (886, 223, 93, 'challenge_organizer'),
  (887, 223, 256, 'challenge_organizer'),
  (888, 223, 119, 'challenge_organizer'),
  (889, 224, 93, 'challenge_organizer'),
  (890, 225, 324, 'challenge_organizer'),
  (891, 225, 93, 'challenge_organizer'),
  (892, 226, 93, 'challenge_organizer'),
  (893, 226, 242, 'challenge_organizer'),
  (894, 226, 93, 'data_contributor'),
  (895, 226, 242, 'data_contributor'),
  (896, 228, 325, 'challenge_organizer'),
  (897, 229, 325, 'challenge_organizer'),
  (898, 227, 326, 'challenge_organizer'),
  (899, 230, 326, 'challenge_organizer'),
  (900, 2, 1, 'sponsor'),
  (901, 38, 1, 'sponsor'),
  (902, 49, 1, 'sponsor'),
  (903, 58, 1, 'sponsor');

-- challenge_incentive data

INSERT INTO challenge_incentive (id, name, challenge_id)
VALUES (1, 'publication', 1),
  (2, 'publication', 2),
  (3, 'speaking_engagement', 2),
  (4, 'monetary', 3),
  (5, 'publication', 5),
  (6, 'speaking_engagement', 5),
  (7, 'publication', 6),
  (8, 'speaking_engagement', 6),
  (9, 'monetary', 7),
  (10, 'publication', 7),
  (11, 'publication', 8),
  (12, 'speaking_engagement', 8),
  (13, 'publication', 9),
  (14, 'publication', 10),
  (15, 'speaking_engagement', 10),
  (16, 'publication', 11),
  (17, 'speaking_engagement', 11),
  (18, 'publication', 12),
  (19, 'speaking_engagement', 12),
  (20, 'monetary', 14),
  (21, 'publication', 14),
  (22, 'speaking_engagement', 14),
  (23, 'monetary', 15),
  (24, 'publication', 15),
  (25, 'speaking_engagement', 15),
  (26, 'publication', 16),
  (27, 'speaking_engagement', 16),
  (28, 'publication', 17),
  (29, 'speaking_engagement', 17),
  (30, 'monetary', 24),
  (31, 'publication', 24),
  (32, 'speaking_engagement', 24),
  (33, 'monetary', 25),
  (34, 'publication', 25),
  (35, 'speaking_engagement', 25),
  (36, 'publication', 26),
  (37, 'monetary', 27),
  (38, 'publication', 27),
  (39, 'publication', 28),
  (40, 'publication', 29),
  (41, 'publication', 30),
  (42, 'publication', 31),
  (43, 'speaking_engagement', 31),
  (44, 'publication', 34),
  (45, 'speaking_engagement', 34),
  (46, 'publication', 37),
  (47, 'speaking_engagement', 37),
  (48, 'publication', 38),
  (49, 'speaking_engagement', 38),
  (50, 'monetary', 46),
  (51, 'publication', 100),
  (52, 'speaking_engagement', 100),
  (53, 'publication', 101),
  (54, 'speaking_engagement', 101),
  (55, 'monetary', 111),
  (56, 'monetary', 113),
  (57, 'monetary', 114),
  (58, 'monetary', 118),
  (59, 'monetary', 123),
  (60, 'monetary', 124),
  (61, 'monetary', 129),
  (62, 'monetary', 131),
  (63, 'monetary', 145),
  (64, 'monetary', 146),
  (65, 'monetary', 147),
  (66, 'monetary', 148),
  (67, 'monetary', 149),
  (68, 'monetary', 156),
  (69, 'publication', 157),
  (70, 'other', 157),
  (71, 'publication', 158),
  (72, 'speaking_engagement', 158),
  (73, 'publication', 159),
  (74, 'speaking_engagement', 159),
  (75, 'publication', 160),
  (76, 'speaking_engagement', 160),
  (77, 'publication', 161),
  (78, 'other', 161),
  (79, 'publication', 162),
  (80, 'speaking_engagement', 162),
  (81, 'publication', 163),
  (82, 'speaking_engagement', 163),
  (83, 'publication', 165),
  (84, 'speaking_engagement', 166),
  (85, 'other', 166),
  (86, 'publication', 168),
  (87, 'publication', 169),
  (88, 'speaking_engagement', 169),
  (89, 'monetary', 171),
  (90, 'speaking_engagement', 171),
  (91, 'publication', 173),
  (92, 'monetary', 175),
  (93, 'monetary', 176),
  (94, 'monetary', 177),
  (95, 'speaking_engagement', 177),
  (96, 'monetary', 178),
  (97, 'monetary', 179),
  (98, 'monetary', 180),
  (99, 'speaking_engagement', 180),
  (100, 'monetary', 182),
  (101, 'speaking_engagement', 182),
  (102, 'monetary', 183),
  (103, 'monetary', 184),
  (104, 'monetary', 185),
  (105, 'publication', 186),
  (106, 'publication', 187),
  (107, 'publication', 188),
  (108, 'publication', 189),
  (109, 'publication', 190),
  (110, 'publication', 191),
  (111, 'monetary', 192),
  (112, 'monetary', 193),
  (113, 'speaking_engagement', 193),
  (114, 'other', 193),
  (115, 'monetary', 194),
  (116, 'speaking_engagement', 194),
  (117, 'monetary', 195),
  (118, 'publication', 195),
  (119, 'publication', 196),
  (120, 'speaking_engagement', 196),
  (121, 'other', 196),
  (122, 'publication', 197),
  (123, 'speaking_engagement', 197),
  (124, 'other', 198),
  (125, 'publication', 199),
  (126, 'speaking_engagement', 199),
  (127, 'monetary,publication', 200),
  (128, 'speaking_engagement', 200),
  (129, 'monetary', 201),
  (130, 'publication', 201),
  (131, 'speaking_engagement', 201),
  (132, 'publication', 202),
  (133, 'speaking_engagement', 202),
  (134, 'publication', 203),
  (135, 'speaking_engagement', 203),
  (136, 'publication', 204),
  (137, 'speaking_engagement', 204),
  (138, 'publication', 205),
  (139, 'speaking_engagement', 205),
  (140, 'publication', 206),
  (141, 'publication', 207),
  (142, 'publication', 208),
  (143, 'publication', 209),
  (144, 'publication', 210),
  (145, 'monetary', 227),
  (146, 'publication', 227),
  (147, 'monetary', 228),
  (148, 'publication', 228),
  (149, 'monetary', 229),
  (150, 'publication', 229),
  (151, 'monetary', 230),
  (152, 'publication', 230);

-- challenge_submission_type data

INSERT INTO challenge_submission_type (id, name, challenge_id)
VALUES (1, 'prediction_file', 1),
  (2, 'prediction_file', 2),
  (3, 'prediction_file', 3),
  (4, 'prediction_file', 4),
  (5, 'prediction_file', 5),
  (6, 'prediction_file', 6),
  (7, 'prediction_file', 7),
  (8, 'prediction_file', 8),
  (9, 'prediction_file', 9),
  (10, 'prediction_file', 10),
  (11, 'prediction_file', 11),
  (12, 'prediction_file', 12),
  (13, 'prediction_file', 13),
  (14, 'prediction_file', 14),
  (15, 'prediction_file', 15),
  (16, 'prediction_file', 16),
  (17, 'prediction_file', 17),
  (18, 'prediction_file', 18),
  (19, 'prediction_file', 19),
  (20, 'prediction_file', 20),
  (21, 'prediction_file', 21),
  (22, 'other', 22),
  (23, 'prediction_file', 23),
  (24, 'container_image', 24),
  (25, 'container_image', 25),
  (26, 'prediction_file', 26),
  (27, 'prediction_file', 27),
  (28, 'container_image', 28),
  (29, 'prediction_file', 29),
  (30, 'prediction_file', 30),
  (31, 'prediction_file', 31),
  (32, 'prediction_file', 32),
  (33, 'prediction_file', 33),
  (34, 'prediction_file', 34),
  (35, 'container_image', 35),
  (36, 'prediction_file', 36),
  (37, 'container_image', 37),
  (38, 'prediction_file', 38),
  (39, 'container_image', 39),
  (40, 'container_image', 40),
  (41, 'container_image', 41),
  (42, 'prediction_file', 42),
  (43, 'prediction_file', 43),
  (44, 'container_image', 44),
  (45, 'container_image', 45),
  (46, 'prediction_file', 46),
  (47, 'container_image', 46),
  (48, 'container_image', 47),
  (49, 'other', 48),
  (50, 'prediction_file', 49),
  (51, 'container_image', 49),
  (52, 'prediction_file', 50),
  (53, 'container_image', 50),
  (54, 'prediction_file', 51),
  (55, 'container_image', 52),
  (56, 'container_image', 53),
  (57, 'container_image', 54),
  (58, 'container_image', 55),
  (59, 'other', 56),
  (60, 'prediction_file', 57),
  (61, 'container_image', 58),
  (62, 'prediction_file', 59),
  (63, 'prediction_file', 60),
  (64, 'prediction_file', 61),
  (65, 'prediction_file', 62),
  (66, 'prediction_file', 63),
  (67, 'prediction_file', 64),
  (68, 'prediction_file', 65),
  (69, 'prediction_file', 66),
  (70, 'prediction_file', 67),
  (71, 'prediction_file', 68),
  (72, 'prediction_file', 69),
  (73, 'prediction_file', 70),
  (74, 'prediction_file', 71),
  (75, 'prediction_file', 72),
  (76, 'prediction_file', 73),
  (77, 'prediction_file', 74),
  (78, 'prediction_file', 75),
  (79, 'prediction_file', 76),
  (80, 'prediction_file', 77),
  (81, 'prediction_file', 78),
  (82, 'prediction_file', 79),
  (83, 'prediction_file', 80),
  (84, 'prediction_file', 81),
  (85, 'prediction_file', 82),
  (86, 'prediction_file', 83),
  (87, 'prediction_file', 84),
  (88, 'prediction_file', 85),
  (89, 'prediction_file', 86),
  (90, 'container_image', 86),
  (91, 'other', 87),
  (92, 'other', 88),
  (93, 'other', 89),
  (94, 'prediction_file', 90),
  (95, 'prediction_file', 91),
  (96, 'prediction_file', 92),
  (97, 'prediction_file', 93),
  (98, 'prediction_file', 94),
  (99, 'prediction_file', 95),
  (100, 'prediction_file', 96),
  (101, 'prediction_file', 97),
  (102, 'prediction_file', 98),
  (103, 'prediction_file', 99),
  (104, 'prediction_file', 100),
  (105, 'prediction_file', 101),
  (106, 'prediction_file', 102),
  (107, 'prediction_file', 103),
  (108, 'container_image', 104),
  (109, 'prediction_file', 105),
  (110, 'container_image', 105),
  (111, 'prediction_file', 106),
  (112, 'container_image', 107),
  (113, 'notebook', 108),
  (114, 'notebook', 109),
  (115, 'notebook', 110),
  (116, 'notebook', 111),
  (117, 'notebook', 112),
  (118, 'notebook', 113),
  (119, 'notebook', 114),
  (120, 'notebook', 115),
  (121, 'notebook', 116),
  (122, 'notebook', 117),
  (123, 'notebook', 118),
  (124, 'notebook', 119),
  (125, 'notebook', 120),
  (126, 'notebook', 121),
  (127, 'notebook', 122),
  (128, 'notebook', 123),
  (129, 'notebook', 124),
  (130, 'notebook', 125),
  (131, 'notebook', 126),
  (132, 'notebook', 127),
  (133, 'notebook', 128),
  (134, 'notebook', 129),
  (135, 'notebook', 130),
  (136, 'notebook', 131),
  (137, 'notebook', 132),
  (138, 'notebook', 133),
  (139, 'notebook', 134),
  (140, 'notebook', 135),
  (141, 'notebook', 136),
  (142, 'notebook', 137),
  (143, 'notebook', 138),
  (144, 'notebook', 139),
  (145, 'notebook', 140),
  (146, 'notebook', 141),
  (147, 'notebook', 142),
  (148, 'notebook', 143),
  (149, 'notebook', 144),
  (150, 'notebook', 145),
  (151, 'notebook', 146),
  (152, 'notebook', 147),
  (153, 'notebook', 148),
  (154, 'notebook', 149),
  (155, 'notebook', 150),
  (156, 'notebook', 151),
  (157, 'notebook', 152),
  (158, 'notebook', 153),
  (159, 'notebook', 154),
  (160, 'notebook', 155),
  (161, 'notebook', 156),
  (162, 'prediction_file', 157),
  (163, 'other', 158),
  (164, 'other', 159),
  (165, 'other', 160),
  (166, 'prediction_file', 161),
  (167, 'prediction_file', 162),
  (168, 'other', 162),
  (169, 'prediction_file', 163),
  (170, 'other', 163),
  (171, 'prediction_file', 164),
  (172, 'prediction_file', 165),
  (173, 'prediction_file', 166),
  (174, 'other', 166),
  (175, 'prediction_file', 167),
  (176, 'container_image', 167),
  (177, 'prediction_file', 169),
  (178, 'container_image', 170),
  (179, 'prediction_file', 174),
  (180, 'container_image', 174),
  (181, 'notebook', 175),
  (182, 'notebook', 176),
  (183, 'notebook', 177),
  (184, 'notebook', 178),
  (185, 'notebook', 179),
  (186, 'notebook', 180),
  (187, 'prediction_file', 181),
  (188, 'notebook', 182),
  (189, 'notebook', 183),
  (190, 'notebook', 184),
  (191, 'notebook', 185),
  (192, 'other', 186),
  (193, 'other', 187),
  (194, 'other', 188),
  (195, 'other', 189),
  (196, 'other', 190),
  (197, 'other', 191),
  (198, 'notebook', 192),
  (199, 'container_image', 192),
  (200, 'notebook', 193),
  (201, 'notebook', 194),
  (202, 'container_image', 195),
  (203, 'container_image', 196),
  (204, 'container_image', 197),
  (205, 'notebook', 198),
  (206, 'prediction_file', 199),
  (207, 'other', 199),
  (208, 'prediction_file', 200),
  (209, 'prediction_file', 201),
  (210, 'prediction_file', 202),
  (211, 'prediction_file', 203),
  (212, 'prediction_file', 204),
  (213, 'prediction_file', 205),
  (214, 'container_image', 206),
  (215, 'container_image', 207),
  (216, 'container_image', 208),
  (217, 'container_image', 209),
  (218, 'container_image', 210),
  (219, 'prediction_file', 211),
  (220, 'prediction_file', 212),
  (221, 'prediction_file', 213),
  (222, 'prediction_file', 214),
  (223, 'prediction_file', 215),
  (224, 'prediction_file', 216),
  (225, 'prediction_file', 217),
  (226, 'prediction_file', 218),
  (227, 'prediction_file', 219),
  (228, 'prediction_file', 220),
  (229, 'prediction_file', 221),
  (230, 'prediction_file', 222),
  (231, 'prediction_file', 223),
  (232, 'prediction_file', 224),
  (233, 'prediction_file', 225),
  (234, 'prediction_file', 226),
  (235, 'container_image', 227),
  (236, 'notebook', 228),
  (237, 'notebook', 229),
  (238, 'container_image', 230);

-- challenge_star data

INSERT INTO challenge_star (id, challenge_id, user_id)
VALUES (1, 1, 1),
  (2, 2, 1),
  (3, 1, 2);

-- challenge_x_challenge_input_data_type definition

INSERT INTO challenge_x_challenge_input_data_type (id, challenge_id, challenge_input_data_type_id)
VALUES ('1', 1, 1),
  ('2', 2, 1),
  ('3', 1, 2),
  ('4', 4, 4);

-- challenge_category data

INSERT INTO challenge_category (id, challenge_id, category)
VALUES (1, 161, 'featured'),
  (2, 156, 'featured'),
  (3, 58, 'featured');
