{
 "cells": [
  {
   "cell_type": "markdown",
   "id": "34a39b07-385c-49b7-931f-3631a1aff129",
   "metadata": {
    "tags": []
   },
   "source": [
    "# OpenChallenges REST API Example"
   ]
  },
  {
   "cell_type": "markdown",
   "id": "ed72ae29-13e0-4533-8e7e-f13c9b34da1a",
   "metadata": {},
   "source": [
    "## Overview"
   ]
  },
  {
   "cell_type": "markdown",
   "id": "b876938b-ae3a-49ba-b416-b9fd59f88ae7",
   "metadata": {},
   "source": [
    "This notebook shows how to use OpenChallenges (OC) API client for Python to query challenge resources."
   ]
  },
  {
   "cell_type": "markdown",
   "id": "7e95e24d-3f3a-47d0-bb52-35e718eb7ac6",
   "metadata": {},
   "source": [
    "## Requirements"
   ]
  },
  {
   "cell_type": "markdown",
   "id": "cc586a81-e5b4-4021-9c6b-aa8a69ef221c",
   "metadata": {},
   "source": [
    "The OpenChallenges REST API must be already running."
   ]
  },
  {
   "cell_type": "markdown",
   "id": "c90710c2-f053-44ae-a3c2-610eecff9073",
   "metadata": {},
   "source": [
    "## List challenges"
   ]
  },
  {
   "cell_type": "code",
   "execution_count": 1,
   "id": "c41a64a1",
   "metadata": {},
   "outputs": [],
   "source": [
    "import openchallenges_client\n",
    "from pprint import pprint\n",
    "from openchallenges_client.api import challenge_api"
   ]
  },
  {
   "cell_type": "code",
   "execution_count": 2,
   "id": "8fc3ac4c-2ceb-4bbc-bdb8-3bb8be08dfc6",
   "metadata": {},
   "outputs": [],
   "source": [
    "# See configuration.py for a list of all supported configuration parameters.\n",
    "configuration = openchallenges_client.Configuration(\n",
    "    host = \"http://localhost:8000/api/v1\"\n",
    ")"
   ]
  },
  {
   "cell_type": "markdown",
   "id": "2f855989-f0a9-4fa0-a8f8-63edba2fdb38",
   "metadata": {},
   "source": [
    "> **Note** The hostname `openchallenges-apex` is defined in `/etc/hosts`. The dev container provided with this project is responsible for definining the required hostnames. By default, the hostnames are mapped to `127.0.0.1`."
   ]
  },
  {
   "cell_type": "code",
   "execution_count": 4,
   "id": "b2f952f5-9140-4702-8a96-3457ca4df841",
   "metadata": {},
   "outputs": [
    {
     "name": "stdout",
     "output_type": "stream",
     "text": [
      "ChallengesPage(number=0, size=100, total_elements=279, total_pages=3, has_next=True, has_previous=False, challenges=[Challenge(id=279, slug='niddk-central-repository-data-centric-challenge', name='NIDDK Central Repository Data-Centric Challenge', headline='Enhancing NIDDK datasets for future Artificial Intelligence (AI) applications.', description='The National Institute of Diabetes and Digestive and Kidney Diseases (NIDDK) Central Repository (https://repository.niddk.nih.gov/home/) is conducting a Data Centric Challenge aimed at augmenting existing Repository data for future secondary research including data-driven discovery by artificial intelligence (AI) researchers. The NIDDK Central Repository (NIDDK-CR) program strives to increase the utilization and impact of the resources under its guardianship. However, lack of standardization and consistent metadata within and across studies limit the ability of secondary researchers to easily combine datasets from related studies to generate new insights using data science methods. In the fall of 2021, the NIDDK-CR began implementing approaches to augment data quality to improve AI-readiness by making research data FAIR (findable, accessible, interoperable, and reusable) via a small pilot project utilizing Natural Language Processing (NLP) to tag study variables. In 2022, the NIDD...', doi='', status=<ChallengeStatus.ACTIVE: 'active'>, difficulty=<ChallengeDifficulty.INTERMEDIATE: 'intermediate'>, platform=SimpleChallengePlatform(id=14, slug='other', name='Other'), website_url='https://www.challenge.gov/?challenge=niddk-central-repository-data-centric-challenge', avatar_url='', incentives=[<ChallengeIncentive.PUBLICATION: 'publication'>, <ChallengeIncentive.SPEAKING_ENGAGEMENT: 'speaking_engagement'>, <ChallengeIncentive.OTHER: 'other'>], submission_types=[<ChallengeSubmissionType.PREDICTION_FILE: 'prediction_file'>, <ChallengeSubmissionType.NOTEBOOK: 'notebook'>], input_data_types=[], start_date=datetime.date(2023, 9, 20), end_date=datetime.date(2023, 11, 3), starred_count=0, created_at=datetime.datetime(2023, 10, 18, 16, 58, 17, tzinfo=datetime.timezone.utc), updated_at=datetime.datetime(2023, 10, 18, 20, 52, 49, tzinfo=datetime.timezone.utc)), Challenge(id=278, slug='qbi-hackathon', name='QBI hackathon', headline='A 48-hour event connecting the Bay Area developer community with scientists ...', description='The QBI hackathon is a 48-hour event connecting the vibrant Bay Area developer community with the scientists from UCSF, UCB and UCSC, during which we work together on the cutting edge biomedical problems. Advances in computer vision, AI, and machine learning have enabled computers to pick out cat videos, recognize people’s faces from photos, play video games and drive cars. More recently, application of deep neural nets to protein structure prediction completely revolutionized the field. We look forward to seeing how far we can push science ahead when we apply these latest algorithms to biomedically relevant light microscopy, electron microscopy, and proteomics data. If you love FFTs, transformers, language models, topological data processing, or simply writing code, this is your chance to apply your skills to make an impact on global healthcare. Beyond the actual event, we hope to establish a better connection between talented developers and scientists in the Bay Area, so that we...', doi='', status=<ChallengeStatus.UPCOMING: 'upcoming'>, difficulty=<ChallengeDifficulty.INTERMEDIATE: 'intermediate'>, platform=SimpleChallengePlatform(id=14, slug='other', name='Other'), website_url='https://www.eventbrite.com/e/qbi-hackathon-2023-tickets-633794304827?aff=oddtdtcreator', avatar_url='', incentives=[<ChallengeIncentive.PUBLICATION: 'publication'>, <ChallengeIncentive.SPEAKING_ENGAGEMENT: 'speaking_engagement'>, <ChallengeIncentive.OTHER: 'other'>], submission_types=[<ChallengeSubmissionType.PREDICTION_FILE: 'prediction_file'>, <ChallengeSubmissionType.CONTAINER_IMAGE: 'container_image'>], input_data_types=[], start_date=datetime.date(2023, 11, 4), end_date=datetime.date(2023, 11, 5), starred_count=0, created_at=datetime.datetime(2023, 10, 6, 21, 22, 51, tzinfo=datetime.timezone.utc), updated_at=datetime.datetime(2023, 10, 19, 23, 49, 11, tzinfo=datetime.timezone.utc)), Challenge(id=266, slug='the-miccai-2014-machine-learning-challenge', name='The MICCAI 2014 Machine Learning Challenge', headline='Predicting Binary and Continuous Phenotypes from Structural Brain MRI Data', description='Machine learning tools have been increasingly applied to structural brain magnetic resonance imaging (MRI) scans, largely for developing models to predict clinical phenotypes at the individual level. Despite significant methodological developments and novel application domains, there has been little effort to conduct benchmark studies with standardized datasets, which researchers can use to validate new tools, and more importantly conduct an objective comparison with state-of-the-art algorithms. The MICCAI 2014 Machine Learning Challenge (MLC) will take a significant step in this direction, where we will employ four separate, carefully compiled, and curated large-scale (each N > 70) structural brain MRI datasets with accompanying clinically relevant phenotypes. Our goal is to provide a snapshot of the current state of the art in the field of neuroimage-based prediction, and attract machine-learning practitioners to the MICCAI community and the field of medical image computing in g...', doi='', status=<ChallengeStatus.COMPLETED: 'completed'>, difficulty=<ChallengeDifficulty.INTERMEDIATE: 'intermediate'>, platform=SimpleChallengePlatform(id=9, slug='codalab', name='CodaLab'), website_url='https://competitions.codalab.org/competitions/1471', avatar_url='', incentives=[], submission_types=[<ChallengeSubmissionType.PREDICTION_FILE: 'prediction_file'>], input_data_types=[], start_date=datetime.date(2014, 4, 16), end_date=datetime.date(2014, 6, 14), starred_count=0, created_at=datetime.datetime(2023, 9, 28, 23, 36, 12, tzinfo=datetime.timezone.utc), updated_at=datetime.datetime(2023, 10, 19, 23, 31, 50, tzinfo=datetime.timezone.utc)), Challenge(id=265, slug='cometh-benchmark', name='COMETH Benchmark', headline='Quantify tumor heterogeneity-how many cell types are present and in which pr...', description='Successful treatment of cancer is still a challenge and this is partly due to a wide heterogeneity of cancer composition across patient population. Unfortunately, accounting for such heterogeneity is very difficult. Clinical evaluation of tumor heterogeneity often requires the expertise of anatomical pathologists and radiologists.This benchmark is dedicated to the quantification of intra-tumor heterogeneity using appropriate statistical methods on cancer omics data.In particular, it focuses on estimating cell types and proportion in biological samples based on methylation and methylome data sets. The goal is to explore various statistical methods for source separation/deconvolution analysis (Non-negative Matrix Factorization, Surrogate Variable Analysis, Principal component Analysis, Latent Factor Models, ...) using both RNA-seq and methylome data.', doi='', status=<ChallengeStatus.COMPLETED: 'completed'>, difficulty=<ChallengeDifficulty.INTERMEDIATE: 'intermediate'>, platform=SimpleChallengePlatform(id=10, slug='codabench', name='CodaBench'), website_url='https://www.codabench.org/competitions/218/', avatar_url='', incentives=[], submission_types=[<ChallengeSubmissionType.OTHER: 'other'>], input_data_types=[], start_date=datetime.date(2020, 6, 14), end_date=datetime.date(2020, 12, 29), starred_count=0, created_at=datetime.datetime(2023, 9, 28, 23, 25, 52, tzinfo=datetime.timezone.utc), updated_at=datetime.datetime(2023, 10, 10, 19, 47, 14, tzinfo=datetime.timezone.utc)), Challenge(id=259, slug='cagi6-arsa', name='CAGI6: ARSA', headline='Predicting the effect of naturally occurring missense mutations on enzymatic...', description='Metachromatic Leukodystrophy (MLD) is an autosomal recessive, lysosomal-storage disease caused by mutations in Arylsulfatase A (ARSA) and toxic accumulation of sulfatide substrate. Genome sequencing has revealed hundreds of protein-altering, ARSA missense variants, but the functional effect of most variants remains unknown. ARSA enzyme activity using a high-throughput cellular assay was measured for a large set of variants of known significance and variants of unknown significance. The challenge is to predict the fractional enzyme activity of each mutant protein compared to the wildtype protein.', doi='', status=<ChallengeStatus.COMPLETED: 'completed'>, difficulty=<ChallengeDifficulty.INTERMEDIATE: 'intermediate'>, platform=SimpleChallengePlatform(id=1, slug='synapse', name='Synapse'), website_url='https://genomeinterpretation.org/CAGI6-lc-arsa.html', avatar_url='', incentives=[<ChallengeIncentive.PUBLICATION: 'publication'>, <ChallengeIncentive.SPEAKING_ENGAGEMENT: 'speaking_engagement'>], submission_types=[<ChallengeSubmissionType.PREDICTION_FILE: 'prediction_file'>], input_data_types=[], start_date=None, end_date=datetime.date(2022, 11, 16), starred_count=0, created_at=datetime.datetime(2023, 9, 28, 18, 20, 23, tzinfo=datetime.timezone.utc), updated_at=datetime.datetime(2023, 10, 12, 18, 11, 51, tzinfo=datetime.timezone.utc)), Challenge(id=252, slug='cagi3-mr-1', name='CAGI3: Shewanella oneidensis strain MR-1', headline='Shewanella oneidensis strain MR-1 (formerly known as S. putrefaciens) is a m...', description='Predictors are asked to submit predictions on how insertions in the given gene of MR-1 affect the fitness of that gene in a given condition (stressor).', doi='', status=<ChallengeStatus.COMPLETED: 'completed'>, difficulty=<ChallengeDifficulty.INTERMEDIATE: 'intermediate'>, platform=SimpleChallengePlatform(id=2, slug='drupal', name='Drupal'), website_url='https://genomeinterpretation.org/CAGI3-mr-1.html', avatar_url='', incentives=[<ChallengeIncentive.PUBLICATION: 'publication'>, <ChallengeIncentive.SPEAKING_ENGAGEMENT: 'speaking_engagement'>], submission_types=[<ChallengeSubmissionType.PREDICTION_FILE: 'prediction_file'>], input_data_types=[], start_date=None, end_date=datetime.date(2013, 4, 25), starred_count=0, created_at=datetime.datetime(2023, 9, 28, 18, 20, 1, tzinfo=datetime.timezone.utc), updated_at=datetime.datetime(2023, 10, 16, 18, 18, 7, tzinfo=datetime.timezone.utc)), Challenge(id=251, slug='cagi2-mr-1', name='CAGI2: Shewanella oneidensis strain MR-1', headline='Shewanella oneidensis strain MR-1 (formerly known as S. putrefaciens) is a m...', description='Predictors are asked to submit predictions on how insertions in the given gene of MR-1 affect the fitness of that gene in a given condition (stressor).', doi='', status=<ChallengeStatus.COMPLETED: 'completed'>, difficulty=<ChallengeDifficulty.INTERMEDIATE: 'intermediate'>, platform=SimpleChallengePlatform(id=2, slug='drupal', name='Drupal'), website_url='https://genomeinterpretation.org/CAGI2-mr-1.html', avatar_url='', incentives=[<ChallengeIncentive.PUBLICATION: 'publication'>, <ChallengeIncentive.SPEAKING_ENGAGEMENT: 'speaking_engagement'>], submission_types=[<ChallengeSubmissionType.PREDICTION_FILE: 'prediction_file'>], input_data_types=[], start_date=None, end_date=datetime.date(2011, 10, 6), starred_count=0, created_at=datetime.datetime(2023, 9, 28, 18, 19, 55, tzinfo=datetime.timezone.utc), updated_at=datetime.datetime(2023, 10, 16, 18, 32, 21, tzinfo=datetime.timezone.utc)), Challenge(id=224, slug='cagi2-breast-cancer-pkg', name='CAGI2: Breast cancer pharmacogenomics', headline='Cancer tissues are specifically responsive to different drugs. For this expe...', description='Cell-cycle-checkpoint kinase 2 (CHEK2; OMIM #604373) is a protein that plays an important role in the maintenance of genome integrity and in the regulation of the G2/M cell cycle checkpoint. CHEK2 has been shown to interact with other proteins involved in DNA repair processes such as BRCA1 and TP53. These findings render CHEK2 an 23 attractive candidate susceptibility gene for a variety of cancers.', doi='', status=<ChallengeStatus.COMPLETED: 'completed'>, difficulty=<ChallengeDifficulty.INTERMEDIATE: 'intermediate'>, platform=SimpleChallengePlatform(id=2, slug='drupal', name='Drupal'), website_url='https://genomeinterpretation.org/CAGI2-breast-cancer-pkg.html', avatar_url='', incentives=[<ChallengeIncentive.PUBLICATION: 'publication'>, <ChallengeIncentive.SPEAKING_ENGAGEMENT: 'speaking_engagement'>], submission_types=[<ChallengeSubmissionType.PREDICTION_FILE: 'prediction_file'>], input_data_types=[], start_date=None, end_date=datetime.date(2011, 10, 6), starred_count=0, created_at=datetime.datetime(2023, 9, 28, 18, 19, 48, tzinfo=datetime.timezone.utc), updated_at=datetime.datetime(2023, 10, 12, 17, 46, 22, tzinfo=datetime.timezone.utc)), Challenge(id=250, slug='cagi2-nav1-5', name='CAGI2: SCN5A', headline='Predictors are asked to submit predictions on the effect of the mutants on t...', description='The cardiac action potential (AP) is the sum of a number of distinct ionic currents. It can be divided into five phases (phase 0‐4). From pacemaker cells of the SA node the initial depolarizing wave front will spread throughout the cardiomyocytes via gap junctions. If the depolarization is sufficient voltage‐dependent sodium channels (Nav1.5) are activated and allow Na+ influx. This results in a further depolarization of the membrane which will lead to opening of even more Nav channels. This positive feedback mechanism is seen as the rapid upstroke in the initial phase (phase 0) of the action potential. Nav1.5 is encoded by SCN5A and mutations in this gene have been associated with various diseases such as Atrial fibrillation, Long QT syndrome, Cardiac Conduction Defect, Sick Sinus Disease, and Brugada Syndrome (BrS).', doi='', status=<ChallengeStatus.COMPLETED: 'completed'>, difficulty=<ChallengeDifficulty.INTERMEDIATE: 'intermediate'>, platform=SimpleChallengePlatform(id=2, slug='drupal', name='Drupal'), website_url='https://genomeinterpretation.org/CAGI2-nav1.5.html', avatar_url='', incentives=[<ChallengeIncentive.PUBLICATION: 'publication'>, <ChallengeIncentive.SPEAKING_ENGAGEMENT: 'speaking_engagement'>], submission_types=[<ChallengeSubmissionType.PREDICTION_FILE: 'prediction_file'>], input_data_types=[], start_date=None, end_date=datetime.date(2011, 10, 6), starred_count=0, created_at=datetime.datetime(2023, 9, 28, 18, 19, 48, tzinfo=datetime.timezone.utc), updated_at=datetime.datetime(2023, 10, 16, 18, 32, 16, tzinfo=datetime.timezone.utc)), Challenge(id=249, slug='cagi3-risksnps', name='CAGI3: riskSNPs', headline='The goal of these challenges is to investigate the community’s ability to id...', description='The goal of this experiment is to explore current understanding of the molecular level mechanisms underlying associations between SNPs and disease risk, incorporating expertise in each of the known mechanism areas, and as far as possible assigning possible mechanisms for each association locus. The correct mechanisms are unknown, so there can be no ranking of accuracy-that is not the point of the experiment. Rather, the goal is to ascertain which mechanisms appear most relevant, how confidently they can be assigned, and what fraction of loci can currently be assigned plausible mechanisms.', doi='', status=<ChallengeStatus.COMPLETED: 'completed'>, difficulty=<ChallengeDifficulty.INTERMEDIATE: 'intermediate'>, platform=SimpleChallengePlatform(id=2, slug='drupal', name='Drupal'), website_url='https://genomeinterpretation.org/CAGI3-risksnps.html', avatar_url='', incentives=[<ChallengeIncentive.PUBLICATION: 'publication'>, <ChallengeIncentive.SPEAKING_ENGAGEMENT: 'speaking_engagement'>], submission_types=[<ChallengeSubmissionType.PREDICTION_FILE: 'prediction_file'>], input_data_types=[], start_date=None, end_date=datetime.date(2013, 4, 25), starred_count=0, created_at=datetime.datetime(2023, 9, 28, 18, 19, 48, tzinfo=datetime.timezone.utc), updated_at=datetime.datetime(2023, 10, 19, 23, 33, 13, tzinfo=datetime.timezone.utc)), Challenge(id=225, slug='cagi4-2eqtl', name='CAGI4: eQTL causal SNPs', headline='Participants are asked to submit predictions of the regulatory sequences tha...', description='Identifying the causal alleles responsible for variation in expression of human genes has been particularly difficult. This is an important problem, as genome-wide association studies (GWAS) suggest that much of the variation underlying common traits and diseases maps within regions of the genome that do not encode protein. A massively parallel reporter assay (MPRA) has been applied to thousands of single nucleotide polymorphisms (SNPs) and small insertion/deletion polymorphisms in linkage disequilibrium (LD) with cis-expression quantitative trait loci (eQTLs). The results identify variants showing differential expression between alleles. The challenge is to identify the regulatory sequences and the expression-modulating variants (emVars) underlying each eQTL and estimate their effects in the assay.', doi='', status=<ChallengeStatus.COMPLETED: 'completed'>, difficulty=<ChallengeDifficulty.INTERMEDIATE: 'intermediate'>, platform=SimpleChallengePlatform(id=2, slug='drupal', name='Drupal'), website_url='https://genomeinterpretation.org/CAGI4-2eqtl.html', avatar_url='', incentives=[<ChallengeIncentive.PUBLICATION: 'publication'>, <ChallengeIncentive.SPEAKING_ENGAGEMENT: 'speaking_engagement'>], submission_types=[<ChallengeSubmissionType.PREDICTION_FILE: 'prediction_file'>], input_data_types=[], start_date=None, end_date=datetime.date(2016, 4, 4), starred_count=0, created_at=datetime.datetime(2023, 9, 28, 18, 19, 48, tzinfo=datetime.timezone.utc), updated_at=datetime.datetime(2023, 9, 29, 3, 58, 33, tzinfo=datetime.timezone.utc)), Challenge(id=227, slug='cagi2-cbs', name='CAGI2: CBS', headline='Participants were asked to submit predictions for the effect of the variants...', description='CBS is a vitamin-dependent enzyme involved in cysteine biosynthesis. The human CBS requires two cofactors for function, vitamin B6 and heme. Homocystinuria due to CBS deficiency (OMIM #236200) is a recessive inborn error of sulfur amino acid metabolism.', doi='', status=<ChallengeStatus.COMPLETED: 'completed'>, difficulty=<ChallengeDifficulty.INTERMEDIATE: 'intermediate'>, platform=SimpleChallengePlatform(id=2, slug='drupal', name='Drupal'), website_url='https://genomeinterpretation.org/CAGI2-cbs.html', avatar_url='', incentives=[<ChallengeIncentive.PUBLICATION: 'publication'>, <ChallengeIncentive.SPEAKING_ENGAGEMENT: 'speaking_engagement'>], submission_types=[<ChallengeSubmissionType.PREDICTION_FILE: 'prediction_file'>], input_data_types=[], start_date=None, end_date=datetime.date(2011, 10, 6), starred_count=0, created_at=datetime.datetime(2023, 9, 28, 18, 19, 48, tzinfo=datetime.timezone.utc), updated_at=datetime.datetime(2023, 10, 12, 17, 45, 56, tzinfo=datetime.timezone.utc)), Challenge(id=228, slug='cagi1-chek2', name='CAGI1: CHEK2', headline='Variants in the ATM & CHEK2 genes are associated with breast cancer.', description='Predictors will be provided with 41 rare missense, nonsense, splicing, and indel variants in CHEK2.', doi='', status=<ChallengeStatus.COMPLETED: 'completed'>, difficulty=<ChallengeDifficulty.INTERMEDIATE: 'intermediate'>, platform=SimpleChallengePlatform(id=2, slug='drupal', name='Drupal'), website_url='https://genomeinterpretation.org/CAGI1-chek2.html', avatar_url='', incentives=[<ChallengeIncentive.PUBLICATION: 'publication'>, <ChallengeIncentive.SPEAKING_ENGAGEMENT: 'speaking_engagement'>], submission_types=[<ChallengeSubmissionType.PREDICTION_FILE: 'prediction_file'>], input_data_types=[], start_date=None, end_date=datetime.date(2010, 12, 10), starred_count=0, created_at=datetime.datetime(2023, 9, 28, 18, 19, 48, tzinfo=datetime.timezone.utc), updated_at=datetime.datetime(2023, 10, 19, 23, 32, 57, tzinfo=datetime.timezone.utc)), Challenge(id=230, slug='cagi3-ha', name='CAGI3: HA', headline='The dataset for this challenge comprises of exome sequencing data for 4 subj...', description='Hypoalphalipoproteinemia (HA; OMIM #604091) is characterized by severely decreased serum high-density lipoprotein cholesterol (HDL-C) levels and low apolipoprotein A1 (APOA1). Low HDL-C is a risk factor for coronary artery disease.', doi='', status=<ChallengeStatus.COMPLETED: 'completed'>, difficulty=<ChallengeDifficulty.INTERMEDIATE: 'intermediate'>, platform=SimpleChallengePlatform(id=2, slug='drupal', name='Drupal'), website_url='https://genomeinterpretation.org/CAGI3-ha.html', avatar_url='', incentives=[<ChallengeIncentive.PUBLICATION: 'publication'>, <ChallengeIncentive.SPEAKING_ENGAGEMENT: 'speaking_engagement'>], submission_types=[<ChallengeSubmissionType.PREDICTION_FILE: 'prediction_file'>], input_data_types=[], start_date=None, end_date=datetime.date(2013, 4, 25), starred_count=0, created_at=datetime.datetime(2023, 9, 28, 18, 19, 48, tzinfo=datetime.timezone.utc), updated_at=datetime.datetime(2023, 10, 12, 17, 45, 41, tzinfo=datetime.timezone.utc)), Challenge(id=256, slug='cagi4-warfarin', name='CAGI4: Warfarin exomes', headline='With the provided exome data and clinical covariates, predict the therapeuti...', description='With over 33 million prescriptions in 2011, warfarin is the most commonly used anticoagulant for preventing thromboembolic events. Warfarin has a twenty-fold inter-individual dose variability and a narrow therapeutic index, and it is responsible for a third of adverse drug event hospitalizations in older Americans [2]. Alternatives to warfarin, such as direct thrombin inhibitors and factor Xa inhibitors, are now available. However, these are more expensive, irreversible, and may cause a higher rate of acute coronary events compared to warfarin [3,4]. Thus, warfarin remains a mainstay of anticoagulant therapy, and better methods of dosing warfarin will lead to fewer adverse events due to overcoagulation.', doi='', status=<ChallengeStatus.COMPLETED: 'completed'>, difficulty=<ChallengeDifficulty.INTERMEDIATE: 'intermediate'>, platform=SimpleChallengePlatform(id=2, slug='drupal', name='Drupal'), website_url='https://genomeinterpretation.org/CAGI4-warfarin.html', avatar_url='', incentives=[<ChallengeIncentive.PUBLICATION: 'publication'>, <ChallengeIncentive.SPEAKING_ENGAGEMENT: 'speaking_engagement'>], submission_types=[<ChallengeSubmissionType.PREDICTION_FILE: 'prediction_file'>], input_data_types=[], start_date=None, end_date=datetime.date(2016, 4, 4), starred_count=0, created_at=datetime.datetime(2023, 9, 28, 18, 19, 48, tzinfo=datetime.timezone.utc), updated_at=datetime.datetime(2023, 9, 28, 21, 19, 3, tzinfo=datetime.timezone.utc)), Challenge(id=243, slug='cagi2-pgp', name='CAGI2: PGP', headline='PGP challenge requires matching of full genome sequences to extensive phenot...', description='Participants in the project make their full sequence and phenotypic profile data publicly available. The four CAGI challenges were based on prerelease samples from this resource.', doi='', status=<ChallengeStatus.COMPLETED: 'completed'>, difficulty=<ChallengeDifficulty.INTERMEDIATE: 'intermediate'>, platform=SimpleChallengePlatform(id=2, slug='drupal', name='Drupal'), website_url='https://genomeinterpretation.org/CAGI2-pgp.html', avatar_url='', incentives=[<ChallengeIncentive.PUBLICATION: 'publication'>, <ChallengeIncentive.SPEAKING_ENGAGEMENT: 'speaking_engagement'>], submission_types=[<ChallengeSubmissionType.PREDICTION_FILE: 'prediction_file'>], input_data_types=[], start_date=None, end_date=datetime.date(2011, 10, 6), starred_count=0, created_at=datetime.datetime(2023, 9, 28, 18, 19, 48, tzinfo=datetime.timezone.utc), updated_at=datetime.datetime(2023, 9, 27, 21, 5, 23, tzinfo=datetime.timezone.utc)), Challenge(id=221, slug='cagi2-asthma-twins', name='CAGI2: Asthma discordant monozygotic twins', headline='With the provided whole genome and RNA sequencing data, identify which two i...', description='The dataset includes whole genomes of 8 pairs of discordant monozygotic twins (randomly numbered from 1 to 16) that is, in each pair identical twins one has asthma and one does not. In addition, RNA sequencing data for each individual is provided. One of the twins in each pair suffers from asthma while the other twin is healthy.', doi='', status=<ChallengeStatus.COMPLETED: 'completed'>, difficulty=<ChallengeDifficulty.INTERMEDIATE: 'intermediate'>, platform=SimpleChallengePlatform(id=2, slug='drupal', name='Drupal'), website_url='https://genomeinterpretation.org/CAGI2-asthma-twins.html', avatar_url='', incentives=[<ChallengeIncentive.PUBLICATION: 'publication'>, <ChallengeIncentive.SPEAKING_ENGAGEMENT: 'speaking_engagement'>], submission_types=[<ChallengeSubmissionType.PREDICTION_FILE: 'prediction_file'>], input_data_types=[], start_date=None, end_date=datetime.date(2011, 10, 6), starred_count=0, created_at=datetime.datetime(2023, 9, 28, 18, 19, 48, tzinfo=datetime.timezone.utc), updated_at=datetime.datetime(2023, 10, 12, 18, 11, 42, tzinfo=datetime.timezone.utc)), Challenge(id=222, slug='cagi4-bipolar', name='CAGI4: Bipolar disorder', headline='With the provided exome data, identify which individuals have BD and which i...', description='Bipolar disorder (BD) is a serious mental illness characterized by recurrent episodes of manias and depression, which are syndromes of abnormal mood, thinking and behavior. It affects 1.0-4.5% of the population [1], and it is among the major causes of disability worldwide. This challenge involved the prediction of which of a set of individuals have been diagnosed with bipolar disorder, given exome data. 500 of the 1000 exome samples were provided for training.', doi='', status=<ChallengeStatus.COMPLETED: 'completed'>, difficulty=<ChallengeDifficulty.INTERMEDIATE: 'intermediate'>, platform=SimpleChallengePlatform(id=2, slug='drupal', name='Drupal'), website_url='https://genomeinterpretation.org/CAGI4-bipolar.html', avatar_url='', incentives=[<ChallengeIncentive.PUBLICATION: 'publication'>, <ChallengeIncentive.SPEAKING_ENGAGEMENT: 'speaking_engagement'>], submission_types=[<ChallengeSubmissionType.PREDICTION_FILE: 'prediction_file'>], input_data_types=[], start_date=None, end_date=datetime.date(2016, 4, 4), starred_count=0, created_at=datetime.datetime(2023, 9, 28, 18, 19, 48, tzinfo=datetime.timezone.utc), updated_at=datetime.datetime(2023, 9, 28, 18, 25, 17, tzinfo=datetime.timezone.utc)), Challenge(id=223, slug='cagi3-brca', name='CAGI3: BRCA1 & BRCA2', headline='For each variant, provide the probability that Myriad Genetics has classifie...', description=\"In normal cells, the BRCA1 and BRCA2 genes are involved in homologous recombination for double strand break repair and ensure the stability of a cell's genetic material. Mutations in these genes have been linked to development of breast and ovarian cancer. Myriad Genetics created the BRACAnalysis test in order to assess a woman’s risk of developing hereditary breast or ovarian cancer based on detection of mutations in the BRCA1 and BRCA2 genes. This test has become the standard of care in identification of individuals with hereditary breast and ovarian cancer (HBOC) syndrome. It is based on proprietary methods.\", doi='', status=<ChallengeStatus.COMPLETED: 'completed'>, difficulty=<ChallengeDifficulty.INTERMEDIATE: 'intermediate'>, platform=SimpleChallengePlatform(id=2, slug='drupal', name='Drupal'), website_url='https://genomeinterpretation.org/CAGI3-brca.html', avatar_url='', incentives=[<ChallengeIncentive.PUBLICATION: 'publication'>, <ChallengeIncentive.SPEAKING_ENGAGEMENT: 'speaking_engagement'>], submission_types=[<ChallengeSubmissionType.PREDICTION_FILE: 'prediction_file'>], input_data_types=[], start_date=None, end_date=datetime.date(2013, 4, 25), starred_count=0, created_at=datetime.datetime(2023, 9, 28, 18, 19, 48, tzinfo=datetime.timezone.utc), updated_at=datetime.datetime(2023, 10, 19, 23, 32, 48, tzinfo=datetime.timezone.utc)), Challenge(id=242, slug='cagi1-pgp', name='CAGI1: PGP', headline='PGP challenge requires matching of full genome sequences to extensive phenot...', description='Participants in the project make their full sequence and phenotypic profile data publicly available. The four CAGI challenges were based on prerelease samples from this resource.', doi='', status=<ChallengeStatus.COMPLETED: 'completed'>, difficulty=<ChallengeDifficulty.INTERMEDIATE: 'intermediate'>, platform=SimpleChallengePlatform(id=2, slug='drupal', name='Drupal'), website_url='https://genomeinterpretation.org/CAGI1-pgp.html', avatar_url='', incentives=[<ChallengeIncentive.PUBLICATION: 'publication'>, <ChallengeIncentive.SPEAKING_ENGAGEMENT: 'speaking_engagement'>], submission_types=[<ChallengeSubmissionType.PREDICTION_FILE: 'prediction_file'>], input_data_types=[], start_date=None, end_date=datetime.date(2010, 12, 10), starred_count=0, created_at=datetime.datetime(2023, 9, 28, 18, 19, 48, tzinfo=datetime.timezone.utc), updated_at=datetime.datetime(2023, 9, 27, 21, 5, 22, tzinfo=datetime.timezone.utc)), Challenge(id=229, slug='cagi3-fch', name='CAGI3: FCH', headline='The challenge involved exome sequencing data for 5 subjects in an FCH family...', description='Familial combined hyperlipidemia (FCH; OMIM 14380) the most prevalent hyperlipidemia, is a complex metabolic disorder characterized by variable occurrence of elevated low-density lipoprotein cholesterol (LDL-C) level and high triglycerides (TG)—a condition that is commonly associated with coronary artery disease (CAD).', doi='', status=<ChallengeStatus.COMPLETED: 'completed'>, difficulty=<ChallengeDifficulty.INTERMEDIATE: 'intermediate'>, platform=SimpleChallengePlatform(id=2, slug='drupal', name='Drupal'), website_url='https://genomeinterpretation.org/CAGI3-fch.html', avatar_url='', incentives=[<ChallengeIncentive.PUBLICATION: 'publication'>, <ChallengeIncentive.SPEAKING_ENGAGEMENT: 'speaking_engagement'>], submission_types=[<ChallengeSubmissionType.PREDICTION_FILE: 'prediction_file'>], input_data_types=[], start_date=None, end_date=datetime.date(2013, 4, 25), starred_count=0, created_at=datetime.datetime(2023, 9, 28, 18, 19, 48, tzinfo=datetime.timezone.utc), updated_at=datetime.datetime(2023, 10, 12, 17, 45, 43, tzinfo=datetime.timezone.utc)), Challenge(id=226, slug='cagi1-cbs', name='CAGI1: CBS', headline='Participants were asked to submit predictions for the effect of the variants...', description='CBS is a vitamin-dependent enzyme involved in cysteine biosynthesis. The human CBS requires two cofactors for function, vitamin B6 and heme. Homocystinuria due to CBS deficiency (OMIM #236200) is a recessive inborn error of sulfur amino acid metabolism.', doi='', status=<ChallengeStatus.COMPLETED: 'completed'>, difficulty=<ChallengeDifficulty.INTERMEDIATE: 'intermediate'>, platform=SimpleChallengePlatform(id=2, slug='drupal', name='Drupal'), website_url='https://genomeinterpretation.org/CAGI1-cbs.html', avatar_url='', incentives=[<ChallengeIncentive.PUBLICATION: 'publication'>, <ChallengeIncentive.SPEAKING_ENGAGEMENT: 'speaking_engagement'>], submission_types=[<ChallengeSubmissionType.PREDICTION_FILE: 'prediction_file'>], input_data_types=[], start_date=None, end_date=datetime.date(2010, 12, 10), starred_count=0, created_at=datetime.datetime(2023, 9, 28, 18, 19, 48, tzinfo=datetime.timezone.utc), updated_at=datetime.datetime(2023, 10, 12, 17, 46, 7, tzinfo=datetime.timezone.utc)), Challenge(id=231, slug='cagi2-croshn-s', name=\"CAGI2: Crohn's disease\", headline=\"With the provided exome data, identify which individuals have Crohn's diseas...\", description=\"Crohn's disease (CD [MIM 266600]) a form of inflammatory bowel disease (IBD) is a complex genetic disorder characterized by chronic relapsing inflammation that can involve any part of the gastrointestinal tract.\", doi='', status=<ChallengeStatus.COMPLETED: 'completed'>, difficulty=<ChallengeDifficulty.INTERMEDIATE: 'intermediate'>, platform=SimpleChallengePlatform(id=2, slug='drupal', name='Drupal'), website_url='https://genomeinterpretation.org/CAGI2-croshn-s.html', avatar_url='', incentives=[<ChallengeIncentive.PUBLICATION: 'publication'>, <ChallengeIncentive.SPEAKING_ENGAGEMENT: 'speaking_engagement'>], submission_types=[<ChallengeSubmissionType.PREDICTION_FILE: 'prediction_file'>], input_data_types=[], start_date=None, end_date=datetime.date(2011, 10, 6), starred_count=0, created_at=datetime.datetime(2023, 9, 28, 18, 19, 48, tzinfo=datetime.timezone.utc), updated_at=datetime.datetime(2023, 9, 27, 21, 9, 4, tzinfo=datetime.timezone.utc)), Challenge(id=234, slug='cagi4-hopkins', name='CAGI4: Hopkins clinical panel', headline='Participants were tasked with identifying the disease class for each of 106 ...', description='The Johns Hopkins challenge, provided by the Johns Hopkins DNA Diagnostic Laboratory (http://www.hopkinsmedicine.org/dnadiagnostic), comprised of exonic sequence for 83 genes associated with one of 14 disease classes, including 5 decoys', doi='', status=<ChallengeStatus.COMPLETED: 'completed'>, difficulty=<ChallengeDifficulty.INTERMEDIATE: 'intermediate'>, platform=SimpleChallengePlatform(id=2, slug='drupal', name='Drupal'), website_url='https://genomeinterpretation.org/CAGI4-hopkins.html', avatar_url='', incentives=[<ChallengeIncentive.PUBLICATION: 'publication'>, <ChallengeIncentive.SPEAKING_ENGAGEMENT: 'speaking_engagement'>], submission_types=[<ChallengeSubmissionType.PREDICTION_FILE: 'prediction_file'>], input_data_types=[], start_date=None, end_date=datetime.date(2016, 4, 4), starred_count=0, created_at=datetime.datetime(2023, 9, 28, 18, 19, 48, tzinfo=datetime.timezone.utc), updated_at=datetime.datetime(2023, 10, 12, 17, 45, 27, tzinfo=datetime.timezone.utc)), Challenge(id=232, slug='cagi3-crohn-s', name=\"CAGI3: Crohn's disease\", headline=\"With the provided exome data, identify which individuals have Crohn's diseas...\", description=\"Crohn's disease (CD [MIM 266600]) a form of inflammatory bowel disease (IBD) is a complex genetic disorder characterized by chronic relapsing inflammation that can involve any part of the gastrointestinal tract.\", doi='', status=<ChallengeStatus.COMPLETED: 'completed'>, difficulty=<ChallengeDifficulty.INTERMEDIATE: 'intermediate'>, platform=SimpleChallengePlatform(id=2, slug='drupal', name='Drupal'), website_url='https://genomeinterpretation.org/CAGI3-crohn-s.html', avatar_url='', incentives=[<ChallengeIncentive.PUBLICATION: 'publication'>, <ChallengeIncentive.SPEAKING_ENGAGEMENT: 'speaking_engagement'>], submission_types=[<ChallengeSubmissionType.PREDICTION_FILE: 'prediction_file'>], input_data_types=[], start_date=None, end_date=datetime.date(2013, 4, 25), starred_count=0, created_at=datetime.datetime(2023, 9, 28, 18, 19, 48, tzinfo=datetime.timezone.utc), updated_at=datetime.datetime(2023, 9, 28, 18, 25, 20, tzinfo=datetime.timezone.utc)), Challenge(id=233, slug='cagi4-chron-s-exome', name=\"CAGI4: Crohn's exomes\", headline=\"With the provided exome data, identify which individuals have Crohn's diseas...\", description=\"Crohn's disease (CD [MIM 266600]) a form of inflammatory bowel disease (IBD) is a complex genetic disorder characterized by chronic relapsing inflammation that can involve any part of the gastrointestinal tract.\", doi='', status=<ChallengeStatus.COMPLETED: 'completed'>, difficulty=<ChallengeDifficulty.INTERMEDIATE: 'intermediate'>, platform=SimpleChallengePlatform(id=2, slug='drupal', name='Drupal'), website_url='https://genomeinterpretation.org/cagi4-chron-s-exome.html', avatar_url='', incentives=[<ChallengeIncentive.PUBLICATION: 'publication'>, <ChallengeIncentive.SPEAKING_ENGAGEMENT: 'speaking_engagement'>], submission_types=[<ChallengeSubmissionType.PREDICTION_FILE: 'prediction_file'>], input_data_types=[], start_date=None, end_date=datetime.date(2016, 4, 4), starred_count=0, created_at=datetime.datetime(2023, 9, 28, 18, 19, 48, tzinfo=datetime.timezone.utc), updated_at=datetime.datetime(2023, 10, 16, 18, 29, 11, tzinfo=datetime.timezone.utc)), Challenge(id=254, slug='cagi4-sumo-ligase', name='CAGI4: SUMO ligase', headline='Participants are asked to submit predictions of the effect of the variants o...', description='SUMO ligase identifies target proteins and covalently attaches SUMO to them, thereby modulating the functions of hundreds of proteins including proteins implicated in cancer, neurodegeneration, and other diseases. A large library of missense mutations in human SUMO ligase has been assessed for competitive growth in a high-throughput yeast-based complementation assay. The challenge is to predict the effect of mutations on function, as measured by the change in fractional representation of each mutant SUMO ligase clone, relative to wild-type clones, in a competitive yeast growth assay.', doi='', status=<ChallengeStatus.COMPLETED: 'completed'>, difficulty=<ChallengeDifficulty.INTERMEDIATE: 'intermediate'>, platform=SimpleChallengePlatform(id=2, slug='drupal', name='Drupal'), website_url='https://genomeinterpretation.org/CAGI4-sumo-ligase.html', avatar_url='', incentives=[<ChallengeIncentive.PUBLICATION: 'publication'>, <ChallengeIncentive.SPEAKING_ENGAGEMENT: 'speaking_engagement'>], submission_types=[<ChallengeSubmissionType.PREDICTION_FILE: 'prediction_file'>], input_data_types=[], start_date=None, end_date=datetime.date(2016, 4, 4), starred_count=0, created_at=datetime.datetime(2023, 9, 28, 18, 19, 48, tzinfo=datetime.timezone.utc), updated_at=datetime.datetime(2023, 10, 19, 23, 31, 57, tzinfo=datetime.timezone.utc)), Challenge(id=253, slug='cagi4-sickkids', name='CAGI4: SickKids', headline='The challenge presented here is to use computational methods to match each g...', description='Realizing the promise of precision medicine will require developing methods for interpreting genome sequence data to infer individuals’ phenotypic traits and predispositions to disease. This challenge involves 25 children with suspected genetic disorders who were referred for clinical genome sequencing. Predictors are given their genome sequences and their clinical phenotypic descriptions, as provided to the diagnostic laboratory, and asked to predict which genome corresponds to which clinical description. Additionally, identify the diagnostic variants underlying the predictions. Optionally, identify predictive secondary variants conferring high risk of other diseases whose phenotypes are not reported in the clinical descriptions.', doi='', status=<ChallengeStatus.COMPLETED: 'completed'>, difficulty=<ChallengeDifficulty.INTERMEDIATE: 'intermediate'>, platform=SimpleChallengePlatform(id=2, slug='drupal', name='Drupal'), website_url='https://genomeinterpretation.org/CAGI4-sickkids.html', avatar_url='', incentives=[<ChallengeIncentive.PUBLICATION: 'publication'>, <ChallengeIncentive.SPEAKING_ENGAGEMENT: 'speaking_engagement'>], submission_types=[<ChallengeSubmissionType.PREDICTION_FILE: 'prediction_file'>], input_data_types=[], start_date=None, end_date=datetime.date(2016, 4, 4), starred_count=0, created_at=datetime.datetime(2023, 9, 28, 18, 19, 48, tzinfo=datetime.timezone.utc), updated_at=datetime.datetime(2023, 10, 6, 20, 48, 13, tzinfo=datetime.timezone.utc)), Challenge(id=257, slug='cagi6-calmodulin', name='CAGI6: Calmodulin', headline='participants were asked to submit predictions for the competitive growth sco...', description='Calmodulin (CaM) is a ubiquitous calcium (Ca2+) sensor protein interacting with more than 200 molecular partners, thereby regulating a variety of biological processes. Missense point mutations in the genes encoding CaM have been associated with ventricular tachycardia and sudden cardiac death. A library encompassing up to 17 point mutations was assessed by far-UV circular dichroism (CD) by measuring melting temperature (Tm) and percentage of unfolding (%unfold) upon thermal denaturation at pH and salt concentration that mimic the physiological conditions. The challenge is to predict- the Tm and %unfold values for isolated CaM variants under Ca2+-saturating conditions (Ca2+-CaM) and in the Ca2+-free (apo) state;  whether the point mutation stabilizes or destabilizes the protein (based on Tm and %unfold).', doi='', status=<ChallengeStatus.COMPLETED: 'completed'>, difficulty=<ChallengeDifficulty.INTERMEDIATE: 'intermediate'>, platform=SimpleChallengePlatform(id=1, slug='synapse', name='Synapse'), website_url='https://genomeinterpretation.org/CAGI6-cam.html', avatar_url='', incentives=[<ChallengeIncentive.PUBLICATION: 'publication'>, <ChallengeIncentive.SPEAKING_ENGAGEMENT: 'speaking_engagement'>], submission_types=[<ChallengeSubmissionType.PREDICTION_FILE: 'prediction_file'>], input_data_types=[], start_date=None, end_date=datetime.date(2021, 12, 31), starred_count=0, created_at=datetime.datetime(2023, 9, 28, 18, 19, 48, tzinfo=datetime.timezone.utc), updated_at=datetime.datetime(2023, 10, 19, 23, 33, 19, tzinfo=datetime.timezone.utc)), Challenge(id=235, slug='cagi2-mouse-exomes', name='CAGI2: Mouse exomes', headline='The challenge involved identifying the causative variants leading to one of ...', description='Predictors were given SNVs and indels found from exome sequencing. Causative variants had been identified for the L11Jus74 and Sofa phenotypes by the use of traditional breeding crosses,47 and the predictions were compared to these results, which were unpublished at the time of the CAGI submissions. The L11Jus74 phenotype is caused by two SNVs (chr11-102258914A> and chr11-77984176A>T), whereas a 15-nucleotide deletion in the Pfas gene is responsible for the Sofa phenotype. The predictions for Frg and Stn phenotypes could not be compared to experimental data, as the causative variants could not successfully be mapped by linkage', doi='', status=<ChallengeStatus.COMPLETED: 'completed'>, difficulty=<ChallengeDifficulty.INTERMEDIATE: 'intermediate'>, platform=SimpleChallengePlatform(id=2, slug='drupal', name='Drupal'), website_url='https://genomeinterpretation.org/CAGI2-mouse-exomes.html', avatar_url='', incentives=[<ChallengeIncentive.PUBLICATION: 'publication'>, <ChallengeIncentive.SPEAKING_ENGAGEMENT: 'speaking_engagement'>], submission_types=[<ChallengeSubmissionType.PREDICTION_FILE: 'prediction_file'>], input_data_types=[], start_date=None, end_date=datetime.date(2011, 10, 6), starred_count=0, created_at=datetime.datetime(2023, 9, 28, 18, 19, 48, tzinfo=datetime.timezone.utc), updated_at=datetime.datetime(2023, 10, 12, 17, 45, 19, tzinfo=datetime.timezone.utc)), Challenge(id=255, slug='cagi3-splicing', name='CAGI3: TP53 splicing', headline='With the provided data determine which disease-causing mutations in the TP53...', description='The function of exonic splicing regulatory elements can be undermined by DNA sequence variation and in some cases can contribute to pathogenesis. Thousands of disease-causing mutations disrupt exonic splicing regulatory elements. These data suggest that >25 percent of missense mutations may impact pre-mRNA splicing rather than mRNA translation. Using minigene constructs derived from a fragment of the TP53 gene, we have experimentally determined if each mutation influences splicing fidelity in HEK293T cells. We hope that CAGI participants will be able to predict the outcome of our experiments. A long-term goal will be the computational prioritization of disease-causing mutations prior to experimental validation. This contribution is expected to have major impacts in understanding the pathogenic basis of disease-causing mutations.', doi='', status=<ChallengeStatus.COMPLETED: 'completed'>, difficulty=<ChallengeDifficulty.INTERMEDIATE: 'intermediate'>, platform=SimpleChallengePlatform(id=2, slug='drupal', name='Drupal'), website_url='https://genomeinterpretation.org/CAGI3-splicing.html', avatar_url='', incentives=[<ChallengeIncentive.PUBLICATION: 'publication'>, <ChallengeIncentive.SPEAKING_ENGAGEMENT: 'speaking_engagement'>], submission_types=[<ChallengeSubmissionType.PREDICTION_FILE: 'prediction_file'>], input_data_types=[], start_date=None, end_date=datetime.date(2013, 4, 25), starred_count=0, created_at=datetime.datetime(2023, 9, 28, 18, 19, 48, tzinfo=datetime.timezone.utc), updated_at=datetime.datetime(2023, 10, 10, 19, 48, 10, tzinfo=datetime.timezone.utc)), Challenge(id=237, slug='cagi4-naglu', name='CAGI4: NAGLU', headline='Participants are asked to submit predictions on the effect of the variants o...', description='NAGLU is a lysosomal glycohydrolyase. Deficiency of NAGLU causes the rare disorder Mucopolysaccharidosis IIIB or Sanfilippo B disease. Naturally occurring NAGLU mutants have been assayed for enzymatic activity in transfected cell lysates. The challenge is to predict the fractional activity of each mutant protein compared to the wild-type enzyme.', doi='', status=<ChallengeStatus.COMPLETED: 'completed'>, difficulty=<ChallengeDifficulty.INTERMEDIATE: 'intermediate'>, platform=SimpleChallengePlatform(id=2, slug='drupal', name='Drupal'), website_url='https://genomeinterpretation.org/CAGI4-naglu.html', avatar_url='', incentives=[<ChallengeIncentive.PUBLICATION: 'publication'>, <ChallengeIncentive.SPEAKING_ENGAGEMENT: 'speaking_engagement'>], submission_types=[<ChallengeSubmissionType.PREDICTION_FILE: 'prediction_file'>], input_data_types=[], start_date=None, end_date=datetime.date(2016, 4, 4), starred_count=0, created_at=datetime.datetime(2023, 9, 28, 18, 19, 48, tzinfo=datetime.timezone.utc), updated_at=datetime.datetime(2023, 9, 29, 3, 55, 52, tzinfo=datetime.timezone.utc)), Challenge(id=239, slug='cagi3-mrn-nbs1', name='CAGI3: NBS1', headline='Genomes are subject to constant threat by damaging agents that generate DNA ...', description='Predict probability of pathogenicity (a number between 0 and 1) for individual rare variants of MRE11 and NBS1.', doi='', status=<ChallengeStatus.COMPLETED: 'completed'>, difficulty=<ChallengeDifficulty.INTERMEDIATE: 'intermediate'>, platform=SimpleChallengePlatform(id=2, slug='drupal', name='Drupal'), website_url='https://genomeinterpretation.org/CAGI3-mrn.html', avatar_url='', incentives=[<ChallengeIncentive.PUBLICATION: 'publication'>, <ChallengeIncentive.SPEAKING_ENGAGEMENT: 'speaking_engagement'>], submission_types=[<ChallengeSubmissionType.PREDICTION_FILE: 'prediction_file'>], input_data_types=[], start_date=None, end_date=datetime.date(2013, 4, 25), starred_count=0, created_at=datetime.datetime(2023, 9, 28, 18, 19, 48, tzinfo=datetime.timezone.utc), updated_at=datetime.datetime(2023, 10, 16, 18, 38, 55, tzinfo=datetime.timezone.utc)), Challenge(id=240, slug='cagi3-p16', name='CAGI3: p16', headline='CDKN2A is the most common, high penetrance, susceptibility gene identified t...', description='Evaluate how different variants of p16 protein impact its ability to block cell proliferation.', doi='', status=<ChallengeStatus.COMPLETED: 'completed'>, difficulty=<ChallengeDifficulty.INTERMEDIATE: 'intermediate'>, platform=SimpleChallengePlatform(id=2, slug='drupal', name='Drupal'), website_url='https://genomeinterpretation.org/CAGI3-p16.html', avatar_url='', incentives=[<ChallengeIncentive.PUBLICATION: 'publication'>, <ChallengeIncentive.SPEAKING_ENGAGEMENT: 'speaking_engagement'>], submission_types=[<ChallengeSubmissionType.PREDICTION_FILE: 'prediction_file'>], input_data_types=[], start_date=None, end_date=datetime.date(2013, 4, 25), starred_count=0, created_at=datetime.datetime(2023, 9, 28, 18, 19, 48, tzinfo=datetime.timezone.utc), updated_at=datetime.datetime(2023, 9, 29, 3, 55, 38, tzinfo=datetime.timezone.utc)), Challenge(id=258, slug='cagi2-splicing', name='CAGI2: splicing', headline='Predictors are asked to compare exons from wild type and disease-associated ...', description='Accurate precursor mRNA (pre-mRNA) splicing is required for the expression of protein coding genes from the human genome. In this process, intervening sequences (introns) are removed from pre-mRNA and coding/regulatory sequences (exons) are ligated together generating a mature mRNA. A large ribonucleoprotein machine called the spliceosome assembles de novo upon every nascent intron and catalyzes the chemical steps of splicing.', doi='', status=<ChallengeStatus.COMPLETED: 'completed'>, difficulty=<ChallengeDifficulty.INTERMEDIATE: 'intermediate'>, platform=SimpleChallengePlatform(id=2, slug='drupal', name='Drupal'), website_url='https://genomeinterpretation.org/CAGI2-splicing.html', avatar_url='', incentives=[<ChallengeIncentive.PUBLICATION: 'publication'>, <ChallengeIncentive.SPEAKING_ENGAGEMENT: 'speaking_engagement'>], submission_types=[<ChallengeSubmissionType.PREDICTION_FILE: 'prediction_file'>], input_data_types=[], start_date=None, end_date=datetime.date(2011, 10, 6), starred_count=0, created_at=datetime.datetime(2023, 9, 28, 18, 19, 48, tzinfo=datetime.timezone.utc), updated_at=datetime.datetime(2023, 10, 18, 15, 32, 55, tzinfo=datetime.timezone.utc)), Challenge(id=236, slug='cagi3-mrn-mre11', name='CAGI3: MRE11', headline='Genomes are subject to constant threat by damaging agents that generate DNA ...', description='Predict probability of pathogenicity (a number between 0 and 1) for individual rare variants of MRE11 and NBS1.', doi='', status=<ChallengeStatus.COMPLETED: 'completed'>, difficulty=<ChallengeDifficulty.INTERMEDIATE: 'intermediate'>, platform=SimpleChallengePlatform(id=2, slug='drupal', name='Drupal'), website_url='https://genomeinterpretation.org/CAGI3-mrn.html', avatar_url='', incentives=[<ChallengeIncentive.PUBLICATION: 'publication'>, <ChallengeIncentive.SPEAKING_ENGAGEMENT: 'speaking_engagement'>], submission_types=[<ChallengeSubmissionType.PREDICTION_FILE: 'prediction_file'>], input_data_types=[], start_date=None, end_date=datetime.date(2013, 4, 25), starred_count=0, created_at=datetime.datetime(2023, 9, 28, 18, 19, 48, tzinfo=datetime.timezone.utc), updated_at=datetime.datetime(2023, 10, 16, 18, 38, 44, tzinfo=datetime.timezone.utc)), Challenge(id=245, slug='cagi4-pgp', name='CAGI4: PGP', headline='PGP challenge requires matching of full genome sequences to extensive phenot...', description='Participants in the project make their full sequence and phenotypic profile data publicly available. The four CAGI challenges were based on prerelease samples from this resource.', doi='', status=<ChallengeStatus.COMPLETED: 'completed'>, difficulty=<ChallengeDifficulty.INTERMEDIATE: 'intermediate'>, platform=SimpleChallengePlatform(id=2, slug='drupal', name='Drupal'), website_url='https://genomeinterpretation.org/CAGI4-pgp.html', avatar_url='', incentives=[<ChallengeIncentive.PUBLICATION: 'publication'>, <ChallengeIncentive.SPEAKING_ENGAGEMENT: 'speaking_engagement'>], submission_types=[<ChallengeSubmissionType.PREDICTION_FILE: 'prediction_file'>], input_data_types=[], start_date=None, end_date=datetime.date(2016, 4, 4), starred_count=0, created_at=datetime.datetime(2023, 9, 28, 18, 19, 48, tzinfo=datetime.timezone.utc), updated_at=datetime.datetime(2023, 9, 27, 21, 5, 24, tzinfo=datetime.timezone.utc)), Challenge(id=244, slug='cagi3-pgp', name='CAGI3: PGP', headline='PGP challenge requires matching of full genome sequences to extensive phenot...', description='Participants in the project make their full sequence and phenotypic profile data publicly available. The four CAGI challenges were based on prerelease samples from this resource.', doi='', status=<ChallengeStatus.COMPLETED: 'completed'>, difficulty=<ChallengeDifficulty.INTERMEDIATE: 'intermediate'>, platform=SimpleChallengePlatform(id=2, slug='drupal', name='Drupal'), website_url='https://genomeinterpretation.org/CAGI3-pgp.html', avatar_url='', incentives=[<ChallengeIncentive.PUBLICATION: 'publication'>, <ChallengeIncentive.SPEAKING_ENGAGEMENT: 'speaking_engagement'>], submission_types=[<ChallengeSubmissionType.PREDICTION_FILE: 'prediction_file'>], input_data_types=[], start_date=None, end_date=datetime.date(2013, 4, 25), starred_count=0, created_at=datetime.datetime(2023, 9, 28, 18, 19, 48, tzinfo=datetime.timezone.utc), updated_at=datetime.datetime(2023, 9, 27, 21, 5, 23, tzinfo=datetime.timezone.utc)), Challenge(id=246, slug='cagi4-pyruvate-kinase', name='CAGI4: Pyruvate kinase', headline='Participants are asked to submit predictions on the effect of the mutations ...', description='Pyruvate kinase catalyzes the last step in glycolysis and is regulated by allosteric effectors. Variants in the gene encoding the isozymes expressed in red blood cells and liver, including missense variants mapping near the effector binding sites, cause PK deficiency. A large set of single amino acid mutations in the liver enzyme has been assayed in E. coli extracts for the effect on allosteric regulation of enzyme activity. The challenge is to predict the impacts of mutations on enzyme activity and allosteric regulation.', doi='', status=<ChallengeStatus.COMPLETED: 'completed'>, difficulty=<ChallengeDifficulty.INTERMEDIATE: 'intermediate'>, platform=SimpleChallengePlatform(id=2, slug='drupal', name='Drupal'), website_url='https://genomeinterpretation.org/CAGI4-pyruvate-kinase.html', avatar_url='', incentives=[<ChallengeIncentive.PUBLICATION: 'publication'>, <ChallengeIncentive.SPEAKING_ENGAGEMENT: 'speaking_engagement'>], submission_types=[<ChallengeSubmissionType.PREDICTION_FILE: 'prediction_file'>], input_data_types=[], start_date=None, end_date=datetime.date(2015, 1, 11), starred_count=0, created_at=datetime.datetime(2023, 9, 28, 18, 19, 48, tzinfo=datetime.timezone.utc), updated_at=datetime.datetime(2023, 9, 29, 22, 6, 22, tzinfo=datetime.timezone.utc)), Challenge(id=241, slug='cagi2-p53', name='CAGI2: p53 reactivation', headline='Predictors are asked to submit predictions on the effect of the cancer rescu...', description='The transcription factor p53 is a central tumor suppressor protein that controls DNA repair, cell cycle arrest, and apoptosis (programmed cell death). About half of human cancers have p53 mutations that inactivate p53. Over 250,000 US deaths yearly are due to tumors that express full-length p53 that has been inactivated by a single point mutation. For the past several years, the group of Rick Lathrop at University of California, Irvine, has been engaged in a complete functional census of p53 second-site suppressor (“cancer rescue”) mutations. These cancer rescue mutations are additional amino acids changes (to otherwise cancerous p53 mutations), which have been found to rescue p53 tumor suppressor function, reactivating otherwise inactive p53. These intragenic rescue mutations reactivate cancer mutant p53 in yeast and human cell assays by providing structural changes that compensate for the cancer mutation.', doi='', status=<ChallengeStatus.COMPLETED: 'completed'>, difficulty=<ChallengeDifficulty.INTERMEDIATE: 'intermediate'>, platform=SimpleChallengePlatform(id=2, slug='drupal', name='Drupal'), website_url='https://genomeinterpretation.org/CAGI2-p53.html', avatar_url='', incentives=[<ChallengeIncentive.PUBLICATION: 'publication'>, <ChallengeIncentive.SPEAKING_ENGAGEMENT: 'speaking_engagement'>], submission_types=[<ChallengeSubmissionType.PREDICTION_FILE: 'prediction_file'>], input_data_types=[], start_date=None, end_date=datetime.date(2011, 10, 6), starred_count=0, created_at=datetime.datetime(2023, 9, 28, 18, 19, 48, tzinfo=datetime.timezone.utc), updated_at=datetime.datetime(2023, 9, 29, 3, 55, 53, tzinfo=datetime.timezone.utc)), Challenge(id=238, slug='cagi4-npm-alk', name='CAGI4: NPM: ALK', headline='Participants are asked to submit predictions of both the kinase activity and...', description='NPM-ALK is a fusion protein in which the ALK tyrosine kinase is constitutively activated, contributing to cancer. NPM-ALK constructs with mutations in the kinase domain have been assayed in extracts of transfected cells. The challenge is to predict the kinase activity and the Hsp90 binding affinity of the mutant proteins relative to the reference NPM-ALK fusion protein.', doi='', status=<ChallengeStatus.COMPLETED: 'completed'>, difficulty=<ChallengeDifficulty.INTERMEDIATE: 'intermediate'>, platform=SimpleChallengePlatform(id=2, slug='drupal', name='Drupal'), website_url='https://genomeinterpretation.org/CAGI4-npm-alk.html', avatar_url='', incentives=[<ChallengeIncentive.PUBLICATION: 'publication'>, <ChallengeIncentive.SPEAKING_ENGAGEMENT: 'speaking_engagement'>], submission_types=[<ChallengeSubmissionType.PREDICTION_FILE: 'prediction_file'>], input_data_types=[], start_date=None, end_date=datetime.date(2016, 4, 4), starred_count=0, created_at=datetime.datetime(2023, 9, 28, 18, 19, 48, tzinfo=datetime.timezone.utc), updated_at=datetime.datetime(2023, 9, 29, 3, 55, 53, tzinfo=datetime.timezone.utc)), Challenge(id=247, slug='cagi2-rad50', name='CAGI2: RAD50', headline='Predict the probability of the variant occurring in a case individual.', description='RAD50 is a candidate intermediate-risk breast cancer susceptibility gene. The RAD50 data provided for CAGI challenge include a list of potentially interesting sequence variants observed from sequencing RAD50 gene in about 1,400 breast cancer cases and 1,200 ethnically matched controls. Variants in the list were observed between 1 and 20 times.', doi='', status=<ChallengeStatus.COMPLETED: 'completed'>, difficulty=<ChallengeDifficulty.INTERMEDIATE: 'intermediate'>, platform=SimpleChallengePlatform(id=2, slug='drupal', name='Drupal'), website_url='https://genomeinterpretation.org/CAGI2-rad50.html', avatar_url='', incentives=[<ChallengeIncentive.PUBLICATION: 'publication'>, <ChallengeIncentive.SPEAKING_ENGAGEMENT: 'speaking_engagement'>], submission_types=[<ChallengeSubmissionType.PREDICTION_FILE: 'prediction_file'>], input_data_types=[], start_date=None, end_date=datetime.date(2011, 10, 6), starred_count=0, created_at=datetime.datetime(2023, 9, 28, 18, 19, 48, tzinfo=datetime.timezone.utc), updated_at=datetime.datetime(2023, 10, 19, 23, 33, 11, tzinfo=datetime.timezone.utc)), Challenge(id=248, slug='cagi2-risksnps', name='CAGI2: riskSNPs', headline='The goal of these challenges is to investigate the community’s ability to id...', description='The goal of this experiment is to explore current understanding of the molecular level mechanisms underlying associations between SNPs and disease risk, incorporating expertise in each of the known mechanism areas, and as far as possible assigning possible mechanisms for each association locus. The correct mechanisms are unknown, so there can be no ranking of accuracy-that is not the point of the experiment. Rather, the goal is to ascertain which mechanisms appear most relevant, how confidently they can be assigned, and what fraction of loci can currently be assigned plausible mechanisms.', doi='', status=<ChallengeStatus.COMPLETED: 'completed'>, difficulty=<ChallengeDifficulty.INTERMEDIATE: 'intermediate'>, platform=SimpleChallengePlatform(id=2, slug='drupal', name='Drupal'), website_url='https://genomeinterpretation.org/CAGI2-risksnps.html', avatar_url='', incentives=[<ChallengeIncentive.PUBLICATION: 'publication'>, <ChallengeIncentive.SPEAKING_ENGAGEMENT: 'speaking_engagement'>], submission_types=[<ChallengeSubmissionType.PREDICTION_FILE: 'prediction_file'>], input_data_types=[], start_date=None, end_date=datetime.date(2011, 10, 6), starred_count=0, created_at=datetime.datetime(2023, 9, 28, 18, 19, 48, tzinfo=datetime.timezone.utc), updated_at=datetime.datetime(2023, 10, 19, 23, 33, 11, tzinfo=datetime.timezone.utc)), Challenge(id=264, slug='jan2024-rare-disease-ai-hackathon', name='Jan2024: Rare Disease AI Hackathon', headline='Researchers and medical experts are invited to collaborate on our patient ca...', description='Bring AI and medical experts together to build open source models for rare diseases. Create zero-barrier access to rare disease expertise for patients, researchers and physicians. Use AI to Uncover novel links between rare diseases. Establish validation methods for medical AI models. Jumpstart an open source community for rare disease AI models. Launch models for Beta testing on Hypophosphatasia.ai and EhlersDanlos.ai.', doi='', status=<ChallengeStatus.ACTIVE: 'active'>, difficulty=<ChallengeDifficulty.INTERMEDIATE: 'intermediate'>, platform=SimpleChallengePlatform(id=14, slug='other', name='Other'), website_url='https://www.rarediseaseaihackathon.org/', avatar_url='', incentives=[<ChallengeIncentive.PUBLICATION: 'publication'>, <ChallengeIncentive.SPEAKING_ENGAGEMENT: 'speaking_engagement'>], submission_types=[<ChallengeSubmissionType.PREDICTION_FILE: 'prediction_file'>, <ChallengeSubmissionType.CONTAINER_IMAGE: 'container_image'>], input_data_types=[], start_date=datetime.date(2023, 9, 30), end_date=datetime.date(2024, 1, 15), starred_count=0, created_at=datetime.datetime(2023, 9, 27, 19, 10, 40, tzinfo=datetime.timezone.utc), updated_at=datetime.datetime(2023, 10, 12, 18, 13, 38, tzinfo=datetime.timezone.utc)), Challenge(id=263, slug='finding-ligands-targeting-the-tkb-domain-of-cblb', name='CACHE4: Finding ligands targeting the TKB domain of CBLB', headline='Several cancers (PMID-33306199), potential immunotherapy (PMID-24875217), in...', description='Predict compounds that bind to the closed conformation of the CBLB TKB domain with novel chemical templates and KD below 30 micromolar.', doi='', status=<ChallengeStatus.COMPLETED: 'completed'>, difficulty=<ChallengeDifficulty.INTERMEDIATE: 'intermediate'>, platform=SimpleChallengePlatform(id=17, slug='cache', name='CACHE'), website_url='https://cache-challenge.org/challenges/finding-ligands-targeting-the-tkb-domain-of-cblb', avatar_url='', incentives=[<ChallengeIncentive.PUBLICATION: 'publication'>, <ChallengeIncentive.SPEAKING_ENGAGEMENT: 'speaking_engagement'>], submission_types=[<ChallengeSubmissionType.PREDICTION_FILE: 'prediction_file'>, <ChallengeSubmissionType.CONTAINER_IMAGE: 'container_image'>], input_data_types=[], start_date=datetime.date(2023, 3, 9), end_date=datetime.date(2023, 5, 9), starred_count=0, created_at=datetime.datetime(2023, 9, 27, 19, 3, 14, tzinfo=datetime.timezone.utc), updated_at=datetime.datetime(2023, 10, 16, 19, 1, 22, tzinfo=datetime.timezone.utc)), Challenge(id=262, slug='finding-ligands-targeting-the-macrodomain-of-sars-cov-2-nsp3', name='CACHE3: Finding ligands targeting the macrodomain of SARS-CoV-2 Nsp3', headline='Severe acute respiratory syndrome coronavirus 2', description='To predict ligands that bind to the ADPr site of SARS-CoV-2 Nsp3 macrodomain (Mac1).', doi='', status=<ChallengeStatus.COMPLETED: 'completed'>, difficulty=<ChallengeDifficulty.INTERMEDIATE: 'intermediate'>, platform=SimpleChallengePlatform(id=17, slug='cache', name='CACHE'), website_url='https://cache-challenge.org/challenges/finding-ligands-targeting-the-macrodomain-of-sars-cov-2-nsp3', avatar_url='', incentives=[<ChallengeIncentive.PUBLICATION: 'publication'>, <ChallengeIncentive.SPEAKING_ENGAGEMENT: 'speaking_engagement'>], submission_types=[<ChallengeSubmissionType.PREDICTION_FILE: 'prediction_file'>, <ChallengeSubmissionType.CONTAINER_IMAGE: 'container_image'>], input_data_types=[], start_date=datetime.date(2022, 11, 2), end_date=datetime.date(2023, 1, 1), starred_count=0, created_at=datetime.datetime(2023, 9, 27, 19, 3, 13, tzinfo=datetime.timezone.utc), updated_at=datetime.datetime(2023, 10, 16, 19, 1, 19, tzinfo=datetime.timezone.utc)), Challenge(id=261, slug='finding-ligands-targeting-the-conserved-rna-binding-site-of-sars-cov-2-nsp13', name='CACHE2: FINDING LIGANDS TARGETING THE CONSERVED RNA BINDING SITE OF SARS-CoV-2 NSP13', headline='Finding ligands targeting the conserved RNA binding site of SARS-CoV-2 NSP13.', description='Predicted compounds will be procured and tested at CACHE using both enzymatic and binding assays', doi='', status=<ChallengeStatus.COMPLETED: 'completed'>, difficulty=<ChallengeDifficulty.INTERMEDIATE: 'intermediate'>, platform=SimpleChallengePlatform(id=17, slug='cache', name='CACHE'), website_url='https://cache-challenge.org/challenges/finding-ligands-targeting-the-conserved-rna-binding-site-of-sars-cov-2-nsp13', avatar_url='', incentives=[<ChallengeIncentive.PUBLICATION: 'publication'>, <ChallengeIncentive.SPEAKING_ENGAGEMENT: 'speaking_engagement'>], submission_types=[<ChallengeSubmissionType.PREDICTION_FILE: 'prediction_file'>, <ChallengeSubmissionType.CONTAINER_IMAGE: 'container_image'>], input_data_types=[], start_date=datetime.date(2022, 6, 22), end_date=datetime.date(2022, 9, 4), starred_count=0, created_at=datetime.datetime(2023, 9, 27, 19, 2, 43, tzinfo=datetime.timezone.utc), updated_at=datetime.datetime(2023, 10, 16, 19, 1, 17, tzinfo=datetime.timezone.utc)), Challenge(id=260, slug='predict-hits-for-the-wdr-domain-of-lrrk2', name='CACHE1: PREDICT HITS FOR THE WDR DOMAIN OF LRRK2', headline='Finding ligands targeting the central cavity of the WD-40 repeat (WDR) domai...', description=\"The first CACHE Challenge target is LRRK2, the most commonly mutated gene in familial Parkinson's Disease.  Participants are asked to find hits for the WD40 repeat (WDR) domain of LRRK2. Read more under Details below.\", doi='', status=<ChallengeStatus.COMPLETED: 'completed'>, difficulty=<ChallengeDifficulty.INTERMEDIATE: 'intermediate'>, platform=SimpleChallengePlatform(id=17, slug='cache', name='CACHE'), website_url='https://cache-challenge.org/challenges/predict-hits-for-the-wdr-domain-of-lrrk2', avatar_url='', incentives=[<ChallengeIncentive.PUBLICATION: 'publication'>, <ChallengeIncentive.SPEAKING_ENGAGEMENT: 'speaking_engagement'>], submission_types=[<ChallengeSubmissionType.PREDICTION_FILE: 'prediction_file'>, <ChallengeSubmissionType.CONTAINER_IMAGE: 'container_image'>], input_data_types=[], start_date=datetime.date(2021, 12, 1), end_date=datetime.date(2022, 1, 31), starred_count=0, created_at=datetime.datetime(2023, 9, 27, 19, 1, 55, tzinfo=datetime.timezone.utc), updated_at=datetime.datetime(2023, 10, 16, 19, 3, 47, tzinfo=datetime.timezone.utc)), Challenge(id=220, slug='data-management-and-graph-extraction-for-large-models-in-the-biomedical-space', name='Data management and graph extraction for large models in the biomedical space', headline='Collaborative hackathon on the topic of data management and graph extraction...', description='This fall, CMU Libraries is hosting a hackathon in partnership with DNAnexus on the topic of data management and graph extraction for large models in the biomedical space. The hackathon will be held in person at CMU, October 19-21, 2023.  The hackathon is a collaborative, rather than competitive, event, with each team working on a dedicated part of the problem. The teams will be focused on the following topics-1) Knowledge graph-based validation for variant (genomic) assertions; 2) Continuous monitoring for RLHF and flexible infrastructure for layering assertions with rollback; 3) Flexible tokenization of complex data types; 4) Assertion tracking in large models; 5) Column headers for data harmonization.  The outputs are often published as preprints or on the F1000 hackathon channel.  Contact Ben Busby (bbusby@dnanexus.com) with any questions about the hackathon or serving as a team lead.', doi='', status=<ChallengeStatus.ACTIVE: 'active'>, difficulty=<ChallengeDifficulty.INTERMEDIATE: 'intermediate'>, platform=SimpleChallengePlatform(id=14, slug='other', name='Other'), website_url='https://library.cmu.edu/about/news/2023-08/hackathon-2023', avatar_url='', incentives=[<ChallengeIncentive.PUBLICATION: 'publication'>], submission_types=[<ChallengeSubmissionType.OTHER: 'other'>], input_data_types=[], start_date=datetime.date(2023, 10, 19), end_date=datetime.date(2023, 10, 21), starred_count=0, created_at=datetime.datetime(2023, 9, 13, 23, 32, 59, tzinfo=datetime.timezone.utc), updated_at=datetime.datetime(2023, 9, 27, 21, 8, 26, tzinfo=datetime.timezone.utc)), Challenge(id=219, slug='hidden-treasures-warm-up', name='Hidden Treasures: Warm Up', headline='', description='In the context of human genome sequencing, software pipelines typically involve a wide range of processing elements, including aligning sequencing reads to a reference genome and subsequently identifying variants (differences). One way of assessing the performance of such pipelines is by using well-characterized datasets such as Genome in a Bottle’s NA12878. However, because the existing NGS reference datasets are very limited and have been widely used to train/develop software pipelines, benchmarking of pipeline performance would ideally be done on samples with unknown variants.  This challenge will provide a unique opportunity for participants to investigate the accuracy of their pipelines by testing the ability to find in silico injected variants in FASTQ files from exome sequencing of reference cell lines. It will be a warm up for the community ahead of a more difficult in silico challenge to come in the fall. This challenge will provide users with a FASTQ file of a NA12878 se...', doi='', status=<ChallengeStatus.COMPLETED: 'completed'>, difficulty=<ChallengeDifficulty.INTERMEDIATE: 'intermediate'>, platform=SimpleChallengePlatform(id=6, slug='precision-fda', name='precisionFDA'), website_url='https://precision.fda.gov/challenges/1', avatar_url='', incentives=[<ChallengeIncentive.OTHER: 'other'>], submission_types=[<ChallengeSubmissionType.PREDICTION_FILE: 'prediction_file'>], input_data_types=[], start_date=datetime.date(2017, 7, 17), end_date=datetime.date(2017, 9, 13), starred_count=0, created_at=datetime.datetime(2023, 9, 13, 23, 31, 39, tzinfo=datetime.timezone.utc), updated_at=datetime.datetime(2023, 10, 12, 17, 55, 23, tzinfo=datetime.timezone.utc)), Challenge(id=218, slug='genedisco-challenge', name='GeneDisco Challenge', headline='', description='The GeneDisco challenge is a machine learning community challenge for evaluating batch active learning algorithms for exploring the vast experimental design space in genetic perturbation experiments. Genetic perturbation experiments, using for example CRISPR technologies to perturb the genome, are a vital component of early-stage drug discovery, including target discovery and target validation. The GeneDisco challenge is organized in conjunction with the Machine Learning for Drug Discovery workshop at ICLR-22.', doi='https://doi.org/10.48550/arXiv.2110.11875', status=<ChallengeStatus.COMPLETED: 'completed'>, difficulty=<ChallengeDifficulty.INTERMEDIATE: 'intermediate'>, platform=SimpleChallengePlatform(id=16, slug='evalai', name='EvalAI'), website_url='https://www.gsk.ai/genedisco-challenge/', avatar_url='', incentives=[<ChallengeIncentive.MONETARY: 'monetary'>, <ChallengeIncentive.PUBLICATION: 'publication'>], submission_types=[<ChallengeSubmissionType.CONTAINER_IMAGE: 'container_image'>], input_data_types=[], start_date=datetime.date(2022, 1, 31), end_date=datetime.date(2022, 3, 31), starred_count=0, created_at=datetime.datetime(2023, 9, 13, 17, 20, 30, tzinfo=datetime.timezone.utc), updated_at=datetime.datetime(2023, 10, 19, 23, 32, 43, tzinfo=datetime.timezone.utc)), Challenge(id=217, slug='iclr-computational-geometry-and-topology-challenge-2021', name='ICLR Computational Geometry & Topology Challenge 2021', headline='', description='The purpose of this challenge is to push forward the fields of computational differential geometry and topology, by creating the best data analysis, computational method, or numerical experiment relying on state-of-the-art geometric and topological Python packages.', doi='https://doi.org/10.48550/arXiv.2108.09810', status=<ChallengeStatus.COMPLETED: 'completed'>, difficulty=<ChallengeDifficulty.INTERMEDIATE: 'intermediate'>, platform=SimpleChallengePlatform(id=14, slug='other', name='Other'), website_url='https://github.com/geomstats/challenge-iclr-2021', avatar_url='', incentives=[<ChallengeIncentive.MONETARY: 'monetary'>, <ChallengeIncentive.PUBLICATION: 'publication'>], submission_types=[<ChallengeSubmissionType.NOTEBOOK: 'notebook'>], input_data_types=[], start_date=None, end_date=datetime.date(2021, 5, 2), starred_count=0, created_at=datetime.datetime(2023, 9, 13, 17, 2, 12, tzinfo=datetime.timezone.utc), updated_at=datetime.datetime(2023, 10, 19, 23, 28, 44, tzinfo=datetime.timezone.utc)), Challenge(id=216, slug='iclr-computational-geometry-and-topology-challenge-2022', name='ICLR Computational Geometry & Topology Challenge 2022', headline='', description=\"The purpose of this challenge is to foster reproducible research in geometric (deep) learning, by crowdsourcing the open-source implementation of learning algorithms on manifolds. Participants are asked to contribute code for a published/unpublished algorithm, following Scikit-Learn/Geomstats' or pytorch's APIs and computational primitives, benchmark it, and demonstrate its use in real-world scenarios.\", doi='', status=<ChallengeStatus.COMPLETED: 'completed'>, difficulty=<ChallengeDifficulty.INTERMEDIATE: 'intermediate'>, platform=SimpleChallengePlatform(id=14, slug='other', name='Other'), website_url='https://github.com/geomstats/challenge-iclr-2022', avatar_url='', incentives=[<ChallengeIncentive.MONETARY: 'monetary'>, <ChallengeIncentive.PUBLICATION: 'publication'>], submission_types=[<ChallengeSubmissionType.NOTEBOOK: 'notebook'>], input_data_types=[], start_date=None, end_date=datetime.date(2022, 4, 4), starred_count=0, created_at=datetime.datetime(2023, 9, 13, 16, 54, 6, tzinfo=datetime.timezone.utc), updated_at=datetime.datetime(2023, 10, 19, 23, 28, 44, tzinfo=datetime.timezone.utc)), Challenge(id=215, slug='causalbench-challenge', name='CausalBench Challenge', headline='A machine learning contest for gene network inference from single-cell pertu...', description='Mapping gene-gene interactions in cellular systems is a fundamental step in early-stage drug discovery that helps generate hypotheses on what molecular mechanisms may effectively be targeted by potential future medicines. In the CausalBench Challenge, we invite the machine-learning community to advance the state-of-the-art in deriving gene-gene networks from large-scale real-world perturbational single-cell datasets to improve our ability to glean causal insights into disease-relevant biology.', doi='https://doi.org/10.48550/arXiv.2308.15395', status=<ChallengeStatus.COMPLETED: 'completed'>, difficulty=<ChallengeDifficulty.INTERMEDIATE: 'intermediate'>, platform=SimpleChallengePlatform(id=16, slug='evalai', name='EvalAI'), website_url='https://www.gsk.ai/causalbench-challenge/', avatar_url='', incentives=[<ChallengeIncentive.MONETARY: 'monetary'>, <ChallengeIncentive.PUBLICATION: 'publication'>], submission_types=[<ChallengeSubmissionType.CONTAINER_IMAGE: 'container_image'>], input_data_types=[], start_date=datetime.date(2023, 3, 1), end_date=datetime.date(2023, 4, 21), starred_count=0, created_at=datetime.datetime(2023, 9, 12, 21, 28, 25, tzinfo=datetime.timezone.utc), updated_at=datetime.datetime(2023, 10, 19, 23, 32, 34, tzinfo=datetime.timezone.utc)), Challenge(id=214, slug='dream-6-alternative-splicing-challenge', name='DREAM 6 - Alternative Splicing Challenge', headline='', description='The goal of the mRNA-seq alternative splicing challenge is to assess the accuracy of the reconstruction of alternatively spliced mRNA transcripts from Illumina short-read mRNA-seq. Reconstructed transcripts will be scored against Pacific Biosciences long-read mRNA-seq. The ensuing analysis of the transcriptomes from mandrill and rhinoceros fibroblasts and their derived induced pluripotent stem cells (iPSC), as well as the transcriptome for human Embrionic Stem Cells (hESC) is an opportunity to discover novel biology as well as investigate species-bias of different methods.', doi='', status=<ChallengeStatus.COMPLETED: 'completed'>, difficulty=<ChallengeDifficulty.INTERMEDIATE: 'intermediate'>, platform=SimpleChallengePlatform(id=1, slug='synapse', name='Synapse'), website_url='https://www.synapse.org/#!Synapse:syn2817724/wiki/', avatar_url='', incentives=[], submission_types=[<ChallengeSubmissionType.PREDICTION_FILE: 'prediction_file'>], input_data_types=[], start_date=datetime.date(2011, 8, 9), end_date=None, starred_count=0, created_at=datetime.datetime(2023, 9, 12, 21, 28, 25, tzinfo=datetime.timezone.utc), updated_at=datetime.datetime(2023, 10, 12, 17, 54, 50, tzinfo=datetime.timezone.utc)), Challenge(id=213, slug='dream-6-flowcap2-molecular-classification-of-acute-myeloid-leukemia-challenge', name='DREAM 6 - FlowCAP2 Molecular Classification of Acute Myeloid Leukemia Challenge', headline='The goal of this challenge is to diagnose Acute Myeloid Leukaemia from patie...', description='Flow cytometry (FCM) has been widely used by immunologists and cancer biologists for more than 30 years as a biomedical research tool to distinguish different cell types in mixed populations, based on the expression of cellular markers. It has also become a widely used diagnostic tool for clinicians to identify abnormal cell populations associated with disease. In the last decade, advances in instrumentation and reagent technologies have enabled simultaneous single-cell measurement of tens of surface and intracellular markers, as well as tens of signaling molecules, positioning FCM to play an even bigger role in medicine and systems biology [1,2]. However, the rapid expansion of FCM applications has outpaced the functionality of traditional analysis tools used to interpret FCM data such that scientists are faced with the daunting prospect of manually identifying interesting cell populations in 20 dimensional data from a collection of millions of cells. For these reasons a reliable...', doi='https://doi.org/10.1038/nmeth.2365', status=<ChallengeStatus.COMPLETED: 'completed'>, difficulty=<ChallengeDifficulty.INTERMEDIATE: 'intermediate'>, platform=SimpleChallengePlatform(id=1, slug='synapse', name='Synapse'), website_url='https://www.synapse.org/#!Synapse:syn2887788/wiki/72178', avatar_url='', incentives=[], submission_types=[<ChallengeSubmissionType.PREDICTION_FILE: 'prediction_file'>], input_data_types=[], start_date=datetime.date(2011, 6, 1), end_date=datetime.date(2011, 9, 30), starred_count=0, created_at=datetime.datetime(2023, 9, 12, 21, 28, 19, tzinfo=datetime.timezone.utc), updated_at=datetime.datetime(2023, 10, 12, 17, 54, 47, tzinfo=datetime.timezone.utc)), Challenge(id=212, slug='dream-6-estimation-of-model-parameters-challenge', name='DREAM 6 - Estimation of Model Parameters Challenge', headline='', description='Given the complete model structures (including expressions for the kinetic rate laws) for three gene regulatory networks, participants are asked to develop and/or apply optimization methods, including the selection of the most informative experiments, to accurately estimate parameters and predict outcomes of perturbations in Systems Biology models.', doi='', status=<ChallengeStatus.COMPLETED: 'completed'>, difficulty=<ChallengeDifficulty.INTERMEDIATE: 'intermediate'>, platform=SimpleChallengePlatform(id=1, slug='synapse', name='Synapse'), website_url='https://www.synapse.org/#!Synapse:syn2841366/wiki/71372', avatar_url='', incentives=[], submission_types=[<ChallengeSubmissionType.PREDICTION_FILE: 'prediction_file'>], input_data_types=[], start_date=datetime.date(2011, 6, 1), end_date=datetime.date(2011, 10, 31), starred_count=0, created_at=datetime.datetime(2023, 9, 12, 21, 28, 12, tzinfo=datetime.timezone.utc), updated_at=datetime.datetime(2023, 10, 12, 17, 54, 45, tzinfo=datetime.timezone.utc)), Challenge(id=211, slug='dream-5-systems-genetics-challenge', name='DREAM 5 - Systems Genetics Challenge', headline='Predict disease phenotypes and infer Gene Networks from Systems Genetics data', description='The central goal of systems biology is to gain a predictive, system-level understanding of biological networks. This can be done, for example, by inferring causal networks from observations on a perturbed biological system. An ideal experimental design for causal inference is randomized, multifactorial perturbation. The recognition that the genetic variation in a segregating population represents randomized, multifactorial perturbations (Jansen and Nap (2001), Jansen (2003)) gave rise to Systems Genetics (SG), where a segregating or genetically randomized population is genotyped for many DNA variants, and profiled for phenotypes of interest (e.g. disease phenotypes), gene expression, and potentially other ‘omics’ variables (protein expression, metabolomics, DNA methylation, etc.; Figure 1. Figure 1 was taken from Jansen and Nap (2001)). In this challenge we explore the use of Systems Genetics data for elucidating causal network models among genes, i.e. Gene Networks (DREAM5 SYSGEN...', doi='', status=<ChallengeStatus.COMPLETED: 'completed'>, difficulty=<ChallengeDifficulty.INTERMEDIATE: 'intermediate'>, platform=SimpleChallengePlatform(id=1, slug='synapse', name='Synapse'), website_url='https://www.synapse.org/#!Synapse:syn2820440/wiki/', avatar_url='', incentives=[], submission_types=[<ChallengeSubmissionType.PREDICTION_FILE: 'prediction_file'>], input_data_types=[], start_date=datetime.date(2010, 7, 9), end_date=None, starred_count=0, created_at=datetime.datetime(2023, 9, 12, 21, 28, 10, tzinfo=datetime.timezone.utc), updated_at=datetime.datetime(2023, 10, 12, 17, 54, 42, tzinfo=datetime.timezone.utc)), Challenge(id=210, slug='dream-gene-expression-prediction-challenge', name='DREAM Gene Expression Prediction Challenge', headline='Predict gene expression levels from promoter sequences in eukaryotes', description='The level by which genes are transcribed is determined in large part by the DNA sequence upstream to the gene, known as the promoter region. Although widely studied, we are still far from a quantitative and predictive understanding of how transcriptional regulation is encoded in gene promoters. One obstacle in the field is obtaining accurate measurements of transcription derived by different promoters. To address this, an experimental system was designed to measure the transcription derived by different promoters, all of which are inserted into the same genomic location upstream to a reporter gene -a yellow florescence protein gene (YFP). The challenge consists of the prediction of the promoter activity given a promoter sequence and a specific experimental condition. To study a set of promoters that share many elements of the regulatory program, and thus are suitable for computational learning, the data pertains to promoters of most of the ribosomal protein genes (RP) of yeast (S....', doi='', status=<ChallengeStatus.COMPLETED: 'completed'>, difficulty=<ChallengeDifficulty.INTERMEDIATE: 'intermediate'>, platform=SimpleChallengePlatform(id=1, slug='synapse', name='Synapse'), website_url='https://www.synapse.org/#!Synapse:syn2820426/wiki/71010', avatar_url='', incentives=[], submission_types=[<ChallengeSubmissionType.PREDICTION_FILE: 'prediction_file'>], input_data_types=[], start_date=datetime.date(2010, 7, 9), end_date=None, starred_count=0, created_at=datetime.datetime(2023, 9, 12, 21, 28, tzinfo=datetime.timezone.utc), updated_at=datetime.datetime(2023, 10, 19, 23, 32, 10, tzinfo=datetime.timezone.utc)), Challenge(id=209, slug='dream-5-epitope-antibody-recognition-ear-challenge', name='DREAM 5 - Epitope-Antibody Recognition (EAR) Challenge', headline='Predict the binding specificity of peptide-antibody interactions.', description='Humoral immune responses are mediated through antibodies. About 1010 to 1012 different antigen binding sites called paratopes are generated by genomic recombination. These antibodies are capable to bind to a variety of structures ranging from small molecules to protein complexes, including any posttranslational modification thereof. When studying protein-antibody interactions, two types of epitopes (the region paratopes interact with) are to be distinguished from each other-i) conformational and ii) linear epitopes. All potential linear epitopes of a protein can be represented by short peptides derived from the primary amino acid sequence. These peptides can be synthesized and arrayed on solid supports, e.g. glass slides (see Lorenz et al., 2009 [1]). By incubating these peptide arrays with antibody mixtures such as human serum or plasma, peptides can be determined that interact with antibodies in a specific fashion.', doi='', status=<ChallengeStatus.COMPLETED: 'completed'>, difficulty=<ChallengeDifficulty.INTERMEDIATE: 'intermediate'>, platform=SimpleChallengePlatform(id=1, slug='synapse', name='Synapse'), website_url='https://www.synapse.org/#!Synapse:syn2820433/wiki/71017', avatar_url='', incentives=[], submission_types=[<ChallengeSubmissionType.PREDICTION_FILE: 'prediction_file'>], input_data_types=[], start_date=datetime.date(2010, 6, 9), end_date=None, starred_count=0, created_at=datetime.datetime(2023, 9, 12, 21, 27, 44, tzinfo=datetime.timezone.utc), updated_at=datetime.datetime(2023, 10, 12, 17, 54, 39, tzinfo=datetime.timezone.utc)), Challenge(id=208, slug='dream-5-transcription-factor-dna-motif-recognition-challenge', name='DREAM 5 - Transcription-Factor, DNA-Motif Recognition Challenge', headline='', description='Transcription factors (TFs) control the expression of genes through sequence-specific interactions with genomic DNA. Different TFs bind preferentially to different sequences, with the majority recognizing short (6-12 base), degenerate ‘motifs’. Modeling the sequence specificities of TFs is a central problem in understanding the function and evolution of the genome, because many types of genomic analyses involve scanning for potential TF binding sites. Models of TF binding specificity are also important for understanding the function and evolution of the TFs themselves. The challenge consists of predicting the signal intensities for the remaining TFs.', doi='https://doi.org/10.1038/nbt.2486', status=<ChallengeStatus.COMPLETED: 'completed'>, difficulty=<ChallengeDifficulty.INTERMEDIATE: 'intermediate'>, platform=SimpleChallengePlatform(id=1, slug='synapse', name='Synapse'), website_url='https://www.synapse.org/#!Synapse:syn2887863/wiki/72185', avatar_url='', incentives=[], submission_types=[<ChallengeSubmissionType.PREDICTION_FILE: 'prediction_file'>], input_data_types=[], start_date=datetime.date(2011, 6, 1), end_date=datetime.date(2011, 9, 30), starred_count=0, created_at=datetime.datetime(2023, 9, 12, 21, 27, 41, tzinfo=datetime.timezone.utc), updated_at=datetime.datetime(2023, 10, 12, 17, 54, 36, tzinfo=datetime.timezone.utc)), Challenge(id=207, slug='dream-4-peptide-recognition-domain-prd-specificity-prediction', name='DREAM 4 - Peptide Recognition Domain (PRD) Specificity Prediction', headline='', description='Many important protein-protein interactions are mediated by peptide recognition domains (PRD), which bind short linear sequence motifs in other proteins. For example, SH3 domains typically recognize proline-rich motifs, PDZ domains recognize hydrophobic C-terminal tails, and kinases recognize short sequence regions around a phosphorylatable residue (Pawson, 2003).  Given the sequence of the domains, the challenge consists of predicting a position weight matrix (PWM) that describes the specificity profile of each of the given domains to their target peptides. Any publicly accessible peptide specificity information available for the domain may be used.', doi='', status=<ChallengeStatus.COMPLETED: 'completed'>, difficulty=<ChallengeDifficulty.INTERMEDIATE: 'intermediate'>, platform=SimpleChallengePlatform(id=1, slug='synapse', name='Synapse'), website_url='https://www.synapse.org/#!Synapse:syn2925957/wiki/72976', avatar_url='', incentives=[], submission_types=[<ChallengeSubmissionType.PREDICTION_FILE: 'prediction_file'>], input_data_types=[], start_date=datetime.date(2009, 6, 1), end_date=datetime.date(2009, 10, 31), starred_count=0, created_at=datetime.datetime(2023, 9, 12, 21, 27, 35, tzinfo=datetime.timezone.utc), updated_at=datetime.datetime(2023, 10, 12, 17, 54, 35, tzinfo=datetime.timezone.utc)), Challenge(id=206, slug='dream-3-signaling-response-prediction', name='DREAM 3 - Signaling Response Prediction', headline='Predict missing protein concentrations from a large corpus of measurements', description='Approximately 10,000 intracellular measurements (fluorescence signals proportional to the concentrations of phosphorylated proteins) and extracellular measurements (concentrations of cytokines released in response to cell stimulation) were acquired in human normal hepatocytes and the hepatocellular carcinoma cell line HepG2 cells. The datasets consist of measurements of 17 phospho-proteins (at 0 min, 30 min, and 3 hrs) and 20 cytokines (at 0 min, 3 hrs, and 24 hrs) in two cell types (normal and cancer) after perturbations to the pathway induced by the combinatorial treatment of 7 stimuli and 7 selective inhibitors.', doi='https://doi.org/10.1126%2Fscisignal.2002212', status=<ChallengeStatus.COMPLETED: 'completed'>, difficulty=<ChallengeDifficulty.INTERMEDIATE: 'intermediate'>, platform=SimpleChallengePlatform(id=1, slug='synapse', name='Synapse'), website_url='https://www.synapse.org/#!Synapse:syn2825325/wiki/', avatar_url='', incentives=[], submission_types=[<ChallengeSubmissionType.PREDICTION_FILE: 'prediction_file'>], input_data_types=[], start_date=datetime.date(2009, 3, 9), end_date=None, starred_count=0, created_at=datetime.datetime(2023, 9, 12, 21, 27, 20, tzinfo=datetime.timezone.utc), updated_at=datetime.datetime(2023, 10, 12, 17, 54, 33, tzinfo=datetime.timezone.utc)), Challenge(id=205, slug='dream-4-predictive-signaling-network-modelling', name='DREAM 4 - Predictive Signaling Network Modelling', headline='Cell-type specific high-throughput experimental data', description='This challenge explores the extent to which our current knowledge of signaling pathways, collected from a variety of cell types, agrees with cell-type specific high-throughput experimental data. Specifically, we ask the challenge participants to create a cell-type specific model of signal transduction using the measured activity levels of signaling proteins in HepG2 cell lines. The model, which can leverage prior information encoded in a generic signaling pathway provided in the challenge, should be biologically interpretable as a network, and capable of predicting the outcome of new experiments.', doi='', status=<ChallengeStatus.COMPLETED: 'completed'>, difficulty=<ChallengeDifficulty.INTERMEDIATE: 'intermediate'>, platform=SimpleChallengePlatform(id=1, slug='synapse', name='Synapse'), website_url='https://www.synapse.org/#!Synapse:syn2825304/wiki/71129', avatar_url='', incentives=[], submission_types=[<ChallengeSubmissionType.PREDICTION_FILE: 'prediction_file'>], input_data_types=[], start_date=datetime.date(2009, 3, 9), end_date=None, starred_count=0, created_at=datetime.datetime(2023, 9, 12, 21, 27, 14, tzinfo=datetime.timezone.utc), updated_at=datetime.datetime(2023, 10, 12, 17, 54, 30, tzinfo=datetime.timezone.utc)), Challenge(id=204, slug='dream-3-gene-expression-prediction', name='DREAM 3 - Gene Expression Prediction', headline='', description='Gene expression time course data is provided for four different strains of yeast (S. Cerevisiae), after perturbation of the cells. The challenge is to predict the rank order of induction/repression of a small subset of genes (the prediction targets) in one of the four strains, given complete data for three of the strains, and data for all genes except the prediction targets in the other strain. You are also allowed to use any information that is in the public domain and are expected to be forthcoming about what information was used.', doi='', status=<ChallengeStatus.COMPLETED: 'completed'>, difficulty=<ChallengeDifficulty.INTERMEDIATE: 'intermediate'>, platform=SimpleChallengePlatform(id=1, slug='synapse', name='Synapse'), website_url='https://www.synapse.org/#!Synapse:syn3033083/wiki/74369', avatar_url='', incentives=[], submission_types=[<ChallengeSubmissionType.PREDICTION_FILE: 'prediction_file'>], input_data_types=[], start_date=datetime.date(2008, 6, 1), end_date=datetime.date(2008, 10, 31), starred_count=0, created_at=datetime.datetime(2023, 9, 12, 21, 27, 12, tzinfo=datetime.timezone.utc), updated_at=datetime.datetime(2023, 10, 12, 17, 54, 10, tzinfo=datetime.timezone.utc)), Challenge(id=203, slug='dream-3-signaling-cascade-identification', name='DREAM 3 - Signaling Cascade Identification', headline='', description='The concentration of four intracellular proteins or phospho-proteins (X1, X2, X3 and X4) participating in a signaling cascade were measured in about 104 cells by antibody staining and flow cytometry. The idea of this challenge is to explore what key aspects of the dynamics and topology of interactions of a signaling cascade can be inferred from incomplete flow cytometry data.', doi='', status=<ChallengeStatus.COMPLETED: 'completed'>, difficulty=<ChallengeDifficulty.INTERMEDIATE: 'intermediate'>, platform=SimpleChallengePlatform(id=1, slug='synapse', name='Synapse'), website_url='https://www.synapse.org/#!Synapse:syn3033068/wiki/74362', avatar_url='', incentives=[], submission_types=[<ChallengeSubmissionType.PREDICTION_FILE: 'prediction_file'>], input_data_types=[], start_date=datetime.date(2008, 6, 1), end_date=datetime.date(2008, 10, 31), starred_count=0, created_at=datetime.datetime(2023, 9, 12, 21, 27, 4, tzinfo=datetime.timezone.utc), updated_at=datetime.datetime(2023, 10, 12, 17, 54, 8, tzinfo=datetime.timezone.utc)), Challenge(id=202, slug='dream-2-synthetic-five-gene-network-inference', name='DREAM 2 - Synthetic Five-Gene Network Inference', headline='', description='A synthetic-biology network consisting of 5 interacting genes was created and transfected to an in-vivo model organism. The challenge consists of predicting the connectivity of the five-gene network from in-vivo measurements.', doi='https://doi.org/10.1016/j.cell.2009.01.055', status=<ChallengeStatus.COMPLETED: 'completed'>, difficulty=<ChallengeDifficulty.INTERMEDIATE: 'intermediate'>, platform=SimpleChallengePlatform(id=1, slug='synapse', name='Synapse'), website_url='https://www.synapse.org/#!Synapse:syn3034869/wiki/74411', avatar_url='', incentives=[], submission_types=[<ChallengeSubmissionType.PREDICTION_FILE: 'prediction_file'>], input_data_types=[], start_date=datetime.date(2007, 6, 20), end_date=datetime.date(2007, 10, 31), starred_count=0, created_at=datetime.datetime(2023, 9, 12, 21, 26, 56, tzinfo=datetime.timezone.utc), updated_at=datetime.datetime(2023, 10, 12, 17, 54, 5, tzinfo=datetime.timezone.utc)), Challenge(id=201, slug='dream-2-genome-scale-network-inference', name='DREAM 2 - Genome-Scale Network Inference', headline='', description='A panel of single-channel microarrays was collected for a particular microorganism, including some already published and some in-print data. The data was appropriately normalized (to the logarithmic scale). The challenge consists of reconstructing a genome-scale transcriptional network for this organism. The accuracy of network inference will be judged using chromatin precipitation and otherwise experimentally verified Transcription Factor (TF)-target interactions.', doi='https://doi.org/10.1371/journal.pbio.0050008', status=<ChallengeStatus.COMPLETED: 'completed'>, difficulty=<ChallengeDifficulty.INTERMEDIATE: 'intermediate'>, platform=SimpleChallengePlatform(id=1, slug='synapse', name='Synapse'), website_url='https://www.synapse.org/#!Synapse:syn3034894/wiki/74418', avatar_url='', incentives=[], submission_types=[<ChallengeSubmissionType.PREDICTION_FILE: 'prediction_file'>], input_data_types=[], start_date=datetime.date(2007, 6, 5), end_date=datetime.date(2007, 10, 31), starred_count=0, created_at=datetime.datetime(2023, 9, 12, 21, 26, 34, tzinfo=datetime.timezone.utc), updated_at=datetime.datetime(2023, 10, 12, 17, 54, 3, tzinfo=datetime.timezone.utc)), Challenge(id=200, slug='dream-2-protein-protein-interaction-network-inference', name='DREAM 2 - Protein-Protein Interaction Network Inference', headline='Predict a PPI network of 47 proteins', description='For many pairs of bait and prey genes, yeast protein-protein interactions were tested in an unbiased fashion using a high saturation, high-stringency variant of the yeast two-hybrid (Y2H) method. A high confidence subset of gene pairs that were found to interact in at least three repetitions of the experiment but that hadn’t been reported in the literature was extracted. There were 47 yeast genes involved in these pairs. Including self interactions, there are a total of 47*48/2 possible pairs of genes that can be formed with these 47 genes. As mentioned above some of these gene pairs were seen to consistently interact in at least three repetitions of the Y2H experiments-these gene pairs form the gold standard positive set. A second set among these gene pairs were seen never to interact in repeated experiments and were not reported as interacting in the literature; we call this the gold standard negative set. Finally in a third set of gene pairs, which we shall call the undecided s...', doi='https://doi.org/10.1126/science.1158684', status=<ChallengeStatus.COMPLETED: 'completed'>, difficulty=<ChallengeDifficulty.INTERMEDIATE: 'intermediate'>, platform=SimpleChallengePlatform(id=1, slug='synapse', name='Synapse'), website_url='https://www.synapse.org/#!Synapse:syn2825374/wiki/', avatar_url='', incentives=[], submission_types=[<ChallengeSubmissionType.PREDICTION_FILE: 'prediction_file'>], input_data_types=[], start_date=datetime.date(2007, 5, 24), end_date=None, starred_count=0, created_at=datetime.datetime(2023, 9, 12, 21, 26, 28, tzinfo=datetime.timezone.utc), updated_at=datetime.datetime(2023, 10, 12, 17, 54, tzinfo=datetime.timezone.utc)), Challenge(id=199, slug='dream-2-bcl6-transcriptomic-target-prediction', name='DREAM 2 - BCL6 Transcriptomic Target Prediction', headline='', description='A number of potential transcriptional targets of BCL6, a gene that encodes for a transcription factor active in B cells, have been identified with ChIP-on-chip data and functionally validated by perturbing the BCL6 pathway with CD40 and anti-IgM, and by over-expressing exogenous BCL6 in Ramos cell. We subselected a number of targets found in this way (the gold standard positive set), and added a number decoys (genes that have no evidence of being BCL6 targets, named the gold standard negative set), compiling a list of 200 genes in total. Given this list of 200 genes, the challenge consists of identifying which ones are the true targets and which ones are the decoys, using an independent panel of gene topic_3170.', doi='https://doi.org/10.1073/pnas.0437996100', status=<ChallengeStatus.COMPLETED: 'completed'>, difficulty=<ChallengeDifficulty.INTERMEDIATE: 'intermediate'>, platform=SimpleChallengePlatform(id=1, slug='synapse', name='Synapse'), website_url='https://www.synapse.org/#!Synapse:syn3034857/wiki/', avatar_url='', incentives=[], submission_types=[<ChallengeSubmissionType.PREDICTION_FILE: 'prediction_file'>], input_data_types=[], start_date=datetime.date(2007, 4, 19), end_date=None, starred_count=0, created_at=datetime.datetime(2023, 9, 12, 21, 26, 22, tzinfo=datetime.timezone.utc), updated_at=datetime.datetime(2023, 10, 12, 17, 53, 55, tzinfo=datetime.timezone.utc)), Challenge(id=197, slug='nlp-sandbox-contact-annotation', name='NLP Sandbox Contact Annotation', headline='Identify contact information in clinical notes.', description='An NLP Sandbox contact annotator takes as input a clinical note and outputs a list of predicted contact annotations found in the clinical note.', doi='https://doi.org/10.7303/syn22277123', status=<ChallengeStatus.COMPLETED: 'completed'>, difficulty=<ChallengeDifficulty.INTERMEDIATE: 'intermediate'>, platform=SimpleChallengePlatform(id=1, slug='synapse', name='Synapse'), website_url='https://www.synapse.org/#!Synapse:syn22277123/wiki/609134', avatar_url='', incentives=[<ChallengeIncentive.PUBLICATION: 'publication'>], submission_types=[<ChallengeSubmissionType.CONTAINER_IMAGE: 'container_image'>], input_data_types=[], start_date=datetime.date(2021, 6, 4), end_date=datetime.date(2023, 9, 1), starred_count=0, created_at=datetime.datetime(2023, 9, 8, 16, 44, 22, tzinfo=datetime.timezone.utc), updated_at=datetime.datetime(2023, 9, 28, 23, 59, 21, tzinfo=datetime.timezone.utc)), Challenge(id=198, slug='nlp-sandbox-id-annotation', name='NLP Sandbox ID Annotation', headline='Identify identifiers in clinical notes.', description='An NLP Sandbox ID annotator takes as input a clinical note and outputs a list of predicted ID annotations found in the clinical note.', doi='https://doi.org/10.7303/syn22277123', status=<ChallengeStatus.COMPLETED: 'completed'>, difficulty=<ChallengeDifficulty.INTERMEDIATE: 'intermediate'>, platform=SimpleChallengePlatform(id=1, slug='synapse', name='Synapse'), website_url='https://www.synapse.org/#!Synapse:syn22277123/wiki/609134', avatar_url='', incentives=[<ChallengeIncentive.PUBLICATION: 'publication'>], submission_types=[<ChallengeSubmissionType.CONTAINER_IMAGE: 'container_image'>], input_data_types=[], start_date=datetime.date(2021, 6, 4), end_date=datetime.date(2023, 9, 1), starred_count=0, created_at=datetime.datetime(2023, 9, 8, 16, 44, 22, tzinfo=datetime.timezone.utc), updated_at=datetime.datetime(2023, 9, 28, 23, 59, 22, tzinfo=datetime.timezone.utc)), Challenge(id=196, slug='nlp-sandbox-location-annotation', name='NLP Sandbox Location Annotation', headline='Identify location information in clinical notes.', description='An NLP Sandbox Location Annotator takes as input a clinical note and outputs a list of predicted location annotations found in the clinical note.', doi='https://doi.org/10.7303/syn22277123', status=<ChallengeStatus.COMPLETED: 'completed'>, difficulty=<ChallengeDifficulty.INTERMEDIATE: 'intermediate'>, platform=SimpleChallengePlatform(id=1, slug='synapse', name='Synapse'), website_url='https://www.synapse.org/#!Synapse:syn22277123/wiki/609134', avatar_url='', incentives=[<ChallengeIncentive.PUBLICATION: 'publication'>], submission_types=[<ChallengeSubmissionType.CONTAINER_IMAGE: 'container_image'>], input_data_types=[], start_date=datetime.date(2021, 6, 4), end_date=datetime.date(2023, 9, 1), starred_count=0, created_at=datetime.datetime(2023, 9, 8, 16, 44, 21, tzinfo=datetime.timezone.utc), updated_at=datetime.datetime(2023, 9, 28, 23, 59, 21, tzinfo=datetime.timezone.utc)), Challenge(id=195, slug='nlp-sandbox-person-name-annotation', name='NLP Sandbox Person Name Annotation', headline='Identify person names in clinical notes.', description='An NLP Sandbox Person Name Annotator takes as input a clinical note and outputs a list of predicted person name annotations found in the clinical note.', doi='https://doi.org/10.7303/syn22277123', status=<ChallengeStatus.COMPLETED: 'completed'>, difficulty=<ChallengeDifficulty.INTERMEDIATE: 'intermediate'>, platform=SimpleChallengePlatform(id=1, slug='synapse', name='Synapse'), website_url='https://www.synapse.org/#!Synapse:syn22277123/wiki/609134', avatar_url='', incentives=[<ChallengeIncentive.PUBLICATION: 'publication'>], submission_types=[<ChallengeSubmissionType.CONTAINER_IMAGE: 'container_image'>], input_data_types=[], start_date=datetime.date(2021, 6, 4), end_date=datetime.date(2023, 9, 1), starred_count=0, created_at=datetime.datetime(2023, 9, 8, 16, 44, 20, tzinfo=datetime.timezone.utc), updated_at=datetime.datetime(2023, 9, 28, 23, 59, 20, tzinfo=datetime.timezone.utc)), Challenge(id=194, slug='nlp-sandbox-date-annotation', name='NLP Sandbox Date Annotation', headline='Identify dates in clinical notes.', description='An NLP Sandbox Date Annotator takes as input a clinical note and outputs a list of predicted date annotations found in the clinical note.', doi='https://doi.org/10.7303/syn22277123', status=<ChallengeStatus.COMPLETED: 'completed'>, difficulty=<ChallengeDifficulty.INTERMEDIATE: 'intermediate'>, platform=SimpleChallengePlatform(id=1, slug='synapse', name='Synapse'), website_url='https://www.synapse.org/#!Synapse:syn22277123/wiki/609134', avatar_url='', incentives=[<ChallengeIncentive.PUBLICATION: 'publication'>], submission_types=[<ChallengeSubmissionType.CONTAINER_IMAGE: 'container_image'>], input_data_types=[], start_date=datetime.date(2021, 6, 4), end_date=datetime.date(2023, 9, 1), starred_count=0, created_at=datetime.datetime(2023, 8, 25, 16, 45, 22, tzinfo=datetime.timezone.utc), updated_at=datetime.datetime(2023, 9, 28, 23, 59, 2, tzinfo=datetime.timezone.utc)), Challenge(id=193, slug='dream-5-network-inference-challenge', name='DREAM 5 - Network Inference Challenge', headline='The goal of this Network Inference Challenge is to reverse engineer gene reg...', description='The goal of this Network Inference Challenge is to reverse engineer gene regulatory networks from gene topic_3170sets. Participants are given four microarray compendia and are challenged to infer the structure of the underlying transcriptional regulatory networks. Three of the four compendia were obtained from microorganisms, some of which are pathogens of clinical relevance. The fourth compendium is based on an in-silico (i.e., simulated) network. Each compendium consists of hundreds of microarray experiments, which include a wide range of genetic, drug, and environmental perturbations (or in the in-silico network case, simulations thereof). Network predictions will be evaluated on a subset of known interactions for each organism, or on the known network for the in-silico case.', doi='https://doi.org/10.1038/nmeth.2016', status=<ChallengeStatus.COMPLETED: 'completed'>, difficulty=<ChallengeDifficulty.INTERMEDIATE: 'intermediate'>, platform=SimpleChallengePlatform(id=1, slug='synapse', name='Synapse'), website_url='https://www.synapse.org/#!Synapse:syn2787209/wiki/70349', avatar_url='', incentives=[<ChallengeIncentive.PUBLICATION: 'publication'>, <ChallengeIncentive.SPEAKING_ENGAGEMENT: 'speaking_engagement'>], submission_types=[<ChallengeSubmissionType.PREDICTION_FILE: 'prediction_file'>], input_data_types=[], start_date=datetime.date(2010, 6, 9), end_date=datetime.date(2010, 10, 31), starred_count=0, created_at=datetime.datetime(2023, 8, 25, 16, 43, 43, tzinfo=datetime.timezone.utc), updated_at=datetime.datetime(2023, 10, 12, 17, 54, 57, tzinfo=datetime.timezone.utc)), Challenge(id=192, slug='dream-4-in-silico-network-challenge', name='DREAM 4 - In Silico Network Challenge', headline='The goal of the in silico network challenge is to reverse engineer gene regu...', description='The goal of the in silico network challenge is to reverse engineer gene regulation networks from simulated steady-state and time-series data. Participants are challenged to infer the network structure from the given in silico gene topic_3170sets. Optionally, participants may also predict the response of the networks to a set of novel perturbations that were not included in the provided datasets.', doi='https://doi.org/10.1073/pnas.0913357107', status=<ChallengeStatus.COMPLETED: 'completed'>, difficulty=<ChallengeDifficulty.INTERMEDIATE: 'intermediate'>, platform=SimpleChallengePlatform(id=1, slug='synapse', name='Synapse'), website_url='https://www.synapse.org/#!Synapse:syn3049712/wiki/74628', avatar_url='', incentives=[<ChallengeIncentive.PUBLICATION: 'publication'>, <ChallengeIncentive.SPEAKING_ENGAGEMENT: 'speaking_engagement'>], submission_types=[<ChallengeSubmissionType.PREDICTION_FILE: 'prediction_file'>], input_data_types=[], start_date=datetime.date(2009, 6, 9), end_date=None, starred_count=0, created_at=datetime.datetime(2023, 8, 25, 16, 43, 42, tzinfo=datetime.timezone.utc), updated_at=datetime.datetime(2023, 10, 12, 17, 55, tzinfo=datetime.timezone.utc)), Challenge(id=191, slug='dream-3-in-silico-network-challenge', name='DREAM 3 - In Silico Network Challenge', headline='The goal of the in silico challenges is the reverse engineering of gene netw...', description='The goal of the in silico challenges is the reverse engineering of gene networks from steady state and time series data. Participants are challenged to predict the directed unsigned network topology from the given in silico generated gene topic_3170sets.', doi='https://doi.org/10.1089/cmb.2008.09TT', status=<ChallengeStatus.COMPLETED: 'completed'>, difficulty=<ChallengeDifficulty.INTERMEDIATE: 'intermediate'>, platform=SimpleChallengePlatform(id=1, slug='synapse', name='Synapse'), website_url='https://www.synapse.org/#!Synapse:syn2853594/wiki/71567', avatar_url='', incentives=[<ChallengeIncentive.PUBLICATION: 'publication'>, <ChallengeIncentive.SPEAKING_ENGAGEMENT: 'speaking_engagement'>], submission_types=[<ChallengeSubmissionType.PREDICTION_FILE: 'prediction_file'>], input_data_types=[], start_date=datetime.date(2008, 6, 9), end_date=None, starred_count=0, created_at=datetime.datetime(2023, 8, 25, 16, 43, 41, tzinfo=datetime.timezone.utc), updated_at=datetime.datetime(2023, 10, 12, 17, 55, 2, tzinfo=datetime.timezone.utc)), Challenge(id=190, slug='dream-2-in-silico-network-inference', name='DREAM 2 - In Silico Network Inference', headline='Predicting the connectivity and properties of in-silico networks.', description='Three in-silico networks were created and endowed with a dynamics that simulate biological interactions. The challenge consists of predicting the connectivity and some of the properties of one or more of these three networks.', doi='', status=<ChallengeStatus.COMPLETED: 'completed'>, difficulty=<ChallengeDifficulty.INTERMEDIATE: 'intermediate'>, platform=SimpleChallengePlatform(id=1, slug='synapse', name='Synapse'), website_url='https://www.synapse.org/#!Synapse:syn2825394/wiki/71150', avatar_url='', incentives=[<ChallengeIncentive.PUBLICATION: 'publication'>, <ChallengeIncentive.SPEAKING_ENGAGEMENT: 'speaking_engagement'>], submission_types=[<ChallengeSubmissionType.PREDICTION_FILE: 'prediction_file'>], input_data_types=[], start_date=datetime.date(2007, 3, 25), end_date=None, starred_count=0, created_at=datetime.datetime(2023, 8, 24, 18, 54, 5, tzinfo=datetime.timezone.utc), updated_at=datetime.datetime(2023, 10, 12, 17, 55, 3, tzinfo=datetime.timezone.utc)), Challenge(id=189, slug='predicting-high-risk-breast-cancer-phase-2', name='Predicting High Risk Breast Cancer - Phase 2', headline='Predicting High Risk Breast Cancer-a Nightingale OS & AHLI data challenge', description='Every year, 40 million women get a mammogram; some go on to have an invasive biopsy to better examine a concerning area. Underneath these routine tests lies a deep—and disturbing—mystery. Since the 1990s, we have found far more ‘cancers’, which has in turn prompted vastly more surgical procedures and chemotherapy. But death rates from metastatic breast cancer have hardly changed.  When a pathologist looks at a biopsy slide, she is looking for known signs of cancer-tubules, cells with atypical looking nuclei, evidence of rapid cell division. These features, first identified in 1928, still underlie critical decisions today-which women must receive urgent treatment with surgery and chemotherapy? And which can be prescribed “watchful waiting”, sparing them invasive procedures for cancers that would not harm them?  There is already evidence that algorithms can predict which cancers will metastasize and harm patients on the basis of the biopsy image. Fascinatingly, these algorithms also...', doi='', status=<ChallengeStatus.COMPLETED: 'completed'>, difficulty=<ChallengeDifficulty.INTERMEDIATE: 'intermediate'>, platform=SimpleChallengePlatform(id=15, slug='nightingale-os', name='Nightingale OS'), website_url='https://app.nightingalescience.org/contests/vd8g98zv9w0p', avatar_url='', incentives=[<ChallengeIncentive.MONETARY: 'monetary'>, <ChallengeIncentive.PUBLICATION: 'publication'>, <ChallengeIncentive.SPEAKING_ENGAGEMENT: 'speaking_engagement'>], submission_types=[<ChallengeSubmissionType.PREDICTION_FILE: 'prediction_file'>], input_data_types=[], start_date=datetime.date(2023, 2, 3), end_date=datetime.date(2023, 5, 13), starred_count=0, created_at=datetime.datetime(2023, 8, 22, 17, 7, 1, tzinfo=datetime.timezone.utc), updated_at=datetime.datetime(2023, 10, 12, 17, 55, 8, tzinfo=datetime.timezone.utc)), Challenge(id=188, slug='predicting-high-risk-breast-cancer-phase-1', name='Predicting High Risk Breast Cancer - Phase 1', headline='Predicting High Risk Breast Cancer-a Nightingale OS & AHLI data challenge', description='Every year, 40 million women get a mammogram; some go on to have an invasive biopsy to better examine a concerning area. Underneath these routine tests lies a deep—and disturbing—mystery. Since the 1990s, we have found far more ‘cancers’, which has in turn prompted vastly more surgical procedures and chemotherapy. But death rates from metastatic breast cancer have hardly changed. When a pathologist looks at a biopsy slide, she is looking for known signs of cancer-tubules, cells with atypical looking nuclei, evidence of rapid cell division. These features, first identified in 1928, still underlie critical decisions today-which women must receive urgent treatment with surgery and chemotherapy? And which can be prescribed “watchful waiting”, sparing them invasive procedures for cancers that would not harm them? There is already evidence that algorithms can predict which cancers will metastasize and harm patients on the basis of the biopsy image. Fascinatingly, these algorithms also h...', doi='', status=<ChallengeStatus.COMPLETED: 'completed'>, difficulty=<ChallengeDifficulty.INTERMEDIATE: 'intermediate'>, platform=SimpleChallengePlatform(id=15, slug='nightingale-os', name='Nightingale OS'), website_url='https://app.nightingalescience.org/contests/3jmp2y128nxd', avatar_url='', incentives=[<ChallengeIncentive.MONETARY: 'monetary'>, <ChallengeIncentive.PUBLICATION: 'publication'>, <ChallengeIncentive.SPEAKING_ENGAGEMENT: 'speaking_engagement'>], submission_types=[<ChallengeSubmissionType.PREDICTION_FILE: 'prediction_file'>], input_data_types=[], start_date=datetime.date(2022, 6, 1), end_date=datetime.date(2023, 1, 12), starred_count=0, created_at=datetime.datetime(2023, 8, 22, 17, 7, tzinfo=datetime.timezone.utc), updated_at=datetime.datetime(2023, 10, 12, 17, 55, 10, tzinfo=datetime.timezone.utc)), Challenge(id=187, slug='the-veterans-cardiac-health-and-ai-model-predictions-v-champs', name='The Veterans Cardiac Health and AI Model Predictions (V-CHAMPS)', headline='The Veterans Health Administration Innovation Ecosystem, the Digital Health ...', description='To better understand the risk and protective factors in the Veteran population, the VHA IE and its collaborating partners are calling upon the public to develop AI/ML models to predict cardiovascular health outcomes, including readmission and mortality, using synthetically generated Veteran health records.  The Challenge consists of two Phases-Phase 1 is focused on synthetic data. In this Phase of the Challenge, AI/ML models will be developed by Challenge participants and trained and tested on the synthetic data sets provided to them, with a view towards predicting outcome variables for Veterans who have been diagnosed with chronic heart failure (please note that in Phase 1, the data is synthetic Veteran health records). Phase 2 will focus on validating and further exploring the limits of the AI/ML models. During this Phase, high-performing AI/ML models from Phase 1 will be brought into the VA system and validated on the real-world Veterans health data within the VHA. These models...', doi='', status=<ChallengeStatus.COMPLETED: 'completed'>, difficulty=<ChallengeDifficulty.INTERMEDIATE: 'intermediate'>, platform=SimpleChallengePlatform(id=6, slug='precision-fda', name='precisionFDA'), website_url='https://precision.fda.gov/challenges/31', avatar_url='', incentives=[<ChallengeIncentive.PUBLICATION: 'publication'>, <ChallengeIncentive.SPEAKING_ENGAGEMENT: 'speaking_engagement'>], submission_types=[<ChallengeSubmissionType.PREDICTION_FILE: 'prediction_file'>, <ChallengeSubmissionType.OTHER: 'other'>], input_data_types=[], start_date=datetime.date(2023, 5, 25), end_date=datetime.date(2023, 8, 2), starred_count=0, created_at=datetime.datetime(2023, 8, 10, 21, 41, 10, tzinfo=datetime.timezone.utc), updated_at=datetime.datetime(2023, 9, 28, 23, 25, 45, tzinfo=datetime.timezone.utc)), Challenge(id=186, slug='making-sense-of-electronic-health-record-ehr-race-and-ethnicity-data', name='Making Sense of Electronic Health Record (EHR) Race and Ethnicity Data', headline='The US Food and Drug Administration (FDA) calls on stakeholders, including t...', description='The urgency of the coronavirus disease 2019 (COVID-19) pandemic has heightened interest in the use of real-world data (RWD) to obtain timely information about patients and populations and has focused attention on EHRs. The pandemic has also heightened awareness of long-standing racial and ethnic health disparities along a continuum from underlying social determinants of health, exposure to risk, access to insurance and care, quality of care, and responses to treatments. This highlighted the potential that EHRs can be used to describe and contribute to our understanding of racial and ethnic health disparities and their solutions. The OMB Revisions to the Standards for the Classification of Federal Data on Race and Ethnicity provides minimum standards for maintaining, collecting, and presenting data on race and ethnicity for all Federal reporting purposes, and defines the two separate constructs of race and ethnicity.', doi='', status=<ChallengeStatus.COMPLETED: 'completed'>, difficulty=<ChallengeDifficulty.INTERMEDIATE: 'intermediate'>, platform=SimpleChallengePlatform(id=6, slug='precision-fda', name='precisionFDA'), website_url='https://precision.fda.gov/challenges/30', avatar_url='', incentives=[<ChallengeIncentive.OTHER: 'other'>], submission_types=[<ChallengeSubmissionType.NOTEBOOK: 'notebook'>], input_data_types=[], start_date=datetime.date(2023, 5, 31), end_date=datetime.date(2023, 6, 23), starred_count=0, created_at=datetime.datetime(2023, 8, 10, 18, 28, 6, tzinfo=datetime.timezone.utc), updated_at=datetime.datetime(2023, 10, 10, 19, 53, 12, tzinfo=datetime.timezone.utc)), Challenge(id=185, slug='circle-of-willis-intracranial-artery-classification-and-quantification-challenge-2023', name='Circle of Willis Intracranial Artery Classification and Quantification Challenge 2023', headline='', description='The purpose of this challenge is to compare automatic methods for classification of the circle of Willis (CoW) configuration and quantification of the CoW major artery diameters and bifurcation angles.', doi='', status=<ChallengeStatus.COMPLETED: 'completed'>, difficulty=<ChallengeDifficulty.INTERMEDIATE: 'intermediate'>, platform=SimpleChallengePlatform(id=14, slug='other', name='Other'), website_url='https://crown.isi.uu.nl/', avatar_url='', incentives=[<ChallengeIncentive.PUBLICATION: 'publication'>, <ChallengeIncentive.SPEAKING_ENGAGEMENT: 'speaking_engagement'>], submission_types=[<ChallengeSubmissionType.CONTAINER_IMAGE: 'container_image'>], input_data_types=[], start_date=datetime.date(2023, 5, 1), end_date=datetime.date(2023, 8, 15), starred_count=0, created_at=datetime.datetime(2023, 8, 9, 22, 13, 24, tzinfo=datetime.timezone.utc), updated_at=datetime.datetime(2023, 9, 28, 23, 24, 54, tzinfo=datetime.timezone.utc)), Challenge(id=184, slug='topcow23', name='Topology-Aware Anatomical Segmentation of the Circle of Willis for CTA and MRA', headline='', description='The aim of the challenge is to extract the CoW angio-architecture from 3D angiographic imaging by segmentation of the vessel components. There are two sub-tasks-binary segmentation of CoW vessels, and multi-class CoW anatomical segmentation. We release a new dataset of joint-modalities, CTA and MRA of the same patient cohort, both with annotations of the anatomy of CoW. Our challenge has two tracks for the same segmentation task, namely CTA track and MRA track. We made use of the clinical information from both modalities during our annotation. And participants can pick whichever modality they want, both CTA and MRA, and choose to tackle the task for either modality.', doi='', status=<ChallengeStatus.COMPLETED: 'completed'>, difficulty=<ChallengeDifficulty.INTERMEDIATE: 'intermediate'>, platform=SimpleChallengePlatform(id=5, slug='grand-challenge', name='Grand Challenge'), website_url='https://topcow23.grand-challenge.org/', avatar_url='', incentives=[<ChallengeIncentive.PUBLICATION: 'publication'>, <ChallengeIncentive.SPEAKING_ENGAGEMENT: 'speaking_engagement'>, <ChallengeIncentive.OTHER: 'other'>], submission_types=[<ChallengeSubmissionType.CONTAINER_IMAGE: 'container_image'>], input_data_types=[], start_date=datetime.date(2023, 8, 20), end_date=datetime.date(2023, 9, 25), starred_count=0, created_at=datetime.datetime(2023, 8, 9, 17, 16, 22, tzinfo=datetime.timezone.utc), updated_at=datetime.datetime(2023, 9, 28, 23, 24, 41, tzinfo=datetime.timezone.utc)), Challenge(id=183, slug='chaimeleon', name='CHAIMELEON Open Challenges', headline='', description='The CHAIMELEON Open Challenges is a competition designed to train and refine AI models to answer clinical questions about five types of cancer-prostate, lung, breast, colon, and rectal. Participants are challenged to collaborate and develop innovative AI-powered solutions that can significantly impact cancer diagnosis, management, and treatment. They will be evaluated considering a balance between the performance of their AI algorithms to predict different clinical endpoints such as disease staging, treatment response or progression free survival and their trustworthiness.  The challenges are open to the whole scientific and tech community interested in AI. They are a unique opportunity to showcase how AI can be used to advance medical research and improve patient outcomes within the CHAIMELEON project.', doi='', status=<ChallengeStatus.UPCOMING: 'upcoming'>, difficulty=<ChallengeDifficulty.INTERMEDIATE: 'intermediate'>, platform=SimpleChallengePlatform(id=5, slug='grand-challenge', name='Grand Challenge'), website_url='https://chaimeleon.grand-challenge.org/', avatar_url='', incentives=[<ChallengeIncentive.MONETARY: 'monetary'>, <ChallengeIncentive.PUBLICATION: 'publication'>], submission_types=[<ChallengeSubmissionType.CONTAINER_IMAGE: 'container_image'>], input_data_types=[], start_date=None, end_date=datetime.date(2023, 12, 31), starred_count=0, created_at=datetime.datetime(2023, 8, 9, 17, 13, 9, tzinfo=datetime.timezone.utc), updated_at=datetime.datetime(2023, 10, 10, 19, 53, 10, tzinfo=datetime.timezone.utc)), Challenge(id=182, slug='tlvmc-parkinsons-freezing-gait-prediction', name=\"Parkinson's Freezing of Gait Prediction\", headline='Event detection from wearable sensor data', description='The goal of this competition is to detect freezing of gait (FOG), a debilitating symptom that afflicts many people with Parkinson’s disease. You will develop a machine learning model trained on data collected from a wearable 3D lower back sensor. Your work will help researchers better understand when and why FOG episodes occur. This will improve the ability of medical professionals to optimally evaluate, monitor, and ultimately, prevent FOG events.', doi='', status=<ChallengeStatus.COMPLETED: 'completed'>, difficulty=<ChallengeDifficulty.INTERMEDIATE: 'intermediate'>, platform=SimpleChallengePlatform(id=8, slug='kaggle', name='Kaggle'), website_url='https://www.kaggle.com/competitions/tlvmc-parkinsons-freezing-gait-prediction', avatar_url='', incentives=[<ChallengeIncentive.MONETARY: 'monetary'>, <ChallengeIncentive.SPEAKING_ENGAGEMENT: 'speaking_engagement'>], submission_types=[<ChallengeSubmissionType.NOTEBOOK: 'notebook'>], input_data_types=[], start_date=datetime.date(2023, 3, 9), end_date=datetime.date(2023, 6, 8), starred_count=0, created_at=datetime.datetime(2023, 8, 8, 19, 47, 54, tzinfo=datetime.timezone.utc), updated_at=datetime.datetime(2023, 10, 10, 19, 53, 8, tzinfo=datetime.timezone.utc)), Challenge(id=181, slug='recursion-cellular-image-classification', name='Recursion Cellular Image Classification', headline='CellSignal-Disentangling biological signal from experimental noise in cellul...', description='This competition will have you disentangling experimental noise from real biological signals. Your entry will classify images of cells under one of 1,108 different genetic perturbations. You can help eliminate the noise introduced by technical execution and environmental variation between experiments. If successful, you could dramatically improve the industry’s ability to model cellular images according to their relevant biology. In turn, applying AI could greatly decrease the cost of treatments, and ensure these treatments get to patients faster.', doi='', status=<ChallengeStatus.COMPLETED: 'completed'>, difficulty=<ChallengeDifficulty.INTERMEDIATE: 'intermediate'>, platform=SimpleChallengePlatform(id=8, slug='kaggle', name='Kaggle'), website_url='https://www.kaggle.com/competitions/recursion-cellular-image-classification', avatar_url='', incentives=[<ChallengeIncentive.MONETARY: 'monetary'>, <ChallengeIncentive.SPEAKING_ENGAGEMENT: 'speaking_engagement'>, <ChallengeIncentive.OTHER: 'other'>], submission_types=[<ChallengeSubmissionType.NOTEBOOK: 'notebook'>], input_data_types=[], start_date=datetime.date(2019, 6, 27), end_date=datetime.date(2019, 9, 26), starred_count=0, created_at=datetime.datetime(2023, 8, 8, 19, 38, 42, tzinfo=datetime.timezone.utc), updated_at=datetime.datetime(2023, 10, 10, 19, 53, 5, tzinfo=datetime.timezone.utc)), Challenge(id=180, slug='lish-moa', name='Mechanisms of Action (MoA) Prediction', headline='Can you improve the algorithm that classifies drugs based on their biologica...', description='Can you improve the algorithm that classifies drugs based on their biological activity?', doi='', status=<ChallengeStatus.COMPLETED: 'completed'>, difficulty=<ChallengeDifficulty.INTERMEDIATE: 'intermediate'>, platform=SimpleChallengePlatform(id=8, slug='kaggle', name='Kaggle'), website_url='https://www.kaggle.com/competitions/lish-moa', avatar_url='', incentives=[<ChallengeIncentive.MONETARY: 'monetary'>], submission_types=[<ChallengeSubmissionType.CONTAINER_IMAGE: 'container_image'>, <ChallengeSubmissionType.NOTEBOOK: 'notebook'>], input_data_types=[], start_date=datetime.date(2020, 9, 3), end_date=datetime.date(2020, 11, 30), starred_count=0, created_at=datetime.datetime(2023, 8, 8, 19, 9, 31, tzinfo=datetime.timezone.utc), updated_at=datetime.datetime(2023, 9, 28, 23, 18, 4, tzinfo=datetime.timezone.utc)), Challenge(id=179, slug='openribosome', name='OpenRibosome', headline=\"We aim to 1) gain fundamental insights into the ribosome's RNA sequence-fold...\", description=\"Our modern world has many challenges-challenges like climate change, increasing waste production, and human health. Imagine-we could replace petrochemistry with biology, single-use plastics with selectively degradable polymers, broad chemotherapeutics with targeted medicines for fighting specific cancer cells, and complex health equipment with point-of-care diagnostics. These innovations and many more can empower us to confront the challenges affecting humanity, our world, and beyond. But how do we actually create these smart materials and medicines? Is it possible to do so by repurposing one of Nature's molecular machines? We think we can. The answer? Customized ribosomes. In Nature, ribosomes are the catalysts for protein assembly. And proteins are more or less similar, chemically, to the smart materials and medicines we want to synthesize. If we could modify ribosomes to build polymers with diverse components-beyond the canonical amino acids us\", doi='https://doi.org/10.1038/s41467-023-35827-3', status=<ChallengeStatus.ACTIVE: 'active'>, difficulty=<ChallengeDifficulty.INTERMEDIATE: 'intermediate'>, platform=SimpleChallengePlatform(id=13, slug='eterna', name='Eterna'), website_url='https://eternagame.org/challenges/11043833', avatar_url='', incentives=[<ChallengeIncentive.PUBLICATION: 'publication'>], submission_types=[<ChallengeSubmissionType.OTHER: 'other'>], input_data_types=[], start_date=datetime.date(2019, 1, 31), end_date=None, starred_count=0, created_at=datetime.datetime(2023, 8, 8, 18, 43, 27, tzinfo=datetime.timezone.utc), updated_at=datetime.datetime(2023, 10, 10, 19, 53, 1, tzinfo=datetime.timezone.utc)), Challenge(id=178, slug='openaso', name='OpenASO', headline='A research initiative aimed at developing innovative design principles for R...', description='The DNA genome is the blueprint for building and operating cells, but this information must be decoded into RNA molecules to be useful. Transcription is the process of decoding DNA genomic information into RNA, resulting in RNA transcripts. Genes are specific sequences of DNA that contain information to produce a specific RNA transcript. The fate of most mRNA molecules in the cell is to be translated by ribosomes into protein molecules. However, mRNA splicing is a crucial step that occurs between the formation of an RNA transcript and protein translation. This step is essential because genes contain non-protein coding introns and protein-coding exons. Splicing removes introns and joins exons to produce a mature mRNA molecule that can be decoded into the correct protein molecule. When the splicing process is corrupted due to genetic mutations, the resulting RNA can become toxic, leading to the synthesis of non-functional proteins or no protein at all, causing various human diseases...', doi='', status=<ChallengeStatus.ACTIVE: 'active'>, difficulty=<ChallengeDifficulty.INTERMEDIATE: 'intermediate'>, platform=SimpleChallengePlatform(id=13, slug='eterna', name='Eterna'), website_url='https://eternagame.org/challenges/11546273', avatar_url='', incentives=[<ChallengeIncentive.PUBLICATION: 'publication'>], submission_types=[<ChallengeSubmissionType.OTHER: 'other'>], input_data_types=[], start_date=datetime.date(2023, 2, 20), end_date=None, starred_count=0, created_at=datetime.datetime(2023, 8, 8, 18, 43, 25, tzinfo=datetime.timezone.utc), updated_at=datetime.datetime(2023, 10, 10, 19, 52, 57, tzinfo=datetime.timezone.utc)), Challenge(id=177, slug='openknot', name='OpenKnot', headline='Many important biological processes depend on RNAs that form pseudoknots, an...', description='RNA pseudoknots have significant biological importance in various processes. They participate in gene regulation by influencing translation initiation or termination in mRNA molecules. Pseudoknots also play a role in programmed ribosomal frameshifting, leading to the production of different protein products from a single mRNA. RNA viruses, including SARS-CoV-2 and Dengue virus, utilize pseudoknots to regulate their replication and control the synthesis of viral proteins. Additionally, certain RNA molecules with pseudoknot structures exhibit enzymatic activity, acting as ribozymes and catalyzing biochemical reactions. These functions highlight the crucial role of RNA pseudoknots in gene expression, proteomic diversity, viral replication, and enzymatic processes. Several unanswered scientific questions surround RNA pseudoknots. One key area of inquiry is understanding the folding pathways of pseudoknots and how they form from linear RNA sequences. Elucidating the structural dynamics...', doi='', status=<ChallengeStatus.ACTIVE: 'active'>, difficulty=<ChallengeDifficulty.INTERMEDIATE: 'intermediate'>, platform=SimpleChallengePlatform(id=13, slug='eterna', name='Eterna'), website_url='https://eternagame.org/challenges/11843006', avatar_url='', incentives=[<ChallengeIncentive.PUBLICATION: 'publication'>], submission_types=[<ChallengeSubmissionType.OTHER: 'other'>], input_data_types=[], start_date=datetime.date(2022, 6, 17), end_date=None, starred_count=0, created_at=datetime.datetime(2023, 8, 8, 18, 43, 22, tzinfo=datetime.timezone.utc), updated_at=datetime.datetime(2023, 10, 10, 19, 52, 53, tzinfo=datetime.timezone.utc)), Challenge(id=176, slug='opencrispr', name='OpenCRISPR', headline='A project to discover design patterns for guide RNAs to make gene editing mo...', description=\"CRISPR gene editing is a RNA-based method that can target essentially any gene in a living organism for genetic changes. Since its first demonstration, CRISPR has been revolutionizing biology and promises to change how we tackle numerous human diseases from malaria to cancer. Stanford's Center for Personal Dynamic Regulomes and UC Berkeley's Innovative Genomics Institute have challenged Eterna players to solve a remaining hurdle in making this technology safe for use. Scientists want the power to turn on and off CRISPR on demand with small molecules. This is almost a perfect match to the small-molecule switches that the Eterna community has worked on. In fact, the MS2 RNA hairpin often used in Eterna is routinely used to recruit new functionality to CRISPR complexes through other molecules tethered to the MS2 protein. The puzzles began with OpenCRISPR Controls, looking for solutions to lock in or lock out the MS2 RNA hairpin within a special loop in the CRISPR RNA. We hope the res...\", doi='https://doi.org/10.1021/acssynbio.9b00142', status=<ChallengeStatus.COMPLETED: 'completed'>, difficulty=<ChallengeDifficulty.INTERMEDIATE: 'intermediate'>, platform=SimpleChallengePlatform(id=13, slug='eterna', name='Eterna'), website_url='https://eternagame.org/challenges/10845743', avatar_url='', incentives=[<ChallengeIncentive.PUBLICATION: 'publication'>], submission_types=[<ChallengeSubmissionType.OTHER: 'other'>], input_data_types=[], start_date=datetime.date(2017, 8, 26), end_date=None, starred_count=0, created_at=datetime.datetime(2023, 8, 8, 18, 43, 14, tzinfo=datetime.timezone.utc), updated_at=datetime.datetime(2023, 10, 10, 19, 57, 7, tzinfo=datetime.timezone.utc)), Challenge(id=175, slug='opentb', name='OpenTB', headline='What if we could use RNA to detect a gene sequence found to be present only ...', description='OpenTB used a recently reported gene signature for active tuberculosis based on three RNAs in the blood. This signature could form the basis for a fast, color-based test for TB, similar to an over-the-counter pregnancy test. What was needed was a sensor that could detect the concentrations of three RNAs, carry out the needed calculation, and report the result by binding another molecule. Over four rounds, players designed RNA sensors that can do the math on these 3 genes. Through experimental feedback, they honed their skills and techniques, which resulted in the creation of multiple designs that have been shown to be successful. These findings are being prepared to be published, and future work will be done to develop diagnostic devices integrating these designs', doi='', status=<ChallengeStatus.COMPLETED: 'completed'>, difficulty=<ChallengeDifficulty.INTERMEDIATE: 'intermediate'>, platform=SimpleChallengePlatform(id=13, slug='eterna', name='Eterna'), website_url='https://eternagame.org/challenges/10845742', avatar_url='', incentives=[<ChallengeIncentive.PUBLICATION: 'publication'>], submission_types=[<ChallengeSubmissionType.OTHER: 'other'>], input_data_types=[], start_date=datetime.date(2016, 5, 4), end_date=datetime.date(2018, 4, 15), starred_count=0, created_at=datetime.datetime(2023, 8, 8, 18, 43, 9, tzinfo=datetime.timezone.utc), updated_at=datetime.datetime(2023, 9, 28, 23, 17, 9, tzinfo=datetime.timezone.utc)), Challenge(id=174, slug='openvaccine', name='OpenVaccine', headline='To develop mRNA vaccines stable enough to be deployed to everyone in the wor...', description='mRNA vaccines are a relatively new technology that have come into the limelight with the onset of COVID-19. They were the first COVID-19 vaccines to start clinical trials (initially formulated in a matter of days) and the first to be approved and distributed. mRNA vaccines have the potential to transform immunization, being significantly faster to formulate and produce, cheaper, and more effective-including against mutant strains. However, there is one key bottleneck to their widespread viability and our ability to immunize the entire world-poor refrigerator stability in prefilled syringes. The OpenVaccine challenge aims to allow a worldwide community of game players to create an enhanced vaccine to be injected into millions of people. The challenge-design an mRNA that codes for the same amino acid sequence of the spike protein, but is 2x-10x+ more stable. Through a number of academic partnerships and the launch of a Kaggle machine learning challenge to create best-in-class algori...', doi='https://doi.org/10.1038/s41467-022-28776-w', status=<ChallengeStatus.COMPLETED: 'completed'>, difficulty=<ChallengeDifficulty.INTERMEDIATE: 'intermediate'>, platform=SimpleChallengePlatform(id=13, slug='eterna', name='Eterna'), website_url='https://eternagame.org/challenges/10845741', avatar_url='', incentives=[<ChallengeIncentive.PUBLICATION: 'publication'>], submission_types=[<ChallengeSubmissionType.OTHER: 'other'>], input_data_types=[], start_date=None, end_date=datetime.date(2021, 12, 12), starred_count=0, created_at=datetime.datetime(2023, 8, 8, 18, 22, 49, tzinfo=datetime.timezone.utc), updated_at=datetime.datetime(2023, 9, 28, 23, 17, 2, tzinfo=datetime.timezone.utc)), Challenge(id=173, slug='stanford-covid-vaccine', name='OpenVaccine - COVID-19 mRNA Vaccine Degradation Prediction', headline='Urgent need to bring the COVID-19 vaccine to mass production', description='In this competition, we are looking to leverage the data science expertise of the Kaggle community to develop models and design rules for RNA degradation. Your model will predict likely degradation rates at each base of an RNA molecule, trained on a subset of an Eterna dataset comprising over 3000 RNA molecules (which span a panoply of sequences and structures) and their degradation rates at each position. We will then score your models on a second generation of RNA sequences that have just been devised by Eterna players for COVID-19 mRNA vaccines. These final test sequences are currently being synthesized and experimentally characterized at Stanford University in parallel to your modeling efforts--Nature will score your models!', doi='', status=<ChallengeStatus.COMPLETED: 'completed'>, difficulty=<ChallengeDifficulty.INTERMEDIATE: 'intermediate'>, platform=SimpleChallengePlatform(id=8, slug='kaggle', name='Kaggle'), website_url='https://www.kaggle.com/competitions/stanford-covid-vaccine', avatar_url='', incentives=[<ChallengeIncentive.MONETARY: 'monetary'>], submission_types=[<ChallengeSubmissionType.NOTEBOOK: 'notebook'>], input_data_types=[], start_date=datetime.date(2020, 9, 10), end_date=datetime.date(2020, 10, 6), starred_count=0, created_at=datetime.datetime(2023, 8, 8, 18, 6, 17, tzinfo=datetime.timezone.utc), updated_at=datetime.datetime(2023, 10, 12, 18, 14, 27, tzinfo=datetime.timezone.utc)), Challenge(id=172, slug='ventilator-pressure-prediction', name='Google Brain: Ventilator Pressure Prediction', headline=\"Simulate a ventilator connected to a sedated patient's lung\", description=\"In this competition, you’ll simulate a ventilator connected to a sedated patient's lung. The best submissions will take lung attributes compliance and resistance into account. If successful, you'll help overcome the cost barrier of developing new methods for controlling mechanical ventilators. This will pave the way for algorithms that adapt to patients and reduce the burden on clinicians during these novel times and beyond. As a result, ventilator treatments may become more widely available to help patients breathe.\", doi='', status=<ChallengeStatus.COMPLETED: 'completed'>, difficulty=<ChallengeDifficulty.INTERMEDIATE: 'intermediate'>, platform=SimpleChallengePlatform(id=8, slug='kaggle', name='Kaggle'), website_url='https://www.kaggle.com/competitions/ventilator-pressure-prediction', avatar_url='', incentives=[<ChallengeIncentive.MONETARY: 'monetary'>], submission_types=[<ChallengeSubmissionType.NOTEBOOK: 'notebook'>], input_data_types=[], start_date=datetime.date(2021, 9, 22), end_date=datetime.date(2021, 11, 3), starred_count=0, created_at=datetime.datetime(2023, 8, 8, 17, 53, 33, tzinfo=datetime.timezone.utc), updated_at=datetime.datetime(2023, 10, 12, 18, 14, 11, tzinfo=datetime.timezone.utc)), Challenge(id=171, slug='hubmap-kidney-segmentation', name='HuBMAP: Hacking the Kidney', headline='Identify glomeruli in human kidney tissue images', description='This competition, “Hacking the Kidney, starts by mapping the human kidney at single cell resolution. Your challenge is to detect functional tissue units (FTUs) across different tissue preparation pipelines. An FTU is defined as a “three-dimensional block of cells centered around a capillary, such that each cell in this block is within diffusion distance from any other cell in the same block” ([de Bono, 2013](https://www.ncbi.nlm.nih.gov/pubmed/24103658)). The goal of this competition is the implementation of a successful and robust glomeruli FTU detector. You will also have the opportunity to present your findings to a panel of judges for additional consideration. Successful submissions will construct the tools, resources, and cell atlases needed to determine how the relationships between cells can affect the health of an individual. Advancements in HuBMAP will accelerate the world’s understanding of the relationships between cell and tissue organization and function and human health.', doi='', status=<ChallengeStatus.COMPLETED: 'completed'>, difficulty=<ChallengeDifficulty.INTERMEDIATE: 'intermediate'>, platform=SimpleChallengePlatform(id=8, slug='kaggle', name='Kaggle'), website_url='https://www.kaggle.com/competitions/hubmap-kidney-segmentation', avatar_url='', incentives=[<ChallengeIncentive.MONETARY: 'monetary'>], submission_types=[<ChallengeSubmissionType.NOTEBOOK: 'notebook'>], input_data_types=[], start_date=datetime.date(2020, 11, 16), end_date=datetime.date(2021, 5, 10), starred_count=0, created_at=datetime.datetime(2023, 8, 8, 17, 31, 46, tzinfo=datetime.timezone.utc), updated_at=datetime.datetime(2023, 10, 12, 18, 14, 16, tzinfo=datetime.timezone.utc)), Challenge(id=170, slug='hubmap-organ-segmentation', name='HuBMAP + HPA: Hacking the Human Body', headline='Segment multi-organ functional tissue units', description=\"In this competition, you’ll identify and segment functional tissue units (FTUs) across five human organs. You'll build your model using a dataset of tissue section images, with the best submissions segmenting FTUs as accurately as possible. If successful, you'll help accelerate the world’s understanding of the relationships between cell and tissue organization. With a better idea of the relationship of cells, researchers will have more insight into the function of cells that impact human health. Further, the Human Reference Atlas constructed by HuBMAP will be freely available for use by researchers and pharmaceutical companies alike, potentially improving and prolonging human life.\", doi='', status=<ChallengeStatus.COMPLETED: 'completed'>, difficulty=<ChallengeDifficulty.INTERMEDIATE: 'intermediate'>, platform=SimpleChallengePlatform(id=8, slug='kaggle', name='Kaggle'), website_url='https://www.kaggle.com/competitions/hubmap-organ-segmentation', avatar_url='', incentives=[<ChallengeIncentive.MONETARY: 'monetary'>, <ChallengeIncentive.SPEAKING_ENGAGEMENT: 'speaking_engagement'>], submission_types=[<ChallengeSubmissionType.NOTEBOOK: 'notebook'>], input_data_types=[], start_date=datetime.date(2022, 6, 22), end_date=datetime.date(2022, 9, 22), starred_count=0, created_at=datetime.datetime(2023, 8, 8, 16, 30, 22, tzinfo=datetime.timezone.utc), updated_at=datetime.datetime(2023, 10, 12, 18, 14, 20, tzinfo=datetime.timezone.utc)), Challenge(id=169, slug='multi-atlas-labeling-beyond-the-cranial-vault', name='Multi-Atlas Labeling Beyond the Cranial Vault', headline='', description='Multi-atlas labeling has proven to be an effective paradigm for creating segmentation algorithms from training data. These approaches have been extraordinarily successful for brain and cranial structures (e.g., our prior MICCAI workshops-MLSF’11, MAL’12, SATA’13). After the original challenges closed, the data continue to drive scientific innovation; 144 groups have registered for the 2012 challenge (brain only) and 115 groups for the 2013 challenge (brain/heart/canine leg). However, innovation in application outside of the head and to soft tissues has been more limited. This workshop will provide a snapshot of the current progress in the field through extended discussions and provide researchers an opportunity to characterize their methods on a newly created and released standardized dataset of abdominal anatomy on clinically acquired CT. The datasets will be freely available both during and after the challenge. We have two separate new challenges-abdomen and cervix on routinely ...', doi='', status=<ChallengeStatus.ACTIVE: 'active'>, difficulty=<ChallengeDifficulty.INTERMEDIATE: 'intermediate'>, platform=SimpleChallengePlatform(id=1, slug='synapse', name='Synapse'), website_url='https://www.synapse.org/#!Synapse:syn3193805/wiki/89480', avatar_url='', incentives=[], submission_types=[<ChallengeSubmissionType.PREDICTION_FILE: 'prediction_file'>], input_data_types=[], start_date=datetime.date(2015, 4, 15), end_date=None, starred_count=0, created_at=datetime.datetime(2023, 8, 7, 20, 21, 22, tzinfo=datetime.timezone.utc), updated_at=datetime.datetime(2023, 10, 10, 19, 52, 39, tzinfo=datetime.timezone.utc))])\n"
     ]
    }
   ],
   "source": [
    "# Enter a context with an instance of the API client\n",
    "with openchallenges_client.ApiClient(configuration) as api_client:\n",
    "    api_instance = challenge_api.ChallengeApi(api_client)\n",
    "    \n",
    "    query = openchallenges_client.ChallengeSearchQuery(page_number=1, page_size=1)\n",
    "\n",
    "    try:\n",
    "        # Get the first page of the list of challenges\n",
    "        api_response = api_instance.list_challenges(query)\n",
    "        pprint(api_response)\n",
    "    except openchallenges_client.ApiException as e:\n",
    "        print(\"Exception when calling ChallengeApi->list_challenges: %s\\n\" % e)"
   ]
  },
  {
   "cell_type": "code",
   "execution_count": null,
   "id": "d0c0b308-0b58-44a7-8ff6-4987dfbccb17",
   "metadata": {},
   "outputs": [],
   "source": []
  }
 ],
 "metadata": {
  "kernelspec": {
   "display_name": "Python 3 (ipykernel)",
   "language": "python",
   "name": "python3"
  },
  "language_info": {
   "codemirror_mode": {
    "name": "ipython",
    "version": 3
   },
   "file_extension": ".py",
   "mimetype": "text/x-python",
   "name": "python",
   "nbconvert_exporter": "python",
   "pygments_lexer": "ipython3",
   "version": "3.10.12"
  }
 },
 "nbformat": 4,
 "nbformat_minor": 5
}
